<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.2.269">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Modesto">
<meta name="dcterms.date" content="2023-02-27">

<title>Comparison of MDA methods</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1.6em;
  vertical-align: middle;
}
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>


<script src="index_files/libs/clipboard/clipboard.min.js"></script>
<script src="index_files/libs/quarto-html/quarto.js"></script>
<script src="index_files/libs/quarto-html/popper.min.js"></script>
<script src="index_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="index_files/libs/quarto-html/anchor.min.js"></script>
<link href="index_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="index_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="index_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="index_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="index_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">

<script src="index_files/libs/kePrint-0.0.1/kePrint.js"></script>
<link href="index_files/libs/lightable-0.0.1/lightable.css" rel="stylesheet">
<script src="index_files/libs/htmlwidgets-1.6.1/htmlwidgets.js"></script>
<script src="index_files/libs/plotly-binding-4.10.1/plotly.js"></script>
<script src="index_files/libs/typedarray-0.1/typedarray.min.js"></script>
<script src="index_files/libs/jquery-3.5.1/jquery.min.js"></script>
<link href="index_files/libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet">
<script src="index_files/libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="index_files/libs/plotly-htmlwidgets-css-2.11.1/plotly-htmlwidgets.css" rel="stylesheet">
<script src="index_files/libs/plotly-main-2.11.1/plotly-latest.min.js"></script>


<link rel="stylesheet" href="piMDA.css">
</head>

<body>

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article toc-left">
<div id="quarto-sidebar-toc-left" class="sidebar toc-left">
  <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#disclaimer" id="toc-disclaimer" class="nav-link active" data-scroll-target="#disclaimer"><span class="toc-section-number">1</span>  Disclaimer</a></li>
  <li><a href="#input-sample" id="toc-input-sample" class="nav-link" data-scroll-target="#input-sample"><span class="toc-section-number">2</span>  Input sample</a></li>
  <li><a href="#dna-amplification-assays-mda" id="toc-dna-amplification-assays-mda" class="nav-link" data-scroll-target="#dna-amplification-assays-mda"><span class="toc-section-number">3</span>  DNA amplification assays (MDA)</a>
  <ul>
  <li><a href="#buffer-optimization" id="toc-buffer-optimization" class="nav-link" data-scroll-target="#buffer-optimization"><span class="toc-section-number">3.1</span>  Buffer optimization</a></li>
  <li><a href="#methods-comparison" id="toc-methods-comparison" class="nav-link" data-scroll-target="#methods-comparison"><span class="toc-section-number">3.2</span>  Methods comparison</a></li>
  </ul></li>
  <li><a href="#illumina-raw-data-processing" id="toc-illumina-raw-data-processing" class="nav-link" data-scroll-target="#illumina-raw-data-processing"><span class="toc-section-number">4</span>  Illumina raw data processing</a>
  <ul>
  <li><a href="#qc-stats-and-trimming" id="toc-qc-stats-and-trimming" class="nav-link" data-scroll-target="#qc-stats-and-trimming"><span class="toc-section-number">4.1</span>  QC stats and trimming</a></li>
  <li><a href="#reads-per-gc-content" id="toc-reads-per-gc-content" class="nav-link" data-scroll-target="#reads-per-gc-content"><span class="toc-section-number">4.2</span>  Reads per GC content</a></li>
  <li><a href="#metagenomes-profiling" id="toc-metagenomes-profiling" class="nav-link" data-scroll-target="#metagenomes-profiling"><span class="toc-section-number">4.3</span>  Metagenomes profiling</a></li>
  </ul></li>
  <li><a href="#assembly-independent-coverage-of-reference-genomes" id="toc-assembly-independent-coverage-of-reference-genomes" class="nav-link" data-scroll-target="#assembly-independent-coverage-of-reference-genomes"><span class="toc-section-number">5</span>  Assembly-independent coverage of reference genomes</a>
  <ul>
  <li><a href="#reference-genomes-breadth-and-coverage" id="toc-reference-genomes-breadth-and-coverage" class="nav-link" data-scroll-target="#reference-genomes-breadth-and-coverage"><span class="toc-section-number">5.1</span>  Reference genomes breadth and coverage</a></li>
  <li><a href="#analysis-of-unclassfied-reads" id="toc-analysis-of-unclassfied-reads" class="nav-link" data-scroll-target="#analysis-of-unclassfied-reads"><span class="toc-section-number">5.2</span>  Analysis of unclassfied reads</a></li>
  <li><a href="#gc-content-bias" id="toc-gc-content-bias" class="nav-link" data-scroll-target="#gc-content-bias"><span class="toc-section-number">5.3</span>  GC content bias</a>
  <ul>
  <li><a href="#gc-bias-per-mda-sample" id="toc-gc-bias-per-mda-sample" class="nav-link" data-scroll-target="#gc-bias-per-mda-sample"><span class="toc-section-number">5.3.1</span>  GC bias per MDA sample</a>
  <ul class="collapse">
  <li><a href="#statistic-comparisons-of-gc-bias" id="toc-statistic-comparisons-of-gc-bias" class="nav-link" data-scroll-target="#statistic-comparisons-of-gc-bias"><span class="toc-section-number">5.3.1.1</span>  Statistic comparisons of GC bias</a></li>
  <li><a href="#correlations-gc-content-vs.-normalized-coverage" id="toc-correlations-gc-content-vs.-normalized-coverage" class="nav-link" data-scroll-target="#correlations-gc-content-vs.-normalized-coverage"><span class="toc-section-number">5.3.1.2</span>  Correlations: GC content vs.&nbsp;Normalized Coverage</a></li>
  </ul></li>
  <li><a href="#gc-bias-detailed-per-reference-genome" id="toc-gc-bias-detailed-per-reference-genome" class="nav-link" data-scroll-target="#gc-bias-detailed-per-reference-genome"><span class="toc-section-number">5.3.2</span>  GC bias detailed per reference genome</a>
  <ul class="collapse">
  <li><a href="#correlations-gc-content-vs.-normalized-coverage-per-reference-genome" id="toc-correlations-gc-content-vs.-normalized-coverage-per-reference-genome" class="nav-link" data-scroll-target="#correlations-gc-content-vs.-normalized-coverage-per-reference-genome"><span class="toc-section-number">5.3.2.1</span>  Correlations: GC content vs.&nbsp;Normalized Coverage per reference genome</a></li>
  </ul></li>
  </ul></li>
  </ul></li>
  <li><a href="#assemblies" id="toc-assemblies" class="nav-link" data-scroll-target="#assemblies"><span class="toc-section-number">6</span>  Assemblies</a>
  <ul>
  <li><a href="#assembly-basic-stats" id="toc-assembly-basic-stats" class="nav-link" data-scroll-target="#assembly-basic-stats"><span class="toc-section-number">6.1</span>  Assembly basic stats</a></li>
  <li><a href="#stats-per-individual-reference-genome" id="toc-stats-per-individual-reference-genome" class="nav-link" data-scroll-target="#stats-per-individual-reference-genome"><span class="toc-section-number">6.2</span>  Stats per individual reference genome</a></li>
  <li><a href="#number-of-contigs-by-gc-content" id="toc-number-of-contigs-by-gc-content" class="nav-link" data-scroll-target="#number-of-contigs-by-gc-content"><span class="toc-section-number">6.3</span>  Number of contigs by GC-content</a></li>
  <li><a href="#assembly-annotation" id="toc-assembly-annotation" class="nav-link" data-scroll-target="#assembly-annotation"><span class="toc-section-number">6.4</span>  Assembly annotation</a></li>
  <li><a href="#variant-calling" id="toc-variant-calling" class="nav-link" data-scroll-target="#variant-calling"><span class="toc-section-number">6.5</span>  Variant calling</a></li>
  <li><a href="#assembly-k-mer-content" id="toc-assembly-k-mer-content" class="nav-link" data-scroll-target="#assembly-k-mer-content"><span class="toc-section-number">6.6</span>  Assembly k-mer content</a></li>
  <li><a href="#assembly-binning" id="toc-assembly-binning" class="nav-link" data-scroll-target="#assembly-binning"><span class="toc-section-number">6.7</span>  Assembly binning</a></li>
  </ul></li>
  <li><a href="#index-of-figures" id="toc-index-of-figures" class="nav-link" data-scroll-target="#index-of-figures"><span class="toc-section-number">7</span>  Index of Figures</a></li>
  <li><a href="#acknowledments-and-funding" id="toc-acknowledments-and-funding" class="nav-link" data-scroll-target="#acknowledments-and-funding"><span class="toc-section-number">8</span>  Acknowledments and Funding</a></li>
  <li><a href="#session-info" id="toc-session-info" class="nav-link" data-scroll-target="#session-info"><span class="toc-section-number">9</span>  Session Info</a></li>
  </ul>
</nav>
</div>
<div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
</div>
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Comparison of MDA methods</h1>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-contents">
             <p>Modesto </p>
          </div>
  </div>
    
    <div>
    <div class="quarto-title-meta-heading">Published</div>
    <div class="quarto-title-meta-contents">
      <p class="date">February 27, 2023</p>
    </div>
  </div>
  
    
  </div>
  

</header>

<hr>
<section id="disclaimer" class="level2" data-link="Disclaimer" data-number="1">
<h2 data-link="Disclaimer" data-number="1" class="anchored" data-anchor-id="disclaimer"><span class="header-section-number">1</span> Disclaimer</h2>
<p>This data analysis is preliminar and will be soon published in a manuscript authored by Carlos D. Ordóñez, Conceçao Egas and <a href="https://orcid.org/0000-0003-0014-4162" target="_blank">Modesto Redrejo-Rodríguez</a>. The <a href="https://github.com/mredrejo/pimda" target="_blank">GitHub repo</a> contains the original result files from all the analyses. Raw reads will be available in a suitable repository along with the manuscript.</p>
<p>Data is shared under creative common license (CC BY-NC-ND 3.0 ES). Contact <a href="mailto:modesto.redrejo@uam.es">modesto.redrejo@uam.es</a> for further info.</p>
</section>
<section id="input-sample" class="level2" data-number="2">
<h2 data-number="2" class="anchored" data-anchor-id="input-sample"><span class="header-section-number">2</span> Input sample</h2>
<p>The input DNA to be amplified is a mock metagenome made up of equal mass of purified genomes from 4 different bacteria, spanning Gram+ and Gram- species, and genomes with high and low GC, as detailed in the Table 1<a href="#fn1" class="footnote-ref" id="fnref1" role="doc-noteref"><sup>1</sup></a>. Note that the used reference genomes were selected later on, using top hits of Blast searches performed the longest contigs from each assembly as query.</p>
<div class="cell">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Define color palette for the whole report to facilitate future modifications</span></span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a>colorines <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">`</span><span class="at">NA</span><span class="st">`</span> <span class="ot">=</span> <span class="st">"grey"</span>, <span class="at">NA2 =</span> <span class="st">"grey"</span>, <span class="st">`</span><span class="at">RP-MDA</span><span class="st">`</span> <span class="ot">=</span> <span class="st">"palegreen4"</span>, <span class="st">`</span><span class="at">RP-MDA2</span><span class="st">`</span> <span class="ot">=</span> <span class="st">"palegreen4"</span>,</span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a>    <span class="st">`</span><span class="at">PrimPol-MDA</span><span class="st">`</span> <span class="ot">=</span> <span class="st">"orangered4"</span>, <span class="at">piPolB =</span> <span class="st">"gold"</span>, <span class="st">`</span><span class="at">piPolB+D</span><span class="st">`</span> <span class="ot">=</span> <span class="st">"darkgoldenrod1"</span>, <span class="at">piMDA =</span> <span class="st">"deepskyblue3"</span>,</span>
<span id="cb1-4"><a href="#cb1-4" aria-hidden="true" tabindex="-1"></a>    <span class="at">piMDA2 =</span> <span class="st">"deepskyblue3"</span>, <span class="st">`</span><span class="at">piMDA+D</span><span class="st">`</span> <span class="ot">=</span> <span class="st">"blue4"</span>, <span class="st">`</span><span class="at">piMDA+D2</span><span class="st">`</span> <span class="ot">=</span> <span class="st">"blue4"</span>, <span class="st">`</span><span class="at">B. subtilis</span><span class="st">`</span> <span class="ot">=</span> <span class="st">"orange2"</span>,</span>
<span id="cb1-5"><a href="#cb1-5" aria-hidden="true" tabindex="-1"></a>    <span class="st">`</span><span class="at">E. coli</span><span class="st">`</span> <span class="ot">=</span> <span class="st">"gold1"</span>, <span class="st">`</span><span class="at">K. rhizophila</span><span class="st">`</span> <span class="ot">=</span> <span class="st">"skyblue1"</span>, <span class="st">`</span><span class="at">P. aeruginosa</span><span class="st">`</span> <span class="ot">=</span> <span class="st">"slateblue2"</span>)</span>
<span id="cb1-6"><a href="#cb1-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-7"><a href="#cb1-7" aria-hidden="true" tabindex="-1"></a><span class="co"># load data</span></span>
<span id="cb1-8"><a href="#cb1-8" aria-hidden="true" tabindex="-1"></a>samples <span class="ot">&lt;-</span> <span class="fu">read.table</span>(<span class="st">"amplification_yield/input_sample.csv"</span>, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">";"</span>)</span>
<span id="cb1-9"><a href="#cb1-9" aria-hidden="true" tabindex="-1"></a><span class="fu">names</span>(samples) <span class="ot">=</span> <span class="fu">c</span>(<span class="st">"Strain"</span>, <span class="st">"Model strain"</span>, <span class="st">"Reference genome"</span>, <span class="st">"Length (MB)"</span>, <span class="st">"GC content"</span>)</span>
<span id="cb1-10"><a href="#cb1-10" aria-hidden="true" tabindex="-1"></a><span class="fu">kbl</span>(samples, <span class="at">align =</span> <span class="st">"c"</span>, <span class="at">caption =</span> <span class="st">"Table 1. Composition of the amplified mock-metagenome"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-11"><a href="#cb1-11" aria-hidden="true" tabindex="-1"></a>    <span class="fu">kable_styling</span>(<span class="at">bootstrap_options =</span> <span class="st">"striped"</span>, <span class="at">full_width =</span> F, <span class="at">position =</span> <span class="st">"center"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-12"><a href="#cb1-12" aria-hidden="true" tabindex="-1"></a>    <span class="fu">column_spec</span>(<span class="dv">1</span>, <span class="at">italic =</span> T)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">

<table class="table table-striped" style="width: auto !important; margin-left: auto; margin-right: auto;">
<caption>Table 1. Composition of the amplified mock-metagenome</caption>
 <thead>
  <tr>
   <th style="text-align:center;"> Strain </th>
   <th style="text-align:center;"> Model strain </th>
   <th style="text-align:center;"> Reference genome </th>
   <th style="text-align:center;"> Length (MB) </th>
   <th style="text-align:center;"> GC content </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:center;font-style: italic;"> E.coli </td>
   <td style="text-align:center;"> E. coli K12 MG1655 </td>
   <td style="text-align:center;"> NC_000913.3 </td>
   <td style="text-align:center;"> 4.64 </td>
   <td style="text-align:center;"> 50.79 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> B. subtilis 110NA </td>
   <td style="text-align:center;"> B. subtilis 168 </td>
   <td style="text-align:center;"> NC_000964.3 </td>
   <td style="text-align:center;"> 4.21 </td>
   <td style="text-align:center;"> 43.51 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> P. aeruginosa </td>
   <td style="text-align:center;"> P. aeruginosa PAO1 </td>
   <td style="text-align:center;"> NC_002516.2 </td>
   <td style="text-align:center;"> 6.50 </td>
   <td style="text-align:center;"> 66.37 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> Micrococcus luteus </td>
   <td style="text-align:center;"> K. rhizophila ATCC 9341 </td>
   <td style="text-align:center;"> NC_010617.1 </td>
   <td style="text-align:center;"> 2.70 </td>
   <td style="text-align:center;"> 70.56 </td>
  </tr>
</tbody>
</table>

</div>
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb2"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a><span class="co"># %&gt;% save_kable('figures/samples.pdf')</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
</section>
<section id="dna-amplification-assays-mda" class="level2" data-number="3">
<h2 data-number="3" class="anchored" data-anchor-id="dna-amplification-assays-mda"><span class="header-section-number">3</span> DNA amplification assays (MDA)</h2>
<p>DNA amplification was carried out by six different protocols. (1) Random-primer hexamers-based MDA (RP-MDA) was carried out with REPLIg (Qiagen) and (2) TruePrime (4basebio) was used for a control primer-less MDA (PrimPol-MDA). In both cases, amplification protocols followed the manufacturer’s instructions, including the routine alkaline denaturation step. Recombinant piPolB (<a href="https://pubmed.ncbi.nlm.nih.gov/29117562/" target="_blank" title="piPolB description paper">Redrejo-Rodríguez et al.&nbsp;2017</a>) was used for a single-enzyme, primer-free DNA amplification in a single step MDA protocol (3) or an alternative protocol (4) with a previous alkaline denaturation step (piPolB+D). Finally, a new method based on the combination of piPolB and Φ29 DNA polymerases was developed. Two alternative protocols were used, (5) a single-step primer-independent MDA (piMDA) and a modification (6) with a previous alkaline denaturation of input DNA (piMDA+D).</p>
<p>Amplification yield was assessed by absolute measure of DNA amplification product was performed by fluorescence quantitation of dilutions from each sample (triplicates) with the AccuBlue High Sensitivity dsDNA Quantitation Kit (<a href="https://biotium.com/product/accublue-high-sensitivity-dsdna-quantitation-kit-with-8-dna-standards/" target="_blank">Biotum, #31006</a>) using a FLUOstar Omega (BMG Labtech) fluorimeter. AccuBlue High Sensitivity dsDNA Standards (<a href="https://biotium.com/product/accublue-high-sensitivity-dsdna-standards-set-of-eight/" target="_blank">Biotum, #31006C</a>) were used to do the range standard curve in each experiment.</p>
<section id="buffer-optimization" class="level3" data-number="3.1">
<h3 data-number="3.1" class="anchored" data-anchor-id="buffer-optimization"><span class="header-section-number">3.1</span> Buffer optimization</h3>
<p>Some chemicals or biomolecules have been reported to enhance DNA polymerase proficiency and amplification yield, such as salts, reducing agents, BSA, or glycerol, among others (<a href="https://pubmed.ncbi.nlm.nih.gov/30617407/" target="_blank">Gao et al.&nbsp;2019</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/35909465/" target="_blank">Jan and Kim, 2022</a>). In our study, we decided to assess the effect of DTT, BSA and glycerol on the piMDA protocol.</p>
<div class="cell" data-layout-align="left">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb3"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a><span class="co"># read table</span></span>
<span id="cb3-2"><a href="#cb3-2" aria-hidden="true" tabindex="-1"></a>opt <span class="ot">&lt;-</span> <span class="fu">read.csv2</span>(<span class="st">"amplification_yield/buffer_optimization.csv"</span>)</span>
<span id="cb3-3"><a href="#cb3-3" aria-hidden="true" tabindex="-1"></a>bsa <span class="ot">&lt;-</span> opt[opt<span class="sc">$</span>Chem <span class="sc">==</span> <span class="st">"BSA"</span>, ]</span>
<span id="cb3-4"><a href="#cb3-4" aria-hidden="true" tabindex="-1"></a>bsa<span class="sc">$</span>Conc[bsa<span class="sc">$</span>Conc <span class="sc">==</span> <span class="dv">0</span>] <span class="ot">&lt;-</span> <span class="fl">0.001</span></span>
<span id="cb3-5"><a href="#cb3-5" aria-hidden="true" tabindex="-1"></a>dtt <span class="ot">&lt;-</span> opt[opt<span class="sc">$</span>Chem <span class="sc">==</span> <span class="st">"DTT"</span>, ]</span>
<span id="cb3-6"><a href="#cb3-6" aria-hidden="true" tabindex="-1"></a>gly <span class="ot">&lt;-</span> opt[opt<span class="sc">$</span>Chem <span class="sc">==</span> <span class="st">"Glycerol"</span>, ]</span>
<span id="cb3-7"><a href="#cb3-7" aria-hidden="true" tabindex="-1"></a><span class="co"># plots</span></span>
<span id="cb3-8"><a href="#cb3-8" aria-hidden="true" tabindex="-1"></a>p1 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(bsa, <span class="fu">aes</span>(<span class="at">x =</span> Conc, <span class="at">y =</span> Yield, <span class="at">alpha =</span> <span class="fl">0.05</span>)) <span class="sc">+</span> <span class="fu">geom_smooth</span>(<span class="at">color =</span> <span class="st">"deepskyblue3"</span>,</span>
<span id="cb3-9"><a href="#cb3-9" aria-hidden="true" tabindex="-1"></a>    <span class="at">fill =</span> <span class="st">"deepskyblue3"</span>, <span class="at">alpha =</span> <span class="fl">0.25</span>) <span class="sc">+</span> <span class="fu">scale_x_log10</span>(<span class="at">breaks =</span> <span class="fu">c</span>(<span class="fl">0.001</span>, <span class="fl">0.01</span>, <span class="fl">0.1</span>, <span class="dv">1</span>, <span class="dv">10</span>),</span>
<span id="cb3-10"><a href="#cb3-10" aria-hidden="true" tabindex="-1"></a>    <span class="at">labels =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="fl">0.01</span>, <span class="fl">0.1</span>, <span class="dv">1</span>, <span class="dv">10</span>)) <span class="sc">+</span> <span class="fu">scale_y_continuous</span>(<span class="at">limits =</span> <span class="fu">c</span>(<span class="sc">-</span><span class="dv">15</span>, <span class="dv">120</span>), <span class="at">breaks =</span> <span class="fu">c</span>(<span class="dv">0</span>,</span>
<span id="cb3-11"><a href="#cb3-11" aria-hidden="true" tabindex="-1"></a>    <span class="dv">25</span>, <span class="dv">50</span>, <span class="dv">75</span>, <span class="dv">100</span>, <span class="dv">120</span>)) <span class="sc">+</span> <span class="fu">geom_jitter</span>(<span class="at">width =</span> <span class="fl">0.1</span>, <span class="at">pch =</span> <span class="dv">21</span>, <span class="at">color =</span> <span class="st">"deepskyblue3"</span>, <span class="at">fill =</span> <span class="st">"deepskyblue3"</span>) <span class="sc">+</span></span>
<span id="cb3-12"><a href="#cb3-12" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme_bw</span>() <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">15</span>)) <span class="sc">+</span> <span class="fu">guides</span>(<span class="at">alpha =</span> <span class="st">"none"</span>, <span class="at">color =</span> <span class="st">"none"</span>) <span class="sc">+</span></span>
<span id="cb3-13"><a href="#cb3-13" aria-hidden="true" tabindex="-1"></a>    <span class="fu">xlab</span>(<span class="st">"BSA (mg/mL)"</span>) <span class="sc">+</span> <span class="fu">ylab</span>(<span class="st">"DNA product (%)"</span>)</span>
<span id="cb3-14"><a href="#cb3-14" aria-hidden="true" tabindex="-1"></a>p2 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(dtt, <span class="fu">aes</span>(<span class="at">x =</span> Conc, <span class="at">y =</span> Yield, <span class="at">alpha =</span> <span class="fl">0.1</span>)) <span class="sc">+</span> <span class="fu">geom_smooth</span>(<span class="at">color =</span> <span class="st">"deepskyblue3"</span>,</span>
<span id="cb3-15"><a href="#cb3-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">fill =</span> <span class="st">"deepskyblue3"</span>, <span class="at">alpha =</span> <span class="fl">0.25</span>) <span class="sc">+</span> <span class="fu">scale_y_continuous</span>(<span class="at">limits =</span> <span class="fu">c</span>(<span class="sc">-</span><span class="dv">15</span>, <span class="dv">120</span>), <span class="at">breaks =</span> <span class="fu">c</span>(<span class="dv">0</span>,</span>
<span id="cb3-16"><a href="#cb3-16" aria-hidden="true" tabindex="-1"></a>    <span class="dv">25</span>, <span class="dv">50</span>, <span class="dv">75</span>, <span class="dv">100</span>, <span class="dv">120</span>)) <span class="sc">+</span> <span class="fu">geom_jitter</span>(<span class="at">pch =</span> <span class="dv">21</span>, <span class="at">width =</span> <span class="fl">0.1</span>, <span class="at">color =</span> <span class="st">"deepskyblue3"</span>, <span class="at">fill =</span> <span class="st">"deepskyblue3"</span>) <span class="sc">+</span></span>
<span id="cb3-17"><a href="#cb3-17" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme_bw</span>() <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">15</span>)) <span class="sc">+</span> <span class="fu">guides</span>(<span class="at">alpha =</span> <span class="st">"none"</span>, <span class="at">color =</span> <span class="st">"none"</span>) <span class="sc">+</span></span>
<span id="cb3-18"><a href="#cb3-18" aria-hidden="true" tabindex="-1"></a>    <span class="fu">xlab</span>(<span class="st">"DTT (mM)"</span>) <span class="sc">+</span> <span class="fu">ylab</span>(<span class="st">"DNA product (%)"</span>)</span>
<span id="cb3-19"><a href="#cb3-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-20"><a href="#cb3-20" aria-hidden="true" tabindex="-1"></a>p3 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(gly, <span class="fu">aes</span>(<span class="at">x =</span> Conc, <span class="at">y =</span> Yield, <span class="at">alpha =</span> <span class="fl">0.1</span>)) <span class="sc">+</span> <span class="fu">geom_smooth</span>(<span class="at">color =</span> <span class="st">"deepskyblue3"</span>,</span>
<span id="cb3-21"><a href="#cb3-21" aria-hidden="true" tabindex="-1"></a>    <span class="at">fill =</span> <span class="st">"deepskyblue3"</span>, <span class="at">method =</span> <span class="st">"loess"</span>, <span class="at">fullrange =</span> <span class="cn">TRUE</span>, <span class="at">span =</span> <span class="dv">5</span>, <span class="at">alpha =</span> <span class="fl">0.25</span>) <span class="sc">+</span> <span class="fu">scale_y_continuous</span>(<span class="at">limits =</span> <span class="fu">c</span>(<span class="sc">-</span><span class="dv">15</span>,</span>
<span id="cb3-22"><a href="#cb3-22" aria-hidden="true" tabindex="-1"></a>    <span class="dv">120</span>), <span class="at">breaks =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">25</span>, <span class="dv">50</span>, <span class="dv">75</span>, <span class="dv">100</span>, <span class="dv">120</span>)) <span class="sc">+</span> <span class="fu">geom_jitter</span>(<span class="at">pch =</span> <span class="dv">21</span>, <span class="at">width =</span> <span class="fl">0.1</span>, <span class="at">color =</span> <span class="st">"deepskyblue3"</span>,</span>
<span id="cb3-23"><a href="#cb3-23" aria-hidden="true" tabindex="-1"></a>    <span class="at">fill =</span> <span class="st">"deepskyblue3"</span>) <span class="sc">+</span> <span class="fu">theme_bw</span>() <span class="sc">+</span> <span class="fu">scale_x_continuous</span>(<span class="at">breaks =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">2</span>, <span class="dv">4</span>, <span class="dv">6</span>, <span class="dv">8</span>, <span class="dv">10</span>,</span>
<span id="cb3-24"><a href="#cb3-24" aria-hidden="true" tabindex="-1"></a>    <span class="dv">12</span>, <span class="dv">14</span>, <span class="dv">16</span>), <span class="at">labels =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">2</span>, <span class="dv">4</span>, <span class="dv">6</span>, <span class="dv">8</span>, <span class="dv">10</span>, <span class="dv">12</span>, <span class="dv">14</span>, <span class="dv">16</span>)) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">15</span>)) <span class="sc">+</span></span>
<span id="cb3-25"><a href="#cb3-25" aria-hidden="true" tabindex="-1"></a>    <span class="fu">guides</span>(<span class="at">alpha =</span> <span class="st">"none"</span>, <span class="at">color =</span> <span class="st">"none"</span>) <span class="sc">+</span> <span class="fu">xlab</span>(<span class="st">"Glycerol (%)"</span>) <span class="sc">+</span> <span class="fu">ylab</span>(<span class="st">"DNA product (%)"</span>)</span>
<span id="cb3-26"><a href="#cb3-26" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-27"><a href="#cb3-27" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-28"><a href="#cb3-28" aria-hidden="true" tabindex="-1"></a>optimization <span class="ot">&lt;-</span> p1<span class="sc">/</span>p3<span class="sc">/</span>p2</span>
<span id="cb3-29"><a href="#cb3-29" aria-hidden="true" tabindex="-1"></a>optimization</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stderr">
<pre><code>`geom_smooth()` using method = 'loess' and formula = 'y ~ x'
`geom_smooth()` using formula = 'y ~ x'
`geom_smooth()` using method = 'loess' and formula = 'y ~ x'</code></pre>
</div>
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-left">
<figure class="figure">
<p><img src="Plotoptimization-1.png" class="img-fluid figure-img" width="576"></p>
<p></p><figcaption class="figure-caption">Figure 1. Optimization of piMDA buffer components.</figcaption><p></p>
</figure>
</div>
</div>
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb5"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb5-1"><a href="#cb5-1" aria-hidden="true" tabindex="-1"></a><span class="co"># ggsave('figures/opti.svg',optimization,width=6,height=11)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<p>The presence of BSA does not affect the yield of the amplification up to 1 mg/mL. However, BSA is being highly inhibitory at high concentration, likewise increasing concentration of glycerol. Surprisingly, DTT results strongly stimulatory, an effect that seems generalized for reducing agents (see Carlos PhD’s Thesis).</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<figure class="figure">
<img src="images/reducing.png" title="Effect of different reducing agents on piPolB-MDA assays" class="img-fluid figure-img">
</figure>
<p></p><figcaption class="figure-caption">External Figure 1. Effect of different reducing agents on piPolB-MDA assays.</figcaption><p></p>
</figure>
</div>
</section>
<section id="methods-comparison" class="level3" data-number="3.2">
<h3 data-number="3.2" class="anchored" data-anchor-id="methods-comparison"><span class="header-section-number">3.2</span> Methods comparison</h3>
<div class="cell">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb6"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1" aria-hidden="true" tabindex="-1"></a><span class="co"># read GraphPad Prism file from Carlos experiment</span></span>
<span id="cb6-2"><a href="#cb6-2" aria-hidden="true" tabindex="-1"></a>df <span class="ot">&lt;-</span> <span class="fu">read_pzfx</span>(<span class="st">"amplification_yield/Exp34_Datos.pzfx"</span>, <span class="at">table =</span> <span class="dv">4</span>)</span>
<span id="cb6-3"><a href="#cb6-3" aria-hidden="true" tabindex="-1"></a>free <span class="ot">&lt;-</span> reshape2<span class="sc">::</span><span class="fu">melt</span>(df[, <span class="sc">-</span><span class="dv">2</span>], <span class="at">id.vars =</span> <span class="st">"ROWTITLE"</span>)</span>
<span id="cb6-4"><a href="#cb6-4" aria-hidden="true" tabindex="-1"></a>free<span class="sc">$</span>variable <span class="ot">&lt;-</span> <span class="fu">as.character</span>(free<span class="sc">$</span>variable)</span>
<span id="cb6-5"><a href="#cb6-5" aria-hidden="true" tabindex="-1"></a><span class="co"># recode variable factor for consistent nomenclature</span></span>
<span id="cb6-6"><a href="#cb6-6" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">nrow</span>(free)) {</span>
<span id="cb6-7"><a href="#cb6-7" aria-hidden="true" tabindex="-1"></a>    free<span class="sc">$</span>variable[i] <span class="ot">&lt;-</span> <span class="fu">substr</span>(free<span class="sc">$</span>variable[i], <span class="dv">1</span>, (<span class="fu">nchar</span>(free<span class="sc">$</span>variable[i]) <span class="sc">-</span> <span class="dv">2</span>))</span>
<span id="cb6-8"><a href="#cb6-8" aria-hidden="true" tabindex="-1"></a>    free<span class="sc">$</span>variable[i] <span class="ot">&lt;-</span> <span class="cf">switch</span>(free<span class="sc">$</span>variable[i], <span class="st">`</span><span class="at">REPLI-g</span><span class="st">`</span> <span class="ot">=</span> <span class="st">"RP-MDA"</span>, <span class="at">TruePrime =</span> <span class="st">"PrimPol-MDA"</span>,</span>
<span id="cb6-9"><a href="#cb6-9" aria-hidden="true" tabindex="-1"></a>        <span class="st">`</span><span class="at">pi-mgDNA one-step</span><span class="st">`</span> <span class="ot">=</span> <span class="st">"piMDA"</span>, <span class="st">`</span><span class="at">pi-mgDNA alk. denat.</span><span class="st">`</span> <span class="ot">=</span> <span class="st">"piMDA+D"</span>)</span>
<span id="cb6-10"><a href="#cb6-10" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb6-11"><a href="#cb6-11" aria-hidden="true" tabindex="-1"></a><span class="co"># change decimal separator</span></span>
<span id="cb6-12"><a href="#cb6-12" aria-hidden="true" tabindex="-1"></a>free<span class="sc">$</span>value <span class="ot">&lt;-</span> <span class="fu">as.numeric</span>(<span class="fu">gsub</span>(<span class="st">","</span>, <span class="st">"."</span>, free<span class="sc">$</span>value))</span>
<span id="cb6-13"><a href="#cb6-13" aria-hidden="true" tabindex="-1"></a><span class="co"># reorder</span></span>
<span id="cb6-14"><a href="#cb6-14" aria-hidden="true" tabindex="-1"></a>free<span class="sc">$</span>variable <span class="ot">&lt;-</span> <span class="fu">factor</span>(free<span class="sc">$</span>variable, <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">"RP-MDA"</span>, <span class="st">"PrimPol-MDA"</span>, <span class="st">"piMDA"</span>, <span class="st">"piMDA+D"</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<p>We plot first the DNA input vs.&nbsp;the product yield for the four methods.</p>
<div class="cell" data-layout-align="left">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb7"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb7-1"><a href="#cb7-1" aria-hidden="true" tabindex="-1"></a>p <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(free, <span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">as.factor</span>(ROWTITLE), <span class="at">y =</span> value, <span class="at">color =</span> variable, <span class="at">fill =</span> variable,</span>
<span id="cb7-2"><a href="#cb7-2" aria-hidden="true" tabindex="-1"></a>    <span class="at">alpha =</span> <span class="fl">0.12</span>)) <span class="sc">+</span> <span class="fu">geom_boxplot</span>() <span class="sc">+</span> <span class="fu">scale_y_continuous</span>(<span class="at">limits =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">30</span>), <span class="at">expand =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">0</span>)) <span class="sc">+</span></span>
<span id="cb7-3"><a href="#cb7-3" aria-hidden="true" tabindex="-1"></a>    <span class="fu">guides</span>(<span class="at">alpha =</span> <span class="st">"none"</span>, <span class="at">color =</span> <span class="st">"none"</span>) <span class="sc">+</span> <span class="fu">geom_point</span>(<span class="at">pch =</span> <span class="dv">21</span>, <span class="at">position =</span> <span class="fu">position_jitterdodge</span>(),</span>
<span id="cb7-4"><a href="#cb7-4" aria-hidden="true" tabindex="-1"></a>    <span class="at">alpha =</span> <span class="fl">0.5</span>) <span class="sc">+</span> <span class="fu">facet_grid</span>(<span class="sc">~</span>variable) <span class="sc">+</span> <span class="fu">theme_bw</span>() <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">15</span>)) <span class="sc">+</span></span>
<span id="cb7-5"><a href="#cb7-5" aria-hidden="true" tabindex="-1"></a>    <span class="fu">xlab</span>(<span class="st">"Input DNA (ng)"</span>) <span class="sc">+</span> <span class="fu">ylab</span>(<span class="st">"DNA product (µg)"</span>) <span class="sc">+</span> <span class="fu">labs</span>(<span class="at">fill =</span> <span class="st">"MDA protocol"</span>) <span class="sc">+</span> <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> colorines[<span class="fu">c</span>(<span class="dv">3</span>,</span>
<span id="cb7-6"><a href="#cb7-6" aria-hidden="true" tabindex="-1"></a>    <span class="dv">5</span>, <span class="dv">8</span>, <span class="dv">10</span>)]) <span class="sc">+</span> <span class="fu">scale_color_manual</span>(<span class="at">values =</span> colorines)</span>
<span id="cb7-7"><a href="#cb7-7" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplotly</span>(p)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-99f1d7aaadbc44eff5ba" style="width:100%;height:413px;"></div>
<script type="application/json" data-for="htmlwidget-99f1d7aaadbc44eff5ba">{"x":{"data":[{"x":[1,1,1,1,2,2,2,2,3,3,3,3,4,4,4,4,5,5,5,5],"y":[0.389478,0.000178,0.099965,0.01837006,0.15047,0.923169,1.21415673,0.942836,0.083507,2.76911266,4.511168,1.460909,3.42462905,5.049576,0.213514,2.659323,6.380671,5.116456,6.90471487,1.510188],"hoverinfo":"y","type":"box","fillcolor":"rgba(84,139,84,0.55)","marker":{"opacity":null,"outliercolor":"rgba(0,0,0,1)","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"},"size":5.66929133858268},"line":{"color":"rgba(84,139,84,1)","width":1.88976377952756},"name":"RP-MDA","legendgroup":"RP-MDA","showlegend":true,"xaxis":"x","yaxis":"y","frame":null},{"x":[1,1,1,1,2,2,2,2,3,3,3,3,4,4,4,4,5,5,5,5],"y":[0.180814,null,0.259743,0.2726967,2.607382,2.643574,2.08790265,0.292816,5.279481,6.7520763,6.346972,3.306235,7.37021319,7.667907,6.475921,6.555414,10.913101,8.66595,8.75544662,3.523293],"hoverinfo":"y","type":"box","fillcolor":"rgba(139,37,0,0.55)","marker":{"opacity":null,"outliercolor":"rgba(0,0,0,1)","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"},"size":5.66929133858268},"line":{"color":"rgba(139,37,0,1)","width":1.88976377952756},"name":"PrimPol-MDA","legendgroup":"PrimPol-MDA","showlegend":true,"xaxis":"x2","yaxis":"y","frame":null},{"x":[1,1,1,1,2,2,2,2,3,3,3,3,4,4,4,4,5,5,5,5],"y":[0.004892,0.03469,0.084736,0.04765529,0.224155,0.016125,0.03669276,0.105969,0.940623,0.75269986,0.736413,0.844485,5.14640864,3.830142,3.288347,2.65693,10.272489,7.257752,11.45475466,8.982633],"hoverinfo":"y","type":"box","fillcolor":"rgba(0,154,205,0.55)","marker":{"opacity":null,"outliercolor":"rgba(0,0,0,1)","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"},"size":5.66929133858268},"line":{"color":"rgba(0,154,205,1)","width":1.88976377952756},"name":"piMDA","legendgroup":"piMDA","showlegend":true,"xaxis":"x3","yaxis":"y","frame":null},{"x":[1,1,1,1,2,2,2,2,3,3,3,3,4,4,4,4,5,5,5,5],"y":[0.048647,0.062873,0.053563,0.02944481,0.254553,0.458751,0.18400739,0.185073,7.50242,6.88274625,4.421199,2.276903,11.35291924,17.543596,12.522489,9.576929,26.890394,12.13773,14.26667983,25.81984],"hoverinfo":"y","type":"box","fillcolor":"rgba(0,0,139,0.55)","marker":{"opacity":null,"outliercolor":"rgba(0,0,0,1)","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"},"size":5.66929133858268},"line":{"color":"rgba(0,0,139,1)","width":1.88976377952756},"name":"piMDA+D","legendgroup":"piMDA+D","showlegend":true,"xaxis":"x4","yaxis":"y","frame":null},{"x":[0.944843765565505,1.99502736879513,2.96067388703426,4.02772725503892,4.94008257662257,1.0638214597789,2.02003052042176,3.06269328482449,3.97520681607227,4.94869844745845,1.04582863614584,1.98029217828686,2.94660561187193,4.00254846901322,4.96895367891217,1.06341610237335,1.96988780271883,2.95643398842464,4.04483403240641,5.04725472759455],"y":[0.389478,0.923169,1.460909,2.659323,5.116456,0.000178,0.942836,4.511168,5.049576,6.380671,0.01837006,1.21415673,2.76911266,3.42462905,6.90471487,0.099965,0.15047,0.083507,0.213514,1.510188],"text":["as.factor(ROWTITLE): 0.2<br />value:  0.38947800<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 1<br />value:  0.92316900<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 2<br />value:  1.46090900<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 10<br />value:  2.65932300<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 20<br />value:  5.11645600<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 0.2<br />value:  0.00017800<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 1<br />value:  0.94283600<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 2<br />value:  4.51116800<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 10<br />value:  5.04957600<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 20<br />value:  6.38067100<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 0.2<br />value:  0.01837006<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 1<br />value:  1.21415673<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 2<br />value:  2.76911266<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 10<br />value:  3.42462905<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 20<br />value:  6.90471487<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 0.2<br />value:  0.09996500<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 1<br />value:  0.15047000<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 2<br />value:  0.08350700<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 10<br />value:  0.21351400<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 20<br />value:  1.51018800<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(84,139,84,1)","opacity":0.5,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(84,139,84,1)"}},"hoveron":"points","name":"RP-MDA","legendgroup":"RP-MDA","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0.944843765565505,1.99502736879513,2.96067388703426,4.02772725503892,4.94008257662257,1.0638214597789,2.02003052042176,3.06269328482449,3.97520681607227,4.94869844745845,1.04582863614584,1.98029217828686,2.94660561187193,4.00254846901322,4.96895367891217,1.06341610237335,1.96988780271883,2.95643398842464,4.04483403240641,5.04725472759455],"y":[0.180814,2.643574,3.306235,6.555414,8.66595,null,0.292816,6.346972,7.667907,10.913101,0.2726967,2.08790265,6.7520763,7.37021319,8.75544662,0.259743,2.607382,5.279481,6.475921,3.523293],"text":["as.factor(ROWTITLE): 0.2<br />value:  0.18081400<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 1<br />value:  2.64357400<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 2<br />value:  3.30623500<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 10<br />value:  6.55541400<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 20<br />value:  8.66595000<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 0.2<br />value: -0.01760300<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 1<br />value:  0.29281600<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 2<br />value:  6.34697200<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 10<br />value:  7.66790700<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 20<br />value: 10.91310100<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 0.2<br />value:  0.27269670<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 1<br />value:  2.08790265<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 2<br />value:  6.75207630<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 10<br />value:  7.37021319<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 20<br />value:  8.75544662<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 0.2<br />value:  0.25974300<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 1<br />value:  2.60738200<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 2<br />value:  5.27948100<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 10<br />value:  6.47592100<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 20<br />value:  3.52329300<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(139,37,0,1)","opacity":0.5,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(139,37,0,1)"}},"hoveron":"points","name":"PrimPol-MDA","legendgroup":"PrimPol-MDA","showlegend":false,"xaxis":"x2","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0.944843765565505,1.99502736879513,2.96067388703426,4.02772725503892,4.94008257662257,1.0638214597789,2.02003052042176,3.06269328482449,3.97520681607227,4.94869844745845,1.04582863614584,1.98029217828686,2.94660561187193,4.00254846901322,4.96895367891217,1.06341610237335,1.96988780271883,2.95643398842464,4.04483403240641,5.04725472759455],"y":[0.004892,0.016125,0.844485,2.65693,7.257752,0.03469,0.105969,0.736413,3.830142,10.272489,0.04765529,0.03669276,0.75269986,5.14640864,11.45475466,0.084736,0.224155,0.940623,3.288347,8.982633],"text":["as.factor(ROWTITLE): 0.2<br />value:  0.00489200<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 1<br />value:  0.01612500<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 2<br />value:  0.84448500<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 10<br />value:  2.65693000<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 20<br />value:  7.25775200<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 0.2<br />value:  0.03469000<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 1<br />value:  0.10596900<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 2<br />value:  0.73641300<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 10<br />value:  3.83014200<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 20<br />value: 10.27248900<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 0.2<br />value:  0.04765529<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 1<br />value:  0.03669276<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 2<br />value:  0.75269986<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 10<br />value:  5.14640864<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 20<br />value: 11.45475466<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 0.2<br />value:  0.08473600<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 1<br />value:  0.22415500<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 2<br />value:  0.94062300<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 10<br />value:  3.28834700<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 20<br />value:  8.98263300<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,154,205,1)","opacity":0.5,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,154,205,1)"}},"hoveron":"points","name":"piMDA","legendgroup":"piMDA","showlegend":false,"xaxis":"x3","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0.944843765565505,1.99502736879513,2.96067388703426,4.02772725503892,4.94008257662257,1.0638214597789,2.02003052042176,3.06269328482449,3.97520681607227,4.94869844745845,1.04582863614584,1.98029217828686,2.94660561187193,4.00254846901322,4.96895367891217,1.06341610237335,1.96988780271883,2.95643398842464,4.04483403240641,5.04725472759455],"y":[0.048647,0.458751,2.276903,9.576929,12.13773,0.062873,0.185073,4.421199,17.543596,26.890394,0.02944481,0.18400739,6.88274625,11.35291924,14.26667983,0.053563,0.254553,7.50242,12.522489,25.81984],"text":["as.factor(ROWTITLE): 0.2<br />value:  0.04864700<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 1<br />value:  0.45875100<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 2<br />value:  2.27690300<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 10<br />value:  9.57692900<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 20<br />value: 12.13773000<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 0.2<br />value:  0.06287300<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 1<br />value:  0.18507300<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 2<br />value:  4.42119900<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 10<br />value: 17.54359600<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 20<br />value: 26.89039400<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 0.2<br />value:  0.02944481<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 1<br />value:  0.18400739<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 2<br />value:  6.88274625<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 10<br />value: 11.35291924<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 20<br />value: 14.26667983<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 0.2<br />value:  0.05356300<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 1<br />value:  0.25455300<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 2<br />value:  7.50242000<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 10<br />value: 12.52248900<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 20<br />value: 25.81984000<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,0,139,1)","opacity":0.5,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,139,1)"}},"hoveron":"points","name":"piMDA+D","legendgroup":"piMDA+D","showlegend":false,"xaxis":"x4","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":43.4209808456384,"r":23.2461602324616,"b":50.9992290814209,"l":46.8244084682441},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":19.9252801992528},"xaxis":{"domain":[0,0.243081499930815],"automargin":true,"type":"linear","autorange":false,"range":[0.4,5.6],"tickmode":"array","ticktext":["0.2","1","2","10","20"],"tickvals":[1,2,3,4,5],"categoryorder":"array","categoryarray":["0.2","1","2","10","20"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15.9402241594022},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"annotations":[{"text":"Input DNA (ng)","x":0.5,"y":0,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":19.9252801992528},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"top","annotationType":"axis","yshift":-26.1685346616854},{"text":"DNA product (µg)","x":0,"y":0.5,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":19.9252801992528},"xref":"paper","yref":"paper","textangle":-90,"xanchor":"right","yanchor":"center","annotationType":"axis","xshift":-26.1685346616854},{"text":"RP-MDA","x":0.121540749965407,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(26,26,26,1)","family":"","size":15.9402241594022},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"},{"text":"PrimPol-MDA","x":0.375,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(26,26,26,1)","family":"","size":15.9402241594022},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"},{"text":"piMDA","x":0.625,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(26,26,26,1)","family":"","size":15.9402241594022},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"},{"text":"piMDA+D","x":0.878459250034592,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(26,26,26,1)","family":"","size":15.9402241594022},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"}],"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0,30],"tickmode":"array","ticktext":["0","10","20","30"],"tickvals":[0,10,20,30],"categoryorder":"array","categoryarray":["0","10","20","30"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15.9402241594022},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":0.243081499930815,"y0":0,"y1":1},{"type":"rect","fillcolor":"rgba(217,217,217,1)","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":0.243081499930815,"y0":0,"y1":27.6297218762972,"yanchor":1,"ysizemode":"pixel"},{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.256918500069185,"x1":0.493081499930815,"y0":0,"y1":1},{"type":"rect","fillcolor":"rgba(217,217,217,1)","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.256918500069185,"x1":0.493081499930815,"y0":0,"y1":27.6297218762972,"yanchor":1,"ysizemode":"pixel"},{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.506918500069185,"x1":0.743081499930815,"y0":0,"y1":1},{"type":"rect","fillcolor":"rgba(217,217,217,1)","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.506918500069185,"x1":0.743081499930815,"y0":0,"y1":27.6297218762972,"yanchor":1,"ysizemode":"pixel"},{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.756918500069185,"x1":1,"y0":0,"y1":1},{"type":"rect","fillcolor":"rgba(217,217,217,1)","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.756918500069185,"x1":1,"y0":0,"y1":27.6297218762972,"yanchor":1,"ysizemode":"pixel"}],"xaxis2":{"type":"linear","autorange":false,"range":[0.4,5.6],"tickmode":"array","ticktext":["0.2","1","2","10","20"],"tickvals":[1,2,3,4,5],"categoryorder":"array","categoryarray":["0.2","1","2","10","20"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15.9402241594022},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"domain":[0.256918500069185,0.493081499930815],"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"xaxis3":{"type":"linear","autorange":false,"range":[0.4,5.6],"tickmode":"array","ticktext":["0.2","1","2","10","20"],"tickvals":[1,2,3,4,5],"categoryorder":"array","categoryarray":["0.2","1","2","10","20"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15.9402241594022},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"domain":[0.506918500069185,0.743081499930815],"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"xaxis4":{"type":"linear","autorange":false,"range":[0.4,5.6],"tickmode":"array","ticktext":["0.2","1","2","10","20"],"tickvals":[1,2,3,4,5],"categoryorder":"array","categoryarray":["0.2","1","2","10","20"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15.9402241594022},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"domain":[0.756918500069185,1],"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"showlegend":true,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":15.9402241594022},"title":{"text":"MDA protocol","font":{"color":"rgba(0,0,0,1)","family":"","size":19.9252801992528}}},"hovermode":"closest","barmode":"relative"},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"1701d346c2d06":{"x":{},"y":{},"colour":{},"fill":{},"alpha":{},"type":"box"},"1701d29ffffcd":{"x":{},"y":{},"colour":{},"fill":{},"alpha":{}}},"cur_data":"1701d346c2d06","visdat":{"1701d346c2d06":["function (y) ","x"],"1701d29ffffcd":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<p>Figure 2. DNA amplification yield comparison. All reactions were carried out in 50 µL and incubated at 30ºC for 3 h, except those of RP-MDA protocol that were incubated for 16 h. Shown are results from 4 independent experiments.</p>
</div>
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb8"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb8-1"><a href="#cb8-1" aria-hidden="true" tabindex="-1"></a><span class="co"># ggsave('figures/yieldplot.svg',p,width=8,height=5)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<p>In order to analyze statistically the differences among samples, we will first test normality and homogeneous variance to decide if we should use parametric tests. As shown below, the p-values &lt;0.05 indicate that data are not suitable for parametric tests, leading us to the use of non-parametric pairwise Wilcox comparisons.</p>
<div class="cell">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb9"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb9-1"><a href="#cb9-1" aria-hidden="true" tabindex="-1"></a><span class="co"># statistics comparison of yield/input</span></span>
<span id="cb9-2"><a href="#cb9-2" aria-hidden="true" tabindex="-1"></a>res_aov <span class="ot">&lt;-</span> <span class="fu">aov</span>(value<span class="sc">/</span>ROWTITLE <span class="sc">~</span> variable, <span class="at">data =</span> free)</span>
<span id="cb9-3"><a href="#cb9-3" aria-hidden="true" tabindex="-1"></a><span class="fu">shapiro.test</span>(res_aov<span class="sc">$</span>residuals)  <span class="co">#normality test</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stdout">
<pre><code>
    Shapiro-Wilk normality test

data:  res_aov$residuals
W = 0.90279, p-value = 1.6e-05</code></pre>
</div>
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb11"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb11-1"><a href="#cb11-1" aria-hidden="true" tabindex="-1"></a><span class="fu">leveneTest</span>(value<span class="sc">/</span>ROWTITLE <span class="sc">~</span> variable, <span class="at">data =</span> free)  <span class="co">#NO homogeneus variance</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stdout">
<pre><code>Levene's Test for Homogeneity of Variance (center = median)
      Df F value   Pr(&gt;F)   
group  3  5.6983 0.001423 **
      76                    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</code></pre>
</div>
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb13"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb13-1"><a href="#cb13-1" aria-hidden="true" tabindex="-1"></a><span class="fu">kruskal.test</span>((free<span class="sc">$</span>value<span class="sc">/</span>free<span class="sc">$</span>ROWTITLE), free<span class="sc">$</span>variable, <span class="at">paired =</span> <span class="cn">FALSE</span>, <span class="at">p.adjust.method =</span> <span class="st">"BH"</span>)  <span class="co">#non-parametric multiple groups comparison</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stdout">
<pre><code>
    Kruskal-Wallis rank sum test

data:  (free$value/free$ROWTITLE) and free$variable
Kruskal-Wallis chi-squared = 15.869, df = 3, p-value = 0.001207</code></pre>
</div>
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb15"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb15-1"><a href="#cb15-1" aria-hidden="true" tabindex="-1"></a><span class="fu">pairwise.wilcox.test</span>((free<span class="sc">$</span>value<span class="sc">/</span>free<span class="sc">$</span>ROWTITLE), free<span class="sc">$</span>variable, <span class="at">paired =</span> <span class="cn">FALSE</span>, <span class="at">p.adjust.method =</span> <span class="st">"BH"</span>)  <span class="co">#non-parametric pairwise</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stdout">
<pre><code>
    Pairwise comparisons using Wilcoxon rank sum exact test 

data:  (free$value/free$ROWTITLE) and free$variable 

            RP-MDA  PrimPol-MDA piMDA  
PrimPol-MDA 0.03348 -           -      
piMDA       0.44496 0.00031     -      
piMDA+D     0.18105 0.44496     0.01843

P value adjustment method: BH </code></pre>
</div>
</div>
<p>Statistical analysis indicate that yield differences among MDA methods are statistically significant, particularly the piMDA+D protocol respect to RP-MDA and piPolB. Let’s see about time-course experiments:</p>
<div class="cell" data-layout-align="left">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb17"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb17-1"><a href="#cb17-1" aria-hidden="true" tabindex="-1"></a>df <span class="ot">&lt;-</span> <span class="fu">read_pzfx</span>(<span class="st">"amplification_yield/Exp34_Datos.pzfx"</span>, <span class="at">table =</span> <span class="dv">1</span>)</span>
<span id="cb17-2"><a href="#cb17-2" aria-hidden="true" tabindex="-1"></a>free <span class="ot">&lt;-</span> <span class="fu">melt</span>(df[, <span class="sc">-</span><span class="dv">2</span>], <span class="at">id.vars =</span> <span class="st">"ROWTITLE"</span>)</span>
<span id="cb17-3"><a href="#cb17-3" aria-hidden="true" tabindex="-1"></a>free<span class="sc">$</span>variable <span class="ot">&lt;-</span> <span class="fu">as.character</span>(free<span class="sc">$</span>variable)</span>
<span id="cb17-4"><a href="#cb17-4" aria-hidden="true" tabindex="-1"></a><span class="co"># recode variable factor</span></span>
<span id="cb17-5"><a href="#cb17-5" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">nrow</span>(free)) {</span>
<span id="cb17-6"><a href="#cb17-6" aria-hidden="true" tabindex="-1"></a>    free<span class="sc">$</span>variable[i] <span class="ot">&lt;-</span> <span class="fu">substr</span>(free<span class="sc">$</span>variable[i], <span class="dv">1</span>, (<span class="fu">nchar</span>(free<span class="sc">$</span>variable[i]) <span class="sc">-</span> <span class="dv">2</span>))</span>
<span id="cb17-7"><a href="#cb17-7" aria-hidden="true" tabindex="-1"></a>    free<span class="sc">$</span>variable[i] <span class="ot">&lt;-</span> <span class="cf">switch</span>(free<span class="sc">$</span>variable[i], <span class="st">`</span><span class="at">piDNA Denat.</span><span class="st">`</span> <span class="ot">=</span> <span class="st">"piMDA+D"</span>, <span class="st">`</span><span class="at">piDNA Not denat.</span><span class="st">`</span> <span class="ot">=</span> <span class="st">"piMDA"</span>,</span>
<span id="cb17-8"><a href="#cb17-8" aria-hidden="true" tabindex="-1"></a>        <span class="st">`</span><span class="at">Repli-g</span><span class="st">`</span> <span class="ot">=</span> <span class="st">"RP-MDA"</span>, <span class="at">TruePrime =</span> <span class="st">"PrimPol-MDA"</span>)</span>
<span id="cb17-9"><a href="#cb17-9" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb17-10"><a href="#cb17-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb17-11"><a href="#cb17-11" aria-hidden="true" tabindex="-1"></a><span class="co"># add results from exp42</span></span>
<span id="cb17-12"><a href="#cb17-12" aria-hidden="true" tabindex="-1"></a>df42 <span class="ot">&lt;-</span> <span class="fu">read_pzfx</span>(<span class="st">"amplification_yield/Exp42_Datos.pzfx"</span>, <span class="at">table =</span> <span class="dv">3</span>)</span>
<span id="cb17-13"><a href="#cb17-13" aria-hidden="true" tabindex="-1"></a>df42 <span class="ot">&lt;-</span> df42[<span class="sc">-</span><span class="fu">c</span>(<span class="dv">1</span>, <span class="dv">4</span>, <span class="dv">7</span>), ]  <span class="co">#remove -DNA controls from plot</span></span>
<span id="cb17-14"><a href="#cb17-14" aria-hidden="true" tabindex="-1"></a>df42 <span class="ot">&lt;-</span> df42[, <span class="sc">-</span><span class="fu">c</span>(<span class="dv">3</span>)]  <span class="co">#remove NAs</span></span>
<span id="cb17-15"><a href="#cb17-15" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb17-16"><a href="#cb17-16" aria-hidden="true" tabindex="-1"></a><span class="co"># rename</span></span>
<span id="cb17-17"><a href="#cb17-17" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">nrow</span>(df42)) {</span>
<span id="cb17-18"><a href="#cb17-18" aria-hidden="true" tabindex="-1"></a>    df42<span class="sc">$</span>ROWTITLE[i] <span class="ot">&lt;-</span> <span class="cf">switch</span>(df42<span class="sc">$</span>ROWTITLE[i], <span class="st">`</span><span class="at">piPolB D</span><span class="st">`</span> <span class="ot">=</span> <span class="st">"piPolB+D"</span>, <span class="st">`</span><span class="at">piPolB ND</span><span class="st">`</span> <span class="ot">=</span> <span class="st">"piPolB"</span>,</span>
<span id="cb17-19"><a href="#cb17-19" aria-hidden="true" tabindex="-1"></a>        <span class="st">`</span><span class="at">pi-gAmp D</span><span class="st">`</span> <span class="ot">=</span> <span class="st">"piMDA+D"</span>, <span class="st">`</span><span class="at">pi-gAmp ND</span><span class="st">`</span> <span class="ot">=</span> <span class="st">"piMDA"</span>, <span class="at">TruePrime =</span> <span class="st">"PrimPol-MDA"</span>)</span>
<span id="cb17-20"><a href="#cb17-20" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb17-21"><a href="#cb17-21" aria-hidden="true" tabindex="-1"></a>free42 <span class="ot">&lt;-</span> <span class="fu">melt</span>(df42, <span class="at">id.vars =</span> <span class="st">"ROWTITLE"</span>)</span>
<span id="cb17-22"><a href="#cb17-22" aria-hidden="true" tabindex="-1"></a>free42 <span class="ot">&lt;-</span> free42[, <span class="fu">c</span>(<span class="dv">2</span>, <span class="dv">1</span>, <span class="dv">3</span>)]</span>
<span id="cb17-23"><a href="#cb17-23" aria-hidden="true" tabindex="-1"></a>free42 <span class="ot">&lt;-</span> free42[<span class="sc">-</span><span class="dv">31</span>, ]</span>
<span id="cb17-24"><a href="#cb17-24" aria-hidden="true" tabindex="-1"></a>free42<span class="sc">$</span>variable <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">" h_1"</span>, <span class="st">""</span>, free42<span class="sc">$</span>variable)  <span class="co">#change format</span></span>
<span id="cb17-25"><a href="#cb17-25" aria-hidden="true" tabindex="-1"></a>free42<span class="sc">$</span>variable <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">" h_2"</span>, <span class="st">""</span>, free42<span class="sc">$</span>variable)  <span class="co">#change format</span></span>
<span id="cb17-26"><a href="#cb17-26" aria-hidden="true" tabindex="-1"></a>free42<span class="sc">$</span>variable <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">" h_3"</span>, <span class="st">""</span>, free42<span class="sc">$</span>variable)  <span class="co">#change format</span></span>
<span id="cb17-27"><a href="#cb17-27" aria-hidden="true" tabindex="-1"></a>free42<span class="sc">$</span>variable <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">" h_4"</span>, <span class="st">""</span>, free42<span class="sc">$</span>variable)  <span class="co">#change format</span></span>
<span id="cb17-28"><a href="#cb17-28" aria-hidden="true" tabindex="-1"></a>free42<span class="sc">$</span>variable <span class="ot">&lt;-</span> <span class="fu">as.numeric</span>(free42<span class="sc">$</span>variable)</span>
<span id="cb17-29"><a href="#cb17-29" aria-hidden="true" tabindex="-1"></a><span class="fu">names</span>(free42) <span class="ot">&lt;-</span> <span class="fu">names</span>(free)</span>
<span id="cb17-30"><a href="#cb17-30" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb17-31"><a href="#cb17-31" aria-hidden="true" tabindex="-1"></a>free<span class="sc">$</span>value <span class="ot">&lt;-</span> <span class="fu">as.numeric</span>(<span class="fu">gsub</span>(<span class="st">","</span>, <span class="st">"."</span>, free<span class="sc">$</span>value))  <span class="co">#change decimal separator</span></span>
<span id="cb17-32"><a href="#cb17-32" aria-hidden="true" tabindex="-1"></a>free<span class="sc">$</span>variable <span class="ot">&lt;-</span> <span class="fu">factor</span>(free<span class="sc">$</span>variable, <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">"RP-MDA"</span>, <span class="st">"PrimPol-MDA"</span>, <span class="st">"piPolB"</span>, <span class="st">"piPolB+D"</span>,</span>
<span id="cb17-33"><a href="#cb17-33" aria-hidden="true" tabindex="-1"></a>    <span class="st">"piMDA"</span>, <span class="st">"piMDA+D"</span>))</span>
<span id="cb17-34"><a href="#cb17-34" aria-hidden="true" tabindex="-1"></a>free42<span class="sc">$</span>value <span class="ot">&lt;-</span> <span class="fu">as.numeric</span>(<span class="fu">gsub</span>(<span class="st">","</span>, <span class="st">"."</span>, free42<span class="sc">$</span>value))  <span class="co">#change decimal separator</span></span>
<span id="cb17-35"><a href="#cb17-35" aria-hidden="true" tabindex="-1"></a>free42<span class="sc">$</span>variable <span class="ot">&lt;-</span> <span class="fu">factor</span>(free42<span class="sc">$</span>variable, <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">"RP-MDA"</span>, <span class="st">"PrimPol-MDA"</span>, <span class="st">"piPolB"</span>, <span class="st">"piPolB+D"</span>,</span>
<span id="cb17-36"><a href="#cb17-36" aria-hidden="true" tabindex="-1"></a>    <span class="st">"piMDA"</span>, <span class="st">"piMDA+D"</span>))</span>
<span id="cb17-37"><a href="#cb17-37" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb17-38"><a href="#cb17-38" aria-hidden="true" tabindex="-1"></a>free <span class="ot">&lt;-</span> <span class="fu">rbind</span>(free, free42)</span>
<span id="cb17-39"><a href="#cb17-39" aria-hidden="true" tabindex="-1"></a><span class="co"># plot</span></span>
<span id="cb17-40"><a href="#cb17-40" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb17-41"><a href="#cb17-41" aria-hidden="true" tabindex="-1"></a>q <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(free, <span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">as.factor</span>(ROWTITLE), <span class="at">y =</span> value, <span class="at">colour =</span> variable, <span class="at">fill =</span> variable,</span>
<span id="cb17-42"><a href="#cb17-42" aria-hidden="true" tabindex="-1"></a>    <span class="at">alpha =</span> <span class="fl">0.12</span>)) <span class="sc">+</span> <span class="fu">geom_boxplot</span>() <span class="sc">+</span> <span class="fu">scale_y_continuous</span>(<span class="at">limits =</span> <span class="fu">c</span>(<span class="sc">-</span><span class="fl">0.8</span>, <span class="dv">40</span>), <span class="at">expand =</span> <span class="fu">c</span>(<span class="dv">0</span>,</span>
<span id="cb17-43"><a href="#cb17-43" aria-hidden="true" tabindex="-1"></a>    <span class="dv">0</span>)) <span class="sc">+</span> <span class="fu">guides</span>(<span class="at">alpha =</span> <span class="st">"none"</span>, <span class="at">color =</span> <span class="st">"none"</span>) <span class="sc">+</span> <span class="fu">geom_point</span>(<span class="at">pch =</span> <span class="dv">21</span>, <span class="at">position =</span> <span class="fu">position_jitterdodge</span>(),</span>
<span id="cb17-44"><a href="#cb17-44" aria-hidden="true" tabindex="-1"></a>    <span class="at">alpha =</span> <span class="fl">0.5</span>) <span class="sc">+</span> <span class="fu">facet_grid</span>(<span class="sc">~</span>variable, <span class="at">scales =</span> <span class="st">"free"</span>) <span class="sc">+</span> <span class="fu">theme_bw</span>() <span class="sc">+</span> <span class="fu">force_panelsizes</span>(<span class="at">cols =</span> <span class="fu">c</span>(<span class="fl">1.5</span>,</span>
<span id="cb17-45"><a href="#cb17-45" aria-hidden="true" tabindex="-1"></a>    <span class="dv">3</span>, <span class="fl">1.5</span>, <span class="fl">1.5</span>, <span class="dv">3</span>, <span class="dv">3</span>), <span class="cn">TRUE</span>) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">15</span>)) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">strip.text.x =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">11</span>)) <span class="sc">+</span></span>
<span id="cb17-46"><a href="#cb17-46" aria-hidden="true" tabindex="-1"></a>    <span class="fu">xlab</span>(<span class="st">"Reaction time (h)"</span>) <span class="sc">+</span> <span class="fu">ylab</span>(<span class="st">"DNA product (µg)"</span>) <span class="sc">+</span> <span class="fu">labs</span>(<span class="at">fill =</span> <span class="st">"MDA protocol"</span>) <span class="sc">+</span> <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> colorines[<span class="fu">c</span>(<span class="dv">3</span>,</span>
<span id="cb17-47"><a href="#cb17-47" aria-hidden="true" tabindex="-1"></a>    <span class="dv">5</span>, <span class="dv">6</span>, <span class="dv">7</span>, <span class="dv">8</span>, <span class="dv">10</span>)]) <span class="sc">+</span> <span class="fu">scale_color_manual</span>(<span class="at">values =</span> colorines)</span>
<span id="cb17-48"><a href="#cb17-48" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplotly</span>(q)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Removed 27 rows containing non-finite values (`stat_boxplot()`).</code></pre>
</div>
<div class="cell-output-display">
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-9f522f3366b641371563" style="width:100%;height:413px;"></div>
<script type="application/json" data-for="htmlwidget-9f522f3366b641371563">{"x":{"data":[{"x":[4,4,4,4,5,5,5,5],"y":[0.671343,0.764225,1.128017,1.076899,1.910175,2.798151,1.526284,2.751485],"hoverinfo":"y","type":"box","fillcolor":"rgba(84,139,84,0.55)","marker":{"opacity":null,"outliercolor":"rgba(0,0,0,1)","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"},"size":5.66929133858268},"line":{"color":"rgba(84,139,84,1)","width":1.88976377952756},"name":"RP-MDA","legendgroup":"RP-MDA","showlegend":true,"xaxis":"x","yaxis":"y","frame":null},{"x":[1,1,1,1,2,2,2,2,3,3,3,3,4,4,4,4,4,4,4,5,5,5,5],"y":[1.712829,1.853547,2.113019,1.528933,2.919301,3.482054,3.362202,2.669649,5.989767,5.054349,5.732208,5.630136,7.190141,6.683902,8.464336,2.8868494,7.92515,6.04370663,7.30249416,14.16087203,12.80835108,11.64365916,5.29669112],"hoverinfo":"y","type":"box","fillcolor":"rgba(139,37,0,0.55)","marker":{"opacity":null,"outliercolor":"rgba(0,0,0,1)","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"},"size":5.66929133858268},"line":{"color":"rgba(139,37,0,1)","width":1.88976377952756},"name":"PrimPol-MDA","legendgroup":"PrimPol-MDA","showlegend":true,"xaxis":"x2","yaxis":"y","frame":null},{"x":[1,1,1,2,2,2,2],"y":[-0.11357168,0.00815794,-0.38708556,-0.16202976,5.5828678,2.22566101,0.24329362],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,215,0,0.55)","marker":{"opacity":null,"outliercolor":"rgba(0,0,0,1)","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"},"size":5.66929133858268},"line":{"color":"rgba(255,215,0,1)","width":1.88976377952756},"name":"piPolB","legendgroup":"piPolB","showlegend":true,"xaxis":"x3","yaxis":"y","frame":null},{"x":[1,1,1,2,2,2],"y":[0.00840268,0.04503239,0.1215516,21.26401077,13.40951455,21.55170408],"hoverinfo":"y","type":"box","fillcolor":"rgba(255,185,15,0.55)","marker":{"opacity":null,"outliercolor":"rgba(0,0,0,1)","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"},"size":5.66929133858268},"line":{"color":"rgba(255,185,15,1)","width":1.88976377952756},"name":"piPolB+D","legendgroup":"piPolB+D","showlegend":true,"xaxis":"x4","yaxis":"y","frame":null},{"x":[1,1,1,1,2,2,2,2,3,3,3,3,4,4,4,4,4,4,4,5,5,5,5],"y":[0.008529,0.044687,0.007122,-0.022524,0.041366,0.190759,0.269145,0.118282,1.377841,4.048378,2.518987,3.838436,6.485821,8.79845,9.095781,8.36089315,9.24173431,6.04503182,10.221992,null,35.73614456,36.61536591,29.98517801],"hoverinfo":"y","type":"box","fillcolor":"rgba(0,154,205,0.55)","marker":{"opacity":null,"outliercolor":"rgba(0,0,0,1)","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"},"size":5.66929133858268},"line":{"color":"rgba(0,154,205,1)","width":1.88976377952756},"name":"piMDA","legendgroup":"piMDA","showlegend":true,"xaxis":"x5","yaxis":"y","frame":null},{"x":[1,1,1,1,2,2,2,2,3,3,3,3,4,4,4,4,4,4,4,5,5,5,5],"y":[0.322025,0.664821,0.581926,0.635621,1.978889,2.47953,2.772197,1.566821,10.019175,7.428052,4.875586,7.180053,19.66041797,13.043967,22.92878038,16.308639,12.63950073,17.545357,13.236082,null,null,30.09935975,34.76416894],"hoverinfo":"y","type":"box","fillcolor":"rgba(0,0,139,0.55)","marker":{"opacity":null,"outliercolor":"rgba(0,0,0,1)","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"},"size":5.66929133858268},"line":{"color":"rgba(0,0,139,1)","width":1.88976377952756},"name":"piMDA+D","legendgroup":"piMDA+D","showlegend":true,"xaxis":"x6","yaxis":"y","frame":null},{"x":[1.04591691333335,2.04090196352918,2.99764990175609,3.98389046459924,4.97121384928469,1.0499198412057,2.01535701439716,3.01055299467407,4.03691935690586,5.0061031639576,0.989412086666562,2.04839073149487,3.01033344026655,4.02461909260601,5.03479084724095,1.03274935325608,1.98603042541072,2.95812158754561,3.96892069994938,4.98319124996196],"y":[null,null,null,1.076899,2.798151,null,null,null,0.764225,1.910175,null,null,null,0.671343,1.526284,null,null,null,1.128017,2.751485],"text":["as.factor(ROWTITLE): 0.5<br />value:          NA<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 1<br />value:          NA<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 2<br />value:          NA<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 3<br />value:  1.07689900<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 16<br />value:  2.79815100<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 0.5<br />value:          NA<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 1<br />value:          NA<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 2<br />value:          NA<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 3<br />value:  0.76422500<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 16<br />value:  1.91017500<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 0.5<br />value:          NA<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 1<br />value:          NA<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 2<br />value:          NA<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 3<br />value:  0.67134300<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 16<br />value:  1.52628400<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 0.5<br />value:          NA<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 1<br />value:          NA<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 2<br />value:          NA<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 3<br />value:  1.12801700<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 16<br />value:  2.75148500<br />variable: RP-MDA<br />variable: RP-MDA<br />alpha: 0.5"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(84,139,84,1)","opacity":0.5,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(84,139,84,1)"}},"hoveron":"points","name":"RP-MDA","legendgroup":"RP-MDA","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.04591691333335,2.04090196352918,2.99764990175609,3.98389046459924,5.97121384928469,1.0499198412057,2.01535701439716,3.01055299467407,4.03691935690586,6.0061031639576,0.989412086666562,2.04839073149487,3.01033344026655,4.02461909260601,6.03479084724095,1.03274935325608,1.98603042541072,2.95812158754561,3.96892069994938,5.98319124996196,4.01307925870642,3.97437939536758,3.96707545244135,4.96050219247118,4.99752393327653,5.03119676073547,5.03165077953599],"y":[1.528933,2.669649,5.054349,7.190141,null,1.853547,3.362202,5.630136,6.683902,null,2.113019,3.482054,5.989767,7.92515,null,1.712829,2.919301,5.732208,8.464336,null,7.30249416,6.04370663,2.8868494,11.64365916,14.16087203,12.80835108,5.29669112],"text":["as.factor(ROWTITLE): 0.5<br />value:  1.52893300<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 1<br />value:  2.66964900<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 2<br />value:  5.05434900<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 3<br />value:  7.19014100<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 16<br />value:          NA<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 0.5<br />value:  1.85354700<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 1<br />value:  3.36220200<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 2<br />value:  5.63013600<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 3<br />value:  6.68390200<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 16<br />value:          NA<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 0.5<br />value:  2.11301900<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 1<br />value:  3.48205400<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 2<br />value:  5.98976700<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 3<br />value:  7.92515000<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 16<br />value:          NA<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 0.5<br />value:  1.71282900<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 1<br />value:  2.91930100<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 2<br />value:  5.73220800<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 3<br />value:  8.46433600<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 16<br />value:          NA<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 3<br />value:  7.30249416<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 3<br />value:  6.04370663<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 3<br />value:  2.88684940<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 6<br />value: 11.64365916<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 6<br />value: 14.16087203<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 6<br />value: 12.80835108<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5","as.factor(ROWTITLE): 6<br />value:  5.29669112<br />variable: PrimPol-MDA<br />variable: PrimPol-MDA<br />alpha: 0.5"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(139,37,0,1)","opacity":0.5,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(139,37,0,1)"}},"hoveron":"points","name":"PrimPol-MDA","legendgroup":"PrimPol-MDA","showlegend":false,"xaxis":"x2","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.04591691333335,1.04090196352918,0.997649901756085,1.98389046459924,1.97121384928469,2.0499198412057,2.01535701439716],"y":[-0.38708556,-0.11357168,0.00815794,5.5828678,0.24329362,-0.16202976,2.22566101],"text":["as.factor(ROWTITLE): 3<br />value: -0.38708556<br />variable: piPolB<br />variable: piPolB<br />alpha: 0.5","as.factor(ROWTITLE): 3<br />value: -0.11357168<br />variable: piPolB<br />variable: piPolB<br />alpha: 0.5","as.factor(ROWTITLE): 3<br />value:  0.00815794<br />variable: piPolB<br />variable: piPolB<br />alpha: 0.5","as.factor(ROWTITLE): 6<br />value:  5.58286780<br />variable: piPolB<br />variable: piPolB<br />alpha: 0.5","as.factor(ROWTITLE): 6<br />value:  0.24329362<br />variable: piPolB<br />variable: piPolB<br />alpha: 0.5","as.factor(ROWTITLE): 6<br />value: -0.16202976<br />variable: piPolB<br />variable: piPolB<br />alpha: 0.5","as.factor(ROWTITLE): 6<br />value:  2.22566101<br />variable: piPolB<br />variable: piPolB<br />alpha: 0.5"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(255,215,0,1)","opacity":0.5,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(255,215,0,1)"}},"hoveron":"points","name":"piPolB","legendgroup":"piPolB","showlegend":false,"xaxis":"x3","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.04591691333335,1.04090196352918,0.997649901756085,1.98389046459924,1.97121384928469,2.0499198412057],"y":[0.1215516,0.04503239,0.00840268,21.26401077,13.40951455,21.55170408],"text":["as.factor(ROWTITLE): 3<br />value:  0.12155160<br />variable: piPolB+D<br />variable: piPolB+D<br />alpha: 0.5","as.factor(ROWTITLE): 3<br />value:  0.04503239<br />variable: piPolB+D<br />variable: piPolB+D<br />alpha: 0.5","as.factor(ROWTITLE): 3<br />value:  0.00840268<br />variable: piPolB+D<br />variable: piPolB+D<br />alpha: 0.5","as.factor(ROWTITLE): 6<br />value: 21.26401077<br />variable: piPolB+D<br />variable: piPolB+D<br />alpha: 0.5","as.factor(ROWTITLE): 6<br />value: 13.40951455<br />variable: piPolB+D<br />variable: piPolB+D<br />alpha: 0.5","as.factor(ROWTITLE): 6<br />value: 21.55170408<br />variable: piPolB+D<br />variable: piPolB+D<br />alpha: 0.5"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(255,185,15,1)","opacity":0.5,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(255,185,15,1)"}},"hoveron":"points","name":"piPolB+D","legendgroup":"piPolB+D","showlegend":false,"xaxis":"x4","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.04591691333335,2.04090196352918,2.99764990175609,3.98389046459924,5.97121384928469,1.0499198412057,2.01535701439716,3.01055299467407,4.03691935690586,6.0061031639576,0.989412086666562,2.04839073149487,3.01033344026655,4.02461909260601,6.03479084724095,1.03274935325608,1.98603042541072,2.95812158754561,3.96892069994938,5.98319124996196,4.01307925870642,3.97437939536758,3.96707545244135,4.96050219247118,4.99752393327653,5.03119676073547,5.03165077953599],"y":[0.044687,0.190759,3.838436,10.221992,null,0.008529,0.118282,2.518987,8.79845,null,0.007122,0.269145,4.048378,9.095781,null,-0.022524,0.041366,1.377841,6.485821,null,9.24173431,8.36089315,6.04503182,29.98517801,36.61536591,35.73614456,null],"text":["as.factor(ROWTITLE): 0.5<br />value:  0.04468700<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 1<br />value:  0.19075900<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 2<br />value:  3.83843600<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 3<br />value: 10.22199200<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 16<br />value:          NA<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 0.5<br />value:  0.00852900<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 1<br />value:  0.11828200<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 2<br />value:  2.51898700<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 3<br />value:  8.79845000<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 16<br />value:          NA<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 0.5<br />value:  0.00712200<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 1<br />value:  0.26914500<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 2<br />value:  4.04837800<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 3<br />value:  9.09578100<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 16<br />value:          NA<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 0.5<br />value: -0.02252400<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 1<br />value:  0.04136600<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 2<br />value:  1.37784100<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 3<br />value:  6.48582100<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 16<br />value:          NA<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 3<br />value:  9.24173431<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 3<br />value:  8.36089315<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 3<br />value:  6.04503182<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 6<br />value: 29.98517801<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 6<br />value: 36.61536591<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 6<br />value: 35.73614456<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5","as.factor(ROWTITLE): 6<br />value: 44.94175550<br />variable: piMDA<br />variable: piMDA<br />alpha: 0.5"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,154,205,1)","opacity":0.5,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,154,205,1)"}},"hoveron":"points","name":"piMDA","legendgroup":"piMDA","showlegend":false,"xaxis":"x5","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.04591691333335,2.04090196352918,2.99764990175609,3.98389046459924,5.97121384928469,1.0499198412057,2.01535701439716,3.01055299467407,4.03691935690586,6.0061031639576,0.989412086666562,2.04839073149487,3.01033344026655,4.02461909260601,6.03479084724095,1.03274935325608,1.98603042541072,2.95812158754561,3.96892069994938,5.98319124996196,4.01307925870642,3.97437939536758,3.96707545244135,4.96050219247118,4.99752393327653,5.03119676073547,5.03165077953599],"y":[0.664821,2.47953,7.180053,13.236082,null,0.581926,1.566821,4.875586,13.043967,null,0.322025,1.978889,7.428052,16.308639,null,0.635621,2.772197,10.019175,17.545357,null,19.66041797,22.92878038,12.63950073,34.76416894,30.09935975,null,null],"text":["as.factor(ROWTITLE): 0.5<br />value:  0.66482100<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 1<br />value:  2.47953000<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 2<br />value:  7.18005300<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 3<br />value: 13.23608200<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 16<br />value:          NA<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 0.5<br />value:  0.58192600<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 1<br />value:  1.56682100<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 2<br />value:  4.87558600<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 3<br />value: 13.04396700<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 16<br />value:          NA<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 0.5<br />value:  0.32202500<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 1<br />value:  1.97888900<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 2<br />value:  7.42805200<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 3<br />value: 16.30863900<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 16<br />value:          NA<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 0.5<br />value:  0.63562100<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 1<br />value:  2.77219700<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 2<br />value: 10.01917500<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 3<br />value: 17.54535700<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 16<br />value:          NA<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 3<br />value: 19.66041797<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 3<br />value: 22.92878038<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 3<br />value: 12.63950073<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 6<br />value: 34.76416894<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 6<br />value: 30.09935975<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 6<br />value: 47.79950780<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5","as.factor(ROWTITLE): 6<br />value: 99.24776479<br />variable: piMDA+D<br />variable: piMDA+D<br />alpha: 0.5"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(0,0,139,1)","opacity":0.5,"size":5.66929133858268,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,139,1)"}},"hoveron":"points","name":"piMDA+D","legendgroup":"piMDA+D","showlegend":false,"xaxis":"x6","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":42.0926288323549,"r":21.9178082191781,"b":50.9992290814209,"l":46.8244084682441},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":19.9252801992528},"xaxis":{"domain":[0,0.159748166597482],"automargin":true,"type":"linear","autorange":false,"range":[0.4,5.6],"tickmode":"array","ticktext":["0.5","1","2","3","16"],"tickvals":[1,2,3,4,5],"categoryorder":"array","categoryarray":["0.5","1","2","3","16"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15.9402241594022},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"annotations":[{"text":"Reaction time (h)","x":0.5,"y":0,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":19.9252801992528},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"top","annotationType":"axis","yshift":-26.1685346616854},{"text":"DNA product (µg)","x":0,"y":0.5,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":19.9252801992528},"xref":"paper","yref":"paper","textangle":-90,"xanchor":"right","yanchor":"center","annotationType":"axis","xshift":-26.1685346616854},{"text":"RP-MDA","x":0.0798740832987408,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(26,26,26,1)","family":"","size":14.6118721461187},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"},{"text":"PrimPol-MDA","x":0.25,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(26,26,26,1)","family":"","size":14.6118721461187},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"},{"text":"piPolB","x":0.416666666666667,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(26,26,26,1)","family":"","size":14.6118721461187},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"},{"text":"piPolB+D","x":0.583333333333333,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(26,26,26,1)","family":"","size":14.6118721461187},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"},{"text":"piMDA","x":0.75,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(26,26,26,1)","family":"","size":14.6118721461187},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"},{"text":"piMDA+D","x":0.920125916701259,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(26,26,26,1)","family":"","size":14.6118721461187},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"}],"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-0.8,40],"tickmode":"array","ticktext":["0","10","20","30","40"],"tickvals":[0,10,20,30,40],"categoryorder":"array","categoryarray":["0","10","20","30","40"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15.9402241594022},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":0.159748166597482,"y0":0,"y1":1},{"type":"rect","fillcolor":"rgba(217,217,217,1)","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":0.159748166597482,"y0":0,"y1":26.3013698630137,"yanchor":1,"ysizemode":"pixel"},{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.173585166735852,"x1":0.326414833264148,"y0":0,"y1":1},{"type":"rect","fillcolor":"rgba(217,217,217,1)","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.173585166735852,"x1":0.326414833264148,"y0":0,"y1":26.3013698630137,"yanchor":1,"ysizemode":"pixel"},{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.340251833402518,"x1":0.493081499930815,"y0":0,"y1":1},{"type":"rect","fillcolor":"rgba(217,217,217,1)","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.340251833402518,"x1":0.493081499930815,"y0":0,"y1":26.3013698630137,"yanchor":1,"ysizemode":"pixel"},{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.506918500069185,"x1":0.659748166597482,"y0":0,"y1":1},{"type":"rect","fillcolor":"rgba(217,217,217,1)","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.506918500069185,"x1":0.659748166597482,"y0":0,"y1":26.3013698630137,"yanchor":1,"ysizemode":"pixel"},{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.673585166735852,"x1":0.826414833264148,"y0":0,"y1":1},{"type":"rect","fillcolor":"rgba(217,217,217,1)","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.673585166735852,"x1":0.826414833264148,"y0":0,"y1":26.3013698630137,"yanchor":1,"ysizemode":"pixel"},{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.840251833402518,"x1":1,"y0":0,"y1":1},{"type":"rect","fillcolor":"rgba(217,217,217,1)","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.840251833402518,"x1":1,"y0":0,"y1":26.3013698630137,"yanchor":1,"ysizemode":"pixel"}],"xaxis2":{"type":"linear","autorange":false,"range":[0.4,6.6],"tickmode":"array","ticktext":["0.5","1","2","3","6","16"],"tickvals":[1,2,3,4,5,6],"categoryorder":"array","categoryarray":["0.5","1","2","3","6","16"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15.9402241594022},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"domain":[0.173585166735852,0.326414833264148],"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"xaxis3":{"type":"linear","autorange":false,"range":[0.4,2.6],"tickmode":"array","ticktext":["3","6"],"tickvals":[1,2],"categoryorder":"array","categoryarray":["3","6"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15.9402241594022},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"domain":[0.340251833402518,0.493081499930815],"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"xaxis4":{"type":"linear","autorange":false,"range":[0.4,2.6],"tickmode":"array","ticktext":["3","6"],"tickvals":[1,2],"categoryorder":"array","categoryarray":["3","6"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15.9402241594022},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"domain":[0.506918500069185,0.659748166597482],"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"xaxis5":{"type":"linear","autorange":false,"range":[0.4,6.6],"tickmode":"array","ticktext":["0.5","1","2","3","6","16"],"tickvals":[1,2,3,4,5,6],"categoryorder":"array","categoryarray":["0.5","1","2","3","6","16"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15.9402241594022},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"domain":[0.673585166735852,0.826414833264148],"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"xaxis6":{"type":"linear","autorange":false,"range":[0.4,6.6],"tickmode":"array","ticktext":["0.5","1","2","3","6","16"],"tickvals":[1,2,3,4,5,6],"categoryorder":"array","categoryarray":["0.5","1","2","3","6","16"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15.9402241594022},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"domain":[0.840251833402518,1],"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"showlegend":true,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":15.9402241594022},"title":{"text":"MDA protocol","font":{"color":"rgba(0,0,0,1)","family":"","size":19.9252801992528}}},"hovermode":"closest","barmode":"relative"},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"1701d43d4a9e6":{"x":{},"y":{},"colour":{},"fill":{},"alpha":{},"type":"box"},"1701d3ede5dd4":{"x":{},"y":{},"colour":{},"fill":{},"alpha":{}}},"cur_data":"1701d43d4a9e6","visdat":{"1701d43d4a9e6":["function (y) ","x"],"1701d3ede5dd4":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<p>Figure 3. DNA amplification time-course. All the amplification assays contained 10 ng of input DNA sample. Shown is the total DNA product in 50 µL reactions. Amplification yield from 2-4 experiments are shown.</p>
</div>
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb19"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb19-1"><a href="#cb19-1" aria-hidden="true" tabindex="-1"></a><span class="co"># ggsave('figures/timecourse.svg',q,width=9,height=5)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<div class="cell">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb20"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb20-1"><a href="#cb20-1" aria-hidden="true" tabindex="-1"></a><span class="co"># statistics comparison of yield/hour</span></span>
<span id="cb20-2"><a href="#cb20-2" aria-hidden="true" tabindex="-1"></a>res_aov <span class="ot">&lt;-</span> <span class="fu">aov</span>(value<span class="sc">/</span>ROWTITLE <span class="sc">~</span> variable, <span class="at">data =</span> free)</span>
<span id="cb20-3"><a href="#cb20-3" aria-hidden="true" tabindex="-1"></a><span class="fu">shapiro.test</span>(res_aov<span class="sc">$</span>residuals)  <span class="co">#normality test - NOPE</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stdout">
<pre><code>
    Shapiro-Wilk normality test

data:  res_aov$residuals
W = 0.82386, p-value = 5.609e-09</code></pre>
</div>
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb22"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb22-1"><a href="#cb22-1" aria-hidden="true" tabindex="-1"></a><span class="fu">leveneTest</span>(value<span class="sc">/</span>ROWTITLE <span class="sc">~</span> variable, <span class="at">data =</span> free)  <span class="co">#NO homogeneus variance</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stdout">
<pre><code>Levene's Test for Homogeneity of Variance (center = median)
      Df F value    Pr(&gt;F)    
group  5  5.1655 0.0003512 ***
      84                      
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</code></pre>
</div>
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb24"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb24-1"><a href="#cb24-1" aria-hidden="true" tabindex="-1"></a><span class="fu">kruskal.test</span>((free<span class="sc">$</span>value<span class="sc">/</span>free<span class="sc">$</span>ROWTITLE), free<span class="sc">$</span>variable, <span class="at">paired =</span> <span class="cn">FALSE</span>, <span class="at">p.adjust.method =</span> <span class="st">"BH"</span>)  <span class="co">#non-parametric multiple groups comparison</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stdout">
<pre><code>
    Kruskal-Wallis rank sum test

data:  (free$value/free$ROWTITLE) and free$variable
Kruskal-Wallis chi-squared = 35.508, df = 5, p-value = 1.191e-06</code></pre>
</div>
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb26"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb26-1"><a href="#cb26-1" aria-hidden="true" tabindex="-1"></a>pp <span class="ot">&lt;-</span> <span class="fu">pairwise.wilcox.test</span>((free<span class="sc">$</span>value<span class="sc">/</span>free<span class="sc">$</span>ROWTITLE), free<span class="sc">$</span>variable, <span class="at">paired =</span> <span class="cn">FALSE</span>, <span class="at">p.adjust.method =</span> <span class="st">"BH"</span>,</span>
<span id="cb26-2"><a href="#cb26-2" aria-hidden="true" tabindex="-1"></a>    <span class="at">exact =</span> <span class="cn">FALSE</span>)  <span class="co">#non-parametric pairwise</span></span>
<span id="cb26-3"><a href="#cb26-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb26-4"><a href="#cb26-4" aria-hidden="true" tabindex="-1"></a><span class="fu">as.data.frame</span>(<span class="fu">format</span>(pp<span class="sc">$</span>p.value, <span class="at">scientific =</span> <span class="cn">FALSE</span>, <span class="at">digits =</span> <span class="dv">1</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb26-5"><a href="#cb26-5" aria-hidden="true" tabindex="-1"></a>    <span class="fu">replace</span>(., . <span class="sc">&lt;</span> <span class="dv">0</span>, <span class="st">""</span>) <span class="sc">%&gt;%</span></span>
<span id="cb26-6"><a href="#cb26-6" aria-hidden="true" tabindex="-1"></a>    <span class="fu">mutate_all</span>(<span class="sc">~</span><span class="fu">cell_spec</span>(.x, <span class="at">color =</span> <span class="fu">ifelse</span>(.x <span class="sc">&lt;</span> <span class="fl">0.01</span>, <span class="st">"firebrick"</span>, <span class="fu">ifelse</span>(.x <span class="sc">&lt;</span> <span class="fl">0.05</span>, <span class="st">"steelblue"</span>,</span>
<span id="cb26-7"><a href="#cb26-7" aria-hidden="true" tabindex="-1"></a>        <span class="st">"black"</span>)))) <span class="sc">%&gt;%</span></span>
<span id="cb26-8"><a href="#cb26-8" aria-hidden="true" tabindex="-1"></a>    <span class="fu">kable</span>(<span class="at">escape =</span> F, <span class="at">align =</span> <span class="st">"c"</span>, <span class="at">caption =</span> <span class="st">"Table 2. P-value of pairwise comparison of time course assays"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb26-9"><a href="#cb26-9" aria-hidden="true" tabindex="-1"></a>    <span class="fu">kable_styling</span>(<span class="at">bootstrap_options =</span> <span class="st">"striped"</span>, <span class="at">full_width =</span> F, <span class="at">position =</span> <span class="st">"center"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb26-10"><a href="#cb26-10" aria-hidden="true" tabindex="-1"></a>    <span class="fu">column_spec</span>(<span class="dv">1</span>, <span class="at">bold =</span> T)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">

<table class="table table-striped" style="width: auto !important; margin-left: auto; margin-right: auto;">
<caption>Table 2. P-value of pairwise comparison of time course assays</caption>
 <thead>
  <tr>
   <th style="text-align:left;">   </th>
   <th style="text-align:center;"> RP-MDA </th>
   <th style="text-align:center;"> PrimPol-MDA </th>
   <th style="text-align:center;"> piPolB </th>
   <th style="text-align:center;"> piPolB+D </th>
   <th style="text-align:center;"> piMDA </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;font-weight: bold;"> PrimPol-MDA </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;">0.0003</span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
  </tr>
  <tr>
   <td style="text-align:left;font-weight: bold;"> piPolB </td>
   <td style="text-align:center;"> <span style="     color: black !important;">0.1850</span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;">0.0004</span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
  </tr>
  <tr>
   <td style="text-align:left;font-weight: bold;"> piPolB+D </td>
   <td style="text-align:center;"> <span style="     color: black !important;">1.0000</span> </td>
   <td style="text-align:center;"> <span style="     color: black !important;">0.3112</span> </td>
   <td style="text-align:center;"> <span style="     color: black !important;">0.1483</span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
  </tr>
  <tr>
   <td style="text-align:left;font-weight: bold;"> piMDA </td>
   <td style="text-align:center;"> <span style="     color: black !important;">0.1370</span> </td>
   <td style="text-align:center;"> <span style="     color: black !important;">0.1483</span> </td>
   <td style="text-align:center;"> <span style="     color: steelblue !important;">0.0156</span> </td>
   <td style="text-align:center;"> <span style="     color: black !important;">0.6127</span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
  </tr>
  <tr>
   <td style="text-align:left;font-weight: bold;"> piMDA+D </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;">0.0003</span> </td>
   <td style="text-align:center;"> <span style="     color: black !important;">0.0730</span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;">0.0004</span> </td>
   <td style="text-align:center;"> <span style="     color: steelblue !important;">0.0411</span> </td>
   <td style="text-align:center;"> <span style="     color: steelblue !important;">0.0196</span> </td>
  </tr>
</tbody>
</table>

</div>
</div>
<p>These results show that the piMDA method is comparable with other MDA alternatives in terms of sensitivity and DNA production. Moreover, its yield can be increased with the addition of a DNA denaturation step, obtaining statistically significant more amount of DNA per time unit than all the other protocols, except PrimPol-MDA.</p>
<p>It should be also considered that these results apply specifically to this particular sample, and the reduced yield of RP-MDA respect to the other methods is likely by the presence of high GC content sequences, as discussed below.</p>
</section>
</section>
<section id="illumina-raw-data-processing" class="level2" data-number="4">
<h2 data-number="4" class="anchored" data-anchor-id="illumina-raw-data-processing"><span class="header-section-number">4</span> Illumina raw data processing</h2>
<p>Samples from independent amplification experiments, as well as the control non-amplified metagenome (NA), were sequenced by Illumina NextSeq V2.5 High Output, 2x75 bp paired ends sequencing. To reduce the sequencing-derived bias, the sequencing libraries were prepared using TruSeq DNA PCR-Free Low Throughput Library Prep kit (Illumina), at Conceição Egas’ lab (<a href="http://www.genoinseq.com/en/home/" target="_blank">GENOINSEQ - BIOCANT</a>, Portugal). As shown below, NA, RP-MDA, piMDA and piMDA+D were sequenced from duplicated independent experiments.</p>
<section id="qc-stats-and-trimming" class="level3" data-number="4.1">
<h3 data-number="4.1" class="anchored" data-anchor-id="qc-stats-and-trimming"><span class="header-section-number">4.1</span> QC stats and trimming</h3>
<p>The raw files were first analyzed for basic statistics with <a href="https://bioinf.shenwei.me/seqkit/" target="_blank">Seqkit</a> v.2.3 (Table 3).</p>
<div class="cell">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb27"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb27-1"><a href="#cb27-1" aria-hidden="true" tabindex="-1"></a>samples <span class="ot">&lt;-</span> <span class="fu">read.table</span>(<span class="st">"qc/raw_stats.csv"</span>, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">";"</span>, <span class="at">na.strings =</span> <span class="st">""</span>)</span>
<span id="cb27-2"><a href="#cb27-2" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">nrow</span>(samples)) {</span>
<span id="cb27-3"><a href="#cb27-3" aria-hidden="true" tabindex="-1"></a>    samples<span class="sc">$</span>file[i] <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">"freeDNA_data/"</span>, <span class="st">""</span>, samples<span class="sc">$</span>file[i])</span>
<span id="cb27-4"><a href="#cb27-4" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb27-5"><a href="#cb27-5" aria-hidden="true" tabindex="-1"></a><span class="fu">kbl</span>(samples, <span class="at">align =</span> <span class="st">"c"</span>, <span class="at">caption =</span> <span class="st">"Table 3. Stats of raw Illumina data"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb27-6"><a href="#cb27-6" aria-hidden="true" tabindex="-1"></a>    <span class="fu">kable_styling</span>(<span class="at">bootstrap_options =</span> <span class="st">"striped"</span>, <span class="at">full_width =</span> F, <span class="at">position =</span> <span class="st">"center"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb27-7"><a href="#cb27-7" aria-hidden="true" tabindex="-1"></a>    <span class="fu">column_spec</span>(<span class="dv">1</span>, <span class="at">bold =</span> T)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">

<table class="table table-striped" style="width: auto !important; margin-left: auto; margin-right: auto;">
<caption>Table 3. Stats of raw Illumina data</caption>
 <thead>
  <tr>
   <th style="text-align:center;"> Sample </th>
   <th style="text-align:center;"> file </th>
   <th style="text-align:center;"> format </th>
   <th style="text-align:center;"> type </th>
   <th style="text-align:center;"> num_seqs </th>
   <th style="text-align:center;"> sum_len </th>
   <th style="text-align:center;"> min_len </th>
   <th style="text-align:center;"> avg_len </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:center;font-weight: bold;"> piMDA+D2 </td>
   <td style="text-align:center;"> A2_R1.fastq.gz </td>
   <td style="text-align:center;"> FASTQ </td>
   <td style="text-align:center;"> DNA </td>
   <td style="text-align:center;"> 81,578,075 </td>
   <td style="text-align:center;"> 6,172,839,185 </td>
   <td style="text-align:center;"> 35 </td>
   <td style="text-align:center;"> 75.7 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-weight: bold;"> piMDA+D2 </td>
   <td style="text-align:center;"> A2_R2.fastq.gz </td>
   <td style="text-align:center;"> FASTQ </td>
   <td style="text-align:center;"> DNA </td>
   <td style="text-align:center;"> 81,578,075 </td>
   <td style="text-align:center;"> 6,172,767,989 </td>
   <td style="text-align:center;"> 35 </td>
   <td style="text-align:center;"> 75.7 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-weight: bold;"> piMDA2 </td>
   <td style="text-align:center;"> B2_R1.fastq.gz </td>
   <td style="text-align:center;"> FASTQ </td>
   <td style="text-align:center;"> DNA </td>
   <td style="text-align:center;"> 75,890,757 </td>
   <td style="text-align:center;"> 5,742,701,890 </td>
   <td style="text-align:center;"> 35 </td>
   <td style="text-align:center;"> 75.7 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-weight: bold;"> piMDA2 </td>
   <td style="text-align:center;"> B2_R2.fastq.gz </td>
   <td style="text-align:center;"> FASTQ </td>
   <td style="text-align:center;"> DNA </td>
   <td style="text-align:center;"> 75,890,757 </td>
   <td style="text-align:center;"> 5,742,589,467 </td>
   <td style="text-align:center;"> 35 </td>
   <td style="text-align:center;"> 75.7 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-weight: bold;"> Primpol-MDA </td>
   <td style="text-align:center;"> C3_R1.fastq.gz </td>
   <td style="text-align:center;"> FASTQ </td>
   <td style="text-align:center;"> DNA </td>
   <td style="text-align:center;"> 57,026,214 </td>
   <td style="text-align:center;"> 4,309,432,584 </td>
   <td style="text-align:center;"> 35 </td>
   <td style="text-align:center;"> 75.6 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-weight: bold;"> Primpol-MDA </td>
   <td style="text-align:center;"> C3_R2.fastq.gz </td>
   <td style="text-align:center;"> FASTQ </td>
   <td style="text-align:center;"> DNA </td>
   <td style="text-align:center;"> 57,026,214 </td>
   <td style="text-align:center;"> 4,309,441,640 </td>
   <td style="text-align:center;"> 35 </td>
   <td style="text-align:center;"> 75.6 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-weight: bold;"> NA2 </td>
   <td style="text-align:center;"> Ctrl2_R1.fastq.gz </td>
   <td style="text-align:center;"> FASTQ </td>
   <td style="text-align:center;"> DNA </td>
   <td style="text-align:center;"> 76,474,759 </td>
   <td style="text-align:center;"> 5,784,350,652 </td>
   <td style="text-align:center;"> 35 </td>
   <td style="text-align:center;"> 75.6 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-weight: bold;"> NA2 </td>
   <td style="text-align:center;"> Ctrl2_R2.fastq.gz </td>
   <td style="text-align:center;"> FASTQ </td>
   <td style="text-align:center;"> DNA </td>
   <td style="text-align:center;"> 76,474,759 </td>
   <td style="text-align:center;"> 5,784,333,531 </td>
   <td style="text-align:center;"> 35 </td>
   <td style="text-align:center;"> 75.6 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-weight: bold;"> NA2 </td>
   <td style="text-align:center;"> Ctrl_R1.fastq.gz </td>
   <td style="text-align:center;"> FASTQ </td>
   <td style="text-align:center;"> DNA </td>
   <td style="text-align:center;"> 71,838,581 </td>
   <td style="text-align:center;"> 5,432,947,576 </td>
   <td style="text-align:center;"> 35 </td>
   <td style="text-align:center;"> 75.6 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-weight: bold;"> NA2 </td>
   <td style="text-align:center;"> Ctrl_R2.fastq.gz </td>
   <td style="text-align:center;"> FASTQ </td>
   <td style="text-align:center;"> DNA </td>
   <td style="text-align:center;"> 71,838,581 </td>
   <td style="text-align:center;"> 5,432,944,267 </td>
   <td style="text-align:center;"> 35 </td>
   <td style="text-align:center;"> 75.6 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-weight: bold;"> RP-MDA </td>
   <td style="text-align:center;"> D3_R1.fastq.gz </td>
   <td style="text-align:center;"> FASTQ </td>
   <td style="text-align:center;"> DNA </td>
   <td style="text-align:center;"> 67,415,579 </td>
   <td style="text-align:center;"> 5,093,810,770 </td>
   <td style="text-align:center;"> 35 </td>
   <td style="text-align:center;"> 75.6 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-weight: bold;"> RP-MDA </td>
   <td style="text-align:center;"> D3_R2.fastq.gz </td>
   <td style="text-align:center;"> FASTQ </td>
   <td style="text-align:center;"> DNA </td>
   <td style="text-align:center;"> 67,415,579 </td>
   <td style="text-align:center;"> 5,093,837,342 </td>
   <td style="text-align:center;"> 35 </td>
   <td style="text-align:center;"> 75.6 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-weight: bold;"> piPolB+D </td>
   <td style="text-align:center;"> F2_R1.fastq.gz </td>
   <td style="text-align:center;"> FASTQ </td>
   <td style="text-align:center;"> DNA </td>
   <td style="text-align:center;"> 69,333,871 </td>
   <td style="text-align:center;"> 5,181,529,472 </td>
   <td style="text-align:center;"> 35 </td>
   <td style="text-align:center;"> 74.7 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-weight: bold;"> piPolB+D </td>
   <td style="text-align:center;"> F2_R2.fastq.gz </td>
   <td style="text-align:center;"> FASTQ </td>
   <td style="text-align:center;"> DNA </td>
   <td style="text-align:center;"> 69,333,871 </td>
   <td style="text-align:center;"> 5,188,090,985 </td>
   <td style="text-align:center;"> 35 </td>
   <td style="text-align:center;"> 74.8 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-weight: bold;"> piPolB </td>
   <td style="text-align:center;"> N1_R1.fastq.gz </td>
   <td style="text-align:center;"> FASTQ </td>
   <td style="text-align:center;"> DNA </td>
   <td style="text-align:center;"> 66,003,225 </td>
   <td style="text-align:center;"> 4,946,827,786 </td>
   <td style="text-align:center;"> 35 </td>
   <td style="text-align:center;"> 74.9 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-weight: bold;"> piPolB </td>
   <td style="text-align:center;"> N1_R2.fastq.gz </td>
   <td style="text-align:center;"> FASTQ </td>
   <td style="text-align:center;"> DNA </td>
   <td style="text-align:center;"> 66,003,225 </td>
   <td style="text-align:center;"> 4,953,211,772 </td>
   <td style="text-align:center;"> 35 </td>
   <td style="text-align:center;"> 75.0 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-weight: bold;"> piMDA </td>
   <td style="text-align:center;"> N2_R1.fastq.gz </td>
   <td style="text-align:center;"> FASTQ </td>
   <td style="text-align:center;"> DNA </td>
   <td style="text-align:center;"> 69,872,142 </td>
   <td style="text-align:center;"> 5,287,166,316 </td>
   <td style="text-align:center;"> 35 </td>
   <td style="text-align:center;"> 75.7 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-weight: bold;"> piMDA </td>
   <td style="text-align:center;"> N2_R2.fastq.gz </td>
   <td style="text-align:center;"> FASTQ </td>
   <td style="text-align:center;"> DNA </td>
   <td style="text-align:center;"> 69,872,142 </td>
   <td style="text-align:center;"> 5,287,208,173 </td>
   <td style="text-align:center;"> 35 </td>
   <td style="text-align:center;"> 75.7 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-weight: bold;"> RP-MDA </td>
   <td style="text-align:center;"> N4_R1.fastq.gz </td>
   <td style="text-align:center;"> FASTQ </td>
   <td style="text-align:center;"> DNA </td>
   <td style="text-align:center;"> 66,722,442 </td>
   <td style="text-align:center;"> 5,040,940,307 </td>
   <td style="text-align:center;"> 35 </td>
   <td style="text-align:center;"> 75.6 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-weight: bold;"> RP-MDA </td>
   <td style="text-align:center;"> N4_R2.fastq.gz </td>
   <td style="text-align:center;"> FASTQ </td>
   <td style="text-align:center;"> DNA </td>
   <td style="text-align:center;"> 66,722,442 </td>
   <td style="text-align:center;"> 5,041,277,271 </td>
   <td style="text-align:center;"> 35 </td>
   <td style="text-align:center;"> 75.6 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-weight: bold;"> piMDA+D </td>
   <td style="text-align:center;"> N6_R1.fastq.gz </td>
   <td style="text-align:center;"> FASTQ </td>
   <td style="text-align:center;"> DNA </td>
   <td style="text-align:center;"> 64,762,150 </td>
   <td style="text-align:center;"> 4,901,577,518 </td>
   <td style="text-align:center;"> 35 </td>
   <td style="text-align:center;"> 75.7 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-weight: bold;"> piMDA+D </td>
   <td style="text-align:center;"> N6_R2.fastq.gz </td>
   <td style="text-align:center;"> FASTQ </td>
   <td style="text-align:center;"> DNA </td>
   <td style="text-align:center;"> 64,762,150 </td>
   <td style="text-align:center;"> 4,901,629,402 </td>
   <td style="text-align:center;"> 35 </td>
   <td style="text-align:center;"> 75.7 </td>
  </tr>
</tbody>
</table>

</div>
</div>
<p>Then, a detailed quality check (QC) was performed with FastQC v0.11.9 for each sample, and reports aggregated with <a href="https://multiqc.info/" target="_blank">MultiQC 1.11</a>. Reads were subsequently trimmed with <a href="https://github.com/usadellab/Trimmomatic" target="_blank">Trimmomatic</a> 0.39, using a customized script, based on <a href="https://github.com/lakhujanivijay/I-Shell-Do-This" target="_blank">I-Shell-Do-This Github Repo</a> (parameters: LEADING:3 TRAILING:3 SLIDINGWINDOW:4:20 MINLEN:35). A post-trimming QC report was obtained as above. Due to size limitations, Trimmomatic output files are not available in this repository, they’ll be deposited along with the raw reads. Individual QC reports as well as MultiQC-merged reports <a href="qc/multiqc_report.html" target="_blank">before</a> and <a href="qc/multiqc_report_1.html" target="_blank">after</a> Trimmomatic are available in the <em>qc</em> folder of the <a href="https://github.com/mredrejo/pimda" target="_blank">GitHub repository</a>. We include in Figures 4-6 some of the QC results. First, we will have a look to the duplication level of the reads, which is a good indicative of overamplification of some sequences, a common bias source in exponential DNA amplification.</p>
<div class="cell" data-layout-align="left">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb28"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb28-1"><a href="#cb28-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Duplicated reads</span></span>
<span id="cb28-2"><a href="#cb28-2" aria-hidden="true" tabindex="-1"></a>duplication <span class="ot">&lt;-</span> <span class="fu">read.table</span>(<span class="st">"qc/fastqc_sequence_duplication_levels_plot.tsv"</span>, <span class="at">header =</span> <span class="cn">TRUE</span>,</span>
<span id="cb28-3"><a href="#cb28-3" aria-hidden="true" tabindex="-1"></a>    <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>, <span class="at">na.strings =</span> <span class="st">""</span>)</span>
<span id="cb28-4"><a href="#cb28-4" aria-hidden="true" tabindex="-1"></a><span class="co"># reorder</span></span>
<span id="cb28-5"><a href="#cb28-5" aria-hidden="true" tabindex="-1"></a>duplication <span class="ot">&lt;-</span> duplication[<span class="fu">c</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">11</span>, <span class="dv">20</span>, <span class="dv">21</span>, <span class="dv">22</span>, <span class="dv">23</span>, <span class="dv">12</span><span class="sc">:</span><span class="dv">19</span>)]</span>
<span id="cb28-6"><a href="#cb28-6" aria-hidden="true" tabindex="-1"></a>dupli_table <span class="ot">&lt;-</span> <span class="fu">melt</span>(duplication)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stderr">
<pre><code>Using Sequence.Duplication.Level as id variables</code></pre>
</div>
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb30"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb30-1"><a href="#cb30-1" aria-hidden="true" tabindex="-1"></a>dupli_table[, <span class="dv">2</span>] <span class="ot">&lt;-</span> <span class="fu">sub</span>(<span class="st">"RepliG"</span>, <span class="st">"RP-MDA"</span>, dupli_table[, <span class="dv">2</span>])</span>
<span id="cb30-2"><a href="#cb30-2" aria-hidden="true" tabindex="-1"></a>dupli_table[, <span class="dv">2</span>] <span class="ot">&lt;-</span> <span class="fu">sub</span>(<span class="st">"TruePrime"</span>, <span class="st">"PrimPol-MDA"</span>, dupli_table[, <span class="dv">2</span>])</span>
<span id="cb30-3"><a href="#cb30-3" aria-hidden="true" tabindex="-1"></a>dupli_table[, <span class="dv">2</span>] <span class="ot">&lt;-</span> <span class="fu">sub</span>(<span class="st">"piDNA"</span>, <span class="st">"piMDA"</span>, dupli_table[, <span class="dv">2</span>])</span>
<span id="cb30-4"><a href="#cb30-4" aria-hidden="true" tabindex="-1"></a>dupli_table[, <span class="dv">2</span>] <span class="ot">&lt;-</span> <span class="fu">sub</span>(<span class="st">"piMDA.D"</span>, <span class="st">"piMDA+D"</span>, dupli_table[, <span class="dv">2</span>])</span>
<span id="cb30-5"><a href="#cb30-5" aria-hidden="true" tabindex="-1"></a>dupli_table[, <span class="dv">2</span>] <span class="ot">&lt;-</span> <span class="fu">sub</span>(<span class="st">"piPolB.D"</span>, <span class="st">"piPolB+D"</span>, dupli_table[, <span class="dv">2</span>])</span>
<span id="cb30-6"><a href="#cb30-6" aria-hidden="true" tabindex="-1"></a>dupli_table<span class="sc">$</span>Sequence.Duplication.Level <span class="ot">&lt;-</span> <span class="fu">factor</span>(dupli_table<span class="sc">$</span>Sequence.Duplication.Level, <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">"1"</span>,</span>
<span id="cb30-7"><a href="#cb30-7" aria-hidden="true" tabindex="-1"></a>    <span class="st">"2"</span>, <span class="st">"3"</span>, <span class="st">"4"</span>, <span class="st">"5"</span>, <span class="st">"6"</span>, <span class="st">"7"</span>, <span class="st">"8"</span>, <span class="st">"9"</span>, <span class="st">"&gt;10"</span>, <span class="st">"&gt;50"</span>, <span class="st">"&gt;100"</span>, <span class="st">"&gt;500"</span>, <span class="st">"&gt;1k"</span>, <span class="st">"&gt;5k"</span>, <span class="st">"&gt;10k+"</span>))</span>
<span id="cb30-8"><a href="#cb30-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb30-9"><a href="#cb30-9" aria-hidden="true" tabindex="-1"></a><span class="co"># plot</span></span>
<span id="cb30-10"><a href="#cb30-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb30-11"><a href="#cb30-11" aria-hidden="true" tabindex="-1"></a>p <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(dupli_table, <span class="fu">aes</span>(<span class="at">x =</span> Sequence.Duplication.Level, <span class="at">y =</span> value, <span class="at">colour =</span> variable,</span>
<span id="cb30-12"><a href="#cb30-12" aria-hidden="true" tabindex="-1"></a>    <span class="at">fill =</span> variable, <span class="at">label =</span> variable, <span class="at">alpha =</span> <span class="fl">0.8</span>)) <span class="sc">+</span> <span class="fu">geom_point</span>(<span class="at">pch =</span> <span class="dv">21</span>, <span class="at">size =</span> <span class="dv">3</span>, <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="at">width =</span> <span class="fl">0.5</span>),</span>
<span id="cb30-13"><a href="#cb30-13" aria-hidden="true" tabindex="-1"></a>    <span class="at">alpha =</span> <span class="fl">0.8</span>) <span class="sc">+</span> <span class="fu">geom_text_repel</span>(<span class="at">max.overlaps =</span> <span class="dv">15</span>) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">13</span>)) <span class="sc">+</span></span>
<span id="cb30-14"><a href="#cb30-14" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme_bw</span>() <span class="sc">+</span> <span class="fu">ggtitle</span>(<span class="st">"Duplication level in Raw Reads"</span>) <span class="sc">+</span> <span class="fu">xlab</span>(<span class="st">"Duplication level"</span>) <span class="sc">+</span> <span class="fu">ylab</span>(<span class="st">"Reads (%)"</span>) <span class="sc">+</span></span>
<span id="cb30-15"><a href="#cb30-15" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> <span class="fu">rep</span>(<span class="fu">unname</span>(colorines[<span class="fu">c</span>(<span class="dv">1</span>, <span class="dv">2</span>, <span class="dv">8</span>, <span class="dv">10</span>, <span class="dv">11</span>, <span class="dv">9</span>, <span class="dv">6</span>, <span class="dv">7</span>, <span class="dv">5</span>, <span class="dv">3</span>, <span class="dv">4</span>)]),</span>
<span id="cb30-16"><a href="#cb30-16" aria-hidden="true" tabindex="-1"></a>        <span class="at">each =</span> <span class="dv">2</span>)) <span class="sc">+</span> <span class="fu">scale_color_manual</span>(<span class="at">values =</span> <span class="fu">rep</span>(<span class="fu">unname</span>(colorines[<span class="fu">c</span>(<span class="dv">1</span>, <span class="dv">2</span>, <span class="dv">8</span>, <span class="dv">10</span>, <span class="dv">11</span>, <span class="dv">9</span>,</span>
<span id="cb30-17"><a href="#cb30-17" aria-hidden="true" tabindex="-1"></a>    <span class="dv">6</span>, <span class="dv">7</span>, <span class="dv">5</span>, <span class="dv">3</span>, <span class="dv">4</span>)]), <span class="at">each =</span> <span class="dv">2</span>)) <span class="sc">+</span> <span class="fu">guides</span>(<span class="at">alpha =</span> <span class="cn">FALSE</span>, <span class="at">color =</span> <span class="cn">FALSE</span>, <span class="at">fill =</span> <span class="fu">guide_legend</span>(<span class="at">ncol =</span> <span class="dv">2</span>,</span>
<span id="cb30-18"><a href="#cb30-18" aria-hidden="true" tabindex="-1"></a>    <span class="at">title =</span> <span class="st">"MDA Method"</span>))</span>
<span id="cb30-19"><a href="#cb30-19" aria-hidden="true" tabindex="-1"></a><span class="co"># ggsave('figures/duplication.svg',p,width=9,heigh=5)</span></span>
<span id="cb30-20"><a href="#cb30-20" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb30-21"><a href="#cb30-21" aria-hidden="true" tabindex="-1"></a><span class="co"># Duplication level post-trimming</span></span>
<span id="cb30-22"><a href="#cb30-22" aria-hidden="true" tabindex="-1"></a>post_duplication <span class="ot">&lt;-</span> <span class="fu">read.table</span>(<span class="st">"qc/post_fastqc_sequence_duplication_levels_final.tsv"</span>, <span class="at">header =</span> <span class="cn">TRUE</span>,</span>
<span id="cb30-23"><a href="#cb30-23" aria-hidden="true" tabindex="-1"></a>    <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>, <span class="at">na.strings =</span> <span class="st">""</span>)</span>
<span id="cb30-24"><a href="#cb30-24" aria-hidden="true" tabindex="-1"></a>post_dupli_table <span class="ot">&lt;-</span> <span class="fu">melt</span>(post_duplication)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stderr">
<pre><code>Using Sequence.Duplication.Level as id variables</code></pre>
</div>
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb32"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb32-1"><a href="#cb32-1" aria-hidden="true" tabindex="-1"></a>post_dupli_table[, <span class="dv">2</span>] <span class="ot">&lt;-</span> <span class="fu">sub</span>(<span class="st">"piPolB.D"</span>, <span class="st">"piPolB+D"</span>, post_dupli_table[, <span class="dv">2</span>])</span>
<span id="cb32-2"><a href="#cb32-2" aria-hidden="true" tabindex="-1"></a>post_dupli_table[, <span class="dv">2</span>] <span class="ot">&lt;-</span> <span class="fu">sub</span>(<span class="st">"piMDA.D"</span>, <span class="st">"piMDA+D"</span>, post_dupli_table[, <span class="dv">2</span>])</span>
<span id="cb32-3"><a href="#cb32-3" aria-hidden="true" tabindex="-1"></a>post_dupli_table<span class="sc">$</span>Sequence.Duplication.Level <span class="ot">&lt;-</span> <span class="fu">factor</span>(post_dupli_table<span class="sc">$</span>Sequence.Duplication.Level,</span>
<span id="cb32-4"><a href="#cb32-4" aria-hidden="true" tabindex="-1"></a>    <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">"1"</span>, <span class="st">"2"</span>, <span class="st">"3"</span>, <span class="st">"4"</span>, <span class="st">"5"</span>, <span class="st">"6"</span>, <span class="st">"7"</span>, <span class="st">"8"</span>, <span class="st">"9"</span>, <span class="st">"&gt;10"</span>, <span class="st">"&gt;50"</span>, <span class="st">"&gt;100"</span>, <span class="st">"&gt;500"</span>, <span class="st">"&gt;1k"</span>,</span>
<span id="cb32-5"><a href="#cb32-5" aria-hidden="true" tabindex="-1"></a>        <span class="st">"&gt;5k"</span>, <span class="st">"&gt;10k+"</span>))</span>
<span id="cb32-6"><a href="#cb32-6" aria-hidden="true" tabindex="-1"></a>q <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(post_dupli_table, <span class="fu">aes</span>(<span class="at">x =</span> Sequence.Duplication.Level, <span class="at">y =</span> value, <span class="at">colour =</span> variable,</span>
<span id="cb32-7"><a href="#cb32-7" aria-hidden="true" tabindex="-1"></a>    <span class="at">fill =</span> variable, <span class="at">label =</span> variable, <span class="at">alpha =</span> <span class="fl">0.8</span>)) <span class="sc">+</span> <span class="fu">geom_point</span>(<span class="at">pch =</span> <span class="dv">21</span>, <span class="at">size =</span> <span class="dv">3</span>, <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="at">width =</span> <span class="fl">0.5</span>),</span>
<span id="cb32-8"><a href="#cb32-8" aria-hidden="true" tabindex="-1"></a>    <span class="at">alpha =</span> <span class="fl">0.8</span>) <span class="sc">+</span> <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> <span class="fu">rep</span>(<span class="fu">unname</span>(colorines[<span class="fu">c</span>(<span class="dv">1</span>, <span class="dv">2</span>, <span class="dv">8</span>, <span class="dv">9</span>, <span class="dv">10</span>, <span class="dv">11</span>, <span class="dv">6</span>,</span>
<span id="cb32-9"><a href="#cb32-9" aria-hidden="true" tabindex="-1"></a>    <span class="dv">7</span>, <span class="dv">5</span>, <span class="dv">3</span>, <span class="dv">4</span>)]), <span class="at">each =</span> <span class="dv">4</span>)) <span class="sc">+</span> <span class="fu">scale_color_manual</span>(<span class="at">values =</span> <span class="fu">rep</span>(<span class="fu">unname</span>(colorines[<span class="fu">c</span>(<span class="dv">1</span>, <span class="dv">2</span>, <span class="dv">8</span>,</span>
<span id="cb32-10"><a href="#cb32-10" aria-hidden="true" tabindex="-1"></a>    <span class="dv">9</span>, <span class="dv">10</span>, <span class="dv">11</span>, <span class="dv">6</span>, <span class="dv">7</span>, <span class="dv">5</span>, <span class="dv">3</span>, <span class="dv">4</span>)]), <span class="at">each =</span> <span class="dv">4</span>)) <span class="sc">+</span> <span class="fu">geom_text_repel</span>(<span class="at">max.overlaps =</span> <span class="dv">20</span>, <span class="at">fontface =</span> <span class="st">"bold"</span>) <span class="sc">+</span></span>
<span id="cb32-11"><a href="#cb32-11" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme_bw</span>() <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">13</span>, <span class="at">face =</span> <span class="st">"bold"</span>)) <span class="sc">+</span> <span class="fu">ggtitle</span>(<span class="st">"Duplication level post-trimming"</span>) <span class="sc">+</span></span>
<span id="cb32-12"><a href="#cb32-12" aria-hidden="true" tabindex="-1"></a>    <span class="fu">xlab</span>(<span class="st">"Duplication level"</span>) <span class="sc">+</span> <span class="fu">ylab</span>(<span class="st">"Reads (%)"</span>) <span class="sc">+</span> <span class="fu">guides</span>(<span class="at">alpha =</span> <span class="cn">FALSE</span>, <span class="at">color =</span> <span class="cn">FALSE</span>, <span class="at">fill =</span> <span class="fu">guide_legend</span>(<span class="at">ncol =</span> <span class="dv">2</span>,</span>
<span id="cb32-13"><a href="#cb32-13" aria-hidden="true" tabindex="-1"></a>    <span class="at">title =</span> <span class="st">"MDA Method"</span>))</span>
<span id="cb32-14"><a href="#cb32-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb32-15"><a href="#cb32-15" aria-hidden="true" tabindex="-1"></a><span class="co"># ggsave('figures/duplication_post.svg',q,width=12,heigh=5)</span></span>
<span id="cb32-16"><a href="#cb32-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb32-17"><a href="#cb32-17" aria-hidden="true" tabindex="-1"></a>p<span class="sc">/</span>q</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-left">
<figure class="figure">
<p><img src="Plotraw_reads_duplication-1.png" class="img-fluid figure-img" width="1152"></p>
<p></p><figcaption class="figure-caption">Figure 4. Reads duplication level before (top) and after (bottom) trimming</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p><em>Random-primer based MDA</em> (RP-MDA) gave rise to a higher level of duplicated reads than <em>PrimPol-MDA</em> and both <em>piMDA protocols</em>. That suggests a greater level of overamplification of the same sequences. The piPolB <em>solo</em> MDA gave rise to a great level of highly duplicated reads, many of them removed after trimming. However, these overrepresented reads might be of interest as they could represent regions of favored initiation (i.e.&nbsp;DNA priming) events.</p>
<div class="cell" data-layout-align="left">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb33"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb33-1"><a href="#cb33-1" aria-hidden="true" tabindex="-1"></a>trimmo <span class="ot">&lt;-</span> <span class="fu">read.csv2</span>(<span class="st">"trimmo/trimmo_stats_percentage.csv"</span>, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">na.strings =</span> <span class="st">""</span>)</span>
<span id="cb33-2"><a href="#cb33-2" aria-hidden="true" tabindex="-1"></a>trimmo<span class="sc">$</span>sample <span class="ot">&lt;-</span> <span class="fu">factor</span>(trimmo<span class="sc">$</span>sample, <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">"NA"</span>, <span class="st">"NA2"</span>, <span class="st">"RP-MDA"</span>, <span class="st">"RP-MDA2"</span>, <span class="st">"PrimPol-MDA"</span>,</span>
<span id="cb33-3"><a href="#cb33-3" aria-hidden="true" tabindex="-1"></a>    <span class="st">"piPolB"</span>, <span class="st">"piPolB+D"</span>, <span class="st">"piMDA"</span>, <span class="st">"piMDA2"</span>, <span class="st">"piMDA+D"</span>, <span class="st">"piMDA+D2"</span>))</span>
<span id="cb33-4"><a href="#cb33-4" aria-hidden="true" tabindex="-1"></a>g_t <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(trimmo) <span class="sc">+</span> <span class="fu">geom_point</span>(<span class="fu">aes</span>(<span class="at">x =</span> sample, <span class="at">y =</span> <span class="fu">as.numeric</span>(Both), <span class="at">color =</span> sample, <span class="at">fill =</span> sample),</span>
<span id="cb33-5"><a href="#cb33-5" aria-hidden="true" tabindex="-1"></a>    <span class="at">size =</span> <span class="dv">4</span>, <span class="at">alpha =</span> <span class="fl">0.7</span>, <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="at">width =</span> <span class="fl">0.5</span>)) <span class="sc">+</span> <span class="fu">scale_shape_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="dv">22</span>,</span>
<span id="cb33-6"><a href="#cb33-6" aria-hidden="true" tabindex="-1"></a>    <span class="dv">23</span>, <span class="dv">24</span>, <span class="dv">25</span>)) <span class="sc">+</span> <span class="fu">geom_point</span>(<span class="fu">aes</span>(<span class="at">x =</span> sample, <span class="at">y =</span> <span class="fu">as.numeric</span>(Dropped), <span class="at">fill =</span> sample), <span class="at">shape =</span> <span class="dv">21</span>,</span>
<span id="cb33-7"><a href="#cb33-7" aria-hidden="true" tabindex="-1"></a>    <span class="at">color =</span> <span class="st">"black"</span>, <span class="at">size =</span> <span class="dv">2</span>, <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="at">width =</span> <span class="fl">0.5</span>)) <span class="sc">+</span> <span class="fu">scale_y_continuous</span>(<span class="at">name =</span> <span class="st">"Both reads surviving (%)"</span>,</span>
<span id="cb33-8"><a href="#cb33-8" aria-hidden="true" tabindex="-1"></a>    <span class="at">limits =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">100</span>), <span class="at">sec.axis =</span> <span class="fu">sec_axis</span>(<span class="sc">~</span>., <span class="at">name =</span> <span class="st">"Dropped reads (%)"</span>)) <span class="sc">+</span> <span class="fu">theme_bw</span>() <span class="sc">+</span></span>
<span id="cb33-9"><a href="#cb33-9" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> colorines[<span class="dv">1</span><span class="sc">:</span><span class="dv">11</span>]) <span class="sc">+</span> <span class="fu">scale_color_manual</span>(<span class="at">values =</span> colorines[<span class="dv">1</span><span class="sc">:</span><span class="dv">11</span>]) <span class="sc">+</span></span>
<span id="cb33-10"><a href="#cb33-10" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme</span>(<span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">15</span>)) <span class="sc">+</span> <span class="fu">ggtitle</span>(<span class="st">"Reads pairing stats"</span>) <span class="sc">+</span> <span class="fu">xlab</span>(<span class="st">"Sample"</span>) <span class="sc">+</span></span>
<span id="cb33-11"><a href="#cb33-11" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_shape_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="dv">21</span>, <span class="dv">22</span>, <span class="dv">23</span>, <span class="dv">24</span>)) <span class="sc">+</span> <span class="fu">guides</span>(<span class="at">shape =</span> <span class="cn">FALSE</span>, <span class="at">fill =</span> <span class="cn">FALSE</span>, <span class="at">color =</span> <span class="cn">FALSE</span>) <span class="sc">+</span></span>
<span id="cb33-12"><a href="#cb33-12" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme</span>(<span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">45</span>, <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">16</span>)) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">legend.text =</span> <span class="fu">element_text</span>(<span class="at">face =</span> <span class="st">"italic"</span>)) <span class="sc">+</span></span>
<span id="cb33-13"><a href="#cb33-13" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme</span>(<span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">hjust =</span> <span class="fl">0.5</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stderr">
<pre><code>Scale for shape is already present.
Adding another scale for shape, which will replace the existing scale.</code></pre>
</div>
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb35"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb35-1"><a href="#cb35-1" aria-hidden="true" tabindex="-1"></a>g_t</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-left">
<figure class="figure">
<p><img src="Plottrimmo-report-1.png" class="img-fluid figure-img" width="480"></p>
<p></p><figcaption class="figure-caption">Figure 5. Paired reads after trimming</figcaption><p></p>
</figure>
</div>
</div>
</div>
<div class="cell">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb36"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb36-1"><a href="#cb36-1" aria-hidden="true" tabindex="-1"></a><span class="co"># table</span></span>
<span id="cb36-2"><a href="#cb36-2" aria-hidden="true" tabindex="-1"></a>tablita_trimmo <span class="ot">&lt;-</span> trimmo[<span class="fu">order</span>(trimmo<span class="sc">$</span>sample), <span class="sc">-</span><span class="dv">2</span>]</span>
<span id="cb36-3"><a href="#cb36-3" aria-hidden="true" tabindex="-1"></a><span class="fu">kbl</span>(tablita_trimmo, <span class="at">align =</span> <span class="st">"c"</span>, <span class="at">row.names =</span> F, <span class="at">caption =</span> <span class="st">"Table 4. Trimmomatic stats (%)"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb36-4"><a href="#cb36-4" aria-hidden="true" tabindex="-1"></a>    <span class="fu">kable_styling</span>(<span class="at">bootstrap_options =</span> <span class="st">"striped"</span>, <span class="at">full_width =</span> F, <span class="at">position =</span> <span class="st">"center"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb36-5"><a href="#cb36-5" aria-hidden="true" tabindex="-1"></a>    <span class="fu">column_spec</span>(<span class="dv">1</span>, <span class="at">italic =</span> T)  <span class="co">#  %&gt;% save_kable('figures/trimmo_table.pdf')</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">

<table class="table table-striped" style="width: auto !important; margin-left: auto; margin-right: auto;">
<caption>Table 4. Trimmomatic stats (%)</caption>
 <thead>
  <tr>
   <th style="text-align:center;"> sample </th>
   <th style="text-align:center;"> Both </th>
   <th style="text-align:center;"> R1_only </th>
   <th style="text-align:center;"> R2_only </th>
   <th style="text-align:center;"> Dropped </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:center;font-style: italic;"> NA </td>
   <td style="text-align:center;"> 91.41 </td>
   <td style="text-align:center;"> 5.32 </td>
   <td style="text-align:center;"> 1.87 </td>
   <td style="text-align:center;"> 1.40 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> NA2 </td>
   <td style="text-align:center;"> 91.33 </td>
   <td style="text-align:center;"> 5.20 </td>
   <td style="text-align:center;"> 1.53 </td>
   <td style="text-align:center;"> 1.94 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> RP-MDA </td>
   <td style="text-align:center;"> 91.68 </td>
   <td style="text-align:center;"> 5.52 </td>
   <td style="text-align:center;"> 1.34 </td>
   <td style="text-align:center;"> 1.46 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> RP-MDA2 </td>
   <td style="text-align:center;"> 92.05 </td>
   <td style="text-align:center;"> 4.82 </td>
   <td style="text-align:center;"> 1.39 </td>
   <td style="text-align:center;"> 1.74 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> PrimPol-MDA </td>
   <td style="text-align:center;"> 91.13 </td>
   <td style="text-align:center;"> 5.66 </td>
   <td style="text-align:center;"> 1.32 </td>
   <td style="text-align:center;"> 1.89 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> piPolB </td>
   <td style="text-align:center;"> 66.79 </td>
   <td style="text-align:center;"> 15.89 </td>
   <td style="text-align:center;"> 9.06 </td>
   <td style="text-align:center;"> 8.26 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> piPolB+D </td>
   <td style="text-align:center;"> 46.13 </td>
   <td style="text-align:center;"> 26.41 </td>
   <td style="text-align:center;"> 11.12 </td>
   <td style="text-align:center;"> 16.33 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> piMDA </td>
   <td style="text-align:center;"> 90.03 </td>
   <td style="text-align:center;"> 6.07 </td>
   <td style="text-align:center;"> 2.33 </td>
   <td style="text-align:center;"> 1.57 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> piMDA2 </td>
   <td style="text-align:center;"> 91.03 </td>
   <td style="text-align:center;"> 5.28 </td>
   <td style="text-align:center;"> 1.84 </td>
   <td style="text-align:center;"> 1.88 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> piMDA+D </td>
   <td style="text-align:center;"> 90.30 </td>
   <td style="text-align:center;"> 5.80 </td>
   <td style="text-align:center;"> 2.45 </td>
   <td style="text-align:center;"> 1.45 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> piMDA+D2 </td>
   <td style="text-align:center;"> 91.56 </td>
   <td style="text-align:center;"> 5.04 </td>
   <td style="text-align:center;"> 1.64 </td>
   <td style="text-align:center;"> 1.75 </td>
  </tr>
</tbody>
</table>

</div>
</div>
<p>Quality filtering of the samples show that piPolB and piPolB+D samples contain lower quality reads, with less paired reads survival and more dropped reads. Also (Table 4), as expected only-forward survival reads were more (15,89% and 26,41%, respectively) than only-reverse reads (9,06% and 11,12%, respectively). This results indicate a possible library prep or sequencing problem that hinders the analysis of those samples, likely due to the hyperbranched structure of the piPolB-synthesized DNA.</p>
</section>
<section id="reads-per-gc-content" class="level3" data-number="4.2">
<h3 data-number="4.2" class="anchored" data-anchor-id="reads-per-gc-content"><span class="header-section-number">4.2</span> Reads per GC content</h3>
<p>DNA amplification as well as library generation and the sequencing itself may give rise to uneven or poor sequence coverage of GC-poor or rich sequences (see for instance <a href="https://pubmed.ncbi.nlm.nih.gov/24475755/" target="_blank">Marine et al.&nbsp;2014</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/33493489/" target="_blank">Jamal et al.&nbsp;2021</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/22323520/" target="_blank">Benjamini et al.&nbsp;2022</a>). Thus, the ideal MDA method should reflex the real GC pattern of the sample or at least do not neglect DNA sequences with extreme GC composition. GC-bias will be analyzed also in terms of impact on the assembly, but this preliminar analysis is also very informative from a general point of view.</p>
<div class="cell" data-layout-align="left">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb37"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb37-1"><a href="#cb37-1" aria-hidden="true" tabindex="-1"></a><span class="co"># GC content</span></span>
<span id="cb37-2"><a href="#cb37-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb37-3"><a href="#cb37-3" aria-hidden="true" tabindex="-1"></a>gc_reads <span class="ot">&lt;-</span> <span class="fu">read.table</span>(<span class="st">"qc/fastqc_per_sequence_gc_content_plot.tsv"</span>, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>,</span>
<span id="cb37-4"><a href="#cb37-4" aria-hidden="true" tabindex="-1"></a>    <span class="at">na.strings =</span> <span class="st">""</span>)</span>
<span id="cb37-5"><a href="#cb37-5" aria-hidden="true" tabindex="-1"></a>gc_table <span class="ot">&lt;-</span> <span class="fu">melt</span>(gc_reads, <span class="at">id.vars =</span> <span class="st">"X..GC"</span>)</span>
<span id="cb37-6"><a href="#cb37-6" aria-hidden="true" tabindex="-1"></a>gc_table[, <span class="dv">2</span>] <span class="ot">&lt;-</span> <span class="fu">sub</span>(<span class="st">"RepliG"</span>, <span class="st">"RP-MDA"</span>, gc_table[, <span class="dv">2</span>])</span>
<span id="cb37-7"><a href="#cb37-7" aria-hidden="true" tabindex="-1"></a>gc_table[, <span class="dv">2</span>] <span class="ot">&lt;-</span> <span class="fu">sub</span>(<span class="st">"Ctrl"</span>, <span class="st">"NA"</span>, gc_table[, <span class="dv">2</span>])</span>
<span id="cb37-8"><a href="#cb37-8" aria-hidden="true" tabindex="-1"></a>gc_table[, <span class="dv">2</span>] <span class="ot">&lt;-</span> <span class="fu">sub</span>(<span class="st">"TruePrime"</span>, <span class="st">"PrimPol-MDA"</span>, gc_table[, <span class="dv">2</span>])</span>
<span id="cb37-9"><a href="#cb37-9" aria-hidden="true" tabindex="-1"></a>gc_table[, <span class="dv">2</span>] <span class="ot">&lt;-</span> <span class="fu">sub</span>(<span class="st">"piDNA"</span>, <span class="st">"piMDA"</span>, gc_table[, <span class="dv">2</span>])</span>
<span id="cb37-10"><a href="#cb37-10" aria-hidden="true" tabindex="-1"></a>gc_table[, <span class="dv">2</span>] <span class="ot">&lt;-</span> <span class="fu">sub</span>(<span class="st">"piMDA.D"</span>, <span class="st">"piMDA+D"</span>, gc_table[, <span class="dv">2</span>])</span>
<span id="cb37-11"><a href="#cb37-11" aria-hidden="true" tabindex="-1"></a>gc_table[, <span class="dv">2</span>] <span class="ot">&lt;-</span> <span class="fu">sub</span>(<span class="st">"piPolB.D"</span>, <span class="st">"piPolB+D"</span>, gc_table[, <span class="dv">2</span>])</span>
<span id="cb37-12"><a href="#cb37-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb37-13"><a href="#cb37-13" aria-hidden="true" tabindex="-1"></a>post_gc_reads <span class="ot">&lt;-</span> <span class="fu">read.table</span>(<span class="st">"qc/post_fastqc_per_sequence_gc_content_plot.tsv"</span>, <span class="at">header =</span> <span class="cn">TRUE</span>,</span>
<span id="cb37-14"><a href="#cb37-14" aria-hidden="true" tabindex="-1"></a>    <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>, <span class="at">na.strings =</span> <span class="st">""</span>)</span>
<span id="cb37-15"><a href="#cb37-15" aria-hidden="true" tabindex="-1"></a>post_gc_table <span class="ot">&lt;-</span> <span class="fu">melt</span>(post_gc_reads, <span class="at">id.vars =</span> <span class="st">"X..GC"</span>)</span>
<span id="cb37-16"><a href="#cb37-16" aria-hidden="true" tabindex="-1"></a>post_gc_table[, <span class="dv">2</span>] <span class="ot">&lt;-</span> <span class="fu">sub</span>(<span class="st">"RepliG"</span>, <span class="st">"RP-MDA"</span>, post_gc_table[, <span class="dv">2</span>])</span>
<span id="cb37-17"><a href="#cb37-17" aria-hidden="true" tabindex="-1"></a>post_gc_table[, <span class="dv">2</span>] <span class="ot">&lt;-</span> <span class="fu">sub</span>(<span class="st">"Ctrl"</span>, <span class="st">"NA"</span>, post_gc_table[, <span class="dv">2</span>])</span>
<span id="cb37-18"><a href="#cb37-18" aria-hidden="true" tabindex="-1"></a>post_gc_table[, <span class="dv">2</span>] <span class="ot">&lt;-</span> <span class="fu">sub</span>(<span class="st">"TruePrime"</span>, <span class="st">"PrimPol-MDA"</span>, post_gc_table[, <span class="dv">2</span>])</span>
<span id="cb37-19"><a href="#cb37-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb37-20"><a href="#cb37-20" aria-hidden="true" tabindex="-1"></a>p <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(gc_table, <span class="fu">aes</span>(<span class="at">x =</span> X..GC, <span class="at">y =</span> value, <span class="at">colour =</span> variable, <span class="at">fill =</span> variable, <span class="at">label =</span> variable)) <span class="sc">+</span></span>
<span id="cb37-21"><a href="#cb37-21" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_color_manual</span>(<span class="at">values =</span> <span class="fu">rep</span>(<span class="fu">unname</span>(colorines[<span class="fu">c</span>(<span class="dv">1</span>, <span class="dv">2</span>, <span class="dv">8</span>, <span class="dv">9</span>, <span class="dv">10</span>, <span class="dv">11</span>, <span class="dv">6</span>, <span class="dv">7</span>, <span class="dv">5</span>, <span class="dv">3</span>, <span class="dv">4</span>)]),</span>
<span id="cb37-22"><a href="#cb37-22" aria-hidden="true" tabindex="-1"></a>        <span class="at">each =</span> <span class="dv">2</span>)) <span class="sc">+</span> <span class="fu">geom_path</span>(<span class="at">linejoin =</span> <span class="st">"round"</span>, <span class="at">size =</span> <span class="dv">1</span>) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">15</span>)) <span class="sc">+</span></span>
<span id="cb37-23"><a href="#cb37-23" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme_bw</span>() <span class="sc">+</span> <span class="fu">ggtitle</span>(<span class="st">"GC content in Raw Reads"</span>) <span class="sc">+</span> <span class="fu">xlab</span>(<span class="st">"GC"</span>) <span class="sc">+</span> <span class="fu">ylab</span>(<span class="st">"Reads (%)"</span>) <span class="sc">+</span> <span class="fu">labs</span>(<span class="at">color =</span> <span class="st">"Method"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Using `size` aesthetic for lines was deprecated in ggplot2 3.4.0.
ℹ Please use `linewidth` instead.</code></pre>
</div>
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb39"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb39-1"><a href="#cb39-1" aria-hidden="true" tabindex="-1"></a>p <span class="ot">&lt;-</span> p <span class="sc">+</span> <span class="fu">guides</span>(<span class="at">color =</span> <span class="fu">guide_legend</span>(<span class="at">ncol =</span> <span class="dv">2</span>))</span>
<span id="cb39-2"><a href="#cb39-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb39-3"><a href="#cb39-3" aria-hidden="true" tabindex="-1"></a><span class="co"># ggsave('figures/gc_profile.svg', p, width=9,heigh=5)</span></span>
<span id="cb39-4"><a href="#cb39-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb39-5"><a href="#cb39-5" aria-hidden="true" tabindex="-1"></a><span class="co"># And the post-trimming plot add the average GC%</span></span>
<span id="cb39-6"><a href="#cb39-6" aria-hidden="true" tabindex="-1"></a>post_gc_table<span class="sc">$</span>p <span class="ot">&lt;-</span> post_gc_table<span class="sc">$</span>X..GC <span class="sc">*</span> post_gc_table<span class="sc">$</span>value</span>
<span id="cb39-7"><a href="#cb39-7" aria-hidden="true" tabindex="-1"></a>gc_sample <span class="ot">&lt;-</span> <span class="fu">by</span>(post_gc_table<span class="sc">$</span>p, <span class="at">INDICES =</span> post_gc_table<span class="sc">$</span>variable, <span class="at">FUN =</span> mean)</span>
<span id="cb39-8"><a href="#cb39-8" aria-hidden="true" tabindex="-1"></a>leyenda <span class="ot">&lt;-</span> <span class="fu">c</span>()</span>
<span id="cb39-9"><a href="#cb39-9" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">length</span>(<span class="fu">levels</span>(<span class="fu">as.factor</span>(post_gc_table<span class="sc">$</span>variable)))) {</span>
<span id="cb39-10"><a href="#cb39-10" aria-hidden="true" tabindex="-1"></a>    leyenda[i] <span class="ot">&lt;-</span> <span class="fu">paste0</span>(<span class="fu">levels</span>(<span class="fu">as.factor</span>(post_gc_table<span class="sc">$</span>variable))[i], <span class="st">" ("</span>, <span class="fu">round</span>(gc_sample[i],</span>
<span id="cb39-11"><a href="#cb39-11" aria-hidden="true" tabindex="-1"></a>        <span class="dv">1</span>), <span class="st">"%)"</span>)</span>
<span id="cb39-12"><a href="#cb39-12" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb39-13"><a href="#cb39-13" aria-hidden="true" tabindex="-1"></a>q <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(post_gc_table, <span class="fu">aes</span>(<span class="at">x =</span> X..GC, <span class="at">y =</span> value, <span class="at">colour =</span> variable, <span class="at">fill =</span> variable, <span class="at">label =</span> variable)) <span class="sc">+</span></span>
<span id="cb39-14"><a href="#cb39-14" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_color_manual</span>(<span class="at">values =</span> <span class="fu">rep</span>(<span class="fu">unname</span>(colorines[<span class="fu">c</span>(<span class="dv">1</span>, <span class="dv">2</span>, <span class="dv">8</span>, <span class="dv">9</span>, <span class="dv">10</span>, <span class="dv">11</span>, <span class="dv">6</span>, <span class="dv">7</span>, <span class="dv">5</span>, <span class="dv">3</span>, <span class="dv">4</span>)]),</span>
<span id="cb39-15"><a href="#cb39-15" aria-hidden="true" tabindex="-1"></a>        <span class="at">each =</span> <span class="dv">4</span>), <span class="at">labels =</span> leyenda) <span class="sc">+</span> <span class="fu">geom_path</span>(<span class="at">linejoin =</span> <span class="st">"round"</span>, <span class="at">size =</span> <span class="dv">1</span>) <span class="sc">+</span> <span class="fu">theme_bw</span>() <span class="sc">+</span></span>
<span id="cb39-16"><a href="#cb39-16" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme</span>(<span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">15</span>)) <span class="sc">+</span> <span class="fu">ggtitle</span>(<span class="st">"GC content per sample (post-trimming)"</span>) <span class="sc">+</span></span>
<span id="cb39-17"><a href="#cb39-17" aria-hidden="true" tabindex="-1"></a>    <span class="fu">xlab</span>(<span class="st">"GC"</span>) <span class="sc">+</span> <span class="fu">ylab</span>(<span class="st">"Reads (%)"</span>) <span class="sc">+</span> <span class="fu">labs</span>(<span class="at">color =</span> <span class="st">"Sample (Average GC %)"</span>) <span class="sc">+</span> <span class="fu">guides</span>(<span class="at">color =</span> <span class="fu">guide_legend</span>(<span class="at">ncol =</span> <span class="dv">2</span>)) <span class="sc">+</span></span>
<span id="cb39-18"><a href="#cb39-18" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme</span>(<span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">17</span>), <span class="at">legend.text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">9</span>, <span class="at">face =</span> <span class="st">"bold"</span>),</span>
<span id="cb39-19"><a href="#cb39-19" aria-hidden="true" tabindex="-1"></a>        <span class="at">legend.key.height =</span> <span class="fu">unit</span>(<span class="fl">0.4</span>, <span class="st">"cm"</span>))</span>
<span id="cb39-20"><a href="#cb39-20" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb39-21"><a href="#cb39-21" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb39-22"><a href="#cb39-22" aria-hidden="true" tabindex="-1"></a><span class="co"># ggsave('figures/gc_profile_post.svg', q, width=12,heigh=6)</span></span>
<span id="cb39-23"><a href="#cb39-23" aria-hidden="true" tabindex="-1"></a>p<span class="sc">/</span>q</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-left">
<figure class="figure">
<p><img src="PlotGC-content-raw-reads-1.png" class="img-fluid figure-img" width="1152"></p>
<p></p><figcaption class="figure-caption">Figure 6. Reads’ GC content plot. GC content is indicated in the plot leyend for the post-trimming reads</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>As shown in Figure 6, the control <em>non-amplified</em> (NA) samples show a bimodal curve respect to the GC content, in agreement with the selection of the species in the mock-metagenome (see <a href="#disclaimer">Disclaimer</a> and Table 1). None of the MDA protocols achieve a similar curve. The samples from <em>RP-MDA</em> and <em>Primpol-MDA</em> show a similar curve, centered in ~44% and ~48% GC content, respectively, and very few reads (&lt;1%) in regions of GC content above 60%.</p>
<p>The <em>piPolB</em> protocols show a slim peak of about 10% of reads around 40-45% GC that increases to ~47% in the <em>piPolB+D</em> protocol, although it also shows a “shoulder” at 38-40%. Given the limited processivity of piPolB, this pattern suggests that low GC content DNA sequences facilitate DNA priming by piPolB.</p>
<p>On the contrary, the <em>piMDA</em> and <em>piMDA+D</em> methods gave rise to a peak around 68-70% GC, although the raise of the curve is very slow with a good number of reads in the range of the first peak of the NA samples, reaching a 1% at GC content of 40%. Thus, <em>piMDA</em> protocols provide most of their reads of higher GC-content sequences, but that bias is less severe than in the case of the other MDA methods, which centered on 40-50% GC content and seem to neglect high GC sequences.</p>
</section>
<section id="metagenomes-profiling" class="level3" data-number="4.3">
<h3 data-number="4.3" class="anchored" data-anchor-id="metagenomes-profiling"><span class="header-section-number">4.3</span> Metagenomes profiling</h3>
<p>Metagenomes profiling is usually the first approach to measure alpha diversity. In our case, profiling prior to assembly can be very useful to detect contamination. In this case, the trimmed reads were profiled using <a href="https://www.biorxiv.org/content/10.1101/2022.08.22.504593v1" target="_blank">Metaphlan</a> v.4.0.0 installed via Conda. Note that we used the <code>--unclassified_estimation</code> to include the estimation of genomic sequences not identified in the database.</p>
<div class="cell" data-layout-align="left">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb40"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb40-1"><a href="#cb40-1" aria-hidden="true" tabindex="-1"></a><span class="co"># read tsv, select and transform table</span></span>
<span id="cb40-2"><a href="#cb40-2" aria-hidden="true" tabindex="-1"></a>meta <span class="ot">&lt;-</span> <span class="fu">read.table</span>(<span class="at">file =</span> <span class="st">"metaphlan_results/merge_profiled_4.tsv"</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>, <span class="at">header =</span> <span class="cn">TRUE</span>)</span>
<span id="cb40-3"><a href="#cb40-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb40-4"><a href="#cb40-4" aria-hidden="true" tabindex="-1"></a>short_meta <span class="ot">&lt;-</span> meta[<span class="fu">c</span>(<span class="dv">1</span>, <span class="dv">21</span>, <span class="dv">22</span>, <span class="dv">23</span>, <span class="dv">20</span>), ]</span>
<span id="cb40-5"><a href="#cb40-5" aria-hidden="true" tabindex="-1"></a>short_meta[, <span class="dv">1</span>] <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"Unclassified"</span>, <span class="st">"Kokuria rhizophila"</span>, <span class="st">"Pseudomonas aeruginosa"</span>, <span class="st">"Escherichia coli"</span>,</span>
<span id="cb40-6"><a href="#cb40-6" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Bacillus sp"</span>)</span>
<span id="cb40-7"><a href="#cb40-7" aria-hidden="true" tabindex="-1"></a>short_meta <span class="ot">&lt;-</span> short_meta[, <span class="fu">c</span>(<span class="dv">1</span>, <span class="dv">6</span>, <span class="dv">5</span>, <span class="dv">11</span>, <span class="dv">7</span>, <span class="dv">4</span>, <span class="dv">9</span>, <span class="dv">8</span>, <span class="dv">10</span>, <span class="dv">3</span>, <span class="dv">12</span>, <span class="dv">2</span>)]</span>
<span id="cb40-8"><a href="#cb40-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb40-9"><a href="#cb40-9" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(short_meta) <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"Genome"</span>, <span class="st">"NA"</span>, <span class="st">"NA2"</span>, <span class="st">"RP-MDA"</span>, <span class="st">"RP-MDA2"</span>, <span class="st">"PrimPol-MDA"</span>, <span class="st">"piPolB"</span>,</span>
<span id="cb40-10"><a href="#cb40-10" aria-hidden="true" tabindex="-1"></a>    <span class="st">"piPolB+D"</span>, <span class="st">"piMDA"</span>, <span class="st">"piMDA2"</span>, <span class="st">"piMDA+D"</span>, <span class="st">"piMDA+D2"</span>)</span>
<span id="cb40-11"><a href="#cb40-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb40-12"><a href="#cb40-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb40-13"><a href="#cb40-13" aria-hidden="true" tabindex="-1"></a>meta2 <span class="ot">&lt;-</span> <span class="fu">melt</span>(short_meta, <span class="at">id.vars =</span> <span class="st">"Genome"</span>, <span class="at">na.rm =</span> F)</span>
<span id="cb40-14"><a href="#cb40-14" aria-hidden="true" tabindex="-1"></a>meta2[, <span class="dv">4</span>] <span class="ot">&lt;-</span> <span class="fu">grepl</span>(<span class="dv">2</span>, meta2<span class="sc">$</span>variable)</span>
<span id="cb40-15"><a href="#cb40-15" aria-hidden="true" tabindex="-1"></a>meta2<span class="sc">$</span>V4[meta2<span class="sc">$</span>V4 <span class="sc">==</span> <span class="st">"TRUE"</span>] <span class="ot">&lt;-</span> <span class="st">"2"</span></span>
<span id="cb40-16"><a href="#cb40-16" aria-hidden="true" tabindex="-1"></a>meta2<span class="sc">$</span>V4[meta2<span class="sc">$</span>V4 <span class="sc">==</span> <span class="st">"FALSE"</span>] <span class="ot">&lt;-</span> <span class="st">"1"</span></span>
<span id="cb40-17"><a href="#cb40-17" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb40-18"><a href="#cb40-18" aria-hidden="true" tabindex="-1"></a><span class="co"># reorder variables</span></span>
<span id="cb40-19"><a href="#cb40-19" aria-hidden="true" tabindex="-1"></a>meta2<span class="sc">$</span>variable <span class="ot">&lt;-</span> <span class="fu">factor</span>(meta2<span class="sc">$</span>variable, <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">"NA"</span>, <span class="st">"NA2"</span>, <span class="st">"RP-MDA"</span>, <span class="st">"RP-MDA2"</span>, <span class="st">"PrimPol-MDA"</span>,</span>
<span id="cb40-20"><a href="#cb40-20" aria-hidden="true" tabindex="-1"></a>    <span class="st">"piPolB"</span>, <span class="st">"piPolB+D"</span>, <span class="st">"piMDA"</span>, <span class="st">"piMDA2"</span>, <span class="st">"piMDA+D"</span>, <span class="st">"piMDA+D2"</span>))</span>
<span id="cb40-21"><a href="#cb40-21" aria-hidden="true" tabindex="-1"></a>meta2<span class="sc">$</span>Species <span class="ot">&lt;-</span> <span class="fu">factor</span>(meta2<span class="sc">$</span>Genome, <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">"Unclassified"</span>, <span class="st">"Kokuria rhizophila"</span>, <span class="st">"Pseudomonas aeruginosa"</span>,</span>
<span id="cb40-22"><a href="#cb40-22" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Escherichia coli"</span>, <span class="st">"Bacillus sp"</span>))</span>
<span id="cb40-23"><a href="#cb40-23" aria-hidden="true" tabindex="-1"></a><span class="co"># plot</span></span>
<span id="cb40-24"><a href="#cb40-24" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb40-25"><a href="#cb40-25" aria-hidden="true" tabindex="-1"></a>profiling <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(meta2, <span class="fu">aes</span>(<span class="at">y =</span> value, <span class="at">x =</span> Species, <span class="at">group =</span> variable, <span class="at">fill =</span> variable)) <span class="sc">+</span></span>
<span id="cb40-26"><a href="#cb40-26" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_bar</span>(<span class="at">stat =</span> <span class="st">"identity"</span>, <span class="at">position =</span> <span class="st">"dodge"</span>, <span class="at">alpha =</span> <span class="fl">0.8</span>) <span class="sc">+</span> <span class="fu">ylab</span>(<span class="st">"Reads (%)"</span>) <span class="sc">+</span> <span class="fu">geom_text</span>(<span class="fu">aes</span>(<span class="at">label =</span> <span class="fu">round</span>(value,</span>
<span id="cb40-27"><a href="#cb40-27" aria-hidden="true" tabindex="-1"></a>    <span class="at">digits =</span> <span class="dv">2</span>), <span class="at">group =</span> variable), <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="at">width =</span> <span class="fl">0.9</span>), <span class="at">angle =</span> <span class="dv">90</span>, <span class="at">hjust =</span> <span class="dv">0</span>,</span>
<span id="cb40-28"><a href="#cb40-28" aria-hidden="true" tabindex="-1"></a>    <span class="at">vjust =</span> <span class="fl">0.5</span>) <span class="sc">+</span> <span class="fu">theme_bw</span>() <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">face =</span> <span class="st">"bold.italic"</span>, <span class="at">size =</span> <span class="dv">11</span>,</span>
<span id="cb40-29"><a href="#cb40-29" aria-hidden="true" tabindex="-1"></a>    <span class="at">angle =</span> <span class="dv">45</span>, <span class="at">hjust =</span> <span class="dv">1</span>)) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">19</span>)) <span class="sc">+</span> <span class="fu">labs</span>(<span class="at">color =</span> <span class="st">"Sample"</span>) <span class="sc">+</span></span>
<span id="cb40-30"><a href="#cb40-30" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme</span>(<span class="at">axis.text.y =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">17</span>)) <span class="sc">+</span> <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> colorines[<span class="dv">1</span><span class="sc">:</span><span class="dv">11</span>],</span>
<span id="cb40-31"><a href="#cb40-31" aria-hidden="true" tabindex="-1"></a>    <span class="at">name =</span> <span class="st">""</span>) <span class="sc">+</span> <span class="fu">scale_y_continuous</span>(<span class="at">limits =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">100</span>), <span class="at">expand =</span> <span class="fu">c</span>(<span class="fl">0.08</span>, <span class="sc">-</span><span class="fl">0.5</span>))</span>
<span id="cb40-32"><a href="#cb40-32" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb40-33"><a href="#cb40-33" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb40-34"><a href="#cb40-34" aria-hidden="true" tabindex="-1"></a><span class="co"># ggsave('figures/profiling_v4.svg', profiling, width=9,heigh=5)</span></span>
<span id="cb40-35"><a href="#cb40-35" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb40-36"><a href="#cb40-36" aria-hidden="true" tabindex="-1"></a><span class="co"># same plot in a different display</span></span>
<span id="cb40-37"><a href="#cb40-37" aria-hidden="true" tabindex="-1"></a>profilin2 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(meta2, <span class="fu">aes</span>(<span class="at">y =</span> value, <span class="at">x =</span> variable, <span class="at">group =</span> Species, <span class="at">fill =</span> Species, <span class="at">label =</span> <span class="fu">paste</span>(<span class="fu">round</span>(value,</span>
<span id="cb40-38"><a href="#cb40-38" aria-hidden="true" tabindex="-1"></a>    <span class="dv">1</span>), <span class="st">"%"</span>))) <span class="sc">+</span> <span class="fu">geom_bar</span>(<span class="at">stat =</span> <span class="st">"identity"</span>, <span class="at">position =</span> <span class="st">"stack"</span>, <span class="at">alpha =</span> <span class="fl">0.8</span>) <span class="sc">+</span> <span class="fu">ylab</span>(<span class="st">"Reads (%)"</span>) <span class="sc">+</span></span>
<span id="cb40-39"><a href="#cb40-39" aria-hidden="true" tabindex="-1"></a>    <span class="fu">xlab</span>(<span class="st">"Sample"</span>) <span class="sc">+</span> <span class="fu">theme_bw</span>() <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">face =</span> <span class="st">"bold"</span>, <span class="at">size =</span> <span class="dv">13</span>,</span>
<span id="cb40-40"><a href="#cb40-40" aria-hidden="true" tabindex="-1"></a>    <span class="at">angle =</span> <span class="dv">45</span>, <span class="at">vjust =</span> <span class="dv">1</span>, <span class="at">hjust =</span> <span class="dv">1</span>)) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">15</span>)) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">axis.text.y =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">15</span>)) <span class="sc">+</span></span>
<span id="cb40-41"><a href="#cb40-41" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="at">Unclassified =</span> <span class="st">"grey80"</span>, <span class="st">`</span><span class="at">Kokuria rhizophila</span><span class="st">`</span> <span class="ot">=</span> <span class="st">"skyblue1"</span>,</span>
<span id="cb40-42"><a href="#cb40-42" aria-hidden="true" tabindex="-1"></a>        <span class="st">`</span><span class="at">Pseudomonas aeruginosa</span><span class="st">`</span> <span class="ot">=</span> <span class="st">"slateblue2"</span>, <span class="st">`</span><span class="at">Escherichia coli</span><span class="st">`</span> <span class="ot">=</span> <span class="st">"gold1"</span>, <span class="st">`</span><span class="at">Bacillus sp</span><span class="st">`</span> <span class="ot">=</span> <span class="st">"orange2"</span>),</span>
<span id="cb40-43"><a href="#cb40-43" aria-hidden="true" tabindex="-1"></a>        <span class="at">name =</span> <span class="st">"Species"</span>) <span class="sc">+</span> <span class="fu">scale_y_continuous</span>(<span class="at">limits =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">100</span>), <span class="at">expand =</span> <span class="fu">c</span>(<span class="fl">0.08</span>, <span class="sc">-</span><span class="fl">0.5</span>))</span>
<span id="cb40-44"><a href="#cb40-44" aria-hidden="true" tabindex="-1"></a>profiling<span class="sc">/</span>profilin2</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-left">
<figure class="figure">
<p><img src="Plotprofiling-1.png" class="img-fluid figure-img" width="960"></p>
<p></p><figcaption class="figure-caption">Figure 7. Metagenomic samples reads profiling of amplified DNA by different MDA protocols. Note that both plots show the same data, with different representation</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>As shown in Figure 7, the amount of unknown (or junk) DNA is very high in piPolB samples, particularly without denaturation step, but not in piMDA. However, in any of the samples we detected sequences from other organisms than the expected bacteria, downplaying the presence of contaminant DNA. This suggest that, rather than contaminant DNA, piPolB might be performing <em>ab initio</em> DNA synthesis.</p>
<div class="cell">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb41"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb41-1"><a href="#cb41-1" aria-hidden="true" tabindex="-1"></a>res_aov <span class="ot">&lt;-</span> <span class="fu">aov</span>(value <span class="sc">~</span> variable, <span class="at">data =</span> meta2)</span>
<span id="cb41-2"><a href="#cb41-2" aria-hidden="true" tabindex="-1"></a><span class="fu">shapiro.test</span>(res_aov<span class="sc">$</span>residuals)  <span class="co">#normality test</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stdout">
<pre><code>
    Shapiro-Wilk normality test

data:  res_aov$residuals
W = 0.80281, p-value = 3.944e-07</code></pre>
</div>
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb43"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb43-1"><a href="#cb43-1" aria-hidden="true" tabindex="-1"></a><span class="fu">leveneTest</span>(value <span class="sc">~</span> variable, <span class="at">data =</span> meta2)  <span class="co">#homogeneus variance</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stdout">
<pre><code>Levene's Test for Homogeneity of Variance (center = median)
      Df F value Pr(&gt;F)
group 10  0.1652 0.9978
      44               </code></pre>
</div>
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb45"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb45-1"><a href="#cb45-1" aria-hidden="true" tabindex="-1"></a><span class="fu">kruskal.test</span>(value <span class="sc">~</span> variable, <span class="at">data =</span> meta2)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stdout">
<pre><code>
    Kruskal-Wallis rank sum test

data:  value by variable
Kruskal-Wallis chi-squared = 3.7369, df = 10, p-value = 0.9584</code></pre>
</div>
</div>
<p>Although due to the limited number of samples, the differences are not statistically significant, we represent these results as an interactive heatmap.</p>
<div class="cell" data-layout-align="left">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb47"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb47-1"><a href="#cb47-1" aria-hidden="true" tabindex="-1"></a>short <span class="ot">&lt;-</span> <span class="fu">as.matrix</span>(short_meta[, <span class="sc">-</span><span class="dv">1</span>])</span>
<span id="cb47-2"><a href="#cb47-2" aria-hidden="true" tabindex="-1"></a><span class="fu">row.names</span>(short) <span class="ot">&lt;-</span> short_meta[, <span class="dv">1</span>]</span>
<span id="cb47-3"><a href="#cb47-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb47-4"><a href="#cb47-4" aria-hidden="true" tabindex="-1"></a><span class="fu">heatmaply</span>(<span class="fu">t</span>(short), <span class="at">k_row =</span> <span class="dv">3</span>, <span class="at">k_col =</span> <span class="dv">3</span>, <span class="at">fontsize_row =</span> <span class="dv">14</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-3dd85e65aee987291d3e" style="width:100%;height:722px;"></div>
<script type="application/json" data-for="htmlwidget-3dd85e65aee987291d3e">{"x":{"data":[{"x":[4.125,5,null,5,5],"y":[151.86466150634,151.86466150634,null,151.86466150634,0],"text":"y: 151.86466","type":"scatter","mode":"lines","line":{"width":2.26771653543307,"color":"rgba(125,176,221,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[4.125,3.25,null,3.25,3.25,null,3.25,2.5,null,2.5,2.5,null,2.5,2,null,2,2,null,2.5,3,null,3,3,null,3.25,4,null,4,4],"y":[151.86466150634,151.86466150634,null,151.86466150634,76.2453900920711,null,76.2453900920711,76.2453900920711,null,76.2453900920711,41.679611518445,null,41.679611518445,41.679611518445,null,41.679611518445,0,null,41.679611518445,41.679611518445,null,41.679611518445,0,null,76.2453900920711,76.2453900920711,null,76.2453900920711,0],"text":["y: 151.86466","y: 151.86466",null,"y: 151.86466","y: 151.86466",null,"y:  76.24539","y:  76.24539",null,"y:  76.24539","y:  76.24539",null,"y:  41.67961","y:  41.67961",null,"y:  41.67961","y:  41.67961",null,"y:  41.67961","y:  41.67961",null,"y:  41.67961","y:  41.67961",null,"y:  76.24539","y:  76.24539",null,"y:  76.24539","y:  76.24539"],"type":"scatter","mode":"lines","line":{"width":2.26771653543307,"color":"rgba(134,184,117,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[2.5625,1,null,1,1],"y":[179.079354876334,179.079354876334,null,179.079354876334,0],"text":"y: 179.07935","type":"scatter","mode":"lines","line":{"width":2.26771653543307,"color":"rgba(228,149,165,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[2.5625,4.125,null,4.125,4.125],"y":[179.079354876334,179.079354876334,null,179.079354876334,151.86466150634],"text":"y: 179.07935","type":"scatter","mode":"lines","line":{"width":2.26771653543307,"color":"rgba(0,0,0,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[2.5625,1,4.125,3.25,2.5,2,3,4,5],"y":[179.079354876334,0,151.86466150634,76.2453900920711,41.679611518445,0,0,0,0],"text":["y: 179.07935","y:   0.00000","y: 151.86466","y:  76.24539","y:  41.67961","y:   0.00000","y:   0.00000","y:   0.00000","y:   0.00000"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"transparent","opacity":1,"size":null,"symbol":null,"line":{"width":1.88976377952756,"color":"transparent"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"type":"scatter","mode":"markers","marker":{"color":"rgba(140,86,75,1)","line":{"color":"rgba(140,86,75,1)"}},"error_y":{"color":"rgba(140,86,75,1)"},"error_x":{"color":"rgba(140,86,75,1)"},"line":{"color":"rgba(140,86,75,1)"},"xaxis":"x2","yaxis":"y","frame":null},{"x":[1,2,3,4,5],"y":[1,2,3,4,5,6,7,8,9,10,11],"z":[[0.009748412058486,0.00457212117619384,0.00425162488957889,0.00739052052736363,1],[0.193434987625443,0.0758552772294667,0.0443353562028407,0.0720326399743127,0.640250691934897],[0.0589305795673577,0.100406968625958,0.253376058524196,0.613000506928172,0],[0.123362165963215,0.177444763963429,0.246886639226268,0.478012644579947,0],[0.0778236094586769,0.171871828549736,0.307792028355347,0.454093889932453,0.0141442296317933],[0.0863533322171424,0.18220838956702,0.337565814580943,0.412755159892644,0.00685794510135126],[0.291096090333143,0.213095634190125,0.21675011334323,0.304885644658023,0],[0.368235558574622,0.217587504102568,0.188336076244624,0.251663829366808,0],[0.700006761094222,0.268802844132771,0.0330464639015535,0.0240901172808746,0],[0.819448313178809,0.196856286119658,0.00701194210328305,0.00263969944217382,0],[0.88570734434376,0.137848554156929,0.00168678528231877,0.000713351868375551,0]],"text":[["row: piPolB<br>column: Bacillus sp<br>value:  0.95017","row: piPolB<br>column: Escherichia coli<br>value:  0.44564","row: piPolB<br>column: Pseudomonas aeruginosa<br>value:  0.41440","row: piPolB<br>column: Kokuria rhizophila<br>value:  0.72035","row: piPolB<br>column: Unclassified<br>value: 97.46943"],["row: piPolB+D<br>column: Bacillus sp<br>value: 18.85400","row: piPolB+D<br>column: Escherichia coli<br>value:  7.39357","row: piPolB+D<br>column: Pseudomonas aeruginosa<br>value:  4.32134","row: piPolB+D<br>column: Kokuria rhizophila<br>value:  7.02098","row: piPolB+D<br>column: Unclassified<br>value: 62.40487"],["row: piMDA+D<br>column: Bacillus sp<br>value:  5.74393","row: piMDA+D<br>column: Escherichia coli<br>value:  9.78661","row: piMDA+D<br>column: Pseudomonas aeruginosa<br>value: 24.69642","row: piMDA+D<br>column: Kokuria rhizophila<br>value: 59.74881","row: piMDA+D<br>column: Unclassified<br>value:  0.00000"],["row: piMDA+D2<br>column: Bacillus sp<br>value: 12.02404","row: piMDA+D2<br>column: Escherichia coli<br>value: 17.29544","row: piMDA+D2<br>column: Pseudomonas aeruginosa<br>value: 24.06390","row: piMDA+D2<br>column: Kokuria rhizophila<br>value: 46.59162","row: piMDA+D2<br>column: Unclassified<br>value:  0.00000"],["row: piMDA2<br>column: Bacillus sp<br>value:  7.58542","row: piMDA2<br>column: Escherichia coli<br>value: 16.75225","row: piMDA2<br>column: Pseudomonas aeruginosa<br>value: 30.00031","row: piMDA2<br>column: Kokuria rhizophila<br>value: 44.26027","row: piMDA2<br>column: Unclassified<br>value:  1.37863"],["row: piMDA<br>column: Bacillus sp<br>value:  8.41681","row: piMDA<br>column: Escherichia coli<br>value: 17.75975","row: piMDA<br>column: Pseudomonas aeruginosa<br>value: 32.90235","row: piMDA<br>column: Kokuria rhizophila<br>value: 40.23101","row: piMDA<br>column: Unclassified<br>value:  0.66844"],["row: NA2<br>column: Bacillus sp<br>value: 28.37297","row: NA2<br>column: Escherichia coli<br>value: 20.77031","row: NA2<br>column: Pseudomonas aeruginosa<br>value: 21.12651","row: NA2<br>column: Kokuria rhizophila<br>value: 29.71703","row: NA2<br>column: Unclassified<br>value:  0.00000"],["row: NA<br>column: Bacillus sp<br>value: 35.89171","row: NA<br>column: Escherichia coli<br>value: 21.20813","row: NA<br>column: Pseudomonas aeruginosa<br>value: 18.35701","row: NA<br>column: Kokuria rhizophila<br>value: 24.52953","row: NA<br>column: Unclassified<br>value:  0.00000"],["row: PrimPol-MDA<br>column: Bacillus sp<br>value: 68.22926","row: PrimPol-MDA<br>column: Escherichia coli<br>value: 26.20006","row: PrimPol-MDA<br>column: Pseudomonas aeruginosa<br>value:  3.22102","row: PrimPol-MDA<br>column: Kokuria rhizophila<br>value:  2.34805","row: PrimPol-MDA<br>column: Unclassified<br>value:  0.00000"],["row: RP-MDA2<br>column: Bacillus sp<br>value: 79.87116","row: RP-MDA2<br>column: Escherichia coli<br>value: 19.18747","row: RP-MDA2<br>column: Pseudomonas aeruginosa<br>value:  0.68345","row: RP-MDA2<br>column: Kokuria rhizophila<br>value:  0.25729","row: RP-MDA2<br>column: Unclassified<br>value:  0.00000"],["row: RP-MDA<br>column: Bacillus sp<br>value: 86.32939","row: RP-MDA<br>column: Escherichia coli<br>value: 13.43602","row: RP-MDA<br>column: Pseudomonas aeruginosa<br>value:  0.16441","row: RP-MDA<br>column: Kokuria rhizophila<br>value:  0.06953","row: RP-MDA<br>column: Unclassified<br>value:  0.00000"]],"colorscale":[[0,"#440154"],[0.000713351868375551,"#440154"],[0.00168678528231877,"#440155"],[0.00263969944217382,"#440255"],[0.00425162488957889,"#440256"],[0.00457212117619384,"#440256"],[0.00685794510135126,"#450357"],[0.00701194210328305,"#450457"],[0.00739052052736363,"#450457"],[0.009748412058486,"#450458"],[0.0141442296317933,"#46065A"],[0.0240901172808746,"#460A5D"],[0.0330464639015535,"#470D60"],[0.0443353562028407,"#471264"],[0.0589305795673577,"#481769"],[0.0720326399743127,"#481B6D"],[0.0758552772294667,"#481C6E"],[0.0778236094586769,"#481D6F"],[0.0863533322171424,"#482071"],[0.100406968625958,"#482576"],[0.123362165963215,"#472C7A"],[0.137848554156929,"#46307E"],[0.171871828549736,"#443B84"],[0.177444763963429,"#433D84"],[0.18220838956702,"#433E85"],[0.188336076244624,"#424086"],[0.193434987625443,"#424186"],[0.196856286119658,"#414287"],[0.213095634190125,"#3F4788"],[0.21675011334323,"#3F4889"],[0.217587504102568,"#3F4889"],[0.246886639226268,"#3B518B"],[0.251663829366808,"#3B528B"],[0.253376058524196,"#3A538B"],[0.268802844132771,"#38578C"],[0.291096090333143,"#365D8D"],[0.304885644658023,"#34618D"],[0.307792028355347,"#34618D"],[0.337565814580943,"#30698E"],[0.368235558574622,"#2D718E"],[0.412755159892644,"#297B8E"],[0.454093889932453,"#25858E"],[0.478012644579947,"#228B8D"],[0.613000506928172,"#25AB82"],[0.640250691934897,"#2CB17E"],[0.700006761094222,"#43BF70"],[0.819448313178809,"#86D549"],[0.88570734434376,"#B2DD2D"],[1,"#FDE725"]],"type":"heatmap","showscale":false,"autocolorscale":false,"showlegend":false,"xaxis":"x","yaxis":"y2","hoverinfo":"text","frame":null},{"x":[1],"y":[1],"name":"99_6c62444e55ec1e1ab615cef6d81f4a8d","type":"scatter","mode":"markers","opacity":0,"hoverinfo":"skip","showlegend":false,"marker":{"color":[0,1],"colorscale":[[0,"#440154"],[0.00334448160535117,"#440256"],[0.00668896321070234,"#450357"],[0.0100334448160535,"#450558"],[0.0133779264214047,"#450659"],[0.0167224080267559,"#46075B"],[0.020066889632107,"#46085C"],[0.0234113712374582,"#460A5D"],[0.0267558528428094,"#460B5E"],[0.0301003344481605,"#470C5F"],[0.0334448160535117,"#470E61"],[0.0367892976588629,"#470F62"],[0.040133779264214,"#471063"],[0.0434782608695652,"#471164"],[0.0468227424749164,"#471365"],[0.0501672240802676,"#481467"],[0.0535117056856187,"#481568"],[0.0568561872909699,"#481768"],[0.0602006688963211,"#481769"],[0.0635451505016722,"#48186A"],[0.0668896321070234,"#481A6C"],[0.0702341137123746,"#481B6D"],[0.0735785953177258,"#481C6E"],[0.0769230769230769,"#481D6F"],[0.0802675585284281,"#481E6F"],[0.0836120401337793,"#481F70"],[0.0869565217391304,"#482071"],[0.0903010033444816,"#482173"],[0.0936454849498328,"#482374"],[0.096989966555184,"#482475"],[0.100334448160535,"#482576"],[0.103678929765886,"#482576"],[0.107023411371237,"#482777"],[0.110367892976589,"#482878"],[0.11371237458194,"#482979"],[0.117056856187291,"#472A7A"],[0.120401337792642,"#472B7A"],[0.123745819397993,"#472D7B"],[0.127090301003344,"#472D7B"],[0.130434782608696,"#472E7C"],[0.133779264214047,"#472F7D"],[0.137123745819398,"#46307E"],[0.140468227424749,"#46327E"],[0.1438127090301,"#46337F"],[0.147157190635452,"#463480"],[0.150501672240803,"#463480"],[0.153846153846154,"#453581"],[0.157190635451505,"#453781"],[0.160535117056856,"#453882"],[0.163879598662207,"#443983"],[0.167224080267559,"#443A83"],[0.17056856187291,"#443A83"],[0.173913043478261,"#443C84"],[0.177257525083612,"#433D84"],[0.180602006688963,"#433E85"],[0.183946488294314,"#423F85"],[0.187290969899666,"#424086"],[0.190635451505017,"#424186"],[0.193979933110368,"#424186"],[0.197324414715719,"#414387"],[0.20066889632107,"#414487"],[0.204013377926421,"#404588"],[0.207357859531773,"#404688"],[0.210702341137124,"#3F4788"],[0.214046822742475,"#3F4889"],[0.217391304347826,"#3F4889"],[0.220735785953177,"#3E4989"],[0.224080267558528,"#3E4A89"],[0.22742474916388,"#3E4C8A"],[0.230769230769231,"#3D4D8A"],[0.234113712374582,"#3D4E8A"],[0.237458193979933,"#3C4F8A"],[0.240802675585284,"#3C4F8A"],[0.244147157190635,"#3C508B"],[0.247491638795987,"#3B518B"],[0.250836120401338,"#3B528B"],[0.254180602006689,"#3A538B"],[0.25752508361204,"#3A548C"],[0.260869565217391,"#39558C"],[0.264214046822743,"#39558C"],[0.267558528428094,"#39568C"],[0.270903010033445,"#38588C"],[0.274247491638796,"#38598C"],[0.277591973244147,"#375A8C"],[0.280936454849498,"#375B8D"],[0.28428093645485,"#375B8D"],[0.287625418060201,"#365C8D"],[0.290969899665552,"#365D8D"],[0.294314381270903,"#355E8D"],[0.297658862876254,"#355F8D"],[0.301003344481605,"#34608D"],[0.304347826086957,"#34618D"],[0.307692307692308,"#34618D"],[0.311036789297659,"#33628D"],[0.31438127090301,"#33638D"],[0.317725752508361,"#32648E"],[0.321070234113712,"#32658E"],[0.324414715719064,"#31668E"],[0.327759197324415,"#31678E"],[0.331103678929766,"#31678E"],[0.334448160535117,"#31688E"],[0.337792642140468,"#30698E"],[0.341137123745819,"#306A8E"],[0.344481605351171,"#2F6B8E"],[0.347826086956522,"#2F6C8E"],[0.351170568561873,"#2E6D8E"],[0.354515050167224,"#2E6D8E"],[0.357859531772575,"#2E6E8E"],[0.361204013377926,"#2E6F8E"],[0.364548494983278,"#2D708E"],[0.367892976588629,"#2D718E"],[0.37123745819398,"#2C718E"],[0.374581939799331,"#2C728E"],[0.377926421404682,"#2C728E"],[0.381270903010033,"#2C738E"],[0.384615384615385,"#2B748E"],[0.387959866220736,"#2B758E"],[0.391304347826087,"#2A768E"],[0.394648829431438,"#2A778E"],[0.397993311036789,"#2A778E"],[0.40133779264214,"#2A788E"],[0.404682274247492,"#29798E"],[0.408026755852843,"#297A8E"],[0.411371237458194,"#297B8E"],[0.414715719063545,"#287C8E"],[0.418060200668896,"#287D8E"],[0.421404682274248,"#287D8E"],[0.424749163879599,"#277E8E"],[0.42809364548495,"#277F8E"],[0.431438127090301,"#27808E"],[0.434782608695652,"#26818E"],[0.438127090301003,"#26828E"],[0.441471571906354,"#26828E"],[0.444816053511706,"#26828E"],[0.448160535117057,"#25838E"],[0.451505016722408,"#25848E"],[0.454849498327759,"#25858E"],[0.45819397993311,"#24868E"],[0.461538461538462,"#24878E"],[0.464882943143813,"#23888E"],[0.468227424749164,"#23888E"],[0.471571906354515,"#23898E"],[0.474916387959866,"#238A8D"],[0.478260869565217,"#228B8D"],[0.481605351170569,"#228C8D"],[0.48494983277592,"#228D8D"],[0.488294314381271,"#218E8D"],[0.491638795986622,"#218E8D"],[0.494983277591973,"#218F8D"],[0.498327759197324,"#21908D"],[0.501672240802676,"#21918C"],[0.505016722408027,"#20928C"],[0.508361204013378,"#20928C"],[0.511705685618729,"#20928C"],[0.51505016722408,"#20938C"],[0.518394648829432,"#1F948C"],[0.521739130434783,"#1F958B"],[0.525083612040134,"#1F968B"],[0.528428093645485,"#1F978B"],[0.531772575250836,"#1F988B"],[0.535117056856187,"#1F988B"],[0.538461538461538,"#1F998A"],[0.54180602006689,"#1F9A8A"],[0.545150501672241,"#1E9B8A"],[0.548494983277592,"#1E9C89"],[0.551839464882943,"#1E9D89"],[0.555183946488294,"#1F9E89"],[0.558528428093646,"#1F9E89"],[0.561872909698997,"#1F9F88"],[0.565217391304348,"#1FA088"],[0.568561872909699,"#1FA188"],[0.57190635451505,"#1FA187"],[0.575250836120401,"#1FA287"],[0.578595317725753,"#20A386"],[0.581939799331104,"#20A386"],[0.585284280936455,"#20A486"],[0.588628762541806,"#21A585"],[0.591973244147157,"#21A685"],[0.595317725752508,"#22A785"],[0.598662207357859,"#22A884"],[0.602006688963211,"#23A983"],[0.605351170568562,"#23A983"],[0.608695652173913,"#24AA83"],[0.612040133779264,"#25AB82"],[0.615384615384615,"#25AC82"],[0.618729096989967,"#26AD81"],[0.622073578595318,"#27AD81"],[0.625418060200669,"#27AD81"],[0.62876254180602,"#28AE80"],[0.632107023411371,"#29AF7F"],[0.635451505016722,"#2AB07F"],[0.638795986622074,"#2CB17E"],[0.642140468227425,"#2DB27D"],[0.645484949832776,"#2EB37C"],[0.648829431438127,"#2EB37C"],[0.652173913043478,"#30B47C"],[0.655518394648829,"#31B57B"],[0.658862876254181,"#32B67A"],[0.662207357859532,"#34B679"],[0.665551839464883,"#35B779"],[0.668896321070234,"#36B878"],[0.672240802675585,"#37B878"],[0.675585284280936,"#39B977"],[0.678929765886288,"#3ABA76"],[0.682274247491639,"#3BBB75"],[0.68561872909699,"#3DBC74"],[0.688963210702341,"#3EBC73"],[0.692307692307692,"#40BD72"],[0.695652173913043,"#41BD72"],[0.698996655518395,"#42BE71"],[0.702341137123746,"#44BF70"],[0.705685618729097,"#46C06F"],[0.709030100334448,"#48C16E"],[0.712374581939799,"#49C16D"],[0.71571906354515,"#4BC26C"],[0.719063545150502,"#4DC26C"],[0.722408026755853,"#4EC36B"],[0.725752508361204,"#50C46A"],[0.729096989966555,"#52C569"],[0.732441471571906,"#54C568"],[0.735785953177258,"#55C667"],[0.739130434782609,"#57C666"],[0.74247491638796,"#59C765"],[0.745819397993311,"#5AC864"],[0.749163879598662,"#5CC863"],[0.752508361204013,"#5EC962"],[0.755852842809365,"#5FCA61"],[0.759197324414716,"#62CB5F"],[0.762541806020067,"#64CB5F"],[0.765886287625418,"#66CB5D"],[0.769230769230769,"#67CC5C"],[0.77257525083612,"#69CD5B"],[0.775919732441472,"#6CCD5A"],[0.779264214046823,"#6DCE59"],[0.782608695652174,"#6FCF57"],[0.785953177257525,"#71CF57"],[0.789297658862876,"#74D055"],[0.792642140468227,"#75D054"],[0.795986622073579,"#77D153"],[0.79933110367893,"#79D151"],[0.802675585284281,"#7BD250"],[0.806020066889632,"#7ED34F"],[0.809364548494983,"#80D34E"],[0.812709030100334,"#82D34D"],[0.816053511705686,"#84D44B"],[0.819397993311037,"#86D549"],[0.822742474916388,"#88D548"],[0.826086956521739,"#8AD647"],[0.82943143812709,"#8DD645"],[0.832775919732442,"#8FD644"],[0.836120401337793,"#91D743"],[0.839464882943144,"#93D741"],[0.842809364548495,"#95D840"],[0.846153846153846,"#97D83E"],[0.849498327759197,"#9AD93D"],[0.852842809364549,"#9CD93C"],[0.8561872909699,"#9ED93A"],[0.859531772575251,"#A0DA39"],[0.862876254180602,"#A2DA37"],[0.866220735785953,"#A5DB36"],[0.869565217391304,"#A7DB35"],[0.872909698996656,"#A9DC33"],[0.876254180602007,"#ABDC31"],[0.879598662207358,"#AEDC30"],[0.882943143812709,"#B0DD2F"],[0.88628762541806,"#B2DD2D"],[0.889632107023411,"#B5DE2B"],[0.892976588628763,"#B7DE2A"],[0.896321070234114,"#B9DE28"],[0.899665551839465,"#BBDE27"],[0.903010033444816,"#BEDF26"],[0.906354515050167,"#C0DF25"],[0.909698996655518,"#C2DF23"],[0.91304347826087,"#C4E021"],[0.916387959866221,"#C7E020"],[0.919732441471572,"#C9E11F"],[0.923076923076923,"#CBE11E"],[0.926421404682274,"#CEE11D"],[0.929765886287625,"#D0E11C"],[0.933110367892977,"#D2E21B"],[0.936454849498328,"#D4E21A"],[0.939799331103679,"#D7E219"],[0.94314381270903,"#D9E319"],[0.946488294314381,"#DBE319"],[0.949832775919733,"#DDE318"],[0.953177257525084,"#DFE318"],[0.956521739130435,"#E2E418"],[0.959866220735786,"#E4E419"],[0.963210702341137,"#E6E419"],[0.966555183946488,"#E8E419"],[0.969899665551839,"#EBE51A"],[0.973244147157191,"#EDE51B"],[0.976588628762542,"#EFE51C"],[0.979933110367893,"#F1E51D"],[0.983277591973244,"#F3E61E"],[0.986622073578595,"#F5E61F"],[0.989966555183946,"#F7E620"],[0.993311036789298,"#F9E622"],[0.996655518394649,"#FBE723"],[1,"#FDE725"]],"colorbar":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"thickness":23.04,"title":null,"titlefont":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"tickmode":"array","ticktext":["0","25","50","75"],"tickvals":[0,0.25649067610224,0.512981352204481,0.769472028306721],"tickfont":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895},"ticklen":2,"len":0.5}},"xaxis":"x","yaxis":"y2","frame":null},{"x":[103.299421744355,103.299421744355,null,103.299421744355,22.4737611721091,null,22.4737611721091,22.4737611721091,null,22.4737611721091,0,null,22.4737611721091,22.4737611721091,null,22.4737611721091,8.66559681583444,null,8.66559681583444,8.66559681583444,null,8.66559681583444,0,null,8.66559681583444,8.66559681583444,null,8.66559681583444,0],"y":[7.71875,9.75,null,9.75,9.75,null,9.75,9,null,9,9,null,9.75,10.5,null,10.5,10.5,null,10.5,10,null,10,10,null,10.5,11,null,11,11],"text":["y: 103.299422","y: 103.299422",null,"y: 103.299422","y: 103.299422",null,"y:  22.473761","y:  22.473761",null,"y:  22.473761","y:  22.473761",null,"y:  22.473761","y:  22.473761",null,"y:  22.473761","y:  22.473761",null,"y:   8.665597","y:   8.665597",null,"y:   8.665597","y:   8.665597",null,"y:   8.665597","y:   8.665597",null,"y:   8.665597","y:   8.665597"],"type":"scatter","mode":"lines","line":{"width":2.26771653543307,"color":"rgba(125,176,221,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x2","yaxis":"y2","hoverinfo":"text","frame":null},{"x":[103.299421744355,103.299421744355,null,103.299421744355,48.1656055801783,null,48.1656055801783,48.1656055801783,null,48.1656055801783,22.7913094087056,null,22.7913094087056,22.7913094087056,null,22.7913094087056,0,null,22.7913094087056,22.7913094087056,null,22.7913094087056,11.5000006047272,null,11.5000006047272,11.5000006047272,null,11.5000006047272,0,null,11.5000006047272,11.5000006047272,null,11.5000006047272,5.18337585832964,null,5.18337585832964,5.18337585832964,null,5.18337585832964,0,null,5.18337585832964,5.18337585832964,null,5.18337585832964,0,null,48.1656055801783,48.1656055801783,null,48.1656055801783,9.55528252015607,null,9.55528252015607,9.55528252015607,null,9.55528252015607,0,null,9.55528252015607,9.55528252015607,null,9.55528252015607,0],"y":[7.71875,5.6875,null,5.6875,5.6875,null,5.6875,3.875,null,3.875,3.875,null,3.875,3,null,3,3,null,3.875,4.75,null,4.75,4.75,null,4.75,4,null,4,4,null,4.75,5.5,null,5.5,5.5,null,5.5,5,null,5,5,null,5.5,6,null,6,6,null,5.6875,7.5,null,7.5,7.5,null,7.5,7,null,7,7,null,7.5,8,null,8,8],"text":["y: 103.299422","y: 103.299422",null,"y: 103.299422","y: 103.299422",null,"y:  48.165606","y:  48.165606",null,"y:  48.165606","y:  48.165606",null,"y:  22.791309","y:  22.791309",null,"y:  22.791309","y:  22.791309",null,"y:  22.791309","y:  22.791309",null,"y:  22.791309","y:  22.791309",null,"y:  11.500001","y:  11.500001",null,"y:  11.500001","y:  11.500001",null,"y:  11.500001","y:  11.500001",null,"y:  11.500001","y:  11.500001",null,"y:   5.183376","y:   5.183376",null,"y:   5.183376","y:   5.183376",null,"y:   5.183376","y:   5.183376",null,"y:   5.183376","y:   5.183376",null,"y:  48.165606","y:  48.165606",null,"y:  48.165606","y:  48.165606",null,"y:   9.555283","y:   9.555283",null,"y:   9.555283","y:   9.555283",null,"y:   9.555283","y:   9.555283",null,"y:   9.555283","y:   9.555283"],"type":"scatter","mode":"lines","line":{"width":2.26771653543307,"color":"rgba(134,184,117,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x2","yaxis":"y2","hoverinfo":"text","frame":null},{"x":[130.227249855505,130.227249855505,null,130.227249855505,40.6608678189927,null,40.6608678189927,40.6608678189927,null,40.6608678189927,0,null,40.6608678189927,40.6608678189927,null,40.6608678189927,0],"y":[4.609375,1.5,null,1.5,1.5,null,1.5,1,null,1,1,null,1.5,2,null,2,2],"text":["y: 130.227250","y: 130.227250",null,"y: 130.227250","y: 130.227250",null,"y:  40.660868","y:  40.660868",null,"y:  40.660868","y:  40.660868",null,"y:  40.660868","y:  40.660868",null,"y:  40.660868","y:  40.660868"],"type":"scatter","mode":"lines","line":{"width":2.26771653543307,"color":"rgba(228,149,165,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x2","yaxis":"y2","hoverinfo":"text","frame":null},{"x":[130.227249855505,130.227249855505,null,130.227249855505,103.299421744355],"y":[4.609375,7.71875,null,7.71875,7.71875],"text":"y: 130.227250","type":"scatter","mode":"lines","line":{"width":2.26771653543307,"color":"rgba(0,0,0,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x2","yaxis":"y2","hoverinfo":"text","frame":null},{"x":[130.227249855505,40.6608678189927,0,0,103.299421744355,48.1656055801783,22.7913094087056,0,11.5000006047272,0,5.18337585832964,0,0,9.55528252015607,0,0,22.4737611721091,0,8.66559681583444,0,0],"y":[4.609375,1.5,1,2,7.71875,5.6875,3.875,3,4.75,4,5.5,5,6,7.5,7,8,9.75,9,10.5,10,11],"text":["y: 130.227250","y:  40.660868","y:   0.000000","y:   0.000000","y: 103.299422","y:  48.165606","y:  22.791309","y:   0.000000","y:  11.500001","y:   0.000000","y:   5.183376","y:   0.000000","y:   0.000000","y:   9.555283","y:   0.000000","y:   0.000000","y:  22.473761","y:   0.000000","y:   8.665597","y:   0.000000","y:   0.000000"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"transparent","opacity":1,"size":null,"symbol":null,"line":{"width":1.88976377952756,"color":"transparent"}},"hoveron":"points","showlegend":false,"xaxis":"x2","yaxis":"y2","hoverinfo":"text","frame":null}],"layout":{"xaxis":{"domain":[0,0.8],"automargin":true,"type":"linear","autorange":false,"range":[0.5,5.5],"tickmode":"array","ticktext":["Bacillus sp","Escherichia coli","Pseudomonas aeruginosa","Kokuria rhizophila","Unclassified"],"tickvals":[1,2,3,4,5],"categoryorder":"array","categoryarray":["Bacillus sp","Escherichia coli","Pseudomonas aeruginosa","Kokuria rhizophila","Unclassified"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":13.2835201328352},"tickangle":-45,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y2","title":"","hoverformat":".2f"},"xaxis2":{"domain":[0.8,1],"automargin":true,"showticklabels":false,"showgrid":false,"zeroline":false,"anchor":"y2","type":"linear","autorange":false,"range":[0,130.227249855505],"tickmode":"array","ticktext":["0","25","50","75","100","125"],"tickvals":[0,25,50,75,100,125],"categoryorder":"array","categoryarray":["0","25","50","75","100","125"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"tickfont":{"color":null,"family":null,"size":0},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"gridcolor":null,"gridwidth":0,"title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"yaxis2":{"domain":[0,0.8],"automargin":true,"type":"linear","autorange":false,"range":[0.5,11.5],"tickmode":"array","ticktext":["piPolB","piPolB+D","piMDA+D","piMDA+D2","piMDA2","piMDA","NA2","NA","PrimPol-MDA","RP-MDA2","RP-MDA"],"tickvals":[1,2,3,4,5,6,7,8,9,10,11],"categoryorder":"array","categoryarray":["piPolB","piPolB+D","piMDA+D","piMDA+D2","piMDA2","piMDA","NA2","NA","PrimPol-MDA","RP-MDA2","RP-MDA"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":18.5969281859693},"tickangle":-0,"showline":true,"linecolor":"rgba(0,0,0,1)","linewidth":0.66417600664176,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"yaxis":{"domain":[0.8,1],"automargin":true,"showticklabels":false,"showgrid":false,"zeroline":false,"anchor":"x","type":"linear","autorange":false,"range":[0,179.079354876334],"tickmode":"array","ticktext":["0","50","100","150"],"tickvals":[0,50,100,150],"categoryorder":"array","categoryarray":["0","50","100","150"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"tickfont":{"color":null,"family":null,"size":0},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"gridcolor":null,"gridwidth":0,"title":{"text":"","font":{"color":null,"family":null,"size":0}},"hoverformat":".2f"},"annotations":[],"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":0.8,"y0":0.8,"y1":1},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":0.8,"y0":0,"y1":0.8},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0.8,"x1":1,"y0":0,"y1":0.8}],"images":[],"margin":{"t":0,"r":null,"b":245.479452054795,"l":122.739726027397},"font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895},"title":{"text":"","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187}},"y":1,"yanchor":"top"},"hovermode":"closest","barmode":"relative","paper_bgcolor":"rgba(255,255,255,1)","title":""},"attrs":{"1701daae7e07":{"xend":{},"yend":{},"colour":{},"linetype":{},"size":{},"x":{},"y":{},"type":"scatter"},"1701d45d4030b":{"colour":{},"shape":{},"size":{},"x":{},"y":{}},"1701d9a8ba59":{"alpha_stroke":1,"sizes":[10,100],"spans":[1,20]},"1701d392a84d2":{"x":{},"y":{},"fill":{},"text":{},"type":"heatmap"},"1701d75e5c231":{"xend":{},"yend":{},"colour":{},"linetype":{},"size":{},"x":{},"y":{},"type":"scatter"},"1701d3f305b6f":{"colour":{},"shape":{},"size":{},"x":{},"y":{}}},"source":"A","config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false,"displaylogo":false,"modeBarButtonsToRemove":["sendDataToCloud","select2d","lasso2d","autoScale2d","hoverClosestCartesian","hoverCompareCartesian","sendDataToCloud"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"subplot":true,"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<p>Figure 8. Clustering of samples by the metagenomic profiling results</p>
</div>
</div>
<p>Interestingly, in terms of relative representativity of reads from each reference genome, both NA samples cluster with the <em>piMDA/piMDA+D</em> amplification products. As mentioned above, we found an increased amount of unknown reads in the piPolB(+D) samples. As samples have been quality-trimmed, we only can suggest that those sequences arose from <em>ab initio</em> DNA synthesis.</p>
<p>On the other hand, <em>RP-MDA</em> and <em>PrimPol-MDA</em> contain an overrepresentation of <em>B. subtilis</em> reads, as compared with the NA reads. The <em>piMDA</em> amplified samples, particularly the <em>piMDA+D</em> reads form the first experiment, show a moderate overrepresentation of high GC sequences (<em>K. rhizophyla</em>), in agreement with the GC content profile of these samples (see Figure 6).</p>
</section>
</section>
<section id="assembly-independent-coverage-of-reference-genomes" class="level2" data-number="5">
<h2 data-number="5" class="anchored" data-anchor-id="assembly-independent-coverage-of-reference-genomes"><span class="header-section-number">5</span> Assembly-independent coverage of reference genomes</h2>
<section id="reference-genomes-breadth-and-coverage" class="level3" data-number="5.1">
<h3 data-number="5.1" class="anchored" data-anchor-id="reference-genomes-breadth-and-coverage"><span class="header-section-number">5.1</span> Reference genomes breadth and coverage</h3>
<p>Reads were mapped against the reference genomes with Bowtie2 v2.3.5.1 and further analyzed with <a href="https://bioinformatics.cvr.ac.uk/weesam-version-1-5/" target="_blank">Weesam</a> v1.6 and <a href="https://www.gear-genomics.com/docs/alfred/" target="_blank">Alfred</a> v0.1.16. The Weesam reports can be accessed in a merged <a href="weeSam/weesam_full.html" target="_blank">webpage</a>. Alfred reports are accesible as a <em>csv</em> <a href="alfred/merded/merged_alfred.csv" target="_blank">table</a>. Mean values per sample are shown below in Table 5.</p>
<div class="cell">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb48"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb48-1"><a href="#cb48-1" aria-hidden="true" tabindex="-1"></a><span class="co"># WeeSam html files were merged by copy/paste from the source file by hand and then</span></span>
<span id="cb48-2"><a href="#cb48-2" aria-hidden="true" tabindex="-1"></a><span class="co"># exported into csv</span></span>
<span id="cb48-3"><a href="#cb48-3" aria-hidden="true" tabindex="-1"></a>weesam <span class="ot">&lt;-</span> <span class="fu">read.table</span>(<span class="st">"weeSam/weesam_merged.csv"</span>, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">na.strings =</span> <span class="st">""</span>, <span class="at">sep =</span> <span class="st">";"</span>)</span>
<span id="cb48-4"><a href="#cb48-4" aria-hidden="true" tabindex="-1"></a><span class="co"># change names</span></span>
<span id="cb48-5"><a href="#cb48-5" aria-hidden="true" tabindex="-1"></a>weesam <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="fu">lapply</span>(weesam, <span class="cf">function</span>(x) {</span>
<span id="cb48-6"><a href="#cb48-6" aria-hidden="true" tabindex="-1"></a>    <span class="fu">gsub</span>(<span class="st">"RepliG"</span>, <span class="st">"RP-MDA"</span>, x)</span>
<span id="cb48-7"><a href="#cb48-7" aria-hidden="true" tabindex="-1"></a>}))</span>
<span id="cb48-8"><a href="#cb48-8" aria-hidden="true" tabindex="-1"></a>weesam <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="fu">lapply</span>(weesam, <span class="cf">function</span>(x) {</span>
<span id="cb48-9"><a href="#cb48-9" aria-hidden="true" tabindex="-1"></a>    <span class="fu">gsub</span>(<span class="st">"TruePrime"</span>, <span class="st">"PrimPol-MDA"</span>, x)</span>
<span id="cb48-10"><a href="#cb48-10" aria-hidden="true" tabindex="-1"></a>}))</span>
<span id="cb48-11"><a href="#cb48-11" aria-hidden="true" tabindex="-1"></a><span class="co"># fix variable types</span></span>
<span id="cb48-12"><a href="#cb48-12" aria-hidden="true" tabindex="-1"></a>weesam<span class="sc">$</span>Bam_File <span class="ot">&lt;-</span> <span class="fu">factor</span>(weesam<span class="sc">$</span>Bam_File, <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">"NA"</span>, <span class="st">"RP-MDA"</span>, <span class="st">"PrimPol-MDA"</span>, <span class="st">"piPolB"</span>,</span>
<span id="cb48-13"><a href="#cb48-13" aria-hidden="true" tabindex="-1"></a>    <span class="st">"piPolB+D"</span>, <span class="st">"piMDA"</span>, <span class="st">"piMDA+D"</span>))</span>
<span id="cb48-14"><a href="#cb48-14" aria-hidden="true" tabindex="-1"></a>weesam <span class="ot">&lt;-</span> weesam <span class="sc">%&gt;%</span></span>
<span id="cb48-15"><a href="#cb48-15" aria-hidden="true" tabindex="-1"></a>    <span class="fu">mutate</span>(<span class="fu">across</span>(<span class="fu">c</span>(Ref_Genome, Ref_Name), as.factor))</span>
<span id="cb48-16"><a href="#cb48-16" aria-hidden="true" tabindex="-1"></a>weesam <span class="ot">&lt;-</span> weesam <span class="sc">%&gt;%</span></span>
<span id="cb48-17"><a href="#cb48-17" aria-hidden="true" tabindex="-1"></a>    <span class="fu">mutate</span>(<span class="fu">across</span>(<span class="fu">c</span>(Avg_Depth, Experiment, Mapped_Reads, Breadth, Std_Dev, Variation_Coefficient,</span>
<span id="cb48-18"><a href="#cb48-18" aria-hidden="true" tabindex="-1"></a>        covered), as.numeric))</span>
<span id="cb48-19"><a href="#cb48-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb48-20"><a href="#cb48-20" aria-hidden="true" tabindex="-1"></a><span class="co"># add Bowtie2 overall alignment rate values</span></span>
<span id="cb48-21"><a href="#cb48-21" aria-hidden="true" tabindex="-1"></a>Bam_File <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"NA"</span>, <span class="st">"NA"</span>, <span class="st">"RP-MDA"</span>, <span class="st">"RP-MDA"</span>, <span class="st">"PrimPol-MDA"</span>, <span class="st">"piPolB"</span>, <span class="st">"piPolB+D"</span>, <span class="st">"piMDA"</span>,</span>
<span id="cb48-22"><a href="#cb48-22" aria-hidden="true" tabindex="-1"></a>    <span class="st">"piMDA"</span>, <span class="st">"piMDA+D"</span>, <span class="st">"piMDA+D"</span>)</span>
<span id="cb48-23"><a href="#cb48-23" aria-hidden="true" tabindex="-1"></a>alignment <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="fl">97.85</span>, <span class="fl">97.99</span>, <span class="fl">99.31</span>, <span class="fl">99.24</span>, <span class="fl">99.25</span>, <span class="fl">7.67</span>, <span class="fl">73.15</span>, <span class="fl">94.22</span>, <span class="fl">93.76</span>, <span class="fl">97.68</span>, <span class="fl">97.18</span>)</span>
<span id="cb48-24"><a href="#cb48-24" aria-hidden="true" tabindex="-1"></a>bowtie <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(Bam_File, alignment)</span>
<span id="cb48-25"><a href="#cb48-25" aria-hidden="true" tabindex="-1"></a>weesam <span class="ot">&lt;-</span> <span class="fu">merge</span>(weesam, bowtie, <span class="at">by =</span> <span class="st">"Bam_File"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<p>Among the different parameters that WeeSAM provide, we are going to represent the <em>Alignment Rate, Breadth</em>, <em>Coverage</em> and the <em>Variation Coeficient</em> (Figures 9 and 10). The overall alignment rate is clearly better using selected reference genomes than in the profiling with a reference database (Figures 7 &amp; 8). However, piPolB and piPolB+D samples still show lower alignment rates, suggesting unfaithful DNA amplification (see below).</p>
<p>Breadth and Coverage are good general indicators of very poor amplification of some of the sequences, usually with extreme high or low GC content. On the other hand, the <em>Variation Coeficient</em> indicates variability in coverage. A coefficient of variation &lt; 1 would suggest that the coverage has low-variance, which is good, while a coefficient &gt; 1 would be considered high-variance.</p>
<div class="cell" data-layout-align="left">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb49"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb49-1"><a href="#cb49-1" aria-hidden="true" tabindex="-1"></a>p1 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(weesam, <span class="fu">aes</span>(<span class="at">x =</span> Bam_File, <span class="at">y =</span> <span class="fu">log</span>(Avg_Depth, <span class="dv">10</span>), <span class="at">group =</span> Ref_Name, <span class="at">color =</span> Ref_Name,</span>
<span id="cb49-2"><a href="#cb49-2" aria-hidden="true" tabindex="-1"></a>    <span class="at">fill =</span> Ref_Name, <span class="at">shape =</span> Ref_Name)) <span class="sc">+</span> <span class="fu">geom_pointrange</span>(<span class="fu">aes</span>(<span class="at">ymin =</span> <span class="fu">log</span>(Avg_Depth, <span class="dv">10</span>), <span class="at">ymax =</span> <span class="fu">log</span>(Avg_Depth <span class="sc">+</span></span>
<span id="cb49-3"><a href="#cb49-3" aria-hidden="true" tabindex="-1"></a>    Std_Dev, <span class="dv">10</span>)), <span class="at">size =</span> <span class="dv">1</span>, <span class="at">alpha =</span> <span class="fl">0.7</span>, <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="at">width =</span> <span class="fl">0.5</span>)) <span class="sc">+</span> <span class="fu">theme_bw</span>() <span class="sc">+</span></span>
<span id="cb49-4"><a href="#cb49-4" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> colorines[<span class="dv">12</span><span class="sc">:</span><span class="dv">15</span>]) <span class="sc">+</span> <span class="fu">scale_color_manual</span>(<span class="at">values =</span> colorines[<span class="dv">12</span><span class="sc">:</span><span class="dv">15</span>]) <span class="sc">+</span></span>
<span id="cb49-5"><a href="#cb49-5" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme</span>(<span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">15</span>)) <span class="sc">+</span> <span class="fu">ggtitle</span>(<span class="st">"Coverage Depth"</span>) <span class="sc">+</span> <span class="fu">scale_shape_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="dv">21</span>,</span>
<span id="cb49-6"><a href="#cb49-6" aria-hidden="true" tabindex="-1"></a>    <span class="dv">22</span>, <span class="dv">23</span>, <span class="dv">24</span>)) <span class="sc">+</span> <span class="fu">xlab</span>(<span class="st">"Sample"</span>) <span class="sc">+</span> <span class="fu">ylab</span>(<span class="st">"Depth (Log10)"</span>) <span class="sc">+</span> <span class="fu">labs</span>(<span class="at">color =</span> <span class="st">"Reference genome"</span>) <span class="sc">+</span></span>
<span id="cb49-7"><a href="#cb49-7" aria-hidden="true" tabindex="-1"></a>    <span class="fu">guides</span>(<span class="at">shape =</span> <span class="cn">FALSE</span>, <span class="at">fill =</span> <span class="cn">FALSE</span>, <span class="at">scale =</span> <span class="st">"none"</span>) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">45</span>,</span>
<span id="cb49-8"><a href="#cb49-8" aria-hidden="true" tabindex="-1"></a>    <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">16</span>)) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">legend.text =</span> <span class="fu">element_text</span>(<span class="at">face =</span> <span class="st">"italic"</span>)) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">hjust =</span> <span class="fl">0.5</span>))</span>
<span id="cb49-9"><a href="#cb49-9" aria-hidden="true" tabindex="-1"></a>p2 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(weesam) <span class="sc">+</span> <span class="fu">geom_point</span>(<span class="fu">aes</span>(<span class="at">x =</span> Bam_File, <span class="at">y =</span> <span class="fu">as.numeric</span>(covered), <span class="at">group =</span> Ref_Name,</span>
<span id="cb49-10"><a href="#cb49-10" aria-hidden="true" tabindex="-1"></a>    <span class="at">color =</span> Ref_Name, <span class="at">fill =</span> Ref_Name, <span class="at">shape =</span> Ref_Name), <span class="at">size =</span> <span class="dv">4</span>, <span class="at">alpha =</span> <span class="fl">0.7</span>, <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="at">width =</span> <span class="fl">0.5</span>)) <span class="sc">+</span></span>
<span id="cb49-11"><a href="#cb49-11" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme_bw</span>() <span class="sc">+</span> <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> colorines[<span class="dv">12</span><span class="sc">:</span><span class="dv">15</span>]) <span class="sc">+</span> <span class="fu">scale_color_manual</span>(<span class="at">values =</span> colorines[<span class="dv">12</span><span class="sc">:</span><span class="dv">15</span>]) <span class="sc">+</span></span>
<span id="cb49-12"><a href="#cb49-12" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_shape_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="dv">21</span>, <span class="dv">22</span>, <span class="dv">23</span>, <span class="dv">24</span>)) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">15</span>)) <span class="sc">+</span></span>
<span id="cb49-13"><a href="#cb49-13" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggtitle</span>(<span class="st">"Coverage Breadth"</span>) <span class="sc">+</span> <span class="fu">xlab</span>(<span class="st">"Sample"</span>) <span class="sc">+</span> <span class="fu">labs</span>(<span class="at">color =</span> <span class="st">"Reference genome"</span>) <span class="sc">+</span> <span class="fu">guides</span>(<span class="at">shape =</span> <span class="cn">FALSE</span>,</span>
<span id="cb49-14"><a href="#cb49-14" aria-hidden="true" tabindex="-1"></a>    <span class="at">fill =</span> <span class="cn">FALSE</span>, <span class="at">scale =</span> <span class="st">"none"</span>) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">45</span>, <span class="at">hjust =</span> <span class="dv">1</span>,</span>
<span id="cb49-15"><a href="#cb49-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">size =</span> <span class="dv">16</span>)) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">legend.text =</span> <span class="fu">element_text</span>(<span class="at">face =</span> <span class="st">"italic"</span>)) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">hjust =</span> <span class="fl">0.5</span>)) <span class="sc">+</span></span>
<span id="cb49-16"><a href="#cb49-16" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_y_continuous</span>(<span class="at">name =</span> <span class="st">"Coverage (%)"</span>, <span class="at">breaks =</span> <span class="fu">seq</span>(<span class="dv">80</span>, <span class="dv">100</span>, <span class="dv">5</span>), <span class="at">limit =</span> <span class="fu">c</span>(<span class="dv">80</span>, <span class="dv">100</span>))</span>
<span id="cb49-17"><a href="#cb49-17" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb49-18"><a href="#cb49-18" aria-hidden="true" tabindex="-1"></a>p3 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(weesam, <span class="fu">aes</span>(<span class="at">x =</span> Bam_File, <span class="at">y =</span> Variation_Coefficient, <span class="at">group =</span> Ref_Name, <span class="at">color =</span> Ref_Name,</span>
<span id="cb49-19"><a href="#cb49-19" aria-hidden="true" tabindex="-1"></a>    <span class="at">fill =</span> Ref_Name, <span class="at">shape =</span> Ref_Name)) <span class="sc">+</span> <span class="fu">geom_point</span>(<span class="at">size =</span> <span class="dv">4</span>, <span class="at">alpha =</span> <span class="fl">0.7</span>, <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="at">width =</span> <span class="fl">0.5</span>)) <span class="sc">+</span></span>
<span id="cb49-20"><a href="#cb49-20" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme_bw</span>() <span class="sc">+</span> <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> colorines[<span class="dv">12</span><span class="sc">:</span><span class="dv">15</span>]) <span class="sc">+</span> <span class="fu">scale_color_manual</span>(<span class="at">values =</span> colorines[<span class="dv">12</span><span class="sc">:</span><span class="dv">15</span>]) <span class="sc">+</span></span>
<span id="cb49-21"><a href="#cb49-21" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_shape_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="dv">21</span>, <span class="dv">22</span>, <span class="dv">23</span>, <span class="dv">24</span>)) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">15</span>)) <span class="sc">+</span></span>
<span id="cb49-22"><a href="#cb49-22" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggtitle</span>(<span class="st">"Variation Coefficient"</span>) <span class="sc">+</span> <span class="fu">xlab</span>(<span class="st">"Sample"</span>) <span class="sc">+</span> <span class="fu">ylab</span>(<span class="st">"Coefficient"</span>) <span class="sc">+</span> <span class="fu">labs</span>(<span class="at">color =</span> <span class="st">"Reference genome"</span>) <span class="sc">+</span></span>
<span id="cb49-23"><a href="#cb49-23" aria-hidden="true" tabindex="-1"></a>    <span class="fu">guides</span>(<span class="at">shape =</span> <span class="cn">FALSE</span>, <span class="at">fill =</span> <span class="cn">FALSE</span>, <span class="at">scale =</span> <span class="st">"none"</span>) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">45</span>,</span>
<span id="cb49-24"><a href="#cb49-24" aria-hidden="true" tabindex="-1"></a>    <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">16</span>)) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">legend.text =</span> <span class="fu">element_text</span>(<span class="at">face =</span> <span class="st">"italic"</span>)) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">hjust =</span> <span class="fl">0.5</span>))</span>
<span id="cb49-25"><a href="#cb49-25" aria-hidden="true" tabindex="-1"></a>p4 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(weesam) <span class="sc">+</span> <span class="fu">geom_point</span>(<span class="fu">aes</span>(<span class="at">x =</span> Bam_File, <span class="at">y =</span> <span class="fu">as.numeric</span>(alignment), <span class="at">color =</span> Bam_File),</span>
<span id="cb49-26"><a href="#cb49-26" aria-hidden="true" tabindex="-1"></a>    <span class="at">alpha =</span> <span class="fl">0.4</span>, <span class="at">size =</span> <span class="dv">4</span>, <span class="at">shape =</span> <span class="dv">19</span>) <span class="sc">+</span> <span class="fu">theme_bw</span>() <span class="sc">+</span> <span class="fu">scale_color_manual</span>(<span class="at">values =</span> colorines[<span class="dv">1</span><span class="sc">:</span><span class="dv">11</span>]) <span class="sc">+</span></span>
<span id="cb49-27"><a href="#cb49-27" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme</span>(<span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">15</span>)) <span class="sc">+</span> <span class="fu">ggtitle</span>(<span class="st">"Overall alignment rate"</span>) <span class="sc">+</span> <span class="fu">xlab</span>(<span class="st">"Sample"</span>) <span class="sc">+</span></span>
<span id="cb49-28"><a href="#cb49-28" aria-hidden="true" tabindex="-1"></a>    <span class="fu">labs</span>(<span class="at">color =</span> <span class="st">"Method"</span>) <span class="sc">+</span> <span class="fu">guides</span>(<span class="at">shape =</span> <span class="cn">FALSE</span>, <span class="at">fill =</span> <span class="cn">FALSE</span>, <span class="at">scale =</span> <span class="st">"none"</span>) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">45</span>,</span>
<span id="cb49-29"><a href="#cb49-29" aria-hidden="true" tabindex="-1"></a>    <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">16</span>)) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">legend.text =</span> <span class="fu">element_text</span>(<span class="at">face =</span> <span class="st">"italic"</span>)) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">hjust =</span> <span class="fl">0.5</span>)) <span class="sc">+</span></span>
<span id="cb49-30"><a href="#cb49-30" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_y_continuous</span>(<span class="at">name =</span> <span class="st">"Alignment rate (%)"</span>, <span class="at">breaks =</span> <span class="fu">seq</span>(<span class="dv">0</span>, <span class="dv">100</span>, <span class="dv">20</span>), <span class="at">limit =</span> <span class="fu">c</span>(<span class="dv">0</span>,</span>
<span id="cb49-31"><a href="#cb49-31" aria-hidden="true" tabindex="-1"></a>        <span class="dv">100</span>))</span>
<span id="cb49-32"><a href="#cb49-32" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb49-33"><a href="#cb49-33" aria-hidden="true" tabindex="-1"></a>grid <span class="ot">&lt;-</span> <span class="fu">grid.arrange</span>(p4, p2, p1, p3, <span class="at">nrow =</span> <span class="dv">2</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-left">
<figure class="figure">
<p><img src="Plotdepth-1.png" class="img-fluid figure-img" width="1152"></p>
<p></p><figcaption class="figure-caption">Figure 9. Reference genomes coverage depth by diverse MDA methods</figcaption><p></p>
</figure>
</div>
</div>
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb50"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb50-1"><a href="#cb50-1" aria-hidden="true" tabindex="-1"></a>grid</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stdout">
<pre><code>TableGrob (2 x 2) "arrange": 4 grobs
  z     cells    name           grob
1 1 (1-1,1-1) arrange gtable[layout]
2 2 (1-1,2-2) arrange gtable[layout]
3 3 (2-2,1-1) arrange gtable[layout]
4 4 (2-2,2-2) arrange gtable[layout]</code></pre>
</div>
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb52"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb52-1"><a href="#cb52-1" aria-hidden="true" tabindex="-1"></a><span class="co"># version to export ggsave('figures/depth_plot.svg',grid,width=12,height=12)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<p>The samples of RP-MDA and, to a lesser extent the PrimPol-MDA, show a clear decrease in the sequences of high GC in the profiling (Figures 7 &amp; 8). In agreement with that, they show lower coverage and breadth and higher variation coefficient than all the other samples for those species and higher depth for <em>B. subtilis</em> and <em>E. coli.</em> Interestingly, the piMDA and piMDA+D reads contain a somewhat higher coverage for high GC genomes but lower for <em>B. subtilis</em> and <em>E. coli.</em> In the case of the samples from the piMDA protocol, this lower coverage depth of moderate GC genomes is consequence of a lower alignment rate and it is also in agreement with a higher variation coefficient in the case of <em>B. subtilis</em>, although the coverage depth for this genome is the same for all the samples<em>.</em></p>
<p>In conclusion, the RP-MDA and, to a lesser extent, the PrimPol-MDA amplification methods show a strong bias against very high GC-content sequences and, on the contrary samples from piMDA methods show an overrepresentation of high GC sequences which, in the case of the piMDA protocol (not in the piMDA+D), may give rise to an overall uneven depth.</p>
<p>Another version in a single plot with two axis in the Figure 10 that allow us to see that the very high coverage depth in RP and PrimPol-MDA for intermediate-low GC content genomes correlates with a very high variation coefficient in high GC content genomes, which is not the case for piMDA and specially piMDA+D reads.</p>
<div class="cell" data-layout-align="left">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb53"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb53-1"><a href="#cb53-1" aria-hidden="true" tabindex="-1"></a>pp <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(weesam, <span class="fu">aes</span>(<span class="at">x =</span> Bam_File, <span class="at">y =</span> <span class="fu">log</span>(Avg_Depth, <span class="dv">10</span>), <span class="at">group =</span> Ref_Name, <span class="at">color =</span> Ref_Name,</span>
<span id="cb53-2"><a href="#cb53-2" aria-hidden="true" tabindex="-1"></a>    <span class="at">fill =</span> Ref_Name, <span class="at">shape =</span> Ref_Name)) <span class="sc">+</span> <span class="fu">geom_pointrange</span>(<span class="fu">aes</span>(<span class="at">ymin =</span> <span class="fu">log</span>(Avg_Depth, <span class="dv">10</span>), <span class="at">ymax =</span> <span class="fu">log</span>(Avg_Depth <span class="sc">+</span></span>
<span id="cb53-3"><a href="#cb53-3" aria-hidden="true" tabindex="-1"></a>    Std_Dev, <span class="dv">10</span>)), <span class="at">size =</span> <span class="dv">2</span>, <span class="at">alpha =</span> <span class="fl">0.7</span>, <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="at">width =</span> <span class="fl">0.5</span>)) <span class="sc">+</span> <span class="fu">geom_point</span>(<span class="fu">aes</span>(<span class="at">x =</span> Bam_File,</span>
<span id="cb53-4"><a href="#cb53-4" aria-hidden="true" tabindex="-1"></a>    <span class="at">y =</span> Variation_Coefficient<span class="sc">/</span><span class="dv">10</span>, <span class="at">fill =</span> Ref_Name), <span class="at">shape =</span> <span class="dv">21</span>, <span class="at">color =</span> <span class="st">"black"</span>, <span class="at">size =</span> <span class="dv">2</span>,</span>
<span id="cb53-5"><a href="#cb53-5" aria-hidden="true" tabindex="-1"></a>    <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="at">width =</span> <span class="fl">0.5</span>)) <span class="sc">+</span> <span class="fu">scale_y_continuous</span>(<span class="at">name =</span> <span class="st">"Average Depth (Log10)"</span>,</span>
<span id="cb53-6"><a href="#cb53-6" aria-hidden="true" tabindex="-1"></a>    <span class="at">breaks =</span> <span class="fu">seq</span>(<span class="dv">0</span>, <span class="dv">5</span>, <span class="dv">1</span>), <span class="at">limits =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">4</span>), <span class="at">sec.axis =</span> <span class="fu">sec_axis</span>(<span class="sc">~</span><span class="dv">10</span> <span class="sc">*</span> ., <span class="at">name =</span> <span class="st">"Variation Coefficient"</span>,</span>
<span id="cb53-7"><a href="#cb53-7" aria-hidden="true" tabindex="-1"></a>        <span class="at">breaks =</span> <span class="fu">seq</span>(<span class="dv">0</span>, <span class="dv">125</span>, <span class="dv">25</span>))) <span class="sc">+</span> <span class="fu">theme_bw</span>() <span class="sc">+</span> <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> colorines[<span class="dv">12</span><span class="sc">:</span><span class="dv">15</span>]) <span class="sc">+</span></span>
<span id="cb53-8"><a href="#cb53-8" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_color_manual</span>(<span class="at">values =</span> colorines[<span class="dv">12</span><span class="sc">:</span><span class="dv">15</span>]) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">15</span>)) <span class="sc">+</span></span>
<span id="cb53-9"><a href="#cb53-9" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggtitle</span>(<span class="st">"Coverage Depth"</span>) <span class="sc">+</span> <span class="fu">scale_shape_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="dv">21</span>, <span class="dv">22</span>, <span class="dv">23</span>, <span class="dv">24</span>)) <span class="sc">+</span> <span class="fu">xlab</span>(<span class="st">"Sample"</span>) <span class="sc">+</span></span>
<span id="cb53-10"><a href="#cb53-10" aria-hidden="true" tabindex="-1"></a>    <span class="fu">labs</span>(<span class="at">color =</span> <span class="st">"Reference genome"</span>) <span class="sc">+</span> <span class="fu">guides</span>(<span class="at">shape =</span> <span class="cn">FALSE</span>, <span class="at">fill =</span> <span class="cn">FALSE</span>, <span class="at">scale =</span> <span class="st">"none"</span>) <span class="sc">+</span></span>
<span id="cb53-11"><a href="#cb53-11" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme</span>(<span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">45</span>, <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">16</span>)) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">legend.text =</span> <span class="fu">element_text</span>(<span class="at">face =</span> <span class="st">"italic"</span>)) <span class="sc">+</span></span>
<span id="cb53-12"><a href="#cb53-12" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme</span>(<span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">hjust =</span> <span class="fl">0.5</span>)) <span class="sc">+</span> <span class="fu">geom_hline</span>(<span class="at">yintercept =</span> <span class="dv">1</span>, <span class="at">linetype =</span> <span class="st">"dashed"</span>,</span>
<span id="cb53-13"><a href="#cb53-13" aria-hidden="true" tabindex="-1"></a>    <span class="at">color =</span> <span class="st">"black"</span>)</span>
<span id="cb53-14"><a href="#cb53-14" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplotly</span>(pp)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-90e521d4eb67d1bf271e" style="width:100%;height:454px;"></div>
<script type="application/json" data-for="htmlwidget-90e521d4eb67d1bf271e">{"x":{"data":[{"x":[0.8125,0.8125,0.8125,0.8125,5.8125,5.8125,5.8125,5.8125,6.8125,6.8125,6.8125,6.8125,3.8125,4.8125,2.8125,1.8125,1.8125,1.8125,1.8125],"y":[2.83738007840826,2.83738007840826,2.89867034296553,2.89867034296553,2.25273170272602,2.25273170272602,2.24905152880508,2.24905152880508,2.48395796490097,2.48395796490097,2.10907208097888,2.10907208097888,1.64137494519213,2.27556471506985,3.09270661232552,3.27001932578163,3.27001932578163,3.2393620530744,3.2393620530744],"text":["Bam_File: NA<br />log(Avg_Depth, 10): 2.8373801<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />log(Avg_Depth, 10): 2.8373801<br />log(Avg_Depth + Std_Dev, 10): 2.951677","Bam_File: NA<br />log(Avg_Depth, 10): 2.8373801<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />log(Avg_Depth, 10): 2.8373801<br />log(Avg_Depth + Std_Dev, 10): 2.951677","Bam_File: NA<br />log(Avg_Depth, 10): 2.8986703<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />log(Avg_Depth, 10): 2.8986703<br />log(Avg_Depth + Std_Dev, 10): 2.989147","Bam_File: NA<br />log(Avg_Depth, 10): 2.8986703<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />log(Avg_Depth, 10): 2.8986703<br />log(Avg_Depth + Std_Dev, 10): 2.989147","Bam_File: piMDA<br />log(Avg_Depth, 10): 2.2527317<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />log(Avg_Depth, 10): 2.2527317<br />log(Avg_Depth + Std_Dev, 10): 2.584037","Bam_File: piMDA<br />log(Avg_Depth, 10): 2.2527317<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />log(Avg_Depth, 10): 2.2527317<br />log(Avg_Depth + Std_Dev, 10): 2.584037","Bam_File: piMDA<br />log(Avg_Depth, 10): 2.2490515<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />log(Avg_Depth, 10): 2.2490515<br />log(Avg_Depth + Std_Dev, 10): 2.429542","Bam_File: piMDA<br />log(Avg_Depth, 10): 2.2490515<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />log(Avg_Depth, 10): 2.2490515<br />log(Avg_Depth + Std_Dev, 10): 2.429542","Bam_File: piMDA+D<br />log(Avg_Depth, 10): 2.4839580<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />log(Avg_Depth, 10): 2.4839580<br />log(Avg_Depth + Std_Dev, 10): 2.625055","Bam_File: piMDA+D<br />log(Avg_Depth, 10): 2.4839580<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />log(Avg_Depth, 10): 2.4839580<br />log(Avg_Depth + Std_Dev, 10): 2.625055","Bam_File: piMDA+D<br />log(Avg_Depth, 10): 2.1090721<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />log(Avg_Depth, 10): 2.1090721<br />log(Avg_Depth + Std_Dev, 10): 2.269816","Bam_File: piMDA+D<br />log(Avg_Depth, 10): 2.1090721<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />log(Avg_Depth, 10): 2.1090721<br />log(Avg_Depth + Std_Dev, 10): 2.269816","Bam_File: piPolB<br />log(Avg_Depth, 10): 1.6413749<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />log(Avg_Depth, 10): 1.6413749<br />log(Avg_Depth + Std_Dev, 10): 3.658252","Bam_File: piPolB+D<br />log(Avg_Depth, 10): 2.2755647<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />log(Avg_Depth, 10): 2.2755647<br />log(Avg_Depth + Std_Dev, 10): 3.804932","Bam_File: PrimPol-MDA<br />log(Avg_Depth, 10): 3.0927066<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />log(Avg_Depth, 10): 3.0927066<br />log(Avg_Depth + Std_Dev, 10): 3.209368","Bam_File: RP-MDA<br />log(Avg_Depth, 10): 3.2700193<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />log(Avg_Depth, 10): 3.2700193<br />log(Avg_Depth + Std_Dev, 10): 3.407224","Bam_File: RP-MDA<br />log(Avg_Depth, 10): 3.2700193<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />log(Avg_Depth, 10): 3.2700193<br />log(Avg_Depth + Std_Dev, 10): 3.407224","Bam_File: RP-MDA<br />log(Avg_Depth, 10): 3.2393621<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />log(Avg_Depth, 10): 3.2393621<br />log(Avg_Depth + Std_Dev, 10): 3.353199","Bam_File: RP-MDA<br />log(Avg_Depth, 10): 3.2393621<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />Ref_Name: B. subtilis<br />log(Avg_Depth, 10): 3.2393621<br />log(Avg_Depth + Std_Dev, 10): 3.353199"],"type":"scatter","mode":"lines+markers","opacity":0.7,"line":{"color":"transparent"},"error_y":{"array":[0.11429735893504,0.11429735893504,0.0904767870234373,0.0904767870234373,0.331305368491714,0.331305368491714,0.180490818374348,0.180490818374348,0.141097126360189,0.141097126360189,0.160744296366945,0.160744296366945,2.01687691720906,1.52936758613153,0.116660931524656,0.137204968028944,0.137204968028944,0.113836489809517,0.113836489809517],"arrayminus":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"type":"data","width":0,"symmetric":false,"color":"rgba(238,154,0,1)"},"name":"B. subtilis","legendgroup":"B. subtilis","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","marker":{"autocolorscale":false,"color":"rgba(238,154,0,1)","opacity":0.7,"size":7.55905511811024,"symbol":"circle","line":{"width":3.77952755905512,"color":"rgba(238,154,0,1)"}},"hoveron":"points","frame":null},{"x":[0.9375,0.9375,0.9375,0.9375,5.9375,5.9375,5.9375,5.9375,6.9375,6.9375,6.9375,6.9375,3.9375,4.9375,2.9375,1.9375,1.9375,1.9375,1.9375],"y":[2.71270019804979,2.71270019804979,2.69121414838279,2.69121414838279,2.57174376025517,2.57174376025517,2.54248930979074,2.54248930979074,2.64939328004917,2.64939328004917,2.33581882691516,2.33581882691516,0.819543935541869,1.86593266819319,2.6698001942542,2.48831094232965,2.48831094232965,2.62676111087171,2.62676111087171],"text":["Bam_File: NA<br />log(Avg_Depth, 10): 2.7127002<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />log(Avg_Depth, 10): 2.7127002<br />log(Avg_Depth + Std_Dev, 10): 2.770270","Bam_File: NA<br />log(Avg_Depth, 10): 2.7127002<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />log(Avg_Depth, 10): 2.7127002<br />log(Avg_Depth + Std_Dev, 10): 2.770270","Bam_File: NA<br />log(Avg_Depth, 10): 2.6912141<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />log(Avg_Depth, 10): 2.6912141<br />log(Avg_Depth + Std_Dev, 10): 2.753070","Bam_File: NA<br />log(Avg_Depth, 10): 2.6912141<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />log(Avg_Depth, 10): 2.6912141<br />log(Avg_Depth + Std_Dev, 10): 2.753070","Bam_File: piMDA<br />log(Avg_Depth, 10): 2.5717438<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />log(Avg_Depth, 10): 2.5717438<br />log(Avg_Depth + Std_Dev, 10): 2.708752","Bam_File: piMDA<br />log(Avg_Depth, 10): 2.5717438<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />log(Avg_Depth, 10): 2.5717438<br />log(Avg_Depth + Std_Dev, 10): 2.708752","Bam_File: piMDA<br />log(Avg_Depth, 10): 2.5424893<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />log(Avg_Depth, 10): 2.5424893<br />log(Avg_Depth + Std_Dev, 10): 2.682795","Bam_File: piMDA<br />log(Avg_Depth, 10): 2.5424893<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />log(Avg_Depth, 10): 2.5424893<br />log(Avg_Depth + Std_Dev, 10): 2.682795","Bam_File: piMDA+D<br />log(Avg_Depth, 10): 2.6493933<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />log(Avg_Depth, 10): 2.6493933<br />log(Avg_Depth + Std_Dev, 10): 2.760091","Bam_File: piMDA+D<br />log(Avg_Depth, 10): 2.6493933<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />log(Avg_Depth, 10): 2.6493933<br />log(Avg_Depth + Std_Dev, 10): 2.760091","Bam_File: piMDA+D<br />log(Avg_Depth, 10): 2.3358188<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />log(Avg_Depth, 10): 2.3358188<br />log(Avg_Depth + Std_Dev, 10): 2.458139","Bam_File: piMDA+D<br />log(Avg_Depth, 10): 2.3358188<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />log(Avg_Depth, 10): 2.3358188<br />log(Avg_Depth + Std_Dev, 10): 2.458139","Bam_File: piPolB<br />log(Avg_Depth, 10): 0.8195439<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />log(Avg_Depth, 10): 0.8195439<br />log(Avg_Depth + Std_Dev, 10): 2.188760","Bam_File: piPolB+D<br />log(Avg_Depth, 10): 1.8659327<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />log(Avg_Depth, 10): 1.8659327<br />log(Avg_Depth + Std_Dev, 10): 3.446247","Bam_File: PrimPol-MDA<br />log(Avg_Depth, 10): 2.6698002<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />log(Avg_Depth, 10): 2.6698002<br />log(Avg_Depth + Std_Dev, 10): 2.787843","Bam_File: RP-MDA<br />log(Avg_Depth, 10): 2.4883109<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />log(Avg_Depth, 10): 2.4883109<br />log(Avg_Depth + Std_Dev, 10): 2.660144","Bam_File: RP-MDA<br />log(Avg_Depth, 10): 2.4883109<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />log(Avg_Depth, 10): 2.4883109<br />log(Avg_Depth + Std_Dev, 10): 2.660144","Bam_File: RP-MDA<br />log(Avg_Depth, 10): 2.6267611<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />log(Avg_Depth, 10): 2.6267611<br />log(Avg_Depth + Std_Dev, 10): 2.744778","Bam_File: RP-MDA<br />log(Avg_Depth, 10): 2.6267611<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />Ref_Name: E. coli<br />log(Avg_Depth, 10): 2.6267611<br />log(Avg_Depth + Std_Dev, 10): 2.744778"],"type":"scatter","mode":"lines+markers","opacity":0.7,"line":{"color":"transparent"},"error_y":{"array":[0.0575699109761403,0.0575699109761403,0.061855419582014,0.061855419582014,0.137008471086143,0.137008471086143,0.140305367890363,0.140305367890363,0.110697323869136,0.110697323869136,0.122320241263107,0.122320241263107,1.36921585741014,1.58031394008792,0.118042687744409,0.171833273749234,0.171833273749234,0.118016759263178,0.118016759263178],"arrayminus":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"type":"data","width":0,"symmetric":false,"color":"rgba(255,215,0,1)"},"name":"E. coli","legendgroup":"E. coli","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","marker":{"autocolorscale":false,"color":"rgba(255,215,0,1)","opacity":0.7,"size":7.55905511811024,"symbol":"square","line":{"width":3.77952755905512,"color":"rgba(255,215,0,1)"}},"hoveron":"points","frame":null},{"x":[1.0625,1.0625,1.0625,1.0625,6.0625,6.0625,6.0625,6.0625,7.0625,7.0625,7.0625,7.0625,4.0625,5.0625,3.0625,2.0625,2.0625,2.0625,2.0625],"y":[2.85322960225492,2.85322960225492,2.75894215576734,2.75894215576734,3.04018683247993,3.04018683247993,2.96136888066229,2.96136888066229,3.13958631102151,3.13958631102151,3.15119818265398,3.15119818265398,0.998259338423699,1.82039852270398,1.72411195396121,1.11627558758054,1.11627558758054,0.741939077729199,0.741939077729199],"text":["Bam_File: NA<br />log(Avg_Depth, 10): 2.8532296<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />log(Avg_Depth, 10): 2.8532296<br />log(Avg_Depth + Std_Dev, 10): 2.919486","Bam_File: NA<br />log(Avg_Depth, 10): 2.8532296<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />log(Avg_Depth, 10): 2.8532296<br />log(Avg_Depth + Std_Dev, 10): 2.919486","Bam_File: NA<br />log(Avg_Depth, 10): 2.7589422<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />log(Avg_Depth, 10): 2.7589422<br />log(Avg_Depth + Std_Dev, 10): 2.836773","Bam_File: NA<br />log(Avg_Depth, 10): 2.7589422<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />log(Avg_Depth, 10): 2.7589422<br />log(Avg_Depth + Std_Dev, 10): 2.836773","Bam_File: piMDA<br />log(Avg_Depth, 10): 3.0401868<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />log(Avg_Depth, 10): 3.0401868<br />log(Avg_Depth + Std_Dev, 10): 3.213207","Bam_File: piMDA<br />log(Avg_Depth, 10): 3.0401868<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />log(Avg_Depth, 10): 3.0401868<br />log(Avg_Depth + Std_Dev, 10): 3.213207","Bam_File: piMDA<br />log(Avg_Depth, 10): 2.9613689<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />log(Avg_Depth, 10): 2.9613689<br />log(Avg_Depth + Std_Dev, 10): 3.129439","Bam_File: piMDA<br />log(Avg_Depth, 10): 2.9613689<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />log(Avg_Depth, 10): 2.9613689<br />log(Avg_Depth + Std_Dev, 10): 3.129439","Bam_File: piMDA+D<br />log(Avg_Depth, 10): 3.1395863<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />log(Avg_Depth, 10): 3.1395863<br />log(Avg_Depth + Std_Dev, 10): 3.286737","Bam_File: piMDA+D<br />log(Avg_Depth, 10): 3.1395863<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />log(Avg_Depth, 10): 3.1395863<br />log(Avg_Depth + Std_Dev, 10): 3.286737","Bam_File: piMDA+D<br />log(Avg_Depth, 10): 3.1511982<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />log(Avg_Depth, 10): 3.1511982<br />log(Avg_Depth + Std_Dev, 10): 3.287114","Bam_File: piMDA+D<br />log(Avg_Depth, 10): 3.1511982<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />log(Avg_Depth, 10): 3.1511982<br />log(Avg_Depth + Std_Dev, 10): 3.287114","Bam_File: piPolB<br />log(Avg_Depth, 10): 0.9982593<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />log(Avg_Depth, 10): 0.9982593<br />log(Avg_Depth + Std_Dev, 10): 1.251395","Bam_File: piPolB+D<br />log(Avg_Depth, 10): 1.8203985<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />log(Avg_Depth, 10): 1.8203985<br />log(Avg_Depth + Std_Dev, 10): 2.123590","Bam_File: PrimPol-MDA<br />log(Avg_Depth, 10): 1.7241120<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />log(Avg_Depth, 10): 1.7241120<br />log(Avg_Depth + Std_Dev, 10): 2.148263","Bam_File: RP-MDA<br />log(Avg_Depth, 10): 1.1162756<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />log(Avg_Depth, 10): 1.1162756<br />log(Avg_Depth + Std_Dev, 10): 2.101266","Bam_File: RP-MDA<br />log(Avg_Depth, 10): 1.1162756<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />log(Avg_Depth, 10): 1.1162756<br />log(Avg_Depth + Std_Dev, 10): 2.101266","Bam_File: RP-MDA<br />log(Avg_Depth, 10): 0.7419391<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />log(Avg_Depth, 10): 0.7419391<br />log(Avg_Depth + Std_Dev, 10): 1.869408","Bam_File: RP-MDA<br />log(Avg_Depth, 10): 0.7419391<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />Ref_Name: K. rhizophila<br />log(Avg_Depth, 10): 0.7419391<br />log(Avg_Depth + Std_Dev, 10): 1.869408"],"type":"scatter","mode":"lines+markers","opacity":0.7,"line":{"color":"transparent"},"error_y":{"array":[0.0662564306319888,0.0662564306319888,0.0778312159507304,0.0778312159507304,0.173020033729443,0.173020033729443,0.168069640810063,0.168069640810063,0.147150765107205,0.147150765107205,0.135915743091943,0.135915743091943,0.253135511616405,0.303191809902692,0.424151275675667,0.984990197410791,0.984990197410791,1.127468671653,1.127468671653],"arrayminus":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"type":"data","width":0,"symmetric":false,"color":"rgba(135,206,255,1)"},"name":"K. rhizophila","legendgroup":"K. rhizophila","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","marker":{"autocolorscale":false,"color":"rgba(135,206,255,1)","opacity":0.7,"size":7.55905511811024,"symbol":"diamond","line":{"width":3.77952755905512,"color":"rgba(135,206,255,1)"}},"hoveron":"points","frame":null},{"x":[1.1875,1.1875,1.1875,1.1875,6.1875,6.1875,6.1875,6.1875,7.1875,7.1875,7.1875,7.1875,4.1875,5.1875,3.1875,2.1875,2.1875,2.1875,2.1875],"y":[2.71345694067357,2.71345694067357,2.63122047613855,2.63122047613855,2.84345080523877,2.84345080523877,2.81949129059582,2.81949129059582,2.7992440288438,2.7992440288438,2.72302855634734,2.72302855634734,0.706717782336759,1.55702572238638,1.84825071467704,0.976349979003273,0.976349979003273,1.40891802084678,1.40891802084678],"text":["Bam_File: NA<br />log(Avg_Depth, 10): 2.7134569<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />log(Avg_Depth, 10): 2.7134569<br />log(Avg_Depth + Std_Dev, 10): 2.807494","Bam_File: NA<br />log(Avg_Depth, 10): 2.7134569<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />log(Avg_Depth, 10): 2.7134569<br />log(Avg_Depth + Std_Dev, 10): 2.807494","Bam_File: NA<br />log(Avg_Depth, 10): 2.6312205<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />log(Avg_Depth, 10): 2.6312205<br />log(Avg_Depth + Std_Dev, 10): 2.731282","Bam_File: NA<br />log(Avg_Depth, 10): 2.6312205<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />log(Avg_Depth, 10): 2.6312205<br />log(Avg_Depth + Std_Dev, 10): 2.731282","Bam_File: piMDA<br />log(Avg_Depth, 10): 2.8434508<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />log(Avg_Depth, 10): 2.8434508<br />log(Avg_Depth + Std_Dev, 10): 3.000347","Bam_File: piMDA<br />log(Avg_Depth, 10): 2.8434508<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />log(Avg_Depth, 10): 2.8434508<br />log(Avg_Depth + Std_Dev, 10): 3.000347","Bam_File: piMDA<br />log(Avg_Depth, 10): 2.8194913<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />log(Avg_Depth, 10): 2.8194913<br />log(Avg_Depth + Std_Dev, 10): 2.973696","Bam_File: piMDA<br />log(Avg_Depth, 10): 2.8194913<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />log(Avg_Depth, 10): 2.8194913<br />log(Avg_Depth + Std_Dev, 10): 2.973696","Bam_File: piMDA+D<br />log(Avg_Depth, 10): 2.7992440<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />log(Avg_Depth, 10): 2.7992440<br />log(Avg_Depth + Std_Dev, 10): 2.932621","Bam_File: piMDA+D<br />log(Avg_Depth, 10): 2.7992440<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />log(Avg_Depth, 10): 2.7992440<br />log(Avg_Depth + Std_Dev, 10): 2.932621","Bam_File: piMDA+D<br />log(Avg_Depth, 10): 2.7230286<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />log(Avg_Depth, 10): 2.7230286<br />log(Avg_Depth + Std_Dev, 10): 2.852523","Bam_File: piMDA+D<br />log(Avg_Depth, 10): 2.7230286<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />log(Avg_Depth, 10): 2.7230286<br />log(Avg_Depth + Std_Dev, 10): 2.852523","Bam_File: piPolB<br />log(Avg_Depth, 10): 0.7067178<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />log(Avg_Depth, 10): 0.7067178<br />log(Avg_Depth + Std_Dev, 10): 1.000000","Bam_File: piPolB+D<br />log(Avg_Depth, 10): 1.5570257<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />log(Avg_Depth, 10): 1.5570257<br />log(Avg_Depth + Std_Dev, 10): 1.822430","Bam_File: PrimPol-MDA<br />log(Avg_Depth, 10): 1.8482507<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />log(Avg_Depth, 10): 1.8482507<br />log(Avg_Depth + Std_Dev, 10): 2.174641","Bam_File: RP-MDA<br />log(Avg_Depth, 10): 0.9763500<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />log(Avg_Depth, 10): 0.9763500<br />log(Avg_Depth + Std_Dev, 10): 1.848928","Bam_File: RP-MDA<br />log(Avg_Depth, 10): 0.9763500<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />log(Avg_Depth, 10): 0.9763500<br />log(Avg_Depth + Std_Dev, 10): 1.848928","Bam_File: RP-MDA<br />log(Avg_Depth, 10): 1.4089180<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />log(Avg_Depth, 10): 1.4089180<br />log(Avg_Depth + Std_Dev, 10): 2.033745","Bam_File: RP-MDA<br />log(Avg_Depth, 10): 1.4089180<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />Ref_Name: P. aeruginosa<br />log(Avg_Depth, 10): 1.4089180<br />log(Avg_Depth + Std_Dev, 10): 2.033745"],"type":"scatter","mode":"lines+markers","opacity":0.7,"line":{"color":"transparent"},"error_y":{"array":[0.0940374972291815,0.0940374972291815,0.100061999428542,0.100061999428542,0.156896491446595,0.156896491446595,0.154204470478096,0.154204470478096,0.13337684339218,0.13337684339218,0.12949413268196,0.12949413268196,0.293282217663241,0.265403901392974,0.326390477983406,0.872577734223805,0.872577734223805,0.624827315167194,0.624827315167194],"arrayminus":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"type":"data","width":0,"symmetric":false,"color":"rgba(122,103,238,1)"},"name":"P. aeruginosa","legendgroup":"P. aeruginosa","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","marker":{"autocolorscale":false,"color":"rgba(122,103,238,1)","opacity":0.7,"size":7.55905511811024,"symbol":"triangle-up","line":{"width":3.77952755905512,"color":"rgba(122,103,238,1)"}},"hoveron":"points","frame":null},{"x":[0.9375,0.9375,0.9375,0.9375,5.9375,5.9375,5.9375,5.9375,6.9375,6.9375,6.9375,6.9375,3.9375,4.9375,2.9375,1.9375,1.9375,1.9375,1.9375],"y":[0.014,0.014,0.015,0.015,0.037,0.037,0.038,0.038,0.029,0.029,0.033,0.033,2.239,3.704,0.031,0.049,0.049,0.031,0.031],"text":["Bam_File: NA<br />Variation_Coefficient/10:  0.014<br />Ref_Name: E. coli<br />Ref_Name: black<br />Ref_Name: E. coli<br />Ref_Name: 21","Bam_File: NA<br />Variation_Coefficient/10:  0.014<br />Ref_Name: E. coli<br />Ref_Name: black<br />Ref_Name: E. coli<br />Ref_Name: 21","Bam_File: NA<br />Variation_Coefficient/10:  0.015<br />Ref_Name: E. coli<br />Ref_Name: black<br />Ref_Name: E. coli<br />Ref_Name: 21","Bam_File: NA<br />Variation_Coefficient/10:  0.015<br />Ref_Name: E. coli<br />Ref_Name: black<br />Ref_Name: E. coli<br />Ref_Name: 21","Bam_File: piMDA<br />Variation_Coefficient/10:  0.037<br />Ref_Name: E. coli<br />Ref_Name: black<br />Ref_Name: E. coli<br />Ref_Name: 21","Bam_File: piMDA<br />Variation_Coefficient/10:  0.037<br />Ref_Name: E. coli<br />Ref_Name: black<br />Ref_Name: E. coli<br />Ref_Name: 21","Bam_File: piMDA<br />Variation_Coefficient/10:  0.038<br />Ref_Name: E. coli<br />Ref_Name: black<br />Ref_Name: E. coli<br />Ref_Name: 21","Bam_File: piMDA<br />Variation_Coefficient/10:  0.038<br />Ref_Name: E. coli<br />Ref_Name: black<br />Ref_Name: E. coli<br />Ref_Name: 21","Bam_File: piMDA+D<br />Variation_Coefficient/10:  0.029<br />Ref_Name: E. coli<br />Ref_Name: black<br />Ref_Name: E. coli<br />Ref_Name: 21","Bam_File: piMDA+D<br />Variation_Coefficient/10:  0.029<br />Ref_Name: E. coli<br />Ref_Name: black<br />Ref_Name: E. coli<br />Ref_Name: 21","Bam_File: piMDA+D<br />Variation_Coefficient/10:  0.033<br />Ref_Name: E. coli<br />Ref_Name: black<br />Ref_Name: E. coli<br />Ref_Name: 21","Bam_File: piMDA+D<br />Variation_Coefficient/10:  0.033<br />Ref_Name: E. coli<br />Ref_Name: black<br />Ref_Name: E. coli<br />Ref_Name: 21","Bam_File: piPolB<br />Variation_Coefficient/10:  2.239<br />Ref_Name: E. coli<br />Ref_Name: black<br />Ref_Name: E. coli<br />Ref_Name: 21","Bam_File: piPolB+D<br />Variation_Coefficient/10:  3.704<br />Ref_Name: E. coli<br />Ref_Name: black<br />Ref_Name: E. coli<br />Ref_Name: 21","Bam_File: PrimPol-MDA<br />Variation_Coefficient/10:  0.031<br />Ref_Name: E. coli<br />Ref_Name: black<br />Ref_Name: E. coli<br />Ref_Name: 21","Bam_File: RP-MDA<br />Variation_Coefficient/10:  0.049<br />Ref_Name: E. coli<br />Ref_Name: black<br />Ref_Name: E. coli<br />Ref_Name: 21","Bam_File: RP-MDA<br />Variation_Coefficient/10:  0.049<br />Ref_Name: E. coli<br />Ref_Name: black<br />Ref_Name: E. coli<br />Ref_Name: 21","Bam_File: RP-MDA<br />Variation_Coefficient/10:  0.031<br />Ref_Name: E. coli<br />Ref_Name: black<br />Ref_Name: E. coli<br />Ref_Name: 21","Bam_File: RP-MDA<br />Variation_Coefficient/10:  0.031<br />Ref_Name: E. coli<br />Ref_Name: black<br />Ref_Name: E. coli<br />Ref_Name: 21"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(255,215,0,1)","opacity":1,"size":7.55905511811024,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0.8125,0.8125,0.8125,0.8125,5.8125,5.8125,5.8125,5.8125,6.8125,6.8125,6.8125,6.8125,3.8125,4.8125,2.8125,1.8125,1.8125,1.8125,1.8125],"y":[0.03,0.03,0.023,0.023,0.114,0.114,0.052,0.052,0.038,0.038,0.045,0.045,null,3.283,0.031,0.037,0.037,0.03,0.03],"text":["Bam_File: NA<br />Variation_Coefficient/10:  0.030<br />Ref_Name: B. subtilis<br />Ref_Name: black<br />Ref_Name: B. subtilis<br />Ref_Name: 21","Bam_File: NA<br />Variation_Coefficient/10:  0.030<br />Ref_Name: B. subtilis<br />Ref_Name: black<br />Ref_Name: B. subtilis<br />Ref_Name: 21","Bam_File: NA<br />Variation_Coefficient/10:  0.023<br />Ref_Name: B. subtilis<br />Ref_Name: black<br />Ref_Name: B. subtilis<br />Ref_Name: 21","Bam_File: NA<br />Variation_Coefficient/10:  0.023<br />Ref_Name: B. subtilis<br />Ref_Name: black<br />Ref_Name: B. subtilis<br />Ref_Name: 21","Bam_File: piMDA<br />Variation_Coefficient/10:  0.114<br />Ref_Name: B. subtilis<br />Ref_Name: black<br />Ref_Name: B. subtilis<br />Ref_Name: 21","Bam_File: piMDA<br />Variation_Coefficient/10:  0.114<br />Ref_Name: B. subtilis<br />Ref_Name: black<br />Ref_Name: B. subtilis<br />Ref_Name: 21","Bam_File: piMDA<br />Variation_Coefficient/10:  0.052<br />Ref_Name: B. subtilis<br />Ref_Name: black<br />Ref_Name: B. subtilis<br />Ref_Name: 21","Bam_File: piMDA<br />Variation_Coefficient/10:  0.052<br />Ref_Name: B. subtilis<br />Ref_Name: black<br />Ref_Name: B. subtilis<br />Ref_Name: 21","Bam_File: piMDA+D<br />Variation_Coefficient/10:  0.038<br />Ref_Name: B. subtilis<br />Ref_Name: black<br />Ref_Name: B. subtilis<br />Ref_Name: 21","Bam_File: piMDA+D<br />Variation_Coefficient/10:  0.038<br />Ref_Name: B. subtilis<br />Ref_Name: black<br />Ref_Name: B. subtilis<br />Ref_Name: 21","Bam_File: piMDA+D<br />Variation_Coefficient/10:  0.045<br />Ref_Name: B. subtilis<br />Ref_Name: black<br />Ref_Name: B. subtilis<br />Ref_Name: 21","Bam_File: piMDA+D<br />Variation_Coefficient/10:  0.045<br />Ref_Name: B. subtilis<br />Ref_Name: black<br />Ref_Name: B. subtilis<br />Ref_Name: 21","Bam_File: piPolB<br />Variation_Coefficient/10: 10.296<br />Ref_Name: B. subtilis<br />Ref_Name: black<br />Ref_Name: B. subtilis<br />Ref_Name: 21","Bam_File: piPolB+D<br />Variation_Coefficient/10:  3.283<br />Ref_Name: B. subtilis<br />Ref_Name: black<br />Ref_Name: B. subtilis<br />Ref_Name: 21","Bam_File: PrimPol-MDA<br />Variation_Coefficient/10:  0.031<br />Ref_Name: B. subtilis<br />Ref_Name: black<br />Ref_Name: B. subtilis<br />Ref_Name: 21","Bam_File: RP-MDA<br />Variation_Coefficient/10:  0.037<br />Ref_Name: B. subtilis<br />Ref_Name: black<br />Ref_Name: B. subtilis<br />Ref_Name: 21","Bam_File: RP-MDA<br />Variation_Coefficient/10:  0.037<br />Ref_Name: B. subtilis<br />Ref_Name: black<br />Ref_Name: B. subtilis<br />Ref_Name: 21","Bam_File: RP-MDA<br />Variation_Coefficient/10:  0.030<br />Ref_Name: B. subtilis<br />Ref_Name: black<br />Ref_Name: B. subtilis<br />Ref_Name: 21","Bam_File: RP-MDA<br />Variation_Coefficient/10:  0.030<br />Ref_Name: B. subtilis<br />Ref_Name: black<br />Ref_Name: B. subtilis<br />Ref_Name: 21"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(238,154,0,1)","opacity":1,"size":7.55905511811024,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.0625,1.0625,1.0625,1.0625,6.0625,6.0625,6.0625,6.0625,7.0625,7.0625,7.0625,7.0625,4.0625,5.0625,3.0625,2.0625,2.0625,2.0625,2.0625],"y":[0.016,0.016,0.02,0.02,0.049,0.049,0.047,0.047,0.04,0.04,0.037,0.037,0.079,0.101,0.166,0.866,0.866,1.241,1.241],"text":["Bam_File: NA<br />Variation_Coefficient/10:  0.016<br />Ref_Name: K. rhizophila<br />Ref_Name: black<br />Ref_Name: K. rhizophila<br />Ref_Name: 21","Bam_File: NA<br />Variation_Coefficient/10:  0.016<br />Ref_Name: K. rhizophila<br />Ref_Name: black<br />Ref_Name: K. rhizophila<br />Ref_Name: 21","Bam_File: NA<br />Variation_Coefficient/10:  0.020<br />Ref_Name: K. rhizophila<br />Ref_Name: black<br />Ref_Name: K. rhizophila<br />Ref_Name: 21","Bam_File: NA<br />Variation_Coefficient/10:  0.020<br />Ref_Name: K. rhizophila<br />Ref_Name: black<br />Ref_Name: K. rhizophila<br />Ref_Name: 21","Bam_File: piMDA<br />Variation_Coefficient/10:  0.049<br />Ref_Name: K. rhizophila<br />Ref_Name: black<br />Ref_Name: K. rhizophila<br />Ref_Name: 21","Bam_File: piMDA<br />Variation_Coefficient/10:  0.049<br />Ref_Name: K. rhizophila<br />Ref_Name: black<br />Ref_Name: K. rhizophila<br />Ref_Name: 21","Bam_File: piMDA<br />Variation_Coefficient/10:  0.047<br />Ref_Name: K. rhizophila<br />Ref_Name: black<br />Ref_Name: K. rhizophila<br />Ref_Name: 21","Bam_File: piMDA<br />Variation_Coefficient/10:  0.047<br />Ref_Name: K. rhizophila<br />Ref_Name: black<br />Ref_Name: K. rhizophila<br />Ref_Name: 21","Bam_File: piMDA+D<br />Variation_Coefficient/10:  0.040<br />Ref_Name: K. rhizophila<br />Ref_Name: black<br />Ref_Name: K. rhizophila<br />Ref_Name: 21","Bam_File: piMDA+D<br />Variation_Coefficient/10:  0.040<br />Ref_Name: K. rhizophila<br />Ref_Name: black<br />Ref_Name: K. rhizophila<br />Ref_Name: 21","Bam_File: piMDA+D<br />Variation_Coefficient/10:  0.037<br />Ref_Name: K. rhizophila<br />Ref_Name: black<br />Ref_Name: K. rhizophila<br />Ref_Name: 21","Bam_File: piMDA+D<br />Variation_Coefficient/10:  0.037<br />Ref_Name: K. rhizophila<br />Ref_Name: black<br />Ref_Name: K. rhizophila<br />Ref_Name: 21","Bam_File: piPolB<br />Variation_Coefficient/10:  0.079<br />Ref_Name: K. rhizophila<br />Ref_Name: black<br />Ref_Name: K. rhizophila<br />Ref_Name: 21","Bam_File: piPolB+D<br />Variation_Coefficient/10:  0.101<br />Ref_Name: K. rhizophila<br />Ref_Name: black<br />Ref_Name: K. rhizophila<br />Ref_Name: 21","Bam_File: PrimPol-MDA<br />Variation_Coefficient/10:  0.166<br />Ref_Name: K. rhizophila<br />Ref_Name: black<br />Ref_Name: K. rhizophila<br />Ref_Name: 21","Bam_File: RP-MDA<br />Variation_Coefficient/10:  0.866<br />Ref_Name: K. rhizophila<br />Ref_Name: black<br />Ref_Name: K. rhizophila<br />Ref_Name: 21","Bam_File: RP-MDA<br />Variation_Coefficient/10:  0.866<br />Ref_Name: K. rhizophila<br />Ref_Name: black<br />Ref_Name: K. rhizophila<br />Ref_Name: 21","Bam_File: RP-MDA<br />Variation_Coefficient/10:  1.241<br />Ref_Name: K. rhizophila<br />Ref_Name: black<br />Ref_Name: K. rhizophila<br />Ref_Name: 21","Bam_File: RP-MDA<br />Variation_Coefficient/10:  1.241<br />Ref_Name: K. rhizophila<br />Ref_Name: black<br />Ref_Name: K. rhizophila<br />Ref_Name: 21"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(135,206,255,1)","opacity":1,"size":7.55905511811024,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.1875,1.1875,1.1875,1.1875,6.1875,6.1875,6.1875,6.1875,7.1875,7.1875,7.1875,7.1875,4.1875,5.1875,3.1875,2.1875,2.1875,2.1875,2.1875],"y":[0.024,0.024,0.026,0.026,0.044,0.044,0.043,0.043,0.036,0.036,0.035,0.035,0.096,0.084,0.112,0.646,0.646,0.322,0.322],"text":["Bam_File: NA<br />Variation_Coefficient/10:  0.024<br />Ref_Name: P. aeruginosa<br />Ref_Name: black<br />Ref_Name: P. aeruginosa<br />Ref_Name: 21","Bam_File: NA<br />Variation_Coefficient/10:  0.024<br />Ref_Name: P. aeruginosa<br />Ref_Name: black<br />Ref_Name: P. aeruginosa<br />Ref_Name: 21","Bam_File: NA<br />Variation_Coefficient/10:  0.026<br />Ref_Name: P. aeruginosa<br />Ref_Name: black<br />Ref_Name: P. aeruginosa<br />Ref_Name: 21","Bam_File: NA<br />Variation_Coefficient/10:  0.026<br />Ref_Name: P. aeruginosa<br />Ref_Name: black<br />Ref_Name: P. aeruginosa<br />Ref_Name: 21","Bam_File: piMDA<br />Variation_Coefficient/10:  0.044<br />Ref_Name: P. aeruginosa<br />Ref_Name: black<br />Ref_Name: P. aeruginosa<br />Ref_Name: 21","Bam_File: piMDA<br />Variation_Coefficient/10:  0.044<br />Ref_Name: P. aeruginosa<br />Ref_Name: black<br />Ref_Name: P. aeruginosa<br />Ref_Name: 21","Bam_File: piMDA<br />Variation_Coefficient/10:  0.043<br />Ref_Name: P. aeruginosa<br />Ref_Name: black<br />Ref_Name: P. aeruginosa<br />Ref_Name: 21","Bam_File: piMDA<br />Variation_Coefficient/10:  0.043<br />Ref_Name: P. aeruginosa<br />Ref_Name: black<br />Ref_Name: P. aeruginosa<br />Ref_Name: 21","Bam_File: piMDA+D<br />Variation_Coefficient/10:  0.036<br />Ref_Name: P. aeruginosa<br />Ref_Name: black<br />Ref_Name: P. aeruginosa<br />Ref_Name: 21","Bam_File: piMDA+D<br />Variation_Coefficient/10:  0.036<br />Ref_Name: P. aeruginosa<br />Ref_Name: black<br />Ref_Name: P. aeruginosa<br />Ref_Name: 21","Bam_File: piMDA+D<br />Variation_Coefficient/10:  0.035<br />Ref_Name: P. aeruginosa<br />Ref_Name: black<br />Ref_Name: P. aeruginosa<br />Ref_Name: 21","Bam_File: piMDA+D<br />Variation_Coefficient/10:  0.035<br />Ref_Name: P. aeruginosa<br />Ref_Name: black<br />Ref_Name: P. aeruginosa<br />Ref_Name: 21","Bam_File: piPolB<br />Variation_Coefficient/10:  0.096<br />Ref_Name: P. aeruginosa<br />Ref_Name: black<br />Ref_Name: P. aeruginosa<br />Ref_Name: 21","Bam_File: piPolB+D<br />Variation_Coefficient/10:  0.084<br />Ref_Name: P. aeruginosa<br />Ref_Name: black<br />Ref_Name: P. aeruginosa<br />Ref_Name: 21","Bam_File: PrimPol-MDA<br />Variation_Coefficient/10:  0.112<br />Ref_Name: P. aeruginosa<br />Ref_Name: black<br />Ref_Name: P. aeruginosa<br />Ref_Name: 21","Bam_File: RP-MDA<br />Variation_Coefficient/10:  0.646<br />Ref_Name: P. aeruginosa<br />Ref_Name: black<br />Ref_Name: P. aeruginosa<br />Ref_Name: 21","Bam_File: RP-MDA<br />Variation_Coefficient/10:  0.646<br />Ref_Name: P. aeruginosa<br />Ref_Name: black<br />Ref_Name: P. aeruginosa<br />Ref_Name: 21","Bam_File: RP-MDA<br />Variation_Coefficient/10:  0.322<br />Ref_Name: P. aeruginosa<br />Ref_Name: black<br />Ref_Name: P. aeruginosa<br />Ref_Name: 21","Bam_File: RP-MDA<br />Variation_Coefficient/10:  0.322<br />Ref_Name: P. aeruginosa<br />Ref_Name: black<br />Ref_Name: P. aeruginosa<br />Ref_Name: 21"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(122,103,238,1)","opacity":1,"size":7.55905511811024,"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)"}},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0.4,7.6],"y":[1,1],"text":"yintercept: 1","type":"scatter","mode":"lines","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)","dash":"dash"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":50.3473877720453,"r":7.30593607305936,"b":96.4295491108556,"l":38.854296388543},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":19.9252801992528},"title":{"text":"Coverage Depth","font":{"color":"rgba(0,0,0,1)","family":"","size":23.9103362391034},"x":0.5,"xref":"paper"},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,7.6],"tickmode":"array","ticktext":["NA","RP-MDA","PrimPol-MDA","piPolB","piPolB+D","piMDA","piMDA+D"],"tickvals":[1,2,3,4,5,6,7],"categoryorder":"array","categoryarray":["NA","RP-MDA","PrimPol-MDA","piPolB","piPolB+D","piMDA","piMDA+D"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":21.2536322125363},"tickangle":-45,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"y","title":{"text":"Sample","font":{"color":"rgba(0,0,0,1)","family":"","size":19.9252801992528}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-0.2,4.2],"tickmode":"array","ticktext":["0","1","2","3","4"],"tickvals":[0,1,2,3,4],"categoryorder":"array","categoryarray":["0","1","2","3","4"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":15.9402241594022},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":{"text":"Average Depth (Log10)","font":{"color":"rgba(0,0,0,1)","family":"","size":19.9252801992528}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":true,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":15.9402241594022},"title":{"text":"Reference genome","font":{"color":"rgba(0,0,0,1)","family":"","size":19.9252801992528}}},"hovermode":"closest","barmode":"relative"},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"1701d7b2dc8b1":{"x":{},"y":{},"colour":{},"fill":{},"shape":{},"ymin":{},"ymax":{},"type":"scatter"},"1701d7ad31ba4":{"x":{},"y":{},"colour":{},"fill":{},"shape":{}},"1701d39bbeafb":{"yintercept":{}}},"cur_data":"1701d7b2dc8b1","visdat":{"1701d7b2dc8b1":["function (y) ","x"],"1701d7ad31ba4":["function (y) ","x"],"1701d39bbeafb":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<p>Figure 10. Reference genomes coverage depth (left axis) and variation coefficient (small points, right axis) by MDA methods</p>
</div>
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb54"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb54-1"><a href="#cb54-1" aria-hidden="true" tabindex="-1"></a><span class="co"># ggsave('figures/depth.svg',pp,width=8,height=5)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<p>Alfred stats from Bowtie alignment files allowed us to analyze the mismatch rate in the raw reads mapping (Figure 11). Individual merged tables were re-merged using the script <em>merge_alfred_samples.R.</em>.</p>
<div class="cell" data-layout-align="left">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb55"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb55-1"><a href="#cb55-1" aria-hidden="true" tabindex="-1"></a>alfred <span class="ot">&lt;-</span> <span class="fu">fread</span>(<span class="st">"alfred/merged/merged_alfred.csv"</span>, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">na.strings =</span> <span class="st">""</span>)</span>
<span id="cb55-2"><a href="#cb55-2" aria-hidden="true" tabindex="-1"></a><span class="co"># order samples</span></span>
<span id="cb55-3"><a href="#cb55-3" aria-hidden="true" tabindex="-1"></a>alfred<span class="sc">$</span>sample <span class="ot">&lt;-</span> <span class="fu">factor</span>(alfred<span class="sc">$</span>sample, <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">"NA"</span>, <span class="st">"RP-MDA"</span>, <span class="st">"PrimPol-MDA"</span>, <span class="st">"piPolB"</span>,</span>
<span id="cb55-4"><a href="#cb55-4" aria-hidden="true" tabindex="-1"></a>    <span class="st">"piPolB+D"</span>, <span class="st">"piMDA"</span>, <span class="st">"piMDA+D"</span>))</span>
<span id="cb55-5"><a href="#cb55-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb55-6"><a href="#cb55-6" aria-hidden="true" tabindex="-1"></a><span class="co"># plot</span></span>
<span id="cb55-7"><a href="#cb55-7" aria-hidden="true" tabindex="-1"></a>g1 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(alfred, <span class="fu">aes</span>(<span class="at">x =</span> sample, <span class="at">y =</span> <span class="fu">as.numeric</span>(MismatchRate), <span class="at">group =</span> ref, <span class="at">color =</span> ref,</span>
<span id="cb55-8"><a href="#cb55-8" aria-hidden="true" tabindex="-1"></a>    <span class="at">fill =</span> ref, <span class="at">shape =</span> ref)) <span class="sc">+</span> <span class="fu">geom_point</span>(<span class="at">size =</span> <span class="dv">4</span>, <span class="at">alpha =</span> <span class="fl">0.7</span>, <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="at">width =</span> <span class="fl">0.5</span>)) <span class="sc">+</span></span>
<span id="cb55-9"><a href="#cb55-9" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme_bw</span>() <span class="sc">+</span> <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> colorines[<span class="dv">12</span><span class="sc">:</span><span class="dv">15</span>]) <span class="sc">+</span> <span class="fu">scale_color_manual</span>(<span class="at">values =</span> colorines[<span class="dv">12</span><span class="sc">:</span><span class="dv">15</span>]) <span class="sc">+</span></span>
<span id="cb55-10"><a href="#cb55-10" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_y_continuous</span>(<span class="at">breaks =</span> <span class="fu">seq</span>(<span class="dv">0</span>, <span class="fl">0.05</span>, <span class="fl">0.01</span>), <span class="at">minor_breaks =</span> <span class="fu">seq</span>(<span class="dv">0</span>, <span class="fl">0.05</span>, <span class="at">by =</span> <span class="fl">0.002</span>)) <span class="sc">+</span></span>
<span id="cb55-11"><a href="#cb55-11" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme</span>(<span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">15</span>)) <span class="sc">+</span> <span class="fu">ggtitle</span>(<span class="st">"Mismatch rate"</span>) <span class="sc">+</span> <span class="fu">xlab</span>(<span class="st">"Sample"</span>) <span class="sc">+</span> <span class="fu">ylab</span>(<span class="st">"Mismatch Rate"</span>) <span class="sc">+</span></span>
<span id="cb55-12"><a href="#cb55-12" aria-hidden="true" tabindex="-1"></a>    <span class="fu">labs</span>(<span class="at">color =</span> <span class="st">"Reference genome"</span>) <span class="sc">+</span> <span class="fu">scale_shape_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="dv">21</span>, <span class="dv">22</span>, <span class="dv">23</span>, <span class="dv">24</span>)) <span class="sc">+</span> <span class="fu">guides</span>(<span class="at">shape =</span> <span class="cn">FALSE</span>,</span>
<span id="cb55-13"><a href="#cb55-13" aria-hidden="true" tabindex="-1"></a>    <span class="at">fill =</span> <span class="cn">FALSE</span>, <span class="at">scale =</span> <span class="st">"none"</span>) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">45</span>, <span class="at">hjust =</span> <span class="dv">1</span>,</span>
<span id="cb55-14"><a href="#cb55-14" aria-hidden="true" tabindex="-1"></a>    <span class="at">size =</span> <span class="dv">16</span>)) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">legend.text =</span> <span class="fu">element_text</span>(<span class="at">face =</span> <span class="st">"italic"</span>)) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">hjust =</span> <span class="fl">0.5</span>)) <span class="sc">+</span></span>
<span id="cb55-15"><a href="#cb55-15" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme</span>(<span class="at">panel.grid.minor.y =</span> <span class="fu">element_line</span>(<span class="at">linetype =</span> <span class="dv">2</span>))</span>
<span id="cb55-16"><a href="#cb55-16" aria-hidden="true" tabindex="-1"></a>g2 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(alfred) <span class="sc">+</span> <span class="fu">geom_point</span>(<span class="fu">aes</span>(<span class="at">x =</span> sample, <span class="at">y =</span> <span class="fu">as.numeric</span>(InsertionRate), <span class="at">color =</span> ref,</span>
<span id="cb55-17"><a href="#cb55-17" aria-hidden="true" tabindex="-1"></a>    <span class="at">fill =</span> ref), <span class="at">size =</span> <span class="dv">4</span>, <span class="at">alpha =</span> <span class="fl">0.7</span>, <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="at">width =</span> <span class="fl">0.5</span>)) <span class="sc">+</span> <span class="fu">scale_shape_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="dv">22</span>,</span>
<span id="cb55-18"><a href="#cb55-18" aria-hidden="true" tabindex="-1"></a>    <span class="dv">23</span>, <span class="dv">24</span>, <span class="dv">25</span>)) <span class="sc">+</span> <span class="fu">geom_point</span>(<span class="fu">aes</span>(<span class="at">x =</span> sample, <span class="at">y =</span> <span class="fu">as.numeric</span>(HomopolymerContextIns)<span class="sc">/</span><span class="dv">500</span>, <span class="at">fill =</span> ref),</span>
<span id="cb55-19"><a href="#cb55-19" aria-hidden="true" tabindex="-1"></a>    <span class="at">shape =</span> <span class="dv">21</span>, <span class="at">color =</span> <span class="st">"black"</span>, <span class="at">size =</span> <span class="dv">2</span>, <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="at">width =</span> <span class="fl">0.5</span>)) <span class="sc">+</span> <span class="fu">scale_y_continuous</span>(<span class="at">name =</span> <span class="st">"Insertions"</span>,</span>
<span id="cb55-20"><a href="#cb55-20" aria-hidden="true" tabindex="-1"></a>    <span class="at">breaks =</span> <span class="fu">seq</span>(<span class="dv">0</span>, <span class="fl">0.002</span>, <span class="fl">5e-04</span>), <span class="at">sec.axis =</span> <span class="fu">sec_axis</span>(<span class="sc">~</span><span class="dv">500</span> <span class="sc">*</span> ., <span class="at">name =</span> <span class="st">"Homopolymeric context insertions"</span>,</span>
<span id="cb55-21"><a href="#cb55-21" aria-hidden="true" tabindex="-1"></a>        <span class="at">breaks =</span> <span class="fu">seq</span>(<span class="dv">0</span>, <span class="dv">1</span>, <span class="fl">0.2</span>))) <span class="sc">+</span> <span class="fu">theme_bw</span>() <span class="sc">+</span> <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> colorines[<span class="dv">12</span><span class="sc">:</span><span class="dv">15</span>]) <span class="sc">+</span></span>
<span id="cb55-22"><a href="#cb55-22" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_color_manual</span>(<span class="at">values =</span> colorines[<span class="dv">12</span><span class="sc">:</span><span class="dv">15</span>]) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">15</span>)) <span class="sc">+</span></span>
<span id="cb55-23"><a href="#cb55-23" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggtitle</span>(<span class="st">"Insertions rate"</span>) <span class="sc">+</span> <span class="fu">xlab</span>(<span class="st">"Sample"</span>) <span class="sc">+</span> <span class="fu">ylab</span>(<span class="st">"Insertions Rate"</span>) <span class="sc">+</span> <span class="fu">scale_shape_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="dv">21</span>,</span>
<span id="cb55-24"><a href="#cb55-24" aria-hidden="true" tabindex="-1"></a>    <span class="dv">22</span>, <span class="dv">23</span>, <span class="dv">24</span>)) <span class="sc">+</span> <span class="fu">guides</span>(<span class="at">shape =</span> <span class="cn">FALSE</span>, <span class="at">fill =</span> <span class="cn">FALSE</span>, <span class="at">color =</span> <span class="cn">FALSE</span>, <span class="at">scale =</span> <span class="st">"none"</span>) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">45</span>,</span>
<span id="cb55-25"><a href="#cb55-25" aria-hidden="true" tabindex="-1"></a>    <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">16</span>)) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">legend.text =</span> <span class="fu">element_text</span>(<span class="at">face =</span> <span class="st">"italic"</span>)) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">hjust =</span> <span class="fl">0.5</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stderr">
<pre><code>Scale for shape is already present.
Adding another scale for shape, which will replace the existing scale.</code></pre>
</div>
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb57"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb57-1"><a href="#cb57-1" aria-hidden="true" tabindex="-1"></a>g3 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(alfred) <span class="sc">+</span> <span class="fu">geom_point</span>(<span class="fu">aes</span>(<span class="at">x =</span> sample, <span class="at">y =</span> <span class="fu">as.numeric</span>(DeletionRate), <span class="at">color =</span> ref,</span>
<span id="cb57-2"><a href="#cb57-2" aria-hidden="true" tabindex="-1"></a>    <span class="at">fill =</span> ref), <span class="at">size =</span> <span class="dv">4</span>, <span class="at">alpha =</span> <span class="fl">0.7</span>, <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="at">width =</span> <span class="fl">0.5</span>)) <span class="sc">+</span> <span class="fu">scale_shape_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="dv">22</span>,</span>
<span id="cb57-3"><a href="#cb57-3" aria-hidden="true" tabindex="-1"></a>    <span class="dv">23</span>, <span class="dv">24</span>, <span class="dv">25</span>)) <span class="sc">+</span> <span class="fu">geom_point</span>(<span class="fu">aes</span>(<span class="at">x =</span> sample, <span class="at">y =</span> <span class="fu">as.numeric</span>(HomopolymerContextDel)<span class="sc">/</span><span class="dv">500</span>, <span class="at">fill =</span> ref),</span>
<span id="cb57-4"><a href="#cb57-4" aria-hidden="true" tabindex="-1"></a>    <span class="at">shape =</span> <span class="dv">21</span>, <span class="at">color =</span> <span class="st">"black"</span>, <span class="at">size =</span> <span class="dv">2</span>, <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="at">width =</span> <span class="fl">0.5</span>)) <span class="sc">+</span> <span class="fu">scale_y_continuous</span>(<span class="at">name =</span> <span class="st">"Deletions"</span>,</span>
<span id="cb57-5"><a href="#cb57-5" aria-hidden="true" tabindex="-1"></a>    <span class="at">breaks =</span> <span class="fu">seq</span>(<span class="dv">0</span>, <span class="fl">0.001</span>, <span class="fl">2e-04</span>), <span class="at">sec.axis =</span> <span class="fu">sec_axis</span>(<span class="sc">~</span><span class="dv">500</span> <span class="sc">*</span> ., <span class="at">name =</span> <span class="st">"Homopolymeric context deletions"</span>,</span>
<span id="cb57-6"><a href="#cb57-6" aria-hidden="true" tabindex="-1"></a>        <span class="at">breaks =</span> <span class="fu">seq</span>(<span class="dv">0</span>, <span class="fl">0.5</span>, <span class="fl">0.2</span>))) <span class="sc">+</span> <span class="fu">theme_bw</span>() <span class="sc">+</span> <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> colorines[<span class="dv">12</span><span class="sc">:</span><span class="dv">15</span>]) <span class="sc">+</span></span>
<span id="cb57-7"><a href="#cb57-7" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_color_manual</span>(<span class="at">values =</span> colorines[<span class="dv">12</span><span class="sc">:</span><span class="dv">15</span>]) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">15</span>)) <span class="sc">+</span></span>
<span id="cb57-8"><a href="#cb57-8" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggtitle</span>(<span class="st">"Deletion rate"</span>) <span class="sc">+</span> <span class="fu">xlab</span>(<span class="st">"Sample"</span>) <span class="sc">+</span> <span class="fu">ylab</span>(<span class="st">"Insertions Rate"</span>) <span class="sc">+</span> <span class="fu">scale_shape_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="dv">21</span>,</span>
<span id="cb57-9"><a href="#cb57-9" aria-hidden="true" tabindex="-1"></a>    <span class="dv">22</span>, <span class="dv">23</span>, <span class="dv">24</span>)) <span class="sc">+</span> <span class="fu">guides</span>(<span class="at">shape =</span> <span class="cn">FALSE</span>, <span class="at">fill =</span> <span class="cn">FALSE</span>, <span class="at">color =</span> <span class="cn">FALSE</span>, <span class="at">scale =</span> <span class="st">"none"</span>) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">45</span>,</span>
<span id="cb57-10"><a href="#cb57-10" aria-hidden="true" tabindex="-1"></a>    <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">16</span>)) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">legend.text =</span> <span class="fu">element_text</span>(<span class="at">face =</span> <span class="st">"italic"</span>)) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">hjust =</span> <span class="fl">0.5</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stderr">
<pre><code>Scale for shape is already present.
Adding another scale for shape, which will replace the existing scale.</code></pre>
</div>
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb59"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb59-1"><a href="#cb59-1" aria-hidden="true" tabindex="-1"></a>(grid3 <span class="ot">&lt;-</span> g1<span class="sc">/</span>(g2 <span class="sc">+</span> g3))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-left">
<figure class="figure">
<p><img src="Plotmissmtach-1.png" class="img-fluid figure-img" width="1152"></p>
<p></p><figcaption class="figure-caption">Figure 11. Reads Mismatch rate</figcaption><p></p>
</figure>
</div>
</div>
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb60"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb60-1"><a href="#cb60-1" aria-hidden="true" tabindex="-1"></a><span class="co"># ggsave('figures/mismtaches.svg',grid3,width=10,height=11)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<div class="cell">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb61"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb61-1"><a href="#cb61-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Table</span></span>
<span id="cb61-2"><a href="#cb61-2" aria-hidden="true" tabindex="-1"></a>mismas <span class="ot">&lt;-</span> alfred[, <span class="fu">c</span>(<span class="dv">18</span>, <span class="dv">37</span>)]</span>
<span id="cb61-3"><a href="#cb61-3" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(mismas) <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"MismatchRate"</span>, <span class="st">"Bam_File"</span>)</span>
<span id="cb61-4"><a href="#cb61-4" aria-hidden="true" tabindex="-1"></a>weesam <span class="ot">&lt;-</span> <span class="fu">merge</span>(weesam, mismas, <span class="at">by =</span> <span class="st">"Bam_File"</span>)</span>
<span id="cb61-5"><a href="#cb61-5" aria-hidden="true" tabindex="-1"></a>weesam<span class="sc">$</span>MismatchRate <span class="ot">&lt;-</span> weesam<span class="sc">$</span>MismatchRate <span class="sc">*</span> <span class="dv">100</span></span>
<span id="cb61-6"><a href="#cb61-6" aria-hidden="true" tabindex="-1"></a>tablita <span class="ot">&lt;-</span> <span class="fu">aggregate</span>(<span class="fu">cbind</span>(weesam<span class="sc">$</span>Mapped_Reads, weesam<span class="sc">$</span>Breadth, weesam<span class="sc">$</span>covered, weesam<span class="sc">$</span>Avg_Depth,</span>
<span id="cb61-7"><a href="#cb61-7" aria-hidden="true" tabindex="-1"></a>    weesam<span class="sc">$</span>Variation_Coefficient, weesam<span class="sc">$</span>MismatchRate), <span class="at">by =</span> <span class="fu">list</span>(<span class="at">Sample =</span> weesam<span class="sc">$</span>Bam_File,</span>
<span id="cb61-8"><a href="#cb61-8" aria-hidden="true" tabindex="-1"></a>    <span class="at">Experiment =</span> weesam<span class="sc">$</span>Experiment), <span class="at">FUN =</span> mean)</span>
<span id="cb61-9"><a href="#cb61-9" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(tablita) <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"Sample"</span>, <span class="st">"Experiment"</span>, <span class="st">"Mapped reads"</span>, <span class="st">"Breadth"</span>, <span class="st">"Reference covered (%)"</span>,</span>
<span id="cb61-10"><a href="#cb61-10" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Average Depth"</span>, <span class="st">"Variation coefficient"</span>, <span class="st">"Mismatch Rate (%)"</span>)</span>
<span id="cb61-11"><a href="#cb61-11" aria-hidden="true" tabindex="-1"></a>tablita <span class="ot">&lt;-</span> tablita[<span class="fu">order</span>(tablita<span class="sc">$</span>Sample), ]</span>
<span id="cb61-12"><a href="#cb61-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb61-13"><a href="#cb61-13" aria-hidden="true" tabindex="-1"></a>tablita[, <span class="dv">5</span><span class="sc">:</span><span class="dv">7</span>] <span class="ot">&lt;-</span> <span class="fu">round</span>(tablita[, <span class="dv">5</span><span class="sc">:</span><span class="dv">7</span>], <span class="at">digits =</span> <span class="dv">2</span>)</span>
<span id="cb61-14"><a href="#cb61-14" aria-hidden="true" tabindex="-1"></a>tablita[, <span class="dv">8</span>] <span class="ot">&lt;-</span> <span class="fu">round</span>(tablita[, <span class="dv">8</span>], <span class="at">digits =</span> <span class="dv">4</span>)</span>
<span id="cb61-15"><a href="#cb61-15" aria-hidden="true" tabindex="-1"></a>tablita <span class="ot">&lt;-</span> tablita <span class="sc">%&gt;%</span></span>
<span id="cb61-16"><a href="#cb61-16" aria-hidden="true" tabindex="-1"></a>    <span class="fu">mutate</span>(<span class="st">`</span><span class="at">Mapped reads</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">format</span>(<span class="st">`</span><span class="at">Mapped reads</span><span class="st">`</span>, <span class="at">scientific =</span> <span class="cn">FALSE</span>, <span class="at">big.mark =</span> <span class="st">","</span>))</span>
<span id="cb61-17"><a href="#cb61-17" aria-hidden="true" tabindex="-1"></a>tablita <span class="ot">&lt;-</span> tablita <span class="sc">%&gt;%</span></span>
<span id="cb61-18"><a href="#cb61-18" aria-hidden="true" tabindex="-1"></a>    <span class="fu">mutate</span>(<span class="at">Breadth =</span> <span class="fu">format</span>(Breadth, <span class="at">scientific =</span> <span class="cn">FALSE</span>, <span class="at">big.mark =</span> <span class="st">","</span>))</span>
<span id="cb61-19"><a href="#cb61-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb61-20"><a href="#cb61-20" aria-hidden="true" tabindex="-1"></a><span class="fu">kbl</span>(tablita, <span class="at">align =</span> <span class="st">"c"</span>, <span class="at">row.names =</span> F, <span class="at">caption =</span> <span class="st">"Table 5. Mean mapping and coverage parameters sample."</span>) <span class="sc">%&gt;%</span></span>
<span id="cb61-21"><a href="#cb61-21" aria-hidden="true" tabindex="-1"></a>    <span class="fu">kable_styling</span>(<span class="at">bootstrap_options =</span> <span class="st">"striped"</span>, <span class="at">full_width =</span> F, <span class="at">position =</span> <span class="st">"center"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb61-22"><a href="#cb61-22" aria-hidden="true" tabindex="-1"></a>    <span class="fu">column_spec</span>(<span class="dv">1</span>, <span class="at">italic =</span> T)  <span class="co">#  %&gt;% save_kable('figures/aligned2.pdf')</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">

<table class="table table-striped" style="width: auto !important; margin-left: auto; margin-right: auto;">
<caption>Table 5. Mean mapping and coverage parameters sample.</caption>
 <thead>
  <tr>
   <th style="text-align:center;"> Sample </th>
   <th style="text-align:center;"> Experiment </th>
   <th style="text-align:center;"> Mapped reads </th>
   <th style="text-align:center;"> Breadth </th>
   <th style="text-align:center;"> Reference covered (%) </th>
   <th style="text-align:center;"> Average Depth </th>
   <th style="text-align:center;"> Variation coefficient </th>
   <th style="text-align:center;"> Mismatch Rate (%) </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:center;font-style: italic;"> NA </td>
   <td style="text-align:center;"> 1 </td>
   <td style="text-align:center;"> 33,406,640 </td>
   <td style="text-align:center;"> 4,438,281 </td>
   <td style="text-align:center;"> 98.54 </td>
   <td style="text-align:center;"> 571.22 </td>
   <td style="text-align:center;"> 0.21 </td>
   <td style="text-align:center;"> 0.1077 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> NA </td>
   <td style="text-align:center;"> 2 </td>
   <td style="text-align:center;"> 35,680,689 </td>
   <td style="text-align:center;"> 4,440,442 </td>
   <td style="text-align:center;"> 98.58 </td>
   <td style="text-align:center;"> 608.48 </td>
   <td style="text-align:center;"> 0.21 </td>
   <td style="text-align:center;"> 0.1077 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> RP-MDA </td>
   <td style="text-align:center;"> 1 </td>
   <td style="text-align:center;"> 31,360,430 </td>
   <td style="text-align:center;"> 4,270,082 </td>
   <td style="text-align:center;"> 93.67 </td>
   <td style="text-align:center;"> 546.25 </td>
   <td style="text-align:center;"> 4.93 </td>
   <td style="text-align:center;"> 1.2791 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> RP-MDA </td>
   <td style="text-align:center;"> 2 </td>
   <td style="text-align:center;"> 32,030,479 </td>
   <td style="text-align:center;"> 4,434,691 </td>
   <td style="text-align:center;"> 98.38 </td>
   <td style="text-align:center;"> 549.34 </td>
   <td style="text-align:center;"> 3.12 </td>
   <td style="text-align:center;"> 1.2791 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> PrimPol-MDA </td>
   <td style="text-align:center;"> 2 </td>
   <td style="text-align:center;"> 26,951,539 </td>
   <td style="text-align:center;"> 4,440,242 </td>
   <td style="text-align:center;"> 98.58 </td>
   <td style="text-align:center;"> 457.24 </td>
   <td style="text-align:center;"> 0.85 </td>
   <td style="text-align:center;"> 0.3000 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> piPolB </td>
   <td style="text-align:center;"> 2 </td>
   <td style="text-align:center;"> 2,009,164 </td>
   <td style="text-align:center;"> 4,381,966 </td>
   <td style="text-align:center;"> 97.51 </td>
   <td style="text-align:center;"> 16.36 </td>
   <td style="text-align:center;"> 31.77 </td>
   <td style="text-align:center;"> 0.2673 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> piPolB+D </td>
   <td style="text-align:center;"> 2 </td>
   <td style="text-align:center;"> 16,484,064 </td>
   <td style="text-align:center;"> 4,439,396 </td>
   <td style="text-align:center;"> 98.56 </td>
   <td style="text-align:center;"> 91.06 </td>
   <td style="text-align:center;"> 17.93 </td>
   <td style="text-align:center;"> 0.3443 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> piMDA </td>
   <td style="text-align:center;"> 1 </td>
   <td style="text-align:center;"> 30,765,774 </td>
   <td style="text-align:center;"> 4,438,131 </td>
   <td style="text-align:center;"> 98.54 </td>
   <td style="text-align:center;"> 525.24 </td>
   <td style="text-align:center;"> 0.45 </td>
   <td style="text-align:center;"> 0.1137 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> piMDA </td>
   <td style="text-align:center;"> 2 </td>
   <td style="text-align:center;"> 33,761,506 </td>
   <td style="text-align:center;"> 4,439,853 </td>
   <td style="text-align:center;"> 98.57 </td>
   <td style="text-align:center;"> 586.57 </td>
   <td style="text-align:center;"> 0.61 </td>
   <td style="text-align:center;"> 0.1137 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> piMDA+D </td>
   <td style="text-align:center;"> 1 </td>
   <td style="text-align:center;"> 29,696,732 </td>
   <td style="text-align:center;"> 4,437,996 </td>
   <td style="text-align:center;"> 98.54 </td>
   <td style="text-align:center;"> 572.54 </td>
   <td style="text-align:center;"> 0.38 </td>
   <td style="text-align:center;"> 0.1071 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> piMDA+D </td>
   <td style="text-align:center;"> 2 </td>
   <td style="text-align:center;"> 37,756,486 </td>
   <td style="text-align:center;"> 4,440,189 </td>
   <td style="text-align:center;"> 98.58 </td>
   <td style="text-align:center;"> 689.94 </td>
   <td style="text-align:center;"> 0.36 </td>
   <td style="text-align:center;"> 0.1071 </td>
  </tr>
</tbody>
</table>

</div>
</div>
<p>Similarly to the previous results, Figures 11 and 12 show that the RP-MDA and, to a lesser extent the PrimPol-MDA protocols gave rise to a higher mismatch rate in the <em>P. aeruginosa</em> and <em>K. rhizophila</em> genomes. This is mostly due to misinsertions and deletions. Thus, in our metagenome sample, the RP-MDA protocol generate an amplification product with uneven coverage breadth and higher mismatch rate for high GC-sequences.</p>
<p>The samples generated by the piPolB and piPolB+D protocols also contain a higher mismatch rate, but in this case, the mistmaches are mostly in the <em>B. subtilis</em> sequences and contain a high rate of insertions.</p>
<p>Interestingly, the piMDA and piMDA+D protocols gave rise to similar mismatch rates than NA samples, only with a minor increase in the insertions in moderate GC genomes, mostly in homopolymeric sequence contexts (as expected).</p>
</section>
<section id="analysis-of-unclassfied-reads" class="level3" data-number="5.2">
<h3 data-number="5.2" class="anchored" data-anchor-id="analysis-of-unclassfied-reads"><span class="header-section-number">5.2</span> Analysis of unclassfied reads</h3>
<p>The piPolB and specially piPolB+D samples showed a higher rate of unknown sequences in the profiling (Figure 7) and lower rate of reads mapping against the reference genomes (Figure 9A). After the trimming step, those unmapped reads may correspond either to contaminant amplified DNA or <em>ab initio</em> DNA synthesis (see <a href="https://academic.oup.com/femsle/article/351/1/1/500774">Zyrina et al.&nbsp;2013</a>){target=“_blank”}. Since the profiling with Metaphlan only considers mapped reads, as well as our <em>Bowtie2</em> mapping against the reference, we decided to map the trimmed reads without the <code>–-no-unal</code> option against the Metaphlan database (vJan21), and then extract the unaligned reads an convert to fasta using <em>samtools</em> <em>f -4</em> for further analysis. We used the following bash script:</p>
<p><code>for sample in A2 B2 F2 C3 D3 Ctrl Ctrl2 N1 N2 N4 N6;</code></p>
<p><code>do</code> <code>bowtie2 --very-sensitive -p 32 -x /home/modesto/anaconda3/envs/mpa/lib/python3.7/site-packages/metaphlan/metaphlan_databases/mpa_vJan21_CHOCOPhlAnSGB_202103 -1 ../trimmo/"$sample"_R1_pair.fastq.gz -2 ../trimmo/"$sample"_R2_pair.fastq.gz -U ../trimmo/"$sample"_R1_unpair.fastq.gz,../trimmo/"$sample"_R2_unpair.fastq.gz | samtools view -bS - | samtools sort -@ 32 -o "$sample"_metaphlan_sorted.bam;</code></p>
<p><code>samtools view -u -f 4 "$sample"_metaphlan_sorted.bam &gt; "$sample"_metaphlan_unmapped.sam;</code></p>
<p><code>samtools fasta "$sample"_metaphlan_unmapped.sam &gt; "$sample"_metaphlan_unmapped.fasta;</code> <code>rm "$sample"_metaphlan_unmapped.sam;</code></p>
<p><code>rm "$sample"_metaphlan_sorted.bam;</code></p>
<p><code>done</code></p>
<p>We then analyzed the presence of low complexity sequences using BBduk (from <a href="https://jgi.doe.gov/data-and-tools/software-tools/bbtools/" target="_blank">BBtools</a>) to detect sequences with entropy &lt;0.5. The proportion of filtered low entropy reads was vary low (&lt;0.25%) for all the samples, except for the <strong>piPolB-MDA sample (12.58%)</strong>, supporting the presence of ab initio DNA synthesis (see <a href="https://pubmed.ncbi.nlm.nih.gov/24965874/" target="_blank">Zyrina et al., 2014</a>). Moreover, the presence of <em>ab initio</em> DNA synthesis capacity in piPolB could be confirmed in long incubation MDA assays (≥ 9h), as shown in the figure below.</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/abinitio.png" class="img-fluid figure-img" alt="Ab initio DNA synthesis by piPolB"></p>
<p></p><figcaption class="figure-caption">External Figure 2. Ab initio DNA synthesis by piPolB. Time-course experiments were carried out in the standard conditions in the presence of 0, 2 and 20 ng of metagenomic DNA.</figcaption><p></p>
</figure>
</div>
</section>
<section id="gc-content-bias" class="level3" data-number="5.3">
<h3 data-number="5.3" class="anchored" data-anchor-id="gc-content-bias"><span class="header-section-number">5.3</span> GC content bias</h3>
<p>GC content bias of sequenced reads was analyzed in detail with <a href="https://broadinstitute.github.io/picard/" target="_blank">Picard</a> v.2.25.0 (<em>CollectGcBiasMetrics</em>). This allow us to see, for the whole metagenome and for each reference genome, the frequency of sequences at each GC content window (100 bp). All data is available in the folder <em>gc</em> in the <a href="https://github.com/mredrejo/pimda" target="_blank">GitHub repo</a>.</p>
<p>Note that, contrary to the previous result in Figure 4, the coverage is normalized by the frequency of each GC window in the reference metagenome. According to the <a href="https://gatk.broadinstitute.org/hc/en-us/articles/360037593931-CollectGcBiasMetrics-Picard-" target="_blank">Picard definition</a>:</p>
<p><em>NORMALIZED_COVERAGE is a relative measure of sequence coverage by the reads at a particular GC content. For each run, the corresponding reference sequence is divided into bins or windows based on the percentage of G + C content ranging from 0 - 100%. The percentages of G + C are determined from a defined length of sequence; in this case we used the default value (100 bases).</em></p>
<p>Therefore, if the reference metagenome have higher number of sequences at a particular GC content, the normalized coverage should be also higher at that GC content value.</p>
<section id="gc-bias-per-mda-sample" class="level4" data-number="5.3.1">
<h4 data-number="5.3.1" class="anchored" data-anchor-id="gc-bias-per-mda-sample"><span class="header-section-number">5.3.1</span> GC bias per MDA sample</h4>
<p>Data from Picard output was merged to generate a combined plot with homogeneous scale. Original individual data tables are available in the <em>gc</em> folder of the <a href="https://github.com/mredrejo/pimda" target="_blank">GitHub repository</a>.</p>
<div class="cell">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb62"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb62-1"><a href="#cb62-1" aria-hidden="true" tabindex="-1"></a><span class="co"># load all the files as a list of dataframes</span></span>
<span id="cb62-2"><a href="#cb62-2" aria-hidden="true" tabindex="-1"></a>gc_picard <span class="ot">=</span> <span class="fu">c</span>(<span class="st">"ctrl"</span>, <span class="st">"ctrl2"</span>, <span class="st">"N4"</span>, <span class="st">"D3"</span>, <span class="st">"C3"</span>, <span class="st">"N1"</span>, <span class="st">"F2"</span>, <span class="st">"N2"</span>, <span class="st">"B2"</span>, <span class="st">"N6"</span>, <span class="st">"A2"</span>)</span>
<span id="cb62-3"><a href="#cb62-3" aria-hidden="true" tabindex="-1"></a>gc <span class="ot">&lt;-</span> <span class="fu">lapply</span>(gc_picard, <span class="cf">function</span>(x) <span class="fu">read.csv2</span>(<span class="fu">paste</span>(<span class="st">"gc/"</span>, x, <span class="st">"_gc_bias_metagenome.txt"</span>, <span class="at">sep =</span> <span class="st">""</span>),</span>
<span id="cb62-4"><a href="#cb62-4" aria-hidden="true" tabindex="-1"></a>    <span class="at">skip =</span> <span class="dv">6</span>, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>, <span class="at">colClasses =</span> <span class="fu">c</span>(<span class="st">"NULL"</span>, <span class="st">"NULL"</span>, <span class="st">"numeric"</span>, <span class="st">"numeric"</span>,</span>
<span id="cb62-5"><a href="#cb62-5" aria-hidden="true" tabindex="-1"></a>        <span class="st">"numeric"</span>, <span class="st">"numeric"</span>, <span class="st">"numeric"</span>, <span class="st">"numeric"</span>, <span class="st">"numeric"</span>, <span class="st">"numeric"</span>, <span class="st">"numeric"</span>)))</span>
<span id="cb62-6"><a href="#cb62-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb62-7"><a href="#cb62-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb62-8"><a href="#cb62-8" aria-hidden="true" tabindex="-1"></a>cor_matrix <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="dv">11</span>, <span class="dv">3</span>)</span>
<span id="cb62-9"><a href="#cb62-9" aria-hidden="true" tabindex="-1"></a><span class="fu">names</span>(gc) <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"NA"</span>, <span class="st">"NA2"</span>, <span class="st">"RP-MDA"</span>, <span class="st">"RP-MDA2"</span>, <span class="st">"PrimPol-MDA"</span>, <span class="st">"piPolB"</span>, <span class="st">"piPolB+D"</span>, <span class="st">"piMDA"</span>,</span>
<span id="cb62-10"><a href="#cb62-10" aria-hidden="true" tabindex="-1"></a>    <span class="st">"piMDA2"</span>, <span class="st">"piMDA+D"</span>, <span class="st">"piMDA+D2"</span>)</span>
<span id="cb62-11"><a href="#cb62-11" aria-hidden="true" tabindex="-1"></a><span class="co"># create all the plots in a list</span></span>
<span id="cb62-12"><a href="#cb62-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb62-13"><a href="#cb62-13" aria-hidden="true" tabindex="-1"></a>gc_cov <span class="ot">&lt;-</span> gc[[<span class="dv">1</span>]][, <span class="dv">1</span><span class="sc">:</span><span class="dv">2</span>]</span>
<span id="cb62-14"><a href="#cb62-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb62-15"><a href="#cb62-15" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">length</span>(gc)) {</span>
<span id="cb62-16"><a href="#cb62-16" aria-hidden="true" tabindex="-1"></a>    gc_cov <span class="ot">&lt;-</span> <span class="fu">cbind</span>(gc_cov, gc[[i]]<span class="sc">$</span>NORMALIZED_COVERAGE)</span>
<span id="cb62-17"><a href="#cb62-17" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb62-18"><a href="#cb62-18" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(gc_cov) <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"GC"</span>, <span class="st">"W"</span>, <span class="st">"NA"</span>, <span class="st">"NA2"</span>, <span class="st">"RP-MDA"</span>, <span class="st">"RP-MDA2"</span>, <span class="st">"PrimPol-MDA"</span>, <span class="st">"piPolB"</span>,</span>
<span id="cb62-19"><a href="#cb62-19" aria-hidden="true" tabindex="-1"></a>    <span class="st">"piPolB+D"</span>, <span class="st">"piMDA"</span>, <span class="st">"piMDA2"</span>, <span class="st">"piMDA+D"</span>, <span class="st">"piMDA+D2"</span>)</span>
<span id="cb62-20"><a href="#cb62-20" aria-hidden="true" tabindex="-1"></a>gc_cov <span class="ot">&lt;-</span> <span class="fu">as.data.frame</span>(gc_cov)</span>
<span id="cb62-21"><a href="#cb62-21" aria-hidden="true" tabindex="-1"></a>gc_cov_melted <span class="ot">&lt;-</span> data.table<span class="sc">::</span><span class="fu">melt</span>(gc_cov, <span class="at">id.vars =</span> <span class="fu">c</span>(<span class="st">"GC"</span>, <span class="st">"W"</span>), <span class="at">na.rm =</span> <span class="cn">FALSE</span>)</span>
<span id="cb62-22"><a href="#cb62-22" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(gc_cov_melted) <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"GC"</span>, <span class="st">"W"</span>, <span class="st">"Sample"</span>, <span class="st">"Nor_cov"</span>)</span>
<span id="cb62-23"><a href="#cb62-23" aria-hidden="true" tabindex="-1"></a><span class="co"># mean GC content of mapped reads</span></span>
<span id="cb62-24"><a href="#cb62-24" aria-hidden="true" tabindex="-1"></a>gc_mapped <span class="ot">&lt;-</span> <span class="fu">list</span>()</span>
<span id="cb62-25"><a href="#cb62-25" aria-hidden="true" tabindex="-1"></a>gc_mapped <span class="ot">&lt;-</span> <span class="fu">lapply</span>(<span class="dv">1</span><span class="sc">:</span><span class="fu">length</span>(gc), <span class="at">FUN =</span> <span class="cf">function</span>(i) {</span>
<span id="cb62-26"><a href="#cb62-26" aria-hidden="true" tabindex="-1"></a>    <span class="fu">mean</span>(gc[[i]]<span class="sc">$</span>NORMALIZED_COVERAGE[gc[[i]]<span class="sc">$</span>NORMALIZED_COVERAGE <span class="sc">&gt;</span> <span class="dv">0</span>] <span class="sc">*</span> gc[[i]]<span class="sc">$</span>GC[gc[[i]]<span class="sc">$</span>NORMALIZED_COVERAGE <span class="sc">&gt;</span></span>
<span id="cb62-27"><a href="#cb62-27" aria-hidden="true" tabindex="-1"></a>        <span class="dv">0</span>])</span>
<span id="cb62-28"><a href="#cb62-28" aria-hidden="true" tabindex="-1"></a>})</span>
<span id="cb62-29"><a href="#cb62-29" aria-hidden="true" tabindex="-1"></a><span class="co"># paste in legend</span></span>
<span id="cb62-30"><a href="#cb62-30" aria-hidden="true" tabindex="-1"></a>leyenda <span class="ot">&lt;-</span> <span class="fu">c</span>()</span>
<span id="cb62-31"><a href="#cb62-31" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">length</span>(<span class="fu">levels</span>(gc_cov_melted<span class="sc">$</span>Sample))) {</span>
<span id="cb62-32"><a href="#cb62-32" aria-hidden="true" tabindex="-1"></a>    leyenda[i] <span class="ot">&lt;-</span> <span class="fu">paste0</span>(<span class="fu">levels</span>(gc_cov_melted<span class="sc">$</span>Sample)[i], <span class="st">" ("</span>, <span class="fu">round</span>(gc_mapped[[i]], <span class="dv">1</span>), <span class="st">"%)"</span>)</span>
<span id="cb62-33"><a href="#cb62-33" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb62-34"><a href="#cb62-34" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb62-35"><a href="#cb62-35" aria-hidden="true" tabindex="-1"></a>p <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(gc_cov_melted, <span class="fu">aes</span>(<span class="at">x =</span> GC)) <span class="sc">+</span> <span class="fu">geom_bar</span>(<span class="fu">aes</span>(<span class="at">y =</span> W<span class="sc">/</span><span class="fl">1e+06</span>), <span class="at">stat =</span> <span class="st">"identity"</span>, <span class="at">fill =</span> <span class="st">"steelblue"</span>,</span>
<span id="cb62-36"><a href="#cb62-36" aria-hidden="true" tabindex="-1"></a>    <span class="at">alpha =</span> <span class="fl">0.5</span>) <span class="sc">+</span> <span class="fu">geom_line</span>(<span class="fu">aes</span>(<span class="at">y =</span> Nor_cov, <span class="at">color =</span> Sample), <span class="at">size =</span> <span class="fl">1.5</span>) <span class="sc">+</span> <span class="fu">theme_bw</span>(<span class="at">base_size =</span> <span class="dv">14</span>) <span class="sc">+</span></span>
<span id="cb62-37"><a href="#cb62-37" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_y_continuous</span>(<span class="st">"Normalized coverage"</span>, <span class="at">sec.axis =</span> <span class="fu">sec_axis</span>(<span class="sc">~</span>. <span class="sc">*</span> <span class="fl">1e+06</span>, <span class="at">name =</span> <span class="st">"Frequency at GC"</span>)) <span class="sc">+</span></span>
<span id="cb62-38"><a href="#cb62-38" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme</span>(<span class="at">axis.text.y.right =</span> <span class="fu">element_text</span>(<span class="at">colour =</span> <span class="st">"steelblue"</span>), <span class="at">axis.title.y.right =</span> <span class="fu">element_text</span>(<span class="at">colour =</span> <span class="st">"steelblue"</span>),</span>
<span id="cb62-39"><a href="#cb62-39" aria-hidden="true" tabindex="-1"></a>        <span class="at">axis.line.y.right =</span> <span class="fu">element_line</span>(<span class="at">color =</span> <span class="fu">rgb</span>(<span class="fl">0.275</span>, <span class="fl">0.51</span>, <span class="fl">0.706</span>, <span class="fl">0.55</span>), <span class="at">linewidth =</span> <span class="dv">2</span>)) <span class="sc">+</span></span>
<span id="cb62-40"><a href="#cb62-40" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_color_manual</span>(<span class="at">values =</span> colorines[<span class="dv">1</span><span class="sc">:</span><span class="dv">11</span>], <span class="at">labels =</span> leyenda) <span class="sc">+</span> <span class="fu">geom_hline</span>(<span class="at">yintercept =</span> <span class="dv">1</span>,</span>
<span id="cb62-41"><a href="#cb62-41" aria-hidden="true" tabindex="-1"></a>    <span class="at">linetype =</span> <span class="st">"dashed"</span>, <span class="at">color =</span> <span class="st">"black"</span>)</span>
<span id="cb62-42"><a href="#cb62-42" aria-hidden="true" tabindex="-1"></a>p</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="PlotGCbias_meta_normalized-1.png" class="img-fluid figure-img" width="960"></p>
<p></p><figcaption class="figure-caption">Figure 12. Comparative Normalized coverage by GC distribution of sequence and reads. Sequence coverage (red) is normalized by the GC range frequency in 100 bp windows (blue). The average GC content of each sample mapped reads is also indicated in the leyend</figcaption><p></p>
</figure>
</div>
</div>
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb63"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb63-1"><a href="#cb63-1" aria-hidden="true" tabindex="-1"></a><span class="co"># ggsave('figures/normalized_gc_all_2.svg', p, width=8,heigh=5)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<p>And the interactive plot for detail analysis:</p>
<div class="cell">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb64"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb64-1"><a href="#cb64-1" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplotly</span>(p)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="plotly html-widget html-fill-item-overflow-hidden html-fill-item" id="htmlwidget-06a141d22f76cfec947f" style="width:100%;height:464px;"></div>
<script type="application/json" data-for="htmlwidget-06a141d22f76cfec947f">{"x":{"data":[{"orientation":"v","width":[0.9,0.9,0.9,0.9,0.9,0.9,0.9,0.9,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.900000000000002,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.9,0.9,0.9,0.9,0.9,0.9,0.9,0.9,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.900000000000002,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.9,0.9,0.9,0.9,0.9,0.9,0.9,0.9,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.900000000000002,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.9,0.9,0.9,0.9,0.9,0.9,0.9,0.9,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.900000000000002,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.9,0.9,0.9,0.9,0.9,0.9,0.9,0.9,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.900000000000002,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.9,0.9,0.9,0.9,0.9,0.9,0.9,0.9,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.900000000000002,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.9,0.9,0.9,0.9,0.9,0.9,0.9,0.9,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.900000000000002,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.9,0.9,0.9,0.9,0.9,0.9,0.9,0.9,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.900000000000002,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.9,0.9,0.9,0.9,0.9,0.9,0.9,0.9,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.900000000000002,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.9,0.9,0.9,0.9,0.9,0.9,0.9,0.9,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.900000000000002,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.9,0.9,0.9,0.9,0.9,0.9,0.9,0.9,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.899999999999999,0.900000000000002,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006,0.900000000000006],"base":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,2e-06,3e-06,1.8e-05,6.1e-05,9.9e-05,0.000232,0.000414,0.000657,0.001179,0.001961,0.002947,0.004428,0.006577,0.010038,0.013131,0.016968,0.022322,0.028292,0.03669,0.047001,0.058328,0.070344,0.083849,0.101838,0.119577,0.138734,0.159611,0.181203,0.205339,0.232452,0.262352,0.289362,0.317347,0.34204,0.365406,0.387317,0.406405,0.422871,0.436752,0.445505,0.451658,0.456692,0.45509,0.447841,0.429853,0.405343,0.377543,0.353993,0.338145,0.329892,0.333301,0.351119,0.384616,0.427987,0.475965,0.527593,0.575783,0.614146,0.634249,0.6342,0.617734,0.582209,0.529033,0.467484,0.396863,0.325426,0.258782,0.198358,0.146188,0.105011,0.073676,0.049418,0.033292,0.02213,0.01334,0.007582,0.004134,0.001878,0.000879,0.000326,6.9e-05,1.1e-05,5e-06,4e-06,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,4e-06,6e-06,3.6e-05,0.000122,0.000198,0.000464,0.000828,0.001314,0.002358,0.003922,0.005894,0.008856,0.013154,0.020076,0.026262,0.033936,0.044644,0.056584,0.07338,0.094002,0.116656,0.140688,0.167698,0.203676,0.239154,0.277468,0.319222,0.362406,0.410678,0.464904,0.524704,0.578724,0.634694,0.68408,0.730812,0.774634,0.81281,0.845742,0.873504,0.89101,0.903316,0.913384,0.91018,0.895682,0.859706,0.810686,0.755086,0.707986,0.67629,0.659784,0.666602,0.702238,0.769232,0.855974,0.95193,1.055186,1.151566,1.228292,1.268498,1.2684,1.235468,1.164418,1.058066,0.934968,0.793726,0.650852,0.517564,0.396716,0.292376,0.210022,0.147352,0.098836,0.066584,0.04426,0.02668,0.015164,0.008268,0.003756,0.001758,0.000652,0.000138,2.2e-05,1e-05,8e-06,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,6e-06,9e-06,5.4e-05,0.000183,0.000297,0.000696,0.001242,0.001971,0.003537,0.005883,0.008841,0.013284,0.019731,0.030114,0.039393,0.050904,0.066966,0.084876,0.11007,0.141003,0.174984,0.211032,0.251547,0.305514,0.358731,0.416202,0.478833,0.543609,0.616017,0.697356,0.787056,0.868086,0.952041,1.02612,1.096218,1.161951,1.219215,1.268613,1.310256,1.336515,1.354974,1.370076,1.36527,1.343523,1.289559,1.216029,1.132629,1.061979,1.014435,0.989676,0.999903,1.053357,1.153848,1.283961,1.427895,1.582779,1.727349,1.842438,1.902747,1.9026,1.853202,1.746627,1.587099,1.402452,1.190589,0.976278,0.776346,0.595074,0.438564,0.315033,0.221028,0.148254,0.099876,0.06639,0.04002,0.022746,0.012402,0.005634,0.002637,0.000978,0.000207,3.3e-05,1.5e-05,1.2e-05,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,8e-06,1.2e-05,7.2e-05,0.000244,0.000396,0.000928,0.001656,0.002628,0.004716,0.007844,0.011788,0.017712,0.026308,0.040152,0.052524,0.067872,0.089288,0.113168,0.14676,0.188004,0.233312,0.281376,0.335396,0.407352,0.478308,0.554936,0.638444,0.724812,0.821356,0.929808,1.049408,1.157448,1.269388,1.36816,1.461624,1.549268,1.62562,1.691484,1.747008,1.78202,1.806632,1.826768,1.82036,1.791364,1.719412,1.621372,1.510172,1.415972,1.35258,1.319568,1.333204,1.404476,1.538464,1.711948,1.90386,2.110372,2.303132,2.456584,2.536996,2.5368,2.470936,2.328836,2.116132,1.869936,1.587452,1.301704,1.035128,0.793432,0.584752,0.420044,0.294704,0.197672,0.133168,0.08852,0.05336,0.030328,0.016536,0.007512,0.003516,0.001304,0.000276,4.4e-05,2e-05,1.6e-05,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1e-05,1.5e-05,9e-05,0.000305,0.000495,0.00116,0.00207,0.003285,0.005895,0.009805,0.014735,0.02214,0.032885,0.05019,0.065655,0.08484,0.11161,0.14146,0.18345,0.235005,0.29164,0.35172,0.419245,0.50919,0.597885,0.69367,0.798055,0.906015,1.026695,1.16226,1.31176,1.44681,1.586735,1.7102,1.82703,1.936585,2.032025,2.114355,2.18376,2.227525,2.25829,2.28346,2.27545,2.239205,2.149265,2.026715,1.887715,1.769965,1.690725,1.64946,1.666505,1.755595,1.92308,2.139935,2.379825,2.637965,2.878915,3.07073,3.171245,3.171,3.08867,2.911045,2.645165,2.33742,1.984315,1.62713,1.29391,0.99179,0.73094,0.525055,0.36838,0.24709,0.16646,0.11065,0.0667,0.03791,0.02067,0.00939,0.004395,0.00163,0.000345,5.5e-05,2.5e-05,2e-05,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1.2e-05,1.8e-05,0.000108,0.000366,0.000594,0.001392,0.002484,0.003942,0.007074,0.011766,0.017682,0.026568,0.039462,0.060228,0.078786,0.101808,0.133932,0.169752,0.22014,0.282006,0.349968,0.422064,0.503094,0.611028,0.717462,0.832404,0.957666,1.087218,1.232034,1.394712,1.574112,1.736172,1.904082,2.05224,2.192436,2.323902,2.43843,2.537226,2.620512,2.67303,2.709948,2.740152,2.73054,2.687046,2.579118,2.432058,2.265258,2.123958,2.02887,1.979352,1.999806,2.106714,2.307696,2.567922,2.85579,3.165558,3.454698,3.684876,3.805494,3.8052,3.706404,3.493254,3.174198,2.804904,2.381178,1.952556,1.552692,1.190148,0.877128,0.630066,0.442056,0.296508,0.199752,0.13278,0.08004,0.045492,0.024804,0.011268,0.005274,0.001956,0.000414,6.6e-05,3e-05,2.4e-05,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1.4e-05,2.1e-05,0.000126,0.000427,0.000693,0.001624,0.002898,0.004599,0.008253,0.013727,0.020629,0.030996,0.046039,0.070266,0.091917,0.118776,0.156254,0.198044,0.25683,0.329007,0.408296,0.492408,0.586943,0.712866,0.837039,0.971138,1.117277,1.268421,1.437373,1.627164,1.836464,2.025534,2.221429,2.39428,2.557842,2.711219,2.844835,2.960097,3.057264,3.118535,3.161606,3.196844,3.18563,3.134887,3.008971,2.837401,2.642801,2.477951,2.367015,2.309244,2.333107,2.457833,2.692312,2.995909,3.331755,3.693151,4.030481,4.299022,4.439743,4.4394,4.324138,4.075463,3.703231,3.272388,2.778041,2.277982,1.811474,1.388506,1.023316,0.735077,0.515732,0.345926,0.233044,0.15491,0.09338,0.053074,0.028938,0.013146,0.006153,0.002282,0.000483,7.7e-05,3.5e-05,2.8e-05,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1.6e-05,2.4e-05,0.000144,0.000488,0.000792,0.001856,0.003312,0.005256,0.009432,0.015688,0.023576,0.035424,0.052616,0.080304,0.105048,0.135744,0.178576,0.226336,0.29352,0.376008,0.466624,0.562752,0.670792,0.814704,0.956616,1.109872,1.276888,1.449624,1.642712,1.859616,2.098816,2.314896,2.538776,2.73632,2.923248,3.098536,3.25124,3.382968,3.494016,3.56404,3.613264,3.653536,3.64072,3.582728,3.438824,3.242744,3.020344,2.831944,2.70516,2.639136,2.666408,2.808952,3.076928,3.423896,3.80772,4.220744,4.606264,4.913168,5.073992,5.0736,4.941872,4.657672,4.232264,3.739872,3.174904,2.603408,2.070256,1.586864,1.169504,0.840088,0.589408,0.395344,0.266336,0.17704,0.10672,0.060656,0.033072,0.015024,0.007032,0.002608,0.000552,8.8e-05,4e-05,3.2e-05,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1.8e-05,2.7e-05,0.000162,0.000549,0.000891,0.002088,0.003726,0.005913,0.010611,0.017649,0.026523,0.039852,0.059193,0.090342,0.118179,0.152712,0.200898,0.254628,0.33021,0.423009,0.524952,0.633096,0.754641,0.916542,1.076193,1.248606,1.436499,1.630827,1.848051,2.092068,2.361168,2.604258,2.856123,3.07836,3.288654,3.485853,3.657645,3.805839,3.930768,4.009545,4.064922,4.110228,4.09581,4.030569,3.868677,3.648087,3.397887,3.185937,3.043305,2.969028,2.999709,3.160071,3.461544,3.851883,4.283685,4.748337,5.182047,5.527314,5.708241,5.7078,5.559606,5.239881,4.761297,4.207356,3.571767,2.928834,2.329038,1.785222,1.315692,0.945099,0.663084,0.444762,0.299628,0.19917,0.12006,0.068238,0.037206,0.016902,0.007911,0.002934,0.000621,9.9e-05,4.5e-05,3.6e-05,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,2e-05,3e-05,0.00018,0.00061,0.00099,0.00232,0.00414,0.00657,0.01179,0.01961,0.02947,0.04428,0.06577,0.10038,0.13131,0.16968,0.22322,0.28292,0.3669,0.47001,0.58328,0.70344,0.83849,1.01838,1.19577,1.38734,1.59611,1.81203,2.05339,2.32452,2.62352,2.89362,3.17347,3.4204,3.65406,3.87317,4.06405,4.22871,4.36752,4.45505,4.51658,4.56692,4.5509,4.47841,4.29853,4.05343,3.77543,3.53993,3.38145,3.29892,3.33301,3.51119,3.84616,4.27987,4.75965,5.27593,5.75783,6.14146,6.34249,6.342,6.17734,5.82209,5.29033,4.67484,3.96863,3.25426,2.58782,1.98358,1.46188,1.05011,0.73676,0.49418,0.33292,0.2213,0.1334,0.07582,0.04134,0.01878,0.00879,0.00326,0.00069,0.00011,5e-05,4e-05,0,0,0,0,0,0,0],"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100,0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100,0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100,0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100,0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100,0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100,0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100,0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100,0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100,0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100,0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0,0,0,0,0,0,0,0,0,0,2e-06,3e-06,1.8e-05,6.1e-05,9.9e-05,0.000232,0.000414,0.000657,0.001179,0.001961,0.002947,0.004428,0.006577,0.010038,0.013131,0.016968,0.022322,0.028292,0.03669,0.047001,0.058328,0.070344,0.083849,0.101838,0.119577,0.138734,0.159611,0.181203,0.205339,0.232452,0.262352,0.289362,0.317347,0.34204,0.365406,0.387317,0.406405,0.422871,0.436752,0.445505,0.451658,0.456692,0.45509,0.447841,0.429853,0.405343,0.377543,0.353993,0.338145,0.329892,0.333301,0.351119,0.384616,0.427987,0.475965,0.527593,0.575783,0.614146,0.634249,0.6342,0.617734,0.582209,0.529033,0.467484,0.396863,0.325426,0.258782,0.198358,0.146188,0.105011,0.073676,0.049418,0.033292,0.02213,0.01334,0.007582,0.004134,0.001878,0.000879,0.000326,6.9e-05,1.1e-05,5e-06,4e-06,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,2e-06,3e-06,1.8e-05,6.1e-05,9.9e-05,0.000232,0.000414,0.000657,0.001179,0.001961,0.002947,0.004428,0.006577,0.010038,0.013131,0.016968,0.022322,0.028292,0.03669,0.047001,0.058328,0.070344,0.083849,0.101838,0.119577,0.138734,0.159611,0.181203,0.205339,0.232452,0.262352,0.289362,0.317347,0.34204,0.365406,0.387317,0.406405,0.422871,0.436752,0.445505,0.451658,0.456692,0.45509,0.447841,0.429853,0.405343,0.377543,0.353993,0.338145,0.329892,0.333301,0.351119,0.384616,0.427987,0.475965,0.527593,0.575783,0.614146,0.634249,0.6342,0.617734,0.582209,0.529033,0.467484,0.396863,0.325426,0.258782,0.198358,0.146188,0.105011,0.073676,0.049418,0.033292,0.02213,0.01334,0.007582,0.004134,0.001878,0.000879,0.000326,6.9e-05,1.1e-05,5e-06,4e-06,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,2e-06,3e-06,1.8e-05,6.1e-05,9.9e-05,0.000232,0.000414,0.000657,0.001179,0.001961,0.002947,0.004428,0.006577,0.010038,0.013131,0.016968,0.022322,0.028292,0.03669,0.047001,0.058328,0.070344,0.083849,0.101838,0.119577,0.138734,0.159611,0.181203,0.205339,0.232452,0.262352,0.289362,0.317347,0.34204,0.365406,0.387317,0.406405,0.422871,0.436752,0.445505,0.451658,0.456692,0.45509,0.447841,0.429853,0.405343,0.377543,0.353993,0.338145,0.329892,0.333301,0.351119,0.384616,0.427987,0.475965,0.527593,0.575783,0.614146,0.634249,0.6342,0.617734,0.582209,0.529033,0.467484,0.396863,0.325426,0.258782,0.198358,0.146188,0.105011,0.073676,0.049418,0.033292,0.02213,0.01334,0.007582,0.004134,0.001878,0.000879,0.000326,6.9e-05,1.1e-05,5e-06,4e-06,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,2e-06,3e-06,1.8e-05,6.1e-05,9.9e-05,0.000232,0.000414,0.000657,0.001179,0.001961,0.002947,0.004428,0.006577,0.010038,0.013131,0.016968,0.022322,0.028292,0.03669,0.047001,0.058328,0.070344,0.083849,0.101838,0.119577,0.138734,0.159611,0.181203,0.205339,0.232452,0.262352,0.289362,0.317347,0.34204,0.365406,0.387317,0.406405,0.422871,0.436752,0.445505,0.451658,0.456692,0.45509,0.447841,0.429853,0.405343,0.377543,0.353993,0.338145,0.329892,0.333301,0.351119,0.384616,0.427987,0.475965,0.527593,0.575783,0.614146,0.634249,0.6342,0.617734,0.582209,0.529033,0.467484,0.396863,0.325426,0.258782,0.198358,0.146188,0.105011,0.073676,0.049418,0.033292,0.02213,0.01334,0.007582,0.004134,0.001878,0.000879,0.000326,6.9e-05,1.1e-05,5e-06,4e-06,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,2e-06,3e-06,1.8e-05,6.1e-05,9.9e-05,0.000232,0.000414,0.000657,0.001179,0.001961,0.002947,0.004428,0.006577,0.010038,0.013131,0.016968,0.022322,0.028292,0.03669,0.047001,0.058328,0.070344,0.083849,0.101838,0.119577,0.138734,0.159611,0.181203,0.205339,0.232452,0.262352,0.289362,0.317347,0.34204,0.365406,0.387317,0.406405,0.422871,0.436752,0.445505,0.451658,0.456692,0.45509,0.447841,0.429853,0.405343,0.377543,0.353993,0.338145,0.329892,0.333301,0.351119,0.384616,0.427987,0.475965,0.527593,0.575783,0.614146,0.634249,0.6342,0.617734,0.582209,0.529033,0.467484,0.396863,0.325426,0.258782,0.198358,0.146188,0.105011,0.073676,0.049418,0.033292,0.02213,0.01334,0.007582,0.004134,0.001878,0.000879,0.000326,6.9e-05,1.1e-05,5e-06,4e-06,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,2e-06,3e-06,1.8e-05,6.1e-05,9.89999999999999e-05,0.000232,0.000414,0.000657,0.001179,0.001961,0.002947,0.004428,0.006577,0.010038,0.013131,0.016968,0.022322,0.028292,0.03669,0.047001,0.058328,0.070344,0.083849,0.101838,0.119577,0.138734,0.159611,0.181203,0.205339,0.232452,0.262352,0.289362,0.317347,0.34204,0.365406,0.387317,0.406405,0.422871,0.436752,0.445505,0.451658,0.456692,0.45509,0.447841,0.429853,0.405343,0.377543,0.353993,0.338145,0.329892,0.333301,0.351119,0.384616,0.427987,0.475965,0.527593,0.575783,0.614146,0.634249,0.6342,0.617734,0.582209,0.529033,0.467484,0.396863,0.325426,0.258782,0.198358,0.146188,0.105011,0.073676,0.049418,0.033292,0.02213,0.01334,0.00758200000000001,0.004134,0.001878,0.000879,0.000326,6.9e-05,1.1e-05,5e-06,4e-06,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,2e-06,3e-06,1.8e-05,6.1e-05,9.9e-05,0.000232,0.000414,0.000657,0.001179,0.001961,0.002947,0.004428,0.006577,0.010038,0.013131,0.016968,0.022322,0.028292,0.03669,0.047001,0.058328,0.070344,0.083849,0.101838,0.119577,0.138734,0.159611,0.181203,0.205339,0.232452,0.262352,0.289362,0.317347,0.34204,0.365406,0.387317,0.406405,0.422871,0.436752,0.445505,0.451658,0.456692,0.45509,0.447841,0.429853,0.405343,0.377543,0.353993,0.338145,0.329892,0.333301,0.351119,0.384616,0.427987,0.475965,0.527593,0.575782999999999,0.614146,0.634249000000001,0.6342,0.617734,0.582209,0.529033,0.467484,0.396863,0.325426,0.258782,0.198358,0.146188,0.105011,0.0736760000000001,0.049418,0.033292,0.02213,0.01334,0.007582,0.004134,0.001878,0.000879,0.000326,6.9e-05,1.1e-05,5e-06,4e-06,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,2e-06,3e-06,1.8e-05,6.1e-05,9.9e-05,0.000232,0.000414,0.000657,0.001179,0.001961,0.002947,0.004428,0.006577,0.010038,0.013131,0.016968,0.022322,0.028292,0.03669,0.047001,0.058328,0.070344,0.0838490000000001,0.101838,0.119577,0.138734,0.159611,0.181203,0.205339,0.232452,0.262352,0.289362,0.317347,0.34204,0.365406,0.387317,0.406405,0.422871,0.436752,0.445505,0.451658,0.456692,0.45509,0.447841,0.429853,0.405343,0.377543,0.353993,0.338145,0.329892,0.333301,0.351119,0.384616,0.427987,0.475965,0.527593,0.575783,0.614146,0.634249,0.6342,0.617734,0.582209,0.529033,0.467484,0.396863,0.325426,0.258782,0.198358,0.146188,0.105011,0.073676,0.049418,0.033292,0.02213,0.01334,0.007582,0.004134,0.001878,0.000879,0.000326,6.9e-05,1.1e-05,5e-06,4e-06,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,2e-06,3e-06,1.8e-05,6.1e-05,9.9e-05,0.000232,0.000414,0.000657,0.001179,0.001961,0.002947,0.004428,0.006577,0.010038,0.013131,0.016968,0.022322,0.028292,0.03669,0.047001,0.058328,0.070344,0.0838490000000001,0.101838,0.119577,0.138734,0.159611,0.181203,0.205339,0.232452,0.262352,0.289362,0.317347,0.34204,0.365406,0.387317,0.406405,0.422871,0.436752,0.445505,0.451658,0.456692,0.45509,0.447841,0.429853,0.405343,0.377543,0.353993,0.338145,0.329892,0.333301,0.351119,0.384616,0.427987,0.475965,0.527593,0.575783,0.614146,0.634249,0.6342,0.617734,0.582209,0.529033,0.467484,0.396863,0.325426,0.258782,0.198358,0.146188,0.105011,0.073676,0.049418,0.033292,0.02213,0.01334,0.00758200000000001,0.004134,0.001878,0.000879,0.000326,6.90000000000001e-05,1.1e-05,5e-06,4e-06,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,2e-06,3e-06,1.8e-05,6.1e-05,9.9e-05,0.000232,0.000414,0.000657,0.001179,0.001961,0.002947,0.004428,0.00657700000000001,0.010038,0.013131,0.016968,0.022322,0.028292,0.03669,0.047001,0.058328,0.0703440000000001,0.083849,0.101838,0.119577,0.138734,0.159611,0.181203,0.205339,0.232452,0.262352,0.289362,0.317347,0.34204,0.365406,0.387317,0.406405,0.422871,0.436752,0.445505,0.451658,0.456691999999999,0.45509,0.447841,0.429853,0.405343,0.377543,0.353993,0.338145,0.329892,0.333301,0.351119,0.384616,0.427987,0.475965,0.527593,0.575782999999999,0.614146,0.634249000000001,0.6342,0.617734,0.582209,0.529033,0.467484,0.396863,0.325426,0.258782,0.198358,0.146188,0.105011,0.0736760000000001,0.049418,0.033292,0.02213,0.01334,0.00758199999999999,0.004134,0.001878,0.000879,0.000326,6.89999999999999e-05,1.1e-05,5e-06,4e-06,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,2e-06,3e-06,1.8e-05,6.1e-05,9.89999999999999e-05,0.000232,0.000414,0.000657,0.001179,0.001961,0.002947,0.00442799999999999,0.006577,0.010038,0.013131,0.016968,0.022322,0.028292,0.03669,0.047001,0.0583279999999999,0.070344,0.083849,0.101838,0.119577,0.138734,0.159611,0.181203,0.205339,0.232452,0.262352,0.289362,0.317347,0.34204,0.365406,0.387317,0.406405,0.422871000000001,0.436752,0.445505,0.451658,0.456692,0.455089999999999,0.447840999999999,0.429853,0.405342999999999,0.377543,0.353993,0.338145,0.329892,0.333301,0.351119,0.384616,0.427987,0.475965,0.527593,0.575783,0.614146,0.634249,0.6342,0.617734,0.582209000000001,0.529033,0.467484000000001,0.396863,0.325426,0.258782,0.198358,0.146188,0.105011,0.073676,0.049418,0.033292,0.02213,0.01334,0.00758200000000001,0.004134,0.001878,0.000879000000000001,0.000326,6.89999999999999e-05,1.1e-05,5e-06,4e-06,0,0,0,0,0,0,0],"text":["GC:   0<br />W/1e+06: 0.000000","GC:   1<br />W/1e+06: 0.000000","GC:   2<br />W/1e+06: 0.000000","GC:   3<br />W/1e+06: 0.000000","GC:   4<br />W/1e+06: 0.000000","GC:   5<br />W/1e+06: 0.000000","GC:   6<br />W/1e+06: 0.000000","GC:   7<br />W/1e+06: 0.000000","GC:   8<br />W/1e+06: 0.000000","GC:   9<br />W/1e+06: 0.000000","GC:  10<br />W/1e+06: 0.000002","GC:  11<br />W/1e+06: 0.000003","GC:  12<br />W/1e+06: 0.000018","GC:  13<br />W/1e+06: 0.000061","GC:  14<br />W/1e+06: 0.000099","GC:  15<br />W/1e+06: 0.000232","GC:  16<br />W/1e+06: 0.000414","GC:  17<br />W/1e+06: 0.000657","GC:  18<br />W/1e+06: 0.001179","GC:  19<br />W/1e+06: 0.001961","GC:  20<br />W/1e+06: 0.002947","GC:  21<br />W/1e+06: 0.004428","GC:  22<br />W/1e+06: 0.006577","GC:  23<br />W/1e+06: 0.010038","GC:  24<br />W/1e+06: 0.013131","GC:  25<br />W/1e+06: 0.016968","GC:  26<br />W/1e+06: 0.022322","GC:  27<br />W/1e+06: 0.028292","GC:  28<br />W/1e+06: 0.036690","GC:  29<br />W/1e+06: 0.047001","GC:  30<br />W/1e+06: 0.058328","GC:  31<br />W/1e+06: 0.070344","GC:  32<br />W/1e+06: 0.083849","GC:  33<br />W/1e+06: 0.101838","GC:  34<br />W/1e+06: 0.119577","GC:  35<br />W/1e+06: 0.138734","GC:  36<br />W/1e+06: 0.159611","GC:  37<br />W/1e+06: 0.181203","GC:  38<br />W/1e+06: 0.205339","GC:  39<br />W/1e+06: 0.232452","GC:  40<br />W/1e+06: 0.262352","GC:  41<br />W/1e+06: 0.289362","GC:  42<br />W/1e+06: 0.317347","GC:  43<br />W/1e+06: 0.342040","GC:  44<br />W/1e+06: 0.365406","GC:  45<br />W/1e+06: 0.387317","GC:  46<br />W/1e+06: 0.406405","GC:  47<br />W/1e+06: 0.422871","GC:  48<br />W/1e+06: 0.436752","GC:  49<br />W/1e+06: 0.445505","GC:  50<br />W/1e+06: 0.451658","GC:  51<br />W/1e+06: 0.456692","GC:  52<br />W/1e+06: 0.455090","GC:  53<br />W/1e+06: 0.447841","GC:  54<br />W/1e+06: 0.429853","GC:  55<br />W/1e+06: 0.405343","GC:  56<br />W/1e+06: 0.377543","GC:  57<br />W/1e+06: 0.353993","GC:  58<br />W/1e+06: 0.338145","GC:  59<br />W/1e+06: 0.329892","GC:  60<br />W/1e+06: 0.333301","GC:  61<br />W/1e+06: 0.351119","GC:  62<br />W/1e+06: 0.384616","GC:  63<br />W/1e+06: 0.427987","GC:  64<br />W/1e+06: 0.475965","GC:  65<br />W/1e+06: 0.527593","GC:  66<br />W/1e+06: 0.575783","GC:  67<br />W/1e+06: 0.614146","GC:  68<br />W/1e+06: 0.634249","GC:  69<br />W/1e+06: 0.634200","GC:  70<br />W/1e+06: 0.617734","GC:  71<br />W/1e+06: 0.582209","GC:  72<br />W/1e+06: 0.529033","GC:  73<br />W/1e+06: 0.467484","GC:  74<br />W/1e+06: 0.396863","GC:  75<br />W/1e+06: 0.325426","GC:  76<br />W/1e+06: 0.258782","GC:  77<br />W/1e+06: 0.198358","GC:  78<br />W/1e+06: 0.146188","GC:  79<br />W/1e+06: 0.105011","GC:  80<br />W/1e+06: 0.073676","GC:  81<br />W/1e+06: 0.049418","GC:  82<br />W/1e+06: 0.033292","GC:  83<br />W/1e+06: 0.022130","GC:  84<br />W/1e+06: 0.013340","GC:  85<br />W/1e+06: 0.007582","GC:  86<br />W/1e+06: 0.004134","GC:  87<br />W/1e+06: 0.001878","GC:  88<br />W/1e+06: 0.000879","GC:  89<br />W/1e+06: 0.000326","GC:  90<br />W/1e+06: 0.000069","GC:  91<br />W/1e+06: 0.000011","GC:  92<br />W/1e+06: 0.000005","GC:  93<br />W/1e+06: 0.000004","GC:  94<br />W/1e+06: 0.000000","GC:  95<br />W/1e+06: 0.000000","GC:  96<br />W/1e+06: 0.000000","GC:  97<br />W/1e+06: 0.000000","GC:  98<br />W/1e+06: 0.000000","GC:  99<br />W/1e+06: 0.000000","GC: 100<br />W/1e+06: 0.000000","GC:   0<br />W/1e+06: 0.000000","GC:   1<br />W/1e+06: 0.000000","GC:   2<br />W/1e+06: 0.000000","GC:   3<br />W/1e+06: 0.000000","GC:   4<br />W/1e+06: 0.000000","GC:   5<br />W/1e+06: 0.000000","GC:   6<br />W/1e+06: 0.000000","GC:   7<br />W/1e+06: 0.000000","GC:   8<br />W/1e+06: 0.000000","GC:   9<br />W/1e+06: 0.000000","GC:  10<br />W/1e+06: 0.000002","GC:  11<br />W/1e+06: 0.000003","GC:  12<br />W/1e+06: 0.000018","GC:  13<br />W/1e+06: 0.000061","GC:  14<br />W/1e+06: 0.000099","GC:  15<br />W/1e+06: 0.000232","GC:  16<br />W/1e+06: 0.000414","GC:  17<br />W/1e+06: 0.000657","GC:  18<br />W/1e+06: 0.001179","GC:  19<br />W/1e+06: 0.001961","GC:  20<br />W/1e+06: 0.002947","GC:  21<br />W/1e+06: 0.004428","GC:  22<br />W/1e+06: 0.006577","GC:  23<br />W/1e+06: 0.010038","GC:  24<br />W/1e+06: 0.013131","GC:  25<br />W/1e+06: 0.016968","GC:  26<br />W/1e+06: 0.022322","GC:  27<br />W/1e+06: 0.028292","GC:  28<br />W/1e+06: 0.036690","GC:  29<br />W/1e+06: 0.047001","GC:  30<br />W/1e+06: 0.058328","GC:  31<br />W/1e+06: 0.070344","GC:  32<br />W/1e+06: 0.083849","GC:  33<br />W/1e+06: 0.101838","GC:  34<br />W/1e+06: 0.119577","GC:  35<br />W/1e+06: 0.138734","GC:  36<br />W/1e+06: 0.159611","GC:  37<br />W/1e+06: 0.181203","GC:  38<br />W/1e+06: 0.205339","GC:  39<br />W/1e+06: 0.232452","GC:  40<br />W/1e+06: 0.262352","GC:  41<br />W/1e+06: 0.289362","GC:  42<br />W/1e+06: 0.317347","GC:  43<br />W/1e+06: 0.342040","GC:  44<br />W/1e+06: 0.365406","GC:  45<br />W/1e+06: 0.387317","GC:  46<br />W/1e+06: 0.406405","GC:  47<br />W/1e+06: 0.422871","GC:  48<br />W/1e+06: 0.436752","GC:  49<br />W/1e+06: 0.445505","GC:  50<br />W/1e+06: 0.451658","GC:  51<br />W/1e+06: 0.456692","GC:  52<br />W/1e+06: 0.455090","GC:  53<br />W/1e+06: 0.447841","GC:  54<br />W/1e+06: 0.429853","GC:  55<br />W/1e+06: 0.405343","GC:  56<br />W/1e+06: 0.377543","GC:  57<br />W/1e+06: 0.353993","GC:  58<br />W/1e+06: 0.338145","GC:  59<br />W/1e+06: 0.329892","GC:  60<br />W/1e+06: 0.333301","GC:  61<br />W/1e+06: 0.351119","GC:  62<br />W/1e+06: 0.384616","GC:  63<br />W/1e+06: 0.427987","GC:  64<br />W/1e+06: 0.475965","GC:  65<br />W/1e+06: 0.527593","GC:  66<br />W/1e+06: 0.575783","GC:  67<br />W/1e+06: 0.614146","GC:  68<br />W/1e+06: 0.634249","GC:  69<br />W/1e+06: 0.634200","GC:  70<br />W/1e+06: 0.617734","GC:  71<br />W/1e+06: 0.582209","GC:  72<br />W/1e+06: 0.529033","GC:  73<br />W/1e+06: 0.467484","GC:  74<br />W/1e+06: 0.396863","GC:  75<br />W/1e+06: 0.325426","GC:  76<br />W/1e+06: 0.258782","GC:  77<br />W/1e+06: 0.198358","GC:  78<br />W/1e+06: 0.146188","GC:  79<br />W/1e+06: 0.105011","GC:  80<br />W/1e+06: 0.073676","GC:  81<br />W/1e+06: 0.049418","GC:  82<br />W/1e+06: 0.033292","GC:  83<br />W/1e+06: 0.022130","GC:  84<br />W/1e+06: 0.013340","GC:  85<br />W/1e+06: 0.007582","GC:  86<br />W/1e+06: 0.004134","GC:  87<br />W/1e+06: 0.001878","GC:  88<br />W/1e+06: 0.000879","GC:  89<br />W/1e+06: 0.000326","GC:  90<br />W/1e+06: 0.000069","GC:  91<br />W/1e+06: 0.000011","GC:  92<br />W/1e+06: 0.000005","GC:  93<br />W/1e+06: 0.000004","GC:  94<br />W/1e+06: 0.000000","GC:  95<br />W/1e+06: 0.000000","GC:  96<br />W/1e+06: 0.000000","GC:  97<br />W/1e+06: 0.000000","GC:  98<br />W/1e+06: 0.000000","GC:  99<br />W/1e+06: 0.000000","GC: 100<br />W/1e+06: 0.000000","GC:   0<br />W/1e+06: 0.000000","GC:   1<br />W/1e+06: 0.000000","GC:   2<br />W/1e+06: 0.000000","GC:   3<br />W/1e+06: 0.000000","GC:   4<br />W/1e+06: 0.000000","GC:   5<br />W/1e+06: 0.000000","GC:   6<br />W/1e+06: 0.000000","GC:   7<br />W/1e+06: 0.000000","GC:   8<br />W/1e+06: 0.000000","GC:   9<br />W/1e+06: 0.000000","GC:  10<br />W/1e+06: 0.000002","GC:  11<br />W/1e+06: 0.000003","GC:  12<br />W/1e+06: 0.000018","GC:  13<br />W/1e+06: 0.000061","GC:  14<br />W/1e+06: 0.000099","GC:  15<br />W/1e+06: 0.000232","GC:  16<br />W/1e+06: 0.000414","GC:  17<br />W/1e+06: 0.000657","GC:  18<br />W/1e+06: 0.001179","GC:  19<br />W/1e+06: 0.001961","GC:  20<br />W/1e+06: 0.002947","GC:  21<br />W/1e+06: 0.004428","GC:  22<br />W/1e+06: 0.006577","GC:  23<br />W/1e+06: 0.010038","GC:  24<br />W/1e+06: 0.013131","GC:  25<br />W/1e+06: 0.016968","GC:  26<br />W/1e+06: 0.022322","GC:  27<br />W/1e+06: 0.028292","GC:  28<br />W/1e+06: 0.036690","GC:  29<br />W/1e+06: 0.047001","GC:  30<br />W/1e+06: 0.058328","GC:  31<br />W/1e+06: 0.070344","GC:  32<br />W/1e+06: 0.083849","GC:  33<br />W/1e+06: 0.101838","GC:  34<br />W/1e+06: 0.119577","GC:  35<br />W/1e+06: 0.138734","GC:  36<br />W/1e+06: 0.159611","GC:  37<br />W/1e+06: 0.181203","GC:  38<br />W/1e+06: 0.205339","GC:  39<br />W/1e+06: 0.232452","GC:  40<br />W/1e+06: 0.262352","GC:  41<br />W/1e+06: 0.289362","GC:  42<br />W/1e+06: 0.317347","GC:  43<br />W/1e+06: 0.342040","GC:  44<br />W/1e+06: 0.365406","GC:  45<br />W/1e+06: 0.387317","GC:  46<br />W/1e+06: 0.406405","GC:  47<br />W/1e+06: 0.422871","GC:  48<br />W/1e+06: 0.436752","GC:  49<br />W/1e+06: 0.445505","GC:  50<br />W/1e+06: 0.451658","GC:  51<br />W/1e+06: 0.456692","GC:  52<br />W/1e+06: 0.455090","GC:  53<br />W/1e+06: 0.447841","GC:  54<br />W/1e+06: 0.429853","GC:  55<br />W/1e+06: 0.405343","GC:  56<br />W/1e+06: 0.377543","GC:  57<br />W/1e+06: 0.353993","GC:  58<br />W/1e+06: 0.338145","GC:  59<br />W/1e+06: 0.329892","GC:  60<br />W/1e+06: 0.333301","GC:  61<br />W/1e+06: 0.351119","GC:  62<br />W/1e+06: 0.384616","GC:  63<br />W/1e+06: 0.427987","GC:  64<br />W/1e+06: 0.475965","GC:  65<br />W/1e+06: 0.527593","GC:  66<br />W/1e+06: 0.575783","GC:  67<br />W/1e+06: 0.614146","GC:  68<br />W/1e+06: 0.634249","GC:  69<br />W/1e+06: 0.634200","GC:  70<br />W/1e+06: 0.617734","GC:  71<br />W/1e+06: 0.582209","GC:  72<br />W/1e+06: 0.529033","GC:  73<br />W/1e+06: 0.467484","GC:  74<br />W/1e+06: 0.396863","GC:  75<br />W/1e+06: 0.325426","GC:  76<br />W/1e+06: 0.258782","GC:  77<br />W/1e+06: 0.198358","GC:  78<br />W/1e+06: 0.146188","GC:  79<br />W/1e+06: 0.105011","GC:  80<br />W/1e+06: 0.073676","GC:  81<br />W/1e+06: 0.049418","GC:  82<br />W/1e+06: 0.033292","GC:  83<br />W/1e+06: 0.022130","GC:  84<br />W/1e+06: 0.013340","GC:  85<br />W/1e+06: 0.007582","GC:  86<br />W/1e+06: 0.004134","GC:  87<br />W/1e+06: 0.001878","GC:  88<br />W/1e+06: 0.000879","GC:  89<br />W/1e+06: 0.000326","GC:  90<br />W/1e+06: 0.000069","GC:  91<br />W/1e+06: 0.000011","GC:  92<br />W/1e+06: 0.000005","GC:  93<br />W/1e+06: 0.000004","GC:  94<br />W/1e+06: 0.000000","GC:  95<br />W/1e+06: 0.000000","GC:  96<br />W/1e+06: 0.000000","GC:  97<br />W/1e+06: 0.000000","GC:  98<br />W/1e+06: 0.000000","GC:  99<br />W/1e+06: 0.000000","GC: 100<br />W/1e+06: 0.000000","GC:   0<br />W/1e+06: 0.000000","GC:   1<br />W/1e+06: 0.000000","GC:   2<br />W/1e+06: 0.000000","GC:   3<br />W/1e+06: 0.000000","GC:   4<br />W/1e+06: 0.000000","GC:   5<br />W/1e+06: 0.000000","GC:   6<br />W/1e+06: 0.000000","GC:   7<br />W/1e+06: 0.000000","GC:   8<br />W/1e+06: 0.000000","GC:   9<br />W/1e+06: 0.000000","GC:  10<br />W/1e+06: 0.000002","GC:  11<br />W/1e+06: 0.000003","GC:  12<br />W/1e+06: 0.000018","GC:  13<br />W/1e+06: 0.000061","GC:  14<br />W/1e+06: 0.000099","GC:  15<br />W/1e+06: 0.000232","GC:  16<br />W/1e+06: 0.000414","GC:  17<br />W/1e+06: 0.000657","GC:  18<br />W/1e+06: 0.001179","GC:  19<br />W/1e+06: 0.001961","GC:  20<br />W/1e+06: 0.002947","GC:  21<br />W/1e+06: 0.004428","GC:  22<br />W/1e+06: 0.006577","GC:  23<br />W/1e+06: 0.010038","GC:  24<br />W/1e+06: 0.013131","GC:  25<br />W/1e+06: 0.016968","GC:  26<br />W/1e+06: 0.022322","GC:  27<br />W/1e+06: 0.028292","GC:  28<br />W/1e+06: 0.036690","GC:  29<br />W/1e+06: 0.047001","GC:  30<br />W/1e+06: 0.058328","GC:  31<br />W/1e+06: 0.070344","GC:  32<br />W/1e+06: 0.083849","GC:  33<br />W/1e+06: 0.101838","GC:  34<br />W/1e+06: 0.119577","GC:  35<br />W/1e+06: 0.138734","GC:  36<br />W/1e+06: 0.159611","GC:  37<br />W/1e+06: 0.181203","GC:  38<br />W/1e+06: 0.205339","GC:  39<br />W/1e+06: 0.232452","GC:  40<br />W/1e+06: 0.262352","GC:  41<br />W/1e+06: 0.289362","GC:  42<br />W/1e+06: 0.317347","GC:  43<br />W/1e+06: 0.342040","GC:  44<br />W/1e+06: 0.365406","GC:  45<br />W/1e+06: 0.387317","GC:  46<br />W/1e+06: 0.406405","GC:  47<br />W/1e+06: 0.422871","GC:  48<br />W/1e+06: 0.436752","GC:  49<br />W/1e+06: 0.445505","GC:  50<br />W/1e+06: 0.451658","GC:  51<br />W/1e+06: 0.456692","GC:  52<br />W/1e+06: 0.455090","GC:  53<br />W/1e+06: 0.447841","GC:  54<br />W/1e+06: 0.429853","GC:  55<br />W/1e+06: 0.405343","GC:  56<br />W/1e+06: 0.377543","GC:  57<br />W/1e+06: 0.353993","GC:  58<br />W/1e+06: 0.338145","GC:  59<br />W/1e+06: 0.329892","GC:  60<br />W/1e+06: 0.333301","GC:  61<br />W/1e+06: 0.351119","GC:  62<br />W/1e+06: 0.384616","GC:  63<br />W/1e+06: 0.427987","GC:  64<br />W/1e+06: 0.475965","GC:  65<br />W/1e+06: 0.527593","GC:  66<br />W/1e+06: 0.575783","GC:  67<br />W/1e+06: 0.614146","GC:  68<br />W/1e+06: 0.634249","GC:  69<br />W/1e+06: 0.634200","GC:  70<br />W/1e+06: 0.617734","GC:  71<br />W/1e+06: 0.582209","GC:  72<br />W/1e+06: 0.529033","GC:  73<br />W/1e+06: 0.467484","GC:  74<br />W/1e+06: 0.396863","GC:  75<br />W/1e+06: 0.325426","GC:  76<br />W/1e+06: 0.258782","GC:  77<br />W/1e+06: 0.198358","GC:  78<br />W/1e+06: 0.146188","GC:  79<br />W/1e+06: 0.105011","GC:  80<br />W/1e+06: 0.073676","GC:  81<br />W/1e+06: 0.049418","GC:  82<br />W/1e+06: 0.033292","GC:  83<br />W/1e+06: 0.022130","GC:  84<br />W/1e+06: 0.013340","GC:  85<br />W/1e+06: 0.007582","GC:  86<br />W/1e+06: 0.004134","GC:  87<br />W/1e+06: 0.001878","GC:  88<br />W/1e+06: 0.000879","GC:  89<br />W/1e+06: 0.000326","GC:  90<br />W/1e+06: 0.000069","GC:  91<br />W/1e+06: 0.000011","GC:  92<br />W/1e+06: 0.000005","GC:  93<br />W/1e+06: 0.000004","GC:  94<br />W/1e+06: 0.000000","GC:  95<br />W/1e+06: 0.000000","GC:  96<br />W/1e+06: 0.000000","GC:  97<br />W/1e+06: 0.000000","GC:  98<br />W/1e+06: 0.000000","GC:  99<br />W/1e+06: 0.000000","GC: 100<br />W/1e+06: 0.000000","GC:   0<br />W/1e+06: 0.000000","GC:   1<br />W/1e+06: 0.000000","GC:   2<br />W/1e+06: 0.000000","GC:   3<br />W/1e+06: 0.000000","GC:   4<br />W/1e+06: 0.000000","GC:   5<br />W/1e+06: 0.000000","GC:   6<br />W/1e+06: 0.000000","GC:   7<br />W/1e+06: 0.000000","GC:   8<br />W/1e+06: 0.000000","GC:   9<br />W/1e+06: 0.000000","GC:  10<br />W/1e+06: 0.000002","GC:  11<br />W/1e+06: 0.000003","GC:  12<br />W/1e+06: 0.000018","GC:  13<br />W/1e+06: 0.000061","GC:  14<br />W/1e+06: 0.000099","GC:  15<br />W/1e+06: 0.000232","GC:  16<br />W/1e+06: 0.000414","GC:  17<br />W/1e+06: 0.000657","GC:  18<br />W/1e+06: 0.001179","GC:  19<br />W/1e+06: 0.001961","GC:  20<br />W/1e+06: 0.002947","GC:  21<br />W/1e+06: 0.004428","GC:  22<br />W/1e+06: 0.006577","GC:  23<br />W/1e+06: 0.010038","GC:  24<br />W/1e+06: 0.013131","GC:  25<br />W/1e+06: 0.016968","GC:  26<br />W/1e+06: 0.022322","GC:  27<br />W/1e+06: 0.028292","GC:  28<br />W/1e+06: 0.036690","GC:  29<br />W/1e+06: 0.047001","GC:  30<br />W/1e+06: 0.058328","GC:  31<br />W/1e+06: 0.070344","GC:  32<br />W/1e+06: 0.083849","GC:  33<br />W/1e+06: 0.101838","GC:  34<br />W/1e+06: 0.119577","GC:  35<br />W/1e+06: 0.138734","GC:  36<br />W/1e+06: 0.159611","GC:  37<br />W/1e+06: 0.181203","GC:  38<br />W/1e+06: 0.205339","GC:  39<br />W/1e+06: 0.232452","GC:  40<br />W/1e+06: 0.262352","GC:  41<br />W/1e+06: 0.289362","GC:  42<br />W/1e+06: 0.317347","GC:  43<br />W/1e+06: 0.342040","GC:  44<br />W/1e+06: 0.365406","GC:  45<br />W/1e+06: 0.387317","GC:  46<br />W/1e+06: 0.406405","GC:  47<br />W/1e+06: 0.422871","GC:  48<br />W/1e+06: 0.436752","GC:  49<br />W/1e+06: 0.445505","GC:  50<br />W/1e+06: 0.451658","GC:  51<br />W/1e+06: 0.456692","GC:  52<br />W/1e+06: 0.455090","GC:  53<br />W/1e+06: 0.447841","GC:  54<br />W/1e+06: 0.429853","GC:  55<br />W/1e+06: 0.405343","GC:  56<br />W/1e+06: 0.377543","GC:  57<br />W/1e+06: 0.353993","GC:  58<br />W/1e+06: 0.338145","GC:  59<br />W/1e+06: 0.329892","GC:  60<br />W/1e+06: 0.333301","GC:  61<br />W/1e+06: 0.351119","GC:  62<br />W/1e+06: 0.384616","GC:  63<br />W/1e+06: 0.427987","GC:  64<br />W/1e+06: 0.475965","GC:  65<br />W/1e+06: 0.527593","GC:  66<br />W/1e+06: 0.575783","GC:  67<br />W/1e+06: 0.614146","GC:  68<br />W/1e+06: 0.634249","GC:  69<br />W/1e+06: 0.634200","GC:  70<br />W/1e+06: 0.617734","GC:  71<br />W/1e+06: 0.582209","GC:  72<br />W/1e+06: 0.529033","GC:  73<br />W/1e+06: 0.467484","GC:  74<br />W/1e+06: 0.396863","GC:  75<br />W/1e+06: 0.325426","GC:  76<br />W/1e+06: 0.258782","GC:  77<br />W/1e+06: 0.198358","GC:  78<br />W/1e+06: 0.146188","GC:  79<br />W/1e+06: 0.105011","GC:  80<br />W/1e+06: 0.073676","GC:  81<br />W/1e+06: 0.049418","GC:  82<br />W/1e+06: 0.033292","GC:  83<br />W/1e+06: 0.022130","GC:  84<br />W/1e+06: 0.013340","GC:  85<br />W/1e+06: 0.007582","GC:  86<br />W/1e+06: 0.004134","GC:  87<br />W/1e+06: 0.001878","GC:  88<br />W/1e+06: 0.000879","GC:  89<br />W/1e+06: 0.000326","GC:  90<br />W/1e+06: 0.000069","GC:  91<br />W/1e+06: 0.000011","GC:  92<br />W/1e+06: 0.000005","GC:  93<br />W/1e+06: 0.000004","GC:  94<br />W/1e+06: 0.000000","GC:  95<br />W/1e+06: 0.000000","GC:  96<br />W/1e+06: 0.000000","GC:  97<br />W/1e+06: 0.000000","GC:  98<br />W/1e+06: 0.000000","GC:  99<br />W/1e+06: 0.000000","GC: 100<br />W/1e+06: 0.000000","GC:   0<br />W/1e+06: 0.000000","GC:   1<br />W/1e+06: 0.000000","GC:   2<br />W/1e+06: 0.000000","GC:   3<br />W/1e+06: 0.000000","GC:   4<br />W/1e+06: 0.000000","GC:   5<br />W/1e+06: 0.000000","GC:   6<br />W/1e+06: 0.000000","GC:   7<br />W/1e+06: 0.000000","GC:   8<br />W/1e+06: 0.000000","GC:   9<br />W/1e+06: 0.000000","GC:  10<br />W/1e+06: 0.000002","GC:  11<br />W/1e+06: 0.000003","GC:  12<br />W/1e+06: 0.000018","GC:  13<br />W/1e+06: 0.000061","GC:  14<br />W/1e+06: 0.000099","GC:  15<br />W/1e+06: 0.000232","GC:  16<br />W/1e+06: 0.000414","GC:  17<br />W/1e+06: 0.000657","GC:  18<br />W/1e+06: 0.001179","GC:  19<br />W/1e+06: 0.001961","GC:  20<br />W/1e+06: 0.002947","GC:  21<br />W/1e+06: 0.004428","GC:  22<br />W/1e+06: 0.006577","GC:  23<br />W/1e+06: 0.010038","GC:  24<br />W/1e+06: 0.013131","GC:  25<br />W/1e+06: 0.016968","GC:  26<br />W/1e+06: 0.022322","GC:  27<br />W/1e+06: 0.028292","GC:  28<br />W/1e+06: 0.036690","GC:  29<br />W/1e+06: 0.047001","GC:  30<br />W/1e+06: 0.058328","GC:  31<br />W/1e+06: 0.070344","GC:  32<br />W/1e+06: 0.083849","GC:  33<br />W/1e+06: 0.101838","GC:  34<br />W/1e+06: 0.119577","GC:  35<br />W/1e+06: 0.138734","GC:  36<br />W/1e+06: 0.159611","GC:  37<br />W/1e+06: 0.181203","GC:  38<br />W/1e+06: 0.205339","GC:  39<br />W/1e+06: 0.232452","GC:  40<br />W/1e+06: 0.262352","GC:  41<br />W/1e+06: 0.289362","GC:  42<br />W/1e+06: 0.317347","GC:  43<br />W/1e+06: 0.342040","GC:  44<br />W/1e+06: 0.365406","GC:  45<br />W/1e+06: 0.387317","GC:  46<br />W/1e+06: 0.406405","GC:  47<br />W/1e+06: 0.422871","GC:  48<br />W/1e+06: 0.436752","GC:  49<br />W/1e+06: 0.445505","GC:  50<br />W/1e+06: 0.451658","GC:  51<br />W/1e+06: 0.456692","GC:  52<br />W/1e+06: 0.455090","GC:  53<br />W/1e+06: 0.447841","GC:  54<br />W/1e+06: 0.429853","GC:  55<br />W/1e+06: 0.405343","GC:  56<br />W/1e+06: 0.377543","GC:  57<br />W/1e+06: 0.353993","GC:  58<br />W/1e+06: 0.338145","GC:  59<br />W/1e+06: 0.329892","GC:  60<br />W/1e+06: 0.333301","GC:  61<br />W/1e+06: 0.351119","GC:  62<br />W/1e+06: 0.384616","GC:  63<br />W/1e+06: 0.427987","GC:  64<br />W/1e+06: 0.475965","GC:  65<br />W/1e+06: 0.527593","GC:  66<br />W/1e+06: 0.575783","GC:  67<br />W/1e+06: 0.614146","GC:  68<br />W/1e+06: 0.634249","GC:  69<br />W/1e+06: 0.634200","GC:  70<br />W/1e+06: 0.617734","GC:  71<br />W/1e+06: 0.582209","GC:  72<br />W/1e+06: 0.529033","GC:  73<br />W/1e+06: 0.467484","GC:  74<br />W/1e+06: 0.396863","GC:  75<br />W/1e+06: 0.325426","GC:  76<br />W/1e+06: 0.258782","GC:  77<br />W/1e+06: 0.198358","GC:  78<br />W/1e+06: 0.146188","GC:  79<br />W/1e+06: 0.105011","GC:  80<br />W/1e+06: 0.073676","GC:  81<br />W/1e+06: 0.049418","GC:  82<br />W/1e+06: 0.033292","GC:  83<br />W/1e+06: 0.022130","GC:  84<br />W/1e+06: 0.013340","GC:  85<br />W/1e+06: 0.007582","GC:  86<br />W/1e+06: 0.004134","GC:  87<br />W/1e+06: 0.001878","GC:  88<br />W/1e+06: 0.000879","GC:  89<br />W/1e+06: 0.000326","GC:  90<br />W/1e+06: 0.000069","GC:  91<br />W/1e+06: 0.000011","GC:  92<br />W/1e+06: 0.000005","GC:  93<br />W/1e+06: 0.000004","GC:  94<br />W/1e+06: 0.000000","GC:  95<br />W/1e+06: 0.000000","GC:  96<br />W/1e+06: 0.000000","GC:  97<br />W/1e+06: 0.000000","GC:  98<br />W/1e+06: 0.000000","GC:  99<br />W/1e+06: 0.000000","GC: 100<br />W/1e+06: 0.000000","GC:   0<br />W/1e+06: 0.000000","GC:   1<br />W/1e+06: 0.000000","GC:   2<br />W/1e+06: 0.000000","GC:   3<br />W/1e+06: 0.000000","GC:   4<br />W/1e+06: 0.000000","GC:   5<br />W/1e+06: 0.000000","GC:   6<br />W/1e+06: 0.000000","GC:   7<br />W/1e+06: 0.000000","GC:   8<br />W/1e+06: 0.000000","GC:   9<br />W/1e+06: 0.000000","GC:  10<br />W/1e+06: 0.000002","GC:  11<br />W/1e+06: 0.000003","GC:  12<br />W/1e+06: 0.000018","GC:  13<br />W/1e+06: 0.000061","GC:  14<br />W/1e+06: 0.000099","GC:  15<br />W/1e+06: 0.000232","GC:  16<br />W/1e+06: 0.000414","GC:  17<br />W/1e+06: 0.000657","GC:  18<br />W/1e+06: 0.001179","GC:  19<br />W/1e+06: 0.001961","GC:  20<br />W/1e+06: 0.002947","GC:  21<br />W/1e+06: 0.004428","GC:  22<br />W/1e+06: 0.006577","GC:  23<br />W/1e+06: 0.010038","GC:  24<br />W/1e+06: 0.013131","GC:  25<br />W/1e+06: 0.016968","GC:  26<br />W/1e+06: 0.022322","GC:  27<br />W/1e+06: 0.028292","GC:  28<br />W/1e+06: 0.036690","GC:  29<br />W/1e+06: 0.047001","GC:  30<br />W/1e+06: 0.058328","GC:  31<br />W/1e+06: 0.070344","GC:  32<br />W/1e+06: 0.083849","GC:  33<br />W/1e+06: 0.101838","GC:  34<br />W/1e+06: 0.119577","GC:  35<br />W/1e+06: 0.138734","GC:  36<br />W/1e+06: 0.159611","GC:  37<br />W/1e+06: 0.181203","GC:  38<br />W/1e+06: 0.205339","GC:  39<br />W/1e+06: 0.232452","GC:  40<br />W/1e+06: 0.262352","GC:  41<br />W/1e+06: 0.289362","GC:  42<br />W/1e+06: 0.317347","GC:  43<br />W/1e+06: 0.342040","GC:  44<br />W/1e+06: 0.365406","GC:  45<br />W/1e+06: 0.387317","GC:  46<br />W/1e+06: 0.406405","GC:  47<br />W/1e+06: 0.422871","GC:  48<br />W/1e+06: 0.436752","GC:  49<br />W/1e+06: 0.445505","GC:  50<br />W/1e+06: 0.451658","GC:  51<br />W/1e+06: 0.456692","GC:  52<br />W/1e+06: 0.455090","GC:  53<br />W/1e+06: 0.447841","GC:  54<br />W/1e+06: 0.429853","GC:  55<br />W/1e+06: 0.405343","GC:  56<br />W/1e+06: 0.377543","GC:  57<br />W/1e+06: 0.353993","GC:  58<br />W/1e+06: 0.338145","GC:  59<br />W/1e+06: 0.329892","GC:  60<br />W/1e+06: 0.333301","GC:  61<br />W/1e+06: 0.351119","GC:  62<br />W/1e+06: 0.384616","GC:  63<br />W/1e+06: 0.427987","GC:  64<br />W/1e+06: 0.475965","GC:  65<br />W/1e+06: 0.527593","GC:  66<br />W/1e+06: 0.575783","GC:  67<br />W/1e+06: 0.614146","GC:  68<br />W/1e+06: 0.634249","GC:  69<br />W/1e+06: 0.634200","GC:  70<br />W/1e+06: 0.617734","GC:  71<br />W/1e+06: 0.582209","GC:  72<br />W/1e+06: 0.529033","GC:  73<br />W/1e+06: 0.467484","GC:  74<br />W/1e+06: 0.396863","GC:  75<br />W/1e+06: 0.325426","GC:  76<br />W/1e+06: 0.258782","GC:  77<br />W/1e+06: 0.198358","GC:  78<br />W/1e+06: 0.146188","GC:  79<br />W/1e+06: 0.105011","GC:  80<br />W/1e+06: 0.073676","GC:  81<br />W/1e+06: 0.049418","GC:  82<br />W/1e+06: 0.033292","GC:  83<br />W/1e+06: 0.022130","GC:  84<br />W/1e+06: 0.013340","GC:  85<br />W/1e+06: 0.007582","GC:  86<br />W/1e+06: 0.004134","GC:  87<br />W/1e+06: 0.001878","GC:  88<br />W/1e+06: 0.000879","GC:  89<br />W/1e+06: 0.000326","GC:  90<br />W/1e+06: 0.000069","GC:  91<br />W/1e+06: 0.000011","GC:  92<br />W/1e+06: 0.000005","GC:  93<br />W/1e+06: 0.000004","GC:  94<br />W/1e+06: 0.000000","GC:  95<br />W/1e+06: 0.000000","GC:  96<br />W/1e+06: 0.000000","GC:  97<br />W/1e+06: 0.000000","GC:  98<br />W/1e+06: 0.000000","GC:  99<br />W/1e+06: 0.000000","GC: 100<br />W/1e+06: 0.000000","GC:   0<br />W/1e+06: 0.000000","GC:   1<br />W/1e+06: 0.000000","GC:   2<br />W/1e+06: 0.000000","GC:   3<br />W/1e+06: 0.000000","GC:   4<br />W/1e+06: 0.000000","GC:   5<br />W/1e+06: 0.000000","GC:   6<br />W/1e+06: 0.000000","GC:   7<br />W/1e+06: 0.000000","GC:   8<br />W/1e+06: 0.000000","GC:   9<br />W/1e+06: 0.000000","GC:  10<br />W/1e+06: 0.000002","GC:  11<br />W/1e+06: 0.000003","GC:  12<br />W/1e+06: 0.000018","GC:  13<br />W/1e+06: 0.000061","GC:  14<br />W/1e+06: 0.000099","GC:  15<br />W/1e+06: 0.000232","GC:  16<br />W/1e+06: 0.000414","GC:  17<br />W/1e+06: 0.000657","GC:  18<br />W/1e+06: 0.001179","GC:  19<br />W/1e+06: 0.001961","GC:  20<br />W/1e+06: 0.002947","GC:  21<br />W/1e+06: 0.004428","GC:  22<br />W/1e+06: 0.006577","GC:  23<br />W/1e+06: 0.010038","GC:  24<br />W/1e+06: 0.013131","GC:  25<br />W/1e+06: 0.016968","GC:  26<br />W/1e+06: 0.022322","GC:  27<br />W/1e+06: 0.028292","GC:  28<br />W/1e+06: 0.036690","GC:  29<br />W/1e+06: 0.047001","GC:  30<br />W/1e+06: 0.058328","GC:  31<br />W/1e+06: 0.070344","GC:  32<br />W/1e+06: 0.083849","GC:  33<br />W/1e+06: 0.101838","GC:  34<br />W/1e+06: 0.119577","GC:  35<br />W/1e+06: 0.138734","GC:  36<br />W/1e+06: 0.159611","GC:  37<br />W/1e+06: 0.181203","GC:  38<br />W/1e+06: 0.205339","GC:  39<br />W/1e+06: 0.232452","GC:  40<br />W/1e+06: 0.262352","GC:  41<br />W/1e+06: 0.289362","GC:  42<br />W/1e+06: 0.317347","GC:  43<br />W/1e+06: 0.342040","GC:  44<br />W/1e+06: 0.365406","GC:  45<br />W/1e+06: 0.387317","GC:  46<br />W/1e+06: 0.406405","GC:  47<br />W/1e+06: 0.422871","GC:  48<br />W/1e+06: 0.436752","GC:  49<br />W/1e+06: 0.445505","GC:  50<br />W/1e+06: 0.451658","GC:  51<br />W/1e+06: 0.456692","GC:  52<br />W/1e+06: 0.455090","GC:  53<br />W/1e+06: 0.447841","GC:  54<br />W/1e+06: 0.429853","GC:  55<br />W/1e+06: 0.405343","GC:  56<br />W/1e+06: 0.377543","GC:  57<br />W/1e+06: 0.353993","GC:  58<br />W/1e+06: 0.338145","GC:  59<br />W/1e+06: 0.329892","GC:  60<br />W/1e+06: 0.333301","GC:  61<br />W/1e+06: 0.351119","GC:  62<br />W/1e+06: 0.384616","GC:  63<br />W/1e+06: 0.427987","GC:  64<br />W/1e+06: 0.475965","GC:  65<br />W/1e+06: 0.527593","GC:  66<br />W/1e+06: 0.575783","GC:  67<br />W/1e+06: 0.614146","GC:  68<br />W/1e+06: 0.634249","GC:  69<br />W/1e+06: 0.634200","GC:  70<br />W/1e+06: 0.617734","GC:  71<br />W/1e+06: 0.582209","GC:  72<br />W/1e+06: 0.529033","GC:  73<br />W/1e+06: 0.467484","GC:  74<br />W/1e+06: 0.396863","GC:  75<br />W/1e+06: 0.325426","GC:  76<br />W/1e+06: 0.258782","GC:  77<br />W/1e+06: 0.198358","GC:  78<br />W/1e+06: 0.146188","GC:  79<br />W/1e+06: 0.105011","GC:  80<br />W/1e+06: 0.073676","GC:  81<br />W/1e+06: 0.049418","GC:  82<br />W/1e+06: 0.033292","GC:  83<br />W/1e+06: 0.022130","GC:  84<br />W/1e+06: 0.013340","GC:  85<br />W/1e+06: 0.007582","GC:  86<br />W/1e+06: 0.004134","GC:  87<br />W/1e+06: 0.001878","GC:  88<br />W/1e+06: 0.000879","GC:  89<br />W/1e+06: 0.000326","GC:  90<br />W/1e+06: 0.000069","GC:  91<br />W/1e+06: 0.000011","GC:  92<br />W/1e+06: 0.000005","GC:  93<br />W/1e+06: 0.000004","GC:  94<br />W/1e+06: 0.000000","GC:  95<br />W/1e+06: 0.000000","GC:  96<br />W/1e+06: 0.000000","GC:  97<br />W/1e+06: 0.000000","GC:  98<br />W/1e+06: 0.000000","GC:  99<br />W/1e+06: 0.000000","GC: 100<br />W/1e+06: 0.000000","GC:   0<br />W/1e+06: 0.000000","GC:   1<br />W/1e+06: 0.000000","GC:   2<br />W/1e+06: 0.000000","GC:   3<br />W/1e+06: 0.000000","GC:   4<br />W/1e+06: 0.000000","GC:   5<br />W/1e+06: 0.000000","GC:   6<br />W/1e+06: 0.000000","GC:   7<br />W/1e+06: 0.000000","GC:   8<br />W/1e+06: 0.000000","GC:   9<br />W/1e+06: 0.000000","GC:  10<br />W/1e+06: 0.000002","GC:  11<br />W/1e+06: 0.000003","GC:  12<br />W/1e+06: 0.000018","GC:  13<br />W/1e+06: 0.000061","GC:  14<br />W/1e+06: 0.000099","GC:  15<br />W/1e+06: 0.000232","GC:  16<br />W/1e+06: 0.000414","GC:  17<br />W/1e+06: 0.000657","GC:  18<br />W/1e+06: 0.001179","GC:  19<br />W/1e+06: 0.001961","GC:  20<br />W/1e+06: 0.002947","GC:  21<br />W/1e+06: 0.004428","GC:  22<br />W/1e+06: 0.006577","GC:  23<br />W/1e+06: 0.010038","GC:  24<br />W/1e+06: 0.013131","GC:  25<br />W/1e+06: 0.016968","GC:  26<br />W/1e+06: 0.022322","GC:  27<br />W/1e+06: 0.028292","GC:  28<br />W/1e+06: 0.036690","GC:  29<br />W/1e+06: 0.047001","GC:  30<br />W/1e+06: 0.058328","GC:  31<br />W/1e+06: 0.070344","GC:  32<br />W/1e+06: 0.083849","GC:  33<br />W/1e+06: 0.101838","GC:  34<br />W/1e+06: 0.119577","GC:  35<br />W/1e+06: 0.138734","GC:  36<br />W/1e+06: 0.159611","GC:  37<br />W/1e+06: 0.181203","GC:  38<br />W/1e+06: 0.205339","GC:  39<br />W/1e+06: 0.232452","GC:  40<br />W/1e+06: 0.262352","GC:  41<br />W/1e+06: 0.289362","GC:  42<br />W/1e+06: 0.317347","GC:  43<br />W/1e+06: 0.342040","GC:  44<br />W/1e+06: 0.365406","GC:  45<br />W/1e+06: 0.387317","GC:  46<br />W/1e+06: 0.406405","GC:  47<br />W/1e+06: 0.422871","GC:  48<br />W/1e+06: 0.436752","GC:  49<br />W/1e+06: 0.445505","GC:  50<br />W/1e+06: 0.451658","GC:  51<br />W/1e+06: 0.456692","GC:  52<br />W/1e+06: 0.455090","GC:  53<br />W/1e+06: 0.447841","GC:  54<br />W/1e+06: 0.429853","GC:  55<br />W/1e+06: 0.405343","GC:  56<br />W/1e+06: 0.377543","GC:  57<br />W/1e+06: 0.353993","GC:  58<br />W/1e+06: 0.338145","GC:  59<br />W/1e+06: 0.329892","GC:  60<br />W/1e+06: 0.333301","GC:  61<br />W/1e+06: 0.351119","GC:  62<br />W/1e+06: 0.384616","GC:  63<br />W/1e+06: 0.427987","GC:  64<br />W/1e+06: 0.475965","GC:  65<br />W/1e+06: 0.527593","GC:  66<br />W/1e+06: 0.575783","GC:  67<br />W/1e+06: 0.614146","GC:  68<br />W/1e+06: 0.634249","GC:  69<br />W/1e+06: 0.634200","GC:  70<br />W/1e+06: 0.617734","GC:  71<br />W/1e+06: 0.582209","GC:  72<br />W/1e+06: 0.529033","GC:  73<br />W/1e+06: 0.467484","GC:  74<br />W/1e+06: 0.396863","GC:  75<br />W/1e+06: 0.325426","GC:  76<br />W/1e+06: 0.258782","GC:  77<br />W/1e+06: 0.198358","GC:  78<br />W/1e+06: 0.146188","GC:  79<br />W/1e+06: 0.105011","GC:  80<br />W/1e+06: 0.073676","GC:  81<br />W/1e+06: 0.049418","GC:  82<br />W/1e+06: 0.033292","GC:  83<br />W/1e+06: 0.022130","GC:  84<br />W/1e+06: 0.013340","GC:  85<br />W/1e+06: 0.007582","GC:  86<br />W/1e+06: 0.004134","GC:  87<br />W/1e+06: 0.001878","GC:  88<br />W/1e+06: 0.000879","GC:  89<br />W/1e+06: 0.000326","GC:  90<br />W/1e+06: 0.000069","GC:  91<br />W/1e+06: 0.000011","GC:  92<br />W/1e+06: 0.000005","GC:  93<br />W/1e+06: 0.000004","GC:  94<br />W/1e+06: 0.000000","GC:  95<br />W/1e+06: 0.000000","GC:  96<br />W/1e+06: 0.000000","GC:  97<br />W/1e+06: 0.000000","GC:  98<br />W/1e+06: 0.000000","GC:  99<br />W/1e+06: 0.000000","GC: 100<br />W/1e+06: 0.000000","GC:   0<br />W/1e+06: 0.000000","GC:   1<br />W/1e+06: 0.000000","GC:   2<br />W/1e+06: 0.000000","GC:   3<br />W/1e+06: 0.000000","GC:   4<br />W/1e+06: 0.000000","GC:   5<br />W/1e+06: 0.000000","GC:   6<br />W/1e+06: 0.000000","GC:   7<br />W/1e+06: 0.000000","GC:   8<br />W/1e+06: 0.000000","GC:   9<br />W/1e+06: 0.000000","GC:  10<br />W/1e+06: 0.000002","GC:  11<br />W/1e+06: 0.000003","GC:  12<br />W/1e+06: 0.000018","GC:  13<br />W/1e+06: 0.000061","GC:  14<br />W/1e+06: 0.000099","GC:  15<br />W/1e+06: 0.000232","GC:  16<br />W/1e+06: 0.000414","GC:  17<br />W/1e+06: 0.000657","GC:  18<br />W/1e+06: 0.001179","GC:  19<br />W/1e+06: 0.001961","GC:  20<br />W/1e+06: 0.002947","GC:  21<br />W/1e+06: 0.004428","GC:  22<br />W/1e+06: 0.006577","GC:  23<br />W/1e+06: 0.010038","GC:  24<br />W/1e+06: 0.013131","GC:  25<br />W/1e+06: 0.016968","GC:  26<br />W/1e+06: 0.022322","GC:  27<br />W/1e+06: 0.028292","GC:  28<br />W/1e+06: 0.036690","GC:  29<br />W/1e+06: 0.047001","GC:  30<br />W/1e+06: 0.058328","GC:  31<br />W/1e+06: 0.070344","GC:  32<br />W/1e+06: 0.083849","GC:  33<br />W/1e+06: 0.101838","GC:  34<br />W/1e+06: 0.119577","GC:  35<br />W/1e+06: 0.138734","GC:  36<br />W/1e+06: 0.159611","GC:  37<br />W/1e+06: 0.181203","GC:  38<br />W/1e+06: 0.205339","GC:  39<br />W/1e+06: 0.232452","GC:  40<br />W/1e+06: 0.262352","GC:  41<br />W/1e+06: 0.289362","GC:  42<br />W/1e+06: 0.317347","GC:  43<br />W/1e+06: 0.342040","GC:  44<br />W/1e+06: 0.365406","GC:  45<br />W/1e+06: 0.387317","GC:  46<br />W/1e+06: 0.406405","GC:  47<br />W/1e+06: 0.422871","GC:  48<br />W/1e+06: 0.436752","GC:  49<br />W/1e+06: 0.445505","GC:  50<br />W/1e+06: 0.451658","GC:  51<br />W/1e+06: 0.456692","GC:  52<br />W/1e+06: 0.455090","GC:  53<br />W/1e+06: 0.447841","GC:  54<br />W/1e+06: 0.429853","GC:  55<br />W/1e+06: 0.405343","GC:  56<br />W/1e+06: 0.377543","GC:  57<br />W/1e+06: 0.353993","GC:  58<br />W/1e+06: 0.338145","GC:  59<br />W/1e+06: 0.329892","GC:  60<br />W/1e+06: 0.333301","GC:  61<br />W/1e+06: 0.351119","GC:  62<br />W/1e+06: 0.384616","GC:  63<br />W/1e+06: 0.427987","GC:  64<br />W/1e+06: 0.475965","GC:  65<br />W/1e+06: 0.527593","GC:  66<br />W/1e+06: 0.575783","GC:  67<br />W/1e+06: 0.614146","GC:  68<br />W/1e+06: 0.634249","GC:  69<br />W/1e+06: 0.634200","GC:  70<br />W/1e+06: 0.617734","GC:  71<br />W/1e+06: 0.582209","GC:  72<br />W/1e+06: 0.529033","GC:  73<br />W/1e+06: 0.467484","GC:  74<br />W/1e+06: 0.396863","GC:  75<br />W/1e+06: 0.325426","GC:  76<br />W/1e+06: 0.258782","GC:  77<br />W/1e+06: 0.198358","GC:  78<br />W/1e+06: 0.146188","GC:  79<br />W/1e+06: 0.105011","GC:  80<br />W/1e+06: 0.073676","GC:  81<br />W/1e+06: 0.049418","GC:  82<br />W/1e+06: 0.033292","GC:  83<br />W/1e+06: 0.022130","GC:  84<br />W/1e+06: 0.013340","GC:  85<br />W/1e+06: 0.007582","GC:  86<br />W/1e+06: 0.004134","GC:  87<br />W/1e+06: 0.001878","GC:  88<br />W/1e+06: 0.000879","GC:  89<br />W/1e+06: 0.000326","GC:  90<br />W/1e+06: 0.000069","GC:  91<br />W/1e+06: 0.000011","GC:  92<br />W/1e+06: 0.000005","GC:  93<br />W/1e+06: 0.000004","GC:  94<br />W/1e+06: 0.000000","GC:  95<br />W/1e+06: 0.000000","GC:  96<br />W/1e+06: 0.000000","GC:  97<br />W/1e+06: 0.000000","GC:  98<br />W/1e+06: 0.000000","GC:  99<br />W/1e+06: 0.000000","GC: 100<br />W/1e+06: 0.000000","GC:   0<br />W/1e+06: 0.000000","GC:   1<br />W/1e+06: 0.000000","GC:   2<br />W/1e+06: 0.000000","GC:   3<br />W/1e+06: 0.000000","GC:   4<br />W/1e+06: 0.000000","GC:   5<br />W/1e+06: 0.000000","GC:   6<br />W/1e+06: 0.000000","GC:   7<br />W/1e+06: 0.000000","GC:   8<br />W/1e+06: 0.000000","GC:   9<br />W/1e+06: 0.000000","GC:  10<br />W/1e+06: 0.000002","GC:  11<br />W/1e+06: 0.000003","GC:  12<br />W/1e+06: 0.000018","GC:  13<br />W/1e+06: 0.000061","GC:  14<br />W/1e+06: 0.000099","GC:  15<br />W/1e+06: 0.000232","GC:  16<br />W/1e+06: 0.000414","GC:  17<br />W/1e+06: 0.000657","GC:  18<br />W/1e+06: 0.001179","GC:  19<br />W/1e+06: 0.001961","GC:  20<br />W/1e+06: 0.002947","GC:  21<br />W/1e+06: 0.004428","GC:  22<br />W/1e+06: 0.006577","GC:  23<br />W/1e+06: 0.010038","GC:  24<br />W/1e+06: 0.013131","GC:  25<br />W/1e+06: 0.016968","GC:  26<br />W/1e+06: 0.022322","GC:  27<br />W/1e+06: 0.028292","GC:  28<br />W/1e+06: 0.036690","GC:  29<br />W/1e+06: 0.047001","GC:  30<br />W/1e+06: 0.058328","GC:  31<br />W/1e+06: 0.070344","GC:  32<br />W/1e+06: 0.083849","GC:  33<br />W/1e+06: 0.101838","GC:  34<br />W/1e+06: 0.119577","GC:  35<br />W/1e+06: 0.138734","GC:  36<br />W/1e+06: 0.159611","GC:  37<br />W/1e+06: 0.181203","GC:  38<br />W/1e+06: 0.205339","GC:  39<br />W/1e+06: 0.232452","GC:  40<br />W/1e+06: 0.262352","GC:  41<br />W/1e+06: 0.289362","GC:  42<br />W/1e+06: 0.317347","GC:  43<br />W/1e+06: 0.342040","GC:  44<br />W/1e+06: 0.365406","GC:  45<br />W/1e+06: 0.387317","GC:  46<br />W/1e+06: 0.406405","GC:  47<br />W/1e+06: 0.422871","GC:  48<br />W/1e+06: 0.436752","GC:  49<br />W/1e+06: 0.445505","GC:  50<br />W/1e+06: 0.451658","GC:  51<br />W/1e+06: 0.456692","GC:  52<br />W/1e+06: 0.455090","GC:  53<br />W/1e+06: 0.447841","GC:  54<br />W/1e+06: 0.429853","GC:  55<br />W/1e+06: 0.405343","GC:  56<br />W/1e+06: 0.377543","GC:  57<br />W/1e+06: 0.353993","GC:  58<br />W/1e+06: 0.338145","GC:  59<br />W/1e+06: 0.329892","GC:  60<br />W/1e+06: 0.333301","GC:  61<br />W/1e+06: 0.351119","GC:  62<br />W/1e+06: 0.384616","GC:  63<br />W/1e+06: 0.427987","GC:  64<br />W/1e+06: 0.475965","GC:  65<br />W/1e+06: 0.527593","GC:  66<br />W/1e+06: 0.575783","GC:  67<br />W/1e+06: 0.614146","GC:  68<br />W/1e+06: 0.634249","GC:  69<br />W/1e+06: 0.634200","GC:  70<br />W/1e+06: 0.617734","GC:  71<br />W/1e+06: 0.582209","GC:  72<br />W/1e+06: 0.529033","GC:  73<br />W/1e+06: 0.467484","GC:  74<br />W/1e+06: 0.396863","GC:  75<br />W/1e+06: 0.325426","GC:  76<br />W/1e+06: 0.258782","GC:  77<br />W/1e+06: 0.198358","GC:  78<br />W/1e+06: 0.146188","GC:  79<br />W/1e+06: 0.105011","GC:  80<br />W/1e+06: 0.073676","GC:  81<br />W/1e+06: 0.049418","GC:  82<br />W/1e+06: 0.033292","GC:  83<br />W/1e+06: 0.022130","GC:  84<br />W/1e+06: 0.013340","GC:  85<br />W/1e+06: 0.007582","GC:  86<br />W/1e+06: 0.004134","GC:  87<br />W/1e+06: 0.001878","GC:  88<br />W/1e+06: 0.000879","GC:  89<br />W/1e+06: 0.000326","GC:  90<br />W/1e+06: 0.000069","GC:  91<br />W/1e+06: 0.000011","GC:  92<br />W/1e+06: 0.000005","GC:  93<br />W/1e+06: 0.000004","GC:  94<br />W/1e+06: 0.000000","GC:  95<br />W/1e+06: 0.000000","GC:  96<br />W/1e+06: 0.000000","GC:  97<br />W/1e+06: 0.000000","GC:  98<br />W/1e+06: 0.000000","GC:  99<br />W/1e+06: 0.000000","GC: 100<br />W/1e+06: 0.000000"],"type":"bar","textposition":"none","marker":{"autocolorscale":false,"color":"rgba(70,130,180,0.5)","line":{"width":1.88976377952756,"color":"transparent"}},"showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0,0,0,0,0,0,0,0,0,0,1.960423,1.982957,1.614908,1.212382,1.350651,1.18418,1.305969,1.443385,1.455797,1.49625,1.468729,1.394916,1.405751,1.40204,1.38899,1.347817,1.315069,1.306269,1.276504,1.2693,1.26453,1.271444,1.276227,1.277577,1.286634,1.2953,1.307038,1.316745,1.324915,1.328327,1.325652,1.319101,1.309751,1.293499,1.275996,1.252809,1.224491,1.196815,1.160349,1.124006,1.092694,1.059496,1.028109,1.000004,0.972071,0.949208,0.926144,0.907458,0.891037,0.87392,0.859407,0.852702,0.848998,0.846081,0.846152,0.84888,0.849876,0.851524,0.854222,0.856624,0.860183,0.859328,0.864357,0.864195,0.867073,0.870188,0.872547,0.87071,0.870983,0.86758,0.862225,0.856546,0.849392,0.845899,0.824162,0.816078,0.808168,0.772334,0.742148,0.692183,0.59763,0.848083,0.567847,0.270403,0,0,0,0,0,0,0],"text":["GC:   0<br />Nor_cov: 0.000000<br />Sample: NA","GC:   1<br />Nor_cov: 0.000000<br />Sample: NA","GC:   2<br />Nor_cov: 0.000000<br />Sample: NA","GC:   3<br />Nor_cov: 0.000000<br />Sample: NA","GC:   4<br />Nor_cov: 0.000000<br />Sample: NA","GC:   5<br />Nor_cov: 0.000000<br />Sample: NA","GC:   6<br />Nor_cov: 0.000000<br />Sample: NA","GC:   7<br />Nor_cov: 0.000000<br />Sample: NA","GC:   8<br />Nor_cov: 0.000000<br />Sample: NA","GC:   9<br />Nor_cov: 0.000000<br />Sample: NA","GC:  10<br />Nor_cov: 1.960423<br />Sample: NA","GC:  11<br />Nor_cov: 1.982957<br />Sample: NA","GC:  12<br />Nor_cov: 1.614908<br />Sample: NA","GC:  13<br />Nor_cov: 1.212382<br />Sample: NA","GC:  14<br />Nor_cov: 1.350651<br />Sample: NA","GC:  15<br />Nor_cov: 1.184180<br />Sample: NA","GC:  16<br />Nor_cov: 1.305969<br />Sample: NA","GC:  17<br />Nor_cov: 1.443385<br />Sample: NA","GC:  18<br />Nor_cov: 1.455797<br />Sample: NA","GC:  19<br />Nor_cov: 1.496250<br />Sample: NA","GC:  20<br />Nor_cov: 1.468729<br />Sample: NA","GC:  21<br />Nor_cov: 1.394916<br />Sample: NA","GC:  22<br />Nor_cov: 1.405751<br />Sample: NA","GC:  23<br />Nor_cov: 1.402040<br />Sample: NA","GC:  24<br />Nor_cov: 1.388990<br />Sample: NA","GC:  25<br />Nor_cov: 1.347817<br />Sample: NA","GC:  26<br />Nor_cov: 1.315069<br />Sample: NA","GC:  27<br />Nor_cov: 1.306269<br />Sample: NA","GC:  28<br />Nor_cov: 1.276504<br />Sample: NA","GC:  29<br />Nor_cov: 1.269300<br />Sample: NA","GC:  30<br />Nor_cov: 1.264530<br />Sample: NA","GC:  31<br />Nor_cov: 1.271444<br />Sample: NA","GC:  32<br />Nor_cov: 1.276227<br />Sample: NA","GC:  33<br />Nor_cov: 1.277577<br />Sample: NA","GC:  34<br />Nor_cov: 1.286634<br />Sample: NA","GC:  35<br />Nor_cov: 1.295300<br />Sample: NA","GC:  36<br />Nor_cov: 1.307038<br />Sample: NA","GC:  37<br />Nor_cov: 1.316745<br />Sample: NA","GC:  38<br />Nor_cov: 1.324915<br />Sample: NA","GC:  39<br />Nor_cov: 1.328327<br />Sample: NA","GC:  40<br />Nor_cov: 1.325652<br />Sample: NA","GC:  41<br />Nor_cov: 1.319101<br />Sample: NA","GC:  42<br />Nor_cov: 1.309751<br />Sample: NA","GC:  43<br />Nor_cov: 1.293499<br />Sample: NA","GC:  44<br />Nor_cov: 1.275996<br />Sample: NA","GC:  45<br />Nor_cov: 1.252809<br />Sample: NA","GC:  46<br />Nor_cov: 1.224491<br />Sample: NA","GC:  47<br />Nor_cov: 1.196815<br />Sample: NA","GC:  48<br />Nor_cov: 1.160349<br />Sample: NA","GC:  49<br />Nor_cov: 1.124006<br />Sample: NA","GC:  50<br />Nor_cov: 1.092694<br />Sample: NA","GC:  51<br />Nor_cov: 1.059496<br />Sample: NA","GC:  52<br />Nor_cov: 1.028109<br />Sample: NA","GC:  53<br />Nor_cov: 1.000004<br />Sample: NA","GC:  54<br />Nor_cov: 0.972071<br />Sample: NA","GC:  55<br />Nor_cov: 0.949208<br />Sample: NA","GC:  56<br />Nor_cov: 0.926144<br />Sample: NA","GC:  57<br />Nor_cov: 0.907458<br />Sample: NA","GC:  58<br />Nor_cov: 0.891037<br />Sample: NA","GC:  59<br />Nor_cov: 0.873920<br />Sample: NA","GC:  60<br />Nor_cov: 0.859407<br />Sample: NA","GC:  61<br />Nor_cov: 0.852702<br />Sample: NA","GC:  62<br />Nor_cov: 0.848998<br />Sample: NA","GC:  63<br />Nor_cov: 0.846081<br />Sample: NA","GC:  64<br />Nor_cov: 0.846152<br />Sample: NA","GC:  65<br />Nor_cov: 0.848880<br />Sample: NA","GC:  66<br />Nor_cov: 0.849876<br />Sample: NA","GC:  67<br />Nor_cov: 0.851524<br />Sample: NA","GC:  68<br />Nor_cov: 0.854222<br />Sample: NA","GC:  69<br />Nor_cov: 0.856624<br />Sample: NA","GC:  70<br />Nor_cov: 0.860183<br />Sample: NA","GC:  71<br />Nor_cov: 0.859328<br />Sample: NA","GC:  72<br />Nor_cov: 0.864357<br />Sample: NA","GC:  73<br />Nor_cov: 0.864195<br />Sample: NA","GC:  74<br />Nor_cov: 0.867073<br />Sample: NA","GC:  75<br />Nor_cov: 0.870188<br />Sample: NA","GC:  76<br />Nor_cov: 0.872547<br />Sample: NA","GC:  77<br />Nor_cov: 0.870710<br />Sample: NA","GC:  78<br />Nor_cov: 0.870983<br />Sample: NA","GC:  79<br />Nor_cov: 0.867580<br />Sample: NA","GC:  80<br />Nor_cov: 0.862225<br />Sample: NA","GC:  81<br />Nor_cov: 0.856546<br />Sample: NA","GC:  82<br />Nor_cov: 0.849392<br />Sample: NA","GC:  83<br />Nor_cov: 0.845899<br />Sample: NA","GC:  84<br />Nor_cov: 0.824162<br />Sample: NA","GC:  85<br />Nor_cov: 0.816078<br />Sample: NA","GC:  86<br />Nor_cov: 0.808168<br />Sample: NA","GC:  87<br />Nor_cov: 0.772334<br />Sample: NA","GC:  88<br />Nor_cov: 0.742148<br />Sample: NA","GC:  89<br />Nor_cov: 0.692183<br />Sample: NA","GC:  90<br />Nor_cov: 0.597630<br />Sample: NA","GC:  91<br />Nor_cov: 0.848083<br />Sample: NA","GC:  92<br />Nor_cov: 0.567847<br />Sample: NA","GC:  93<br />Nor_cov: 0.270403<br />Sample: NA","GC:  94<br />Nor_cov: 0.000000<br />Sample: NA","GC:  95<br />Nor_cov: 0.000000<br />Sample: NA","GC:  96<br />Nor_cov: 0.000000<br />Sample: NA","GC:  97<br />Nor_cov: 0.000000<br />Sample: NA","GC:  98<br />Nor_cov: 0.000000<br />Sample: NA","GC:  99<br />Nor_cov: 0.000000<br />Sample: NA","GC: 100<br />Nor_cov: 0.000000<br />Sample: NA"],"type":"scatter","mode":"lines","line":{"width":5.66929133858268,"color":"rgba(190,190,190,1)","dash":"solid"},"hoveron":"points","name":"NA","legendgroup":"NA","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0,0,0,0,0,0,0,0,0,0,2.226714,1.102753,1.229994,1.103449,1.187581,1.177526,1.200182,1.292549,1.321944,1.315557,1.299652,1.24896,1.222243,1.228625,1.211399,1.17649,1.153831,1.12903,1.100256,1.084894,1.080748,1.076314,1.075215,1.073897,1.077931,1.084824,1.089506,1.095114,1.105899,1.109256,1.110377,1.108698,1.105475,1.100068,1.091457,1.080984,1.069975,1.05698,1.036184,1.018179,0.99903,0.980621,0.962741,0.946955,0.931626,0.921032,0.912526,0.905691,0.904099,0.902429,0.905247,0.912256,0.922287,0.929233,0.93873,0.949076,0.956949,0.964414,0.972923,0.981224,0.990464,0.9975,1.00694,1.016528,1.024661,1.033884,1.045007,1.053368,1.06161,1.063254,1.064591,1.064726,1.067027,1.070731,1.058369,1.04711,1.045997,1.030596,1.025309,0.950403,0.870401,1.48062,0.585308,0.381722,0,0,0,0,0,0,0],"text":["GC:   0<br />Nor_cov: 0.000000<br />Sample: NA2","GC:   1<br />Nor_cov: 0.000000<br />Sample: NA2","GC:   2<br />Nor_cov: 0.000000<br />Sample: NA2","GC:   3<br />Nor_cov: 0.000000<br />Sample: NA2","GC:   4<br />Nor_cov: 0.000000<br />Sample: NA2","GC:   5<br />Nor_cov: 0.000000<br />Sample: NA2","GC:   6<br />Nor_cov: 0.000000<br />Sample: NA2","GC:   7<br />Nor_cov: 0.000000<br />Sample: NA2","GC:   8<br />Nor_cov: 0.000000<br />Sample: NA2","GC:   9<br />Nor_cov: 0.000000<br />Sample: NA2","GC:  10<br />Nor_cov: 2.226714<br />Sample: NA2","GC:  11<br />Nor_cov: 1.102753<br />Sample: NA2","GC:  12<br />Nor_cov: 1.229994<br />Sample: NA2","GC:  13<br />Nor_cov: 1.103449<br />Sample: NA2","GC:  14<br />Nor_cov: 1.187581<br />Sample: NA2","GC:  15<br />Nor_cov: 1.177526<br />Sample: NA2","GC:  16<br />Nor_cov: 1.200182<br />Sample: NA2","GC:  17<br />Nor_cov: 1.292549<br />Sample: NA2","GC:  18<br />Nor_cov: 1.321944<br />Sample: NA2","GC:  19<br />Nor_cov: 1.315557<br />Sample: NA2","GC:  20<br />Nor_cov: 1.299652<br />Sample: NA2","GC:  21<br />Nor_cov: 1.248960<br />Sample: NA2","GC:  22<br />Nor_cov: 1.222243<br />Sample: NA2","GC:  23<br />Nor_cov: 1.228625<br />Sample: NA2","GC:  24<br />Nor_cov: 1.211399<br />Sample: NA2","GC:  25<br />Nor_cov: 1.176490<br />Sample: NA2","GC:  26<br />Nor_cov: 1.153831<br />Sample: NA2","GC:  27<br />Nor_cov: 1.129030<br />Sample: NA2","GC:  28<br />Nor_cov: 1.100256<br />Sample: NA2","GC:  29<br />Nor_cov: 1.084894<br />Sample: NA2","GC:  30<br />Nor_cov: 1.080748<br />Sample: NA2","GC:  31<br />Nor_cov: 1.076314<br />Sample: NA2","GC:  32<br />Nor_cov: 1.075215<br />Sample: NA2","GC:  33<br />Nor_cov: 1.073897<br />Sample: NA2","GC:  34<br />Nor_cov: 1.077931<br />Sample: NA2","GC:  35<br />Nor_cov: 1.084824<br />Sample: NA2","GC:  36<br />Nor_cov: 1.089506<br />Sample: NA2","GC:  37<br />Nor_cov: 1.095114<br />Sample: NA2","GC:  38<br />Nor_cov: 1.105899<br />Sample: NA2","GC:  39<br />Nor_cov: 1.109256<br />Sample: NA2","GC:  40<br />Nor_cov: 1.110377<br />Sample: NA2","GC:  41<br />Nor_cov: 1.108698<br />Sample: NA2","GC:  42<br />Nor_cov: 1.105475<br />Sample: NA2","GC:  43<br />Nor_cov: 1.100068<br />Sample: NA2","GC:  44<br />Nor_cov: 1.091457<br />Sample: NA2","GC:  45<br />Nor_cov: 1.080984<br />Sample: NA2","GC:  46<br />Nor_cov: 1.069975<br />Sample: NA2","GC:  47<br />Nor_cov: 1.056980<br />Sample: NA2","GC:  48<br />Nor_cov: 1.036184<br />Sample: NA2","GC:  49<br />Nor_cov: 1.018179<br />Sample: NA2","GC:  50<br />Nor_cov: 0.999030<br />Sample: NA2","GC:  51<br />Nor_cov: 0.980621<br />Sample: NA2","GC:  52<br />Nor_cov: 0.962741<br />Sample: NA2","GC:  53<br />Nor_cov: 0.946955<br />Sample: NA2","GC:  54<br />Nor_cov: 0.931626<br />Sample: NA2","GC:  55<br />Nor_cov: 0.921032<br />Sample: NA2","GC:  56<br />Nor_cov: 0.912526<br />Sample: NA2","GC:  57<br />Nor_cov: 0.905691<br />Sample: NA2","GC:  58<br />Nor_cov: 0.904099<br />Sample: NA2","GC:  59<br />Nor_cov: 0.902429<br />Sample: NA2","GC:  60<br />Nor_cov: 0.905247<br />Sample: NA2","GC:  61<br />Nor_cov: 0.912256<br />Sample: NA2","GC:  62<br />Nor_cov: 0.922287<br />Sample: NA2","GC:  63<br />Nor_cov: 0.929233<br />Sample: NA2","GC:  64<br />Nor_cov: 0.938730<br />Sample: NA2","GC:  65<br />Nor_cov: 0.949076<br />Sample: NA2","GC:  66<br />Nor_cov: 0.956949<br />Sample: NA2","GC:  67<br />Nor_cov: 0.964414<br />Sample: NA2","GC:  68<br />Nor_cov: 0.972923<br />Sample: NA2","GC:  69<br />Nor_cov: 0.981224<br />Sample: NA2","GC:  70<br />Nor_cov: 0.990464<br />Sample: NA2","GC:  71<br />Nor_cov: 0.997500<br />Sample: NA2","GC:  72<br />Nor_cov: 1.006940<br />Sample: NA2","GC:  73<br />Nor_cov: 1.016528<br />Sample: NA2","GC:  74<br />Nor_cov: 1.024661<br />Sample: NA2","GC:  75<br />Nor_cov: 1.033884<br />Sample: NA2","GC:  76<br />Nor_cov: 1.045007<br />Sample: NA2","GC:  77<br />Nor_cov: 1.053368<br />Sample: NA2","GC:  78<br />Nor_cov: 1.061610<br />Sample: NA2","GC:  79<br />Nor_cov: 1.063254<br />Sample: NA2","GC:  80<br />Nor_cov: 1.064591<br />Sample: NA2","GC:  81<br />Nor_cov: 1.064726<br />Sample: NA2","GC:  82<br />Nor_cov: 1.067027<br />Sample: NA2","GC:  83<br />Nor_cov: 1.070731<br />Sample: NA2","GC:  84<br />Nor_cov: 1.058369<br />Sample: NA2","GC:  85<br />Nor_cov: 1.047110<br />Sample: NA2","GC:  86<br />Nor_cov: 1.045997<br />Sample: NA2","GC:  87<br />Nor_cov: 1.030596<br />Sample: NA2","GC:  88<br />Nor_cov: 1.025309<br />Sample: NA2","GC:  89<br />Nor_cov: 0.950403<br />Sample: NA2","GC:  90<br />Nor_cov: 0.870401<br />Sample: NA2","GC:  91<br />Nor_cov: 1.480620<br />Sample: NA2","GC:  92<br />Nor_cov: 0.585308<br />Sample: NA2","GC:  93<br />Nor_cov: 0.381722<br />Sample: NA2","GC:  94<br />Nor_cov: 0.000000<br />Sample: NA2","GC:  95<br />Nor_cov: 0.000000<br />Sample: NA2","GC:  96<br />Nor_cov: 0.000000<br />Sample: NA2","GC:  97<br />Nor_cov: 0.000000<br />Sample: NA2","GC:  98<br />Nor_cov: 0.000000<br />Sample: NA2","GC:  99<br />Nor_cov: 0.000000<br />Sample: NA2","GC: 100<br />Nor_cov: 0.000000<br />Sample: NA2"],"type":"scatter","mode":"lines","line":{"width":5.66929133858268,"color":"rgba(190,190,190,1)","dash":"solid"},"hoveron":"points","name":"NA2","legendgroup":"NA2","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0,0,0,0,0,0,0,0,0,0,3.796911,4.537189,2.873553,2.170334,2.587717,2.315943,2.517084,2.791663,2.997571,3.00405,3.233346,3.068314,3.174874,3.237095,3.182279,3.076865,2.993403,2.96335,2.926365,2.950199,2.945135,2.975521,3.005768,3.030449,3.018703,3.021226,3.039914,3.047614,3.045545,3.006436,2.956498,2.897363,2.81339,2.710874,2.60602,2.479494,2.333693,2.170092,1.988079,1.802099,1.628203,1.456755,1.303494,1.155211,1.016999,0.890315,0.768303,0.64673,0.528894,0.408225,0.29426,0.199966,0.130699,0.081817,0.053203,0.034776,0.024114,0.017482,0.013986,0.011936,0.010634,0.009505,0.008906,0.007903,0.007096,0.006662,0.00615,0.005696,0.005107,0.004872,0.004599,0.004349,0.004127,0.003956,0.003727,0.00395,0.003916,0.003204,0.00326,0.001758,0.004153,0.013025,0,0,0,0,0,0,0,0,0],"text":["GC:   0<br />Nor_cov: 0.000000<br />Sample: RP-MDA","GC:   1<br />Nor_cov: 0.000000<br />Sample: RP-MDA","GC:   2<br />Nor_cov: 0.000000<br />Sample: RP-MDA","GC:   3<br />Nor_cov: 0.000000<br />Sample: RP-MDA","GC:   4<br />Nor_cov: 0.000000<br />Sample: RP-MDA","GC:   5<br />Nor_cov: 0.000000<br />Sample: RP-MDA","GC:   6<br />Nor_cov: 0.000000<br />Sample: RP-MDA","GC:   7<br />Nor_cov: 0.000000<br />Sample: RP-MDA","GC:   8<br />Nor_cov: 0.000000<br />Sample: RP-MDA","GC:   9<br />Nor_cov: 0.000000<br />Sample: RP-MDA","GC:  10<br />Nor_cov: 3.796911<br />Sample: RP-MDA","GC:  11<br />Nor_cov: 4.537189<br />Sample: RP-MDA","GC:  12<br />Nor_cov: 2.873553<br />Sample: RP-MDA","GC:  13<br />Nor_cov: 2.170334<br />Sample: RP-MDA","GC:  14<br />Nor_cov: 2.587717<br />Sample: RP-MDA","GC:  15<br />Nor_cov: 2.315943<br />Sample: RP-MDA","GC:  16<br />Nor_cov: 2.517084<br />Sample: RP-MDA","GC:  17<br />Nor_cov: 2.791663<br />Sample: RP-MDA","GC:  18<br />Nor_cov: 2.997571<br />Sample: RP-MDA","GC:  19<br />Nor_cov: 3.004050<br />Sample: RP-MDA","GC:  20<br />Nor_cov: 3.233346<br />Sample: RP-MDA","GC:  21<br />Nor_cov: 3.068314<br />Sample: RP-MDA","GC:  22<br />Nor_cov: 3.174874<br />Sample: RP-MDA","GC:  23<br />Nor_cov: 3.237095<br />Sample: RP-MDA","GC:  24<br />Nor_cov: 3.182279<br />Sample: RP-MDA","GC:  25<br />Nor_cov: 3.076865<br />Sample: RP-MDA","GC:  26<br />Nor_cov: 2.993403<br />Sample: RP-MDA","GC:  27<br />Nor_cov: 2.963350<br />Sample: RP-MDA","GC:  28<br />Nor_cov: 2.926365<br />Sample: RP-MDA","GC:  29<br />Nor_cov: 2.950199<br />Sample: RP-MDA","GC:  30<br />Nor_cov: 2.945135<br />Sample: RP-MDA","GC:  31<br />Nor_cov: 2.975521<br />Sample: RP-MDA","GC:  32<br />Nor_cov: 3.005768<br />Sample: RP-MDA","GC:  33<br />Nor_cov: 3.030449<br />Sample: RP-MDA","GC:  34<br />Nor_cov: 3.018703<br />Sample: RP-MDA","GC:  35<br />Nor_cov: 3.021226<br />Sample: RP-MDA","GC:  36<br />Nor_cov: 3.039914<br />Sample: RP-MDA","GC:  37<br />Nor_cov: 3.047614<br />Sample: RP-MDA","GC:  38<br />Nor_cov: 3.045545<br />Sample: RP-MDA","GC:  39<br />Nor_cov: 3.006436<br />Sample: RP-MDA","GC:  40<br />Nor_cov: 2.956498<br />Sample: RP-MDA","GC:  41<br />Nor_cov: 2.897363<br />Sample: RP-MDA","GC:  42<br />Nor_cov: 2.813390<br />Sample: RP-MDA","GC:  43<br />Nor_cov: 2.710874<br />Sample: RP-MDA","GC:  44<br />Nor_cov: 2.606020<br />Sample: RP-MDA","GC:  45<br />Nor_cov: 2.479494<br />Sample: RP-MDA","GC:  46<br />Nor_cov: 2.333693<br />Sample: RP-MDA","GC:  47<br />Nor_cov: 2.170092<br />Sample: RP-MDA","GC:  48<br />Nor_cov: 1.988079<br />Sample: RP-MDA","GC:  49<br />Nor_cov: 1.802099<br />Sample: RP-MDA","GC:  50<br />Nor_cov: 1.628203<br />Sample: RP-MDA","GC:  51<br />Nor_cov: 1.456755<br />Sample: RP-MDA","GC:  52<br />Nor_cov: 1.303494<br />Sample: RP-MDA","GC:  53<br />Nor_cov: 1.155211<br />Sample: RP-MDA","GC:  54<br />Nor_cov: 1.016999<br />Sample: RP-MDA","GC:  55<br />Nor_cov: 0.890315<br />Sample: RP-MDA","GC:  56<br />Nor_cov: 0.768303<br />Sample: RP-MDA","GC:  57<br />Nor_cov: 0.646730<br />Sample: RP-MDA","GC:  58<br />Nor_cov: 0.528894<br />Sample: RP-MDA","GC:  59<br />Nor_cov: 0.408225<br />Sample: RP-MDA","GC:  60<br />Nor_cov: 0.294260<br />Sample: RP-MDA","GC:  61<br />Nor_cov: 0.199966<br />Sample: RP-MDA","GC:  62<br />Nor_cov: 0.130699<br />Sample: RP-MDA","GC:  63<br />Nor_cov: 0.081817<br />Sample: RP-MDA","GC:  64<br />Nor_cov: 0.053203<br />Sample: RP-MDA","GC:  65<br />Nor_cov: 0.034776<br />Sample: RP-MDA","GC:  66<br />Nor_cov: 0.024114<br />Sample: RP-MDA","GC:  67<br />Nor_cov: 0.017482<br />Sample: RP-MDA","GC:  68<br />Nor_cov: 0.013986<br />Sample: RP-MDA","GC:  69<br />Nor_cov: 0.011936<br />Sample: RP-MDA","GC:  70<br />Nor_cov: 0.010634<br />Sample: RP-MDA","GC:  71<br />Nor_cov: 0.009505<br />Sample: RP-MDA","GC:  72<br />Nor_cov: 0.008906<br />Sample: RP-MDA","GC:  73<br />Nor_cov: 0.007903<br />Sample: RP-MDA","GC:  74<br />Nor_cov: 0.007096<br />Sample: RP-MDA","GC:  75<br />Nor_cov: 0.006662<br />Sample: RP-MDA","GC:  76<br />Nor_cov: 0.006150<br />Sample: RP-MDA","GC:  77<br />Nor_cov: 0.005696<br />Sample: RP-MDA","GC:  78<br />Nor_cov: 0.005107<br />Sample: RP-MDA","GC:  79<br />Nor_cov: 0.004872<br />Sample: RP-MDA","GC:  80<br />Nor_cov: 0.004599<br />Sample: RP-MDA","GC:  81<br />Nor_cov: 0.004349<br />Sample: RP-MDA","GC:  82<br />Nor_cov: 0.004127<br />Sample: RP-MDA","GC:  83<br />Nor_cov: 0.003956<br />Sample: RP-MDA","GC:  84<br />Nor_cov: 0.003727<br />Sample: RP-MDA","GC:  85<br />Nor_cov: 0.003950<br />Sample: RP-MDA","GC:  86<br />Nor_cov: 0.003916<br />Sample: RP-MDA","GC:  87<br />Nor_cov: 0.003204<br />Sample: RP-MDA","GC:  88<br />Nor_cov: 0.003260<br />Sample: RP-MDA","GC:  89<br />Nor_cov: 0.001758<br />Sample: RP-MDA","GC:  90<br />Nor_cov: 0.004153<br />Sample: RP-MDA","GC:  91<br />Nor_cov: 0.013025<br />Sample: RP-MDA","GC:  92<br />Nor_cov: 0.000000<br />Sample: RP-MDA","GC:  93<br />Nor_cov: 0.000000<br />Sample: RP-MDA","GC:  94<br />Nor_cov: 0.000000<br />Sample: RP-MDA","GC:  95<br />Nor_cov: 0.000000<br />Sample: RP-MDA","GC:  96<br />Nor_cov: 0.000000<br />Sample: RP-MDA","GC:  97<br />Nor_cov: 0.000000<br />Sample: RP-MDA","GC:  98<br />Nor_cov: 0.000000<br />Sample: RP-MDA","GC:  99<br />Nor_cov: 0.000000<br />Sample: RP-MDA","GC: 100<br />Nor_cov: 0.000000<br />Sample: RP-MDA"],"type":"scatter","mode":"lines","line":{"width":5.66929133858268,"color":"rgba(84,139,84,1)","dash":"solid"},"hoveron":"points","name":"RP-MDA","legendgroup":"RP-MDA","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0,0,0,0,0,0,0,0,0,0,5.035215,5.531645,3.83748,2.504238,2.863962,2.545119,2.777128,2.91079,3.135694,3.07058,3.274381,3.111734,3.232966,3.247854,3.193877,3.064652,2.978692,2.946553,2.87076,2.872984,2.859008,2.873319,2.887128,2.888482,2.871671,2.867188,2.873984,2.873741,2.865488,2.836878,2.790464,2.736589,2.662638,2.572886,2.47705,2.363855,2.236344,2.098503,1.942954,1.784167,1.638441,1.49165,1.361388,1.236137,1.113448,1.000022,0.88527,0.764378,0.641028,0.510402,0.381953,0.272087,0.188902,0.128184,0.091647,0.067815,0.053424,0.043999,0.038406,0.034848,0.032221,0.029856,0.027903,0.025938,0.024302,0.022439,0.020962,0.019561,0.018015,0.01669,0.015576,0.0146,0.013608,0.013043,0.013184,0.012253,0.011734,0.010951,0.012263,0.010877,0.014389,0.038683,0,0,0,0,0,0,0,0,0],"text":["GC:   0<br />Nor_cov: 0.000000<br />Sample: RP-MDA2","GC:   1<br />Nor_cov: 0.000000<br />Sample: RP-MDA2","GC:   2<br />Nor_cov: 0.000000<br />Sample: RP-MDA2","GC:   3<br />Nor_cov: 0.000000<br />Sample: RP-MDA2","GC:   4<br />Nor_cov: 0.000000<br />Sample: RP-MDA2","GC:   5<br />Nor_cov: 0.000000<br />Sample: RP-MDA2","GC:   6<br />Nor_cov: 0.000000<br />Sample: RP-MDA2","GC:   7<br />Nor_cov: 0.000000<br />Sample: RP-MDA2","GC:   8<br />Nor_cov: 0.000000<br />Sample: RP-MDA2","GC:   9<br />Nor_cov: 0.000000<br />Sample: RP-MDA2","GC:  10<br />Nor_cov: 5.035215<br />Sample: RP-MDA2","GC:  11<br />Nor_cov: 5.531645<br />Sample: RP-MDA2","GC:  12<br />Nor_cov: 3.837480<br />Sample: RP-MDA2","GC:  13<br />Nor_cov: 2.504238<br />Sample: RP-MDA2","GC:  14<br />Nor_cov: 2.863962<br />Sample: RP-MDA2","GC:  15<br />Nor_cov: 2.545119<br />Sample: RP-MDA2","GC:  16<br />Nor_cov: 2.777128<br />Sample: RP-MDA2","GC:  17<br />Nor_cov: 2.910790<br />Sample: RP-MDA2","GC:  18<br />Nor_cov: 3.135694<br />Sample: RP-MDA2","GC:  19<br />Nor_cov: 3.070580<br />Sample: RP-MDA2","GC:  20<br />Nor_cov: 3.274381<br />Sample: RP-MDA2","GC:  21<br />Nor_cov: 3.111734<br />Sample: RP-MDA2","GC:  22<br />Nor_cov: 3.232966<br />Sample: RP-MDA2","GC:  23<br />Nor_cov: 3.247854<br />Sample: RP-MDA2","GC:  24<br />Nor_cov: 3.193877<br />Sample: RP-MDA2","GC:  25<br />Nor_cov: 3.064652<br />Sample: RP-MDA2","GC:  26<br />Nor_cov: 2.978692<br />Sample: RP-MDA2","GC:  27<br />Nor_cov: 2.946553<br />Sample: RP-MDA2","GC:  28<br />Nor_cov: 2.870760<br />Sample: RP-MDA2","GC:  29<br />Nor_cov: 2.872984<br />Sample: RP-MDA2","GC:  30<br />Nor_cov: 2.859008<br />Sample: RP-MDA2","GC:  31<br />Nor_cov: 2.873319<br />Sample: RP-MDA2","GC:  32<br />Nor_cov: 2.887128<br />Sample: RP-MDA2","GC:  33<br />Nor_cov: 2.888482<br />Sample: RP-MDA2","GC:  34<br />Nor_cov: 2.871671<br />Sample: RP-MDA2","GC:  35<br />Nor_cov: 2.867188<br />Sample: RP-MDA2","GC:  36<br />Nor_cov: 2.873984<br />Sample: RP-MDA2","GC:  37<br />Nor_cov: 2.873741<br />Sample: RP-MDA2","GC:  38<br />Nor_cov: 2.865488<br />Sample: RP-MDA2","GC:  39<br />Nor_cov: 2.836878<br />Sample: RP-MDA2","GC:  40<br />Nor_cov: 2.790464<br />Sample: RP-MDA2","GC:  41<br />Nor_cov: 2.736589<br />Sample: RP-MDA2","GC:  42<br />Nor_cov: 2.662638<br />Sample: RP-MDA2","GC:  43<br />Nor_cov: 2.572886<br />Sample: RP-MDA2","GC:  44<br />Nor_cov: 2.477050<br />Sample: RP-MDA2","GC:  45<br />Nor_cov: 2.363855<br />Sample: RP-MDA2","GC:  46<br />Nor_cov: 2.236344<br />Sample: RP-MDA2","GC:  47<br />Nor_cov: 2.098503<br />Sample: RP-MDA2","GC:  48<br />Nor_cov: 1.942954<br />Sample: RP-MDA2","GC:  49<br />Nor_cov: 1.784167<br />Sample: RP-MDA2","GC:  50<br />Nor_cov: 1.638441<br />Sample: RP-MDA2","GC:  51<br />Nor_cov: 1.491650<br />Sample: RP-MDA2","GC:  52<br />Nor_cov: 1.361388<br />Sample: RP-MDA2","GC:  53<br />Nor_cov: 1.236137<br />Sample: RP-MDA2","GC:  54<br />Nor_cov: 1.113448<br />Sample: RP-MDA2","GC:  55<br />Nor_cov: 1.000022<br />Sample: RP-MDA2","GC:  56<br />Nor_cov: 0.885270<br />Sample: RP-MDA2","GC:  57<br />Nor_cov: 0.764378<br />Sample: RP-MDA2","GC:  58<br />Nor_cov: 0.641028<br />Sample: RP-MDA2","GC:  59<br />Nor_cov: 0.510402<br />Sample: RP-MDA2","GC:  60<br />Nor_cov: 0.381953<br />Sample: RP-MDA2","GC:  61<br />Nor_cov: 0.272087<br />Sample: RP-MDA2","GC:  62<br />Nor_cov: 0.188902<br />Sample: RP-MDA2","GC:  63<br />Nor_cov: 0.128184<br />Sample: RP-MDA2","GC:  64<br />Nor_cov: 0.091647<br />Sample: RP-MDA2","GC:  65<br />Nor_cov: 0.067815<br />Sample: RP-MDA2","GC:  66<br />Nor_cov: 0.053424<br />Sample: RP-MDA2","GC:  67<br />Nor_cov: 0.043999<br />Sample: RP-MDA2","GC:  68<br />Nor_cov: 0.038406<br />Sample: RP-MDA2","GC:  69<br />Nor_cov: 0.034848<br />Sample: RP-MDA2","GC:  70<br />Nor_cov: 0.032221<br />Sample: RP-MDA2","GC:  71<br />Nor_cov: 0.029856<br />Sample: RP-MDA2","GC:  72<br />Nor_cov: 0.027903<br />Sample: RP-MDA2","GC:  73<br />Nor_cov: 0.025938<br />Sample: RP-MDA2","GC:  74<br />Nor_cov: 0.024302<br />Sample: RP-MDA2","GC:  75<br />Nor_cov: 0.022439<br />Sample: RP-MDA2","GC:  76<br />Nor_cov: 0.020962<br />Sample: RP-MDA2","GC:  77<br />Nor_cov: 0.019561<br />Sample: RP-MDA2","GC:  78<br />Nor_cov: 0.018015<br />Sample: RP-MDA2","GC:  79<br />Nor_cov: 0.016690<br />Sample: RP-MDA2","GC:  80<br />Nor_cov: 0.015576<br />Sample: RP-MDA2","GC:  81<br />Nor_cov: 0.014600<br />Sample: RP-MDA2","GC:  82<br />Nor_cov: 0.013608<br />Sample: RP-MDA2","GC:  83<br />Nor_cov: 0.013043<br />Sample: RP-MDA2","GC:  84<br />Nor_cov: 0.013184<br />Sample: RP-MDA2","GC:  85<br />Nor_cov: 0.012253<br />Sample: RP-MDA2","GC:  86<br />Nor_cov: 0.011734<br />Sample: RP-MDA2","GC:  87<br />Nor_cov: 0.010951<br />Sample: RP-MDA2","GC:  88<br />Nor_cov: 0.012263<br />Sample: RP-MDA2","GC:  89<br />Nor_cov: 0.010877<br />Sample: RP-MDA2","GC:  90<br />Nor_cov: 0.014389<br />Sample: RP-MDA2","GC:  91<br />Nor_cov: 0.038683<br />Sample: RP-MDA2","GC:  92<br />Nor_cov: 0.000000<br />Sample: RP-MDA2","GC:  93<br />Nor_cov: 0.000000<br />Sample: RP-MDA2","GC:  94<br />Nor_cov: 0.000000<br />Sample: RP-MDA2","GC:  95<br />Nor_cov: 0.000000<br />Sample: RP-MDA2","GC:  96<br />Nor_cov: 0.000000<br />Sample: RP-MDA2","GC:  97<br />Nor_cov: 0.000000<br />Sample: RP-MDA2","GC:  98<br />Nor_cov: 0.000000<br />Sample: RP-MDA2","GC:  99<br />Nor_cov: 0.000000<br />Sample: RP-MDA2","GC: 100<br />Nor_cov: 0.000000<br />Sample: RP-MDA2"],"type":"scatter","mode":"lines","line":{"width":5.66929133858268,"color":"rgba(84,139,84,1)","dash":"solid"},"hoveron":"points","name":"RP-MDA2","legendgroup":"RP-MDA2","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0,0,0,0,0,0,0,0,0,0,5.142282,6.631576,4.289918,2.921468,3.24937,2.788431,2.628359,2.730694,2.845168,2.826555,2.964649,2.759235,2.80805,2.805001,2.780497,2.661512,2.620013,2.584567,2.54563,2.538235,2.541889,2.5495,2.563477,2.561399,2.55691,2.557787,2.569858,2.579033,2.571929,2.551203,2.51055,2.469833,2.41192,2.341141,2.268073,2.178227,2.077332,1.968274,1.846205,1.720953,1.60773,1.491953,1.38536,1.284256,1.181314,1.084725,0.983315,0.873784,0.757599,0.634114,0.506736,0.397472,0.310209,0.247568,0.20564,0.179516,0.161584,0.149237,0.140612,0.134428,0.128957,0.124043,0.119905,0.11508,0.11174,0.106674,0.103136,0.098714,0.095246,0.092038,0.088872,0.085221,0.083434,0.081565,0.080495,0.07605,0.078141,0.07209,0.075956,0.069819,0.078191,0.091963,0.03372,0,0,0,0,0,0,0,0],"text":["GC:   0<br />Nor_cov: 0.000000<br />Sample: PrimPol-MDA","GC:   1<br />Nor_cov: 0.000000<br />Sample: PrimPol-MDA","GC:   2<br />Nor_cov: 0.000000<br />Sample: PrimPol-MDA","GC:   3<br />Nor_cov: 0.000000<br />Sample: PrimPol-MDA","GC:   4<br />Nor_cov: 0.000000<br />Sample: PrimPol-MDA","GC:   5<br />Nor_cov: 0.000000<br />Sample: PrimPol-MDA","GC:   6<br />Nor_cov: 0.000000<br />Sample: PrimPol-MDA","GC:   7<br />Nor_cov: 0.000000<br />Sample: PrimPol-MDA","GC:   8<br />Nor_cov: 0.000000<br />Sample: PrimPol-MDA","GC:   9<br />Nor_cov: 0.000000<br />Sample: PrimPol-MDA","GC:  10<br />Nor_cov: 5.142282<br />Sample: PrimPol-MDA","GC:  11<br />Nor_cov: 6.631576<br />Sample: PrimPol-MDA","GC:  12<br />Nor_cov: 4.289918<br />Sample: PrimPol-MDA","GC:  13<br />Nor_cov: 2.921468<br />Sample: PrimPol-MDA","GC:  14<br />Nor_cov: 3.249370<br />Sample: PrimPol-MDA","GC:  15<br />Nor_cov: 2.788431<br />Sample: PrimPol-MDA","GC:  16<br />Nor_cov: 2.628359<br />Sample: PrimPol-MDA","GC:  17<br />Nor_cov: 2.730694<br />Sample: PrimPol-MDA","GC:  18<br />Nor_cov: 2.845168<br />Sample: PrimPol-MDA","GC:  19<br />Nor_cov: 2.826555<br />Sample: PrimPol-MDA","GC:  20<br />Nor_cov: 2.964649<br />Sample: PrimPol-MDA","GC:  21<br />Nor_cov: 2.759235<br />Sample: PrimPol-MDA","GC:  22<br />Nor_cov: 2.808050<br />Sample: PrimPol-MDA","GC:  23<br />Nor_cov: 2.805001<br />Sample: PrimPol-MDA","GC:  24<br />Nor_cov: 2.780497<br />Sample: PrimPol-MDA","GC:  25<br />Nor_cov: 2.661512<br />Sample: PrimPol-MDA","GC:  26<br />Nor_cov: 2.620013<br />Sample: PrimPol-MDA","GC:  27<br />Nor_cov: 2.584567<br />Sample: PrimPol-MDA","GC:  28<br />Nor_cov: 2.545630<br />Sample: PrimPol-MDA","GC:  29<br />Nor_cov: 2.538235<br />Sample: PrimPol-MDA","GC:  30<br />Nor_cov: 2.541889<br />Sample: PrimPol-MDA","GC:  31<br />Nor_cov: 2.549500<br />Sample: PrimPol-MDA","GC:  32<br />Nor_cov: 2.563477<br />Sample: PrimPol-MDA","GC:  33<br />Nor_cov: 2.561399<br />Sample: PrimPol-MDA","GC:  34<br />Nor_cov: 2.556910<br />Sample: PrimPol-MDA","GC:  35<br />Nor_cov: 2.557787<br />Sample: PrimPol-MDA","GC:  36<br />Nor_cov: 2.569858<br />Sample: PrimPol-MDA","GC:  37<br />Nor_cov: 2.579033<br />Sample: PrimPol-MDA","GC:  38<br />Nor_cov: 2.571929<br />Sample: PrimPol-MDA","GC:  39<br />Nor_cov: 2.551203<br />Sample: PrimPol-MDA","GC:  40<br />Nor_cov: 2.510550<br />Sample: PrimPol-MDA","GC:  41<br />Nor_cov: 2.469833<br />Sample: PrimPol-MDA","GC:  42<br />Nor_cov: 2.411920<br />Sample: PrimPol-MDA","GC:  43<br />Nor_cov: 2.341141<br />Sample: PrimPol-MDA","GC:  44<br />Nor_cov: 2.268073<br />Sample: PrimPol-MDA","GC:  45<br />Nor_cov: 2.178227<br />Sample: PrimPol-MDA","GC:  46<br />Nor_cov: 2.077332<br />Sample: PrimPol-MDA","GC:  47<br />Nor_cov: 1.968274<br />Sample: PrimPol-MDA","GC:  48<br />Nor_cov: 1.846205<br />Sample: PrimPol-MDA","GC:  49<br />Nor_cov: 1.720953<br />Sample: PrimPol-MDA","GC:  50<br />Nor_cov: 1.607730<br />Sample: PrimPol-MDA","GC:  51<br />Nor_cov: 1.491953<br />Sample: PrimPol-MDA","GC:  52<br />Nor_cov: 1.385360<br />Sample: PrimPol-MDA","GC:  53<br />Nor_cov: 1.284256<br />Sample: PrimPol-MDA","GC:  54<br />Nor_cov: 1.181314<br />Sample: PrimPol-MDA","GC:  55<br />Nor_cov: 1.084725<br />Sample: PrimPol-MDA","GC:  56<br />Nor_cov: 0.983315<br />Sample: PrimPol-MDA","GC:  57<br />Nor_cov: 0.873784<br />Sample: PrimPol-MDA","GC:  58<br />Nor_cov: 0.757599<br />Sample: PrimPol-MDA","GC:  59<br />Nor_cov: 0.634114<br />Sample: PrimPol-MDA","GC:  60<br />Nor_cov: 0.506736<br />Sample: PrimPol-MDA","GC:  61<br />Nor_cov: 0.397472<br />Sample: PrimPol-MDA","GC:  62<br />Nor_cov: 0.310209<br />Sample: PrimPol-MDA","GC:  63<br />Nor_cov: 0.247568<br />Sample: PrimPol-MDA","GC:  64<br />Nor_cov: 0.205640<br />Sample: PrimPol-MDA","GC:  65<br />Nor_cov: 0.179516<br />Sample: PrimPol-MDA","GC:  66<br />Nor_cov: 0.161584<br />Sample: PrimPol-MDA","GC:  67<br />Nor_cov: 0.149237<br />Sample: PrimPol-MDA","GC:  68<br />Nor_cov: 0.140612<br />Sample: PrimPol-MDA","GC:  69<br />Nor_cov: 0.134428<br />Sample: PrimPol-MDA","GC:  70<br />Nor_cov: 0.128957<br />Sample: PrimPol-MDA","GC:  71<br />Nor_cov: 0.124043<br />Sample: PrimPol-MDA","GC:  72<br />Nor_cov: 0.119905<br />Sample: PrimPol-MDA","GC:  73<br />Nor_cov: 0.115080<br />Sample: PrimPol-MDA","GC:  74<br />Nor_cov: 0.111740<br />Sample: PrimPol-MDA","GC:  75<br />Nor_cov: 0.106674<br />Sample: PrimPol-MDA","GC:  76<br />Nor_cov: 0.103136<br />Sample: PrimPol-MDA","GC:  77<br />Nor_cov: 0.098714<br />Sample: PrimPol-MDA","GC:  78<br />Nor_cov: 0.095246<br />Sample: PrimPol-MDA","GC:  79<br />Nor_cov: 0.092038<br />Sample: PrimPol-MDA","GC:  80<br />Nor_cov: 0.088872<br />Sample: PrimPol-MDA","GC:  81<br />Nor_cov: 0.085221<br />Sample: PrimPol-MDA","GC:  82<br />Nor_cov: 0.083434<br />Sample: PrimPol-MDA","GC:  83<br />Nor_cov: 0.081565<br />Sample: PrimPol-MDA","GC:  84<br />Nor_cov: 0.080495<br />Sample: PrimPol-MDA","GC:  85<br />Nor_cov: 0.076050<br />Sample: PrimPol-MDA","GC:  86<br />Nor_cov: 0.078141<br />Sample: PrimPol-MDA","GC:  87<br />Nor_cov: 0.072090<br />Sample: PrimPol-MDA","GC:  88<br />Nor_cov: 0.075956<br />Sample: PrimPol-MDA","GC:  89<br />Nor_cov: 0.069819<br />Sample: PrimPol-MDA","GC:  90<br />Nor_cov: 0.078191<br />Sample: PrimPol-MDA","GC:  91<br />Nor_cov: 0.091963<br />Sample: PrimPol-MDA","GC:  92<br />Nor_cov: 0.033720<br />Sample: PrimPol-MDA","GC:  93<br />Nor_cov: 0.000000<br />Sample: PrimPol-MDA","GC:  94<br />Nor_cov: 0.000000<br />Sample: PrimPol-MDA","GC:  95<br />Nor_cov: 0.000000<br />Sample: PrimPol-MDA","GC:  96<br />Nor_cov: 0.000000<br />Sample: PrimPol-MDA","GC:  97<br />Nor_cov: 0.000000<br />Sample: PrimPol-MDA","GC:  98<br />Nor_cov: 0.000000<br />Sample: PrimPol-MDA","GC:  99<br />Nor_cov: 0.000000<br />Sample: PrimPol-MDA","GC: 100<br />Nor_cov: 0.000000<br />Sample: PrimPol-MDA"],"type":"scatter","mode":"lines","line":{"width":5.66929133858268,"color":"rgba(139,37,0,1)","dash":"solid"},"hoveron":"points","name":"PrimPol-MDA","legendgroup":"PrimPol-MDA","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0,0,0,0,0,0,0,0,0,0,0,0,0.125017,0.07378,0.136382,0.106695,0.244598,0.212357,0.208043,0.200818,0.222969,0.213443,0.200156,0.199519,0.20839,0.194952,0.204243,0.207675,0.215646,0.229526,0.230053,0.262926,0.356967,0.430978,0.400164,0.481126,0.302769,0.482292,0.975184,0.913686,2.517362,3.472525,3.227392,5.113901,4.246328,3.675974,6.359195,2.082858,1.801528,0.891277,0.266504,0.359873,0.753282,1.318317,0.864527,0.911318,1.917165,0.656447,0.23817,0.625762,0.231997,0.868246,0.203677,0.207933,0.201266,0.201575,0.199501,0.199277,0.196775,0.197428,0.197056,0.195818,0.195309,0.193788,0.19302,0.192899,0.194454,0.192496,0.194463,0.192863,0.186589,0.189294,0.18392,0.189644,0.182015,0.170064,0.172556,0.167754,0.171525,0.172569,0.228292,0.204573,0,0,0,0,0,0,0,0,0],"text":["GC:   0<br />Nor_cov: 0.000000<br />Sample: piPolB","GC:   1<br />Nor_cov: 0.000000<br />Sample: piPolB","GC:   2<br />Nor_cov: 0.000000<br />Sample: piPolB","GC:   3<br />Nor_cov: 0.000000<br />Sample: piPolB","GC:   4<br />Nor_cov: 0.000000<br />Sample: piPolB","GC:   5<br />Nor_cov: 0.000000<br />Sample: piPolB","GC:   6<br />Nor_cov: 0.000000<br />Sample: piPolB","GC:   7<br />Nor_cov: 0.000000<br />Sample: piPolB","GC:   8<br />Nor_cov: 0.000000<br />Sample: piPolB","GC:   9<br />Nor_cov: 0.000000<br />Sample: piPolB","GC:  10<br />Nor_cov: 0.000000<br />Sample: piPolB","GC:  11<br />Nor_cov: 0.000000<br />Sample: piPolB","GC:  12<br />Nor_cov: 0.125017<br />Sample: piPolB","GC:  13<br />Nor_cov: 0.073780<br />Sample: piPolB","GC:  14<br />Nor_cov: 0.136382<br />Sample: piPolB","GC:  15<br />Nor_cov: 0.106695<br />Sample: piPolB","GC:  16<br />Nor_cov: 0.244598<br />Sample: piPolB","GC:  17<br />Nor_cov: 0.212357<br />Sample: piPolB","GC:  18<br />Nor_cov: 0.208043<br />Sample: piPolB","GC:  19<br />Nor_cov: 0.200818<br />Sample: piPolB","GC:  20<br />Nor_cov: 0.222969<br />Sample: piPolB","GC:  21<br />Nor_cov: 0.213443<br />Sample: piPolB","GC:  22<br />Nor_cov: 0.200156<br />Sample: piPolB","GC:  23<br />Nor_cov: 0.199519<br />Sample: piPolB","GC:  24<br />Nor_cov: 0.208390<br />Sample: piPolB","GC:  25<br />Nor_cov: 0.194952<br />Sample: piPolB","GC:  26<br />Nor_cov: 0.204243<br />Sample: piPolB","GC:  27<br />Nor_cov: 0.207675<br />Sample: piPolB","GC:  28<br />Nor_cov: 0.215646<br />Sample: piPolB","GC:  29<br />Nor_cov: 0.229526<br />Sample: piPolB","GC:  30<br />Nor_cov: 0.230053<br />Sample: piPolB","GC:  31<br />Nor_cov: 0.262926<br />Sample: piPolB","GC:  32<br />Nor_cov: 0.356967<br />Sample: piPolB","GC:  33<br />Nor_cov: 0.430978<br />Sample: piPolB","GC:  34<br />Nor_cov: 0.400164<br />Sample: piPolB","GC:  35<br />Nor_cov: 0.481126<br />Sample: piPolB","GC:  36<br />Nor_cov: 0.302769<br />Sample: piPolB","GC:  37<br />Nor_cov: 0.482292<br />Sample: piPolB","GC:  38<br />Nor_cov: 0.975184<br />Sample: piPolB","GC:  39<br />Nor_cov: 0.913686<br />Sample: piPolB","GC:  40<br />Nor_cov: 2.517362<br />Sample: piPolB","GC:  41<br />Nor_cov: 3.472525<br />Sample: piPolB","GC:  42<br />Nor_cov: 3.227392<br />Sample: piPolB","GC:  43<br />Nor_cov: 5.113901<br />Sample: piPolB","GC:  44<br />Nor_cov: 4.246328<br />Sample: piPolB","GC:  45<br />Nor_cov: 3.675974<br />Sample: piPolB","GC:  46<br />Nor_cov: 6.359195<br />Sample: piPolB","GC:  47<br />Nor_cov: 2.082858<br />Sample: piPolB","GC:  48<br />Nor_cov: 1.801528<br />Sample: piPolB","GC:  49<br />Nor_cov: 0.891277<br />Sample: piPolB","GC:  50<br />Nor_cov: 0.266504<br />Sample: piPolB","GC:  51<br />Nor_cov: 0.359873<br />Sample: piPolB","GC:  52<br />Nor_cov: 0.753282<br />Sample: piPolB","GC:  53<br />Nor_cov: 1.318317<br />Sample: piPolB","GC:  54<br />Nor_cov: 0.864527<br />Sample: piPolB","GC:  55<br />Nor_cov: 0.911318<br />Sample: piPolB","GC:  56<br />Nor_cov: 1.917165<br />Sample: piPolB","GC:  57<br />Nor_cov: 0.656447<br />Sample: piPolB","GC:  58<br />Nor_cov: 0.238170<br />Sample: piPolB","GC:  59<br />Nor_cov: 0.625762<br />Sample: piPolB","GC:  60<br />Nor_cov: 0.231997<br />Sample: piPolB","GC:  61<br />Nor_cov: 0.868246<br />Sample: piPolB","GC:  62<br />Nor_cov: 0.203677<br />Sample: piPolB","GC:  63<br />Nor_cov: 0.207933<br />Sample: piPolB","GC:  64<br />Nor_cov: 0.201266<br />Sample: piPolB","GC:  65<br />Nor_cov: 0.201575<br />Sample: piPolB","GC:  66<br />Nor_cov: 0.199501<br />Sample: piPolB","GC:  67<br />Nor_cov: 0.199277<br />Sample: piPolB","GC:  68<br />Nor_cov: 0.196775<br />Sample: piPolB","GC:  69<br />Nor_cov: 0.197428<br />Sample: piPolB","GC:  70<br />Nor_cov: 0.197056<br />Sample: piPolB","GC:  71<br />Nor_cov: 0.195818<br />Sample: piPolB","GC:  72<br />Nor_cov: 0.195309<br />Sample: piPolB","GC:  73<br />Nor_cov: 0.193788<br />Sample: piPolB","GC:  74<br />Nor_cov: 0.193020<br />Sample: piPolB","GC:  75<br />Nor_cov: 0.192899<br />Sample: piPolB","GC:  76<br />Nor_cov: 0.194454<br />Sample: piPolB","GC:  77<br />Nor_cov: 0.192496<br />Sample: piPolB","GC:  78<br />Nor_cov: 0.194463<br />Sample: piPolB","GC:  79<br />Nor_cov: 0.192863<br />Sample: piPolB","GC:  80<br />Nor_cov: 0.186589<br />Sample: piPolB","GC:  81<br />Nor_cov: 0.189294<br />Sample: piPolB","GC:  82<br />Nor_cov: 0.183920<br />Sample: piPolB","GC:  83<br />Nor_cov: 0.189644<br />Sample: piPolB","GC:  84<br />Nor_cov: 0.182015<br />Sample: piPolB","GC:  85<br />Nor_cov: 0.170064<br />Sample: piPolB","GC:  86<br />Nor_cov: 0.172556<br />Sample: piPolB","GC:  87<br />Nor_cov: 0.167754<br />Sample: piPolB","GC:  88<br />Nor_cov: 0.171525<br />Sample: piPolB","GC:  89<br />Nor_cov: 0.172569<br />Sample: piPolB","GC:  90<br />Nor_cov: 0.228292<br />Sample: piPolB","GC:  91<br />Nor_cov: 0.204573<br />Sample: piPolB","GC:  92<br />Nor_cov: 0.000000<br />Sample: piPolB","GC:  93<br />Nor_cov: 0.000000<br />Sample: piPolB","GC:  94<br />Nor_cov: 0.000000<br />Sample: piPolB","GC:  95<br />Nor_cov: 0.000000<br />Sample: piPolB","GC:  96<br />Nor_cov: 0.000000<br />Sample: piPolB","GC:  97<br />Nor_cov: 0.000000<br />Sample: piPolB","GC:  98<br />Nor_cov: 0.000000<br />Sample: piPolB","GC:  99<br />Nor_cov: 0.000000<br />Sample: piPolB","GC: 100<br />Nor_cov: 0.000000<br />Sample: piPolB"],"type":"scatter","mode":"lines","line":{"width":5.66929133858268,"color":"rgba(255,215,0,1)","dash":"solid"},"hoveron":"points","name":"piPolB","legendgroup":"piPolB","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0,0,0,0,0,0,0,0,0,0,0.137151,0,0.15239,0.148393,0.166244,0.25302,0.225272,0.299353,0.340144,0.379911,0.368684,0.359294,0.359675,0.3612,0.361141,0.370376,0.376087,0.374727,0.39375,0.401588,0.417144,0.521188,0.656384,3.587968,0.537119,0.52594,1.277829,0.795353,1.491792,0.958059,2.623554,2.89601,2.572525,5.781727,3.153047,4.825296,7.544698,1.562858,1.421858,2.271519,0.374675,0.394507,0.351463,0.385954,0.573219,1.86113,0.387162,0.270095,0.253926,0.259646,0.512164,0.25747,0.201707,0.413203,0.187495,0.175468,0.172149,0.168725,0.166436,0.164002,0.160583,0.157797,0.155655,0.153242,0.150675,0.148143,0.146094,0.144021,0.141669,0.138328,0.137106,0.131384,0.127833,0.129305,0.124999,0.122752,0.12607,0.110276,0.117335,0.095922,0.087459,0.199493,0,0.137151,0,0,0,0,0,0,0],"text":["GC:   0<br />Nor_cov: 0.000000<br />Sample: piPolB+D","GC:   1<br />Nor_cov: 0.000000<br />Sample: piPolB+D","GC:   2<br />Nor_cov: 0.000000<br />Sample: piPolB+D","GC:   3<br />Nor_cov: 0.000000<br />Sample: piPolB+D","GC:   4<br />Nor_cov: 0.000000<br />Sample: piPolB+D","GC:   5<br />Nor_cov: 0.000000<br />Sample: piPolB+D","GC:   6<br />Nor_cov: 0.000000<br />Sample: piPolB+D","GC:   7<br />Nor_cov: 0.000000<br />Sample: piPolB+D","GC:   8<br />Nor_cov: 0.000000<br />Sample: piPolB+D","GC:   9<br />Nor_cov: 0.000000<br />Sample: piPolB+D","GC:  10<br />Nor_cov: 0.137151<br />Sample: piPolB+D","GC:  11<br />Nor_cov: 0.000000<br />Sample: piPolB+D","GC:  12<br />Nor_cov: 0.152390<br />Sample: piPolB+D","GC:  13<br />Nor_cov: 0.148393<br />Sample: piPolB+D","GC:  14<br />Nor_cov: 0.166244<br />Sample: piPolB+D","GC:  15<br />Nor_cov: 0.253020<br />Sample: piPolB+D","GC:  16<br />Nor_cov: 0.225272<br />Sample: piPolB+D","GC:  17<br />Nor_cov: 0.299353<br />Sample: piPolB+D","GC:  18<br />Nor_cov: 0.340144<br />Sample: piPolB+D","GC:  19<br />Nor_cov: 0.379911<br />Sample: piPolB+D","GC:  20<br />Nor_cov: 0.368684<br />Sample: piPolB+D","GC:  21<br />Nor_cov: 0.359294<br />Sample: piPolB+D","GC:  22<br />Nor_cov: 0.359675<br />Sample: piPolB+D","GC:  23<br />Nor_cov: 0.361200<br />Sample: piPolB+D","GC:  24<br />Nor_cov: 0.361141<br />Sample: piPolB+D","GC:  25<br />Nor_cov: 0.370376<br />Sample: piPolB+D","GC:  26<br />Nor_cov: 0.376087<br />Sample: piPolB+D","GC:  27<br />Nor_cov: 0.374727<br />Sample: piPolB+D","GC:  28<br />Nor_cov: 0.393750<br />Sample: piPolB+D","GC:  29<br />Nor_cov: 0.401588<br />Sample: piPolB+D","GC:  30<br />Nor_cov: 0.417144<br />Sample: piPolB+D","GC:  31<br />Nor_cov: 0.521188<br />Sample: piPolB+D","GC:  32<br />Nor_cov: 0.656384<br />Sample: piPolB+D","GC:  33<br />Nor_cov: 3.587968<br />Sample: piPolB+D","GC:  34<br />Nor_cov: 0.537119<br />Sample: piPolB+D","GC:  35<br />Nor_cov: 0.525940<br />Sample: piPolB+D","GC:  36<br />Nor_cov: 1.277829<br />Sample: piPolB+D","GC:  37<br />Nor_cov: 0.795353<br />Sample: piPolB+D","GC:  38<br />Nor_cov: 1.491792<br />Sample: piPolB+D","GC:  39<br />Nor_cov: 0.958059<br />Sample: piPolB+D","GC:  40<br />Nor_cov: 2.623554<br />Sample: piPolB+D","GC:  41<br />Nor_cov: 2.896010<br />Sample: piPolB+D","GC:  42<br />Nor_cov: 2.572525<br />Sample: piPolB+D","GC:  43<br />Nor_cov: 5.781727<br />Sample: piPolB+D","GC:  44<br />Nor_cov: 3.153047<br />Sample: piPolB+D","GC:  45<br />Nor_cov: 4.825296<br />Sample: piPolB+D","GC:  46<br />Nor_cov: 7.544698<br />Sample: piPolB+D","GC:  47<br />Nor_cov: 1.562858<br />Sample: piPolB+D","GC:  48<br />Nor_cov: 1.421858<br />Sample: piPolB+D","GC:  49<br />Nor_cov: 2.271519<br />Sample: piPolB+D","GC:  50<br />Nor_cov: 0.374675<br />Sample: piPolB+D","GC:  51<br />Nor_cov: 0.394507<br />Sample: piPolB+D","GC:  52<br />Nor_cov: 0.351463<br />Sample: piPolB+D","GC:  53<br />Nor_cov: 0.385954<br />Sample: piPolB+D","GC:  54<br />Nor_cov: 0.573219<br />Sample: piPolB+D","GC:  55<br />Nor_cov: 1.861130<br />Sample: piPolB+D","GC:  56<br />Nor_cov: 0.387162<br />Sample: piPolB+D","GC:  57<br />Nor_cov: 0.270095<br />Sample: piPolB+D","GC:  58<br />Nor_cov: 0.253926<br />Sample: piPolB+D","GC:  59<br />Nor_cov: 0.259646<br />Sample: piPolB+D","GC:  60<br />Nor_cov: 0.512164<br />Sample: piPolB+D","GC:  61<br />Nor_cov: 0.257470<br />Sample: piPolB+D","GC:  62<br />Nor_cov: 0.201707<br />Sample: piPolB+D","GC:  63<br />Nor_cov: 0.413203<br />Sample: piPolB+D","GC:  64<br />Nor_cov: 0.187495<br />Sample: piPolB+D","GC:  65<br />Nor_cov: 0.175468<br />Sample: piPolB+D","GC:  66<br />Nor_cov: 0.172149<br />Sample: piPolB+D","GC:  67<br />Nor_cov: 0.168725<br />Sample: piPolB+D","GC:  68<br />Nor_cov: 0.166436<br />Sample: piPolB+D","GC:  69<br />Nor_cov: 0.164002<br />Sample: piPolB+D","GC:  70<br />Nor_cov: 0.160583<br />Sample: piPolB+D","GC:  71<br />Nor_cov: 0.157797<br />Sample: piPolB+D","GC:  72<br />Nor_cov: 0.155655<br />Sample: piPolB+D","GC:  73<br />Nor_cov: 0.153242<br />Sample: piPolB+D","GC:  74<br />Nor_cov: 0.150675<br />Sample: piPolB+D","GC:  75<br />Nor_cov: 0.148143<br />Sample: piPolB+D","GC:  76<br />Nor_cov: 0.146094<br />Sample: piPolB+D","GC:  77<br />Nor_cov: 0.144021<br />Sample: piPolB+D","GC:  78<br />Nor_cov: 0.141669<br />Sample: piPolB+D","GC:  79<br />Nor_cov: 0.138328<br />Sample: piPolB+D","GC:  80<br />Nor_cov: 0.137106<br />Sample: piPolB+D","GC:  81<br />Nor_cov: 0.131384<br />Sample: piPolB+D","GC:  82<br />Nor_cov: 0.127833<br />Sample: piPolB+D","GC:  83<br />Nor_cov: 0.129305<br />Sample: piPolB+D","GC:  84<br />Nor_cov: 0.124999<br />Sample: piPolB+D","GC:  85<br />Nor_cov: 0.122752<br />Sample: piPolB+D","GC:  86<br />Nor_cov: 0.126070<br />Sample: piPolB+D","GC:  87<br />Nor_cov: 0.110276<br />Sample: piPolB+D","GC:  88<br />Nor_cov: 0.117335<br />Sample: piPolB+D","GC:  89<br />Nor_cov: 0.095922<br />Sample: piPolB+D","GC:  90<br />Nor_cov: 0.087459<br />Sample: piPolB+D","GC:  91<br />Nor_cov: 0.199493<br />Sample: piPolB+D","GC:  92<br />Nor_cov: 0.000000<br />Sample: piPolB+D","GC:  93<br />Nor_cov: 0.137151<br />Sample: piPolB+D","GC:  94<br />Nor_cov: 0.000000<br />Sample: piPolB+D","GC:  95<br />Nor_cov: 0.000000<br />Sample: piPolB+D","GC:  96<br />Nor_cov: 0.000000<br />Sample: piPolB+D","GC:  97<br />Nor_cov: 0.000000<br />Sample: piPolB+D","GC:  98<br />Nor_cov: 0.000000<br />Sample: piPolB+D","GC:  99<br />Nor_cov: 0.000000<br />Sample: piPolB+D","GC: 100<br />Nor_cov: 0.000000<br />Sample: piPolB+D"],"type":"scatter","mode":"lines","line":{"width":5.66929133858268,"color":"rgba(255,185,15,1)","dash":"solid"},"hoveron":"points","name":"piPolB+D","legendgroup":"piPolB+D","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0,0,0,0,0,0,0,0,0,0,0.072774,0.145548,0.331525,0.307798,0.294036,0.390219,0.350158,0.385912,0.404423,0.437905,0.396293,0.376721,0.370409,0.363057,0.366757,0.357393,0.348886,0.348065,0.348573,0.344544,0.342427,0.34688,0.34777,0.350609,0.36002,0.36896,0.37722,0.384104,0.396535,0.405657,0.415973,0.425379,0.437138,0.452932,0.46808,0.486169,0.505475,0.5273,0.548652,0.575102,0.602403,0.630888,0.656932,0.683046,0.717527,0.757875,0.811073,0.879951,0.96627,1.061797,1.159691,1.253257,1.33445,1.391704,1.428192,1.453741,1.467683,1.472837,1.477028,1.480501,1.477314,1.472256,1.46805,1.4689,1.463853,1.45657,1.450094,1.437582,1.42408,1.412505,1.387158,1.375932,1.361797,1.363703,1.351728,1.352834,1.347919,1.281099,1.323011,1.098751,0.788911,1.521635,0.436643,0.254709,0,0,0,0,0,0,0],"text":["GC:   0<br />Nor_cov: 0.000000<br />Sample: piMDA","GC:   1<br />Nor_cov: 0.000000<br />Sample: piMDA","GC:   2<br />Nor_cov: 0.000000<br />Sample: piMDA","GC:   3<br />Nor_cov: 0.000000<br />Sample: piMDA","GC:   4<br />Nor_cov: 0.000000<br />Sample: piMDA","GC:   5<br />Nor_cov: 0.000000<br />Sample: piMDA","GC:   6<br />Nor_cov: 0.000000<br />Sample: piMDA","GC:   7<br />Nor_cov: 0.000000<br />Sample: piMDA","GC:   8<br />Nor_cov: 0.000000<br />Sample: piMDA","GC:   9<br />Nor_cov: 0.000000<br />Sample: piMDA","GC:  10<br />Nor_cov: 0.072774<br />Sample: piMDA","GC:  11<br />Nor_cov: 0.145548<br />Sample: piMDA","GC:  12<br />Nor_cov: 0.331525<br />Sample: piMDA","GC:  13<br />Nor_cov: 0.307798<br />Sample: piMDA","GC:  14<br />Nor_cov: 0.294036<br />Sample: piMDA","GC:  15<br />Nor_cov: 0.390219<br />Sample: piMDA","GC:  16<br />Nor_cov: 0.350158<br />Sample: piMDA","GC:  17<br />Nor_cov: 0.385912<br />Sample: piMDA","GC:  18<br />Nor_cov: 0.404423<br />Sample: piMDA","GC:  19<br />Nor_cov: 0.437905<br />Sample: piMDA","GC:  20<br />Nor_cov: 0.396293<br />Sample: piMDA","GC:  21<br />Nor_cov: 0.376721<br />Sample: piMDA","GC:  22<br />Nor_cov: 0.370409<br />Sample: piMDA","GC:  23<br />Nor_cov: 0.363057<br />Sample: piMDA","GC:  24<br />Nor_cov: 0.366757<br />Sample: piMDA","GC:  25<br />Nor_cov: 0.357393<br />Sample: piMDA","GC:  26<br />Nor_cov: 0.348886<br />Sample: piMDA","GC:  27<br />Nor_cov: 0.348065<br />Sample: piMDA","GC:  28<br />Nor_cov: 0.348573<br />Sample: piMDA","GC:  29<br />Nor_cov: 0.344544<br />Sample: piMDA","GC:  30<br />Nor_cov: 0.342427<br />Sample: piMDA","GC:  31<br />Nor_cov: 0.346880<br />Sample: piMDA","GC:  32<br />Nor_cov: 0.347770<br />Sample: piMDA","GC:  33<br />Nor_cov: 0.350609<br />Sample: piMDA","GC:  34<br />Nor_cov: 0.360020<br />Sample: piMDA","GC:  35<br />Nor_cov: 0.368960<br />Sample: piMDA","GC:  36<br />Nor_cov: 0.377220<br />Sample: piMDA","GC:  37<br />Nor_cov: 0.384104<br />Sample: piMDA","GC:  38<br />Nor_cov: 0.396535<br />Sample: piMDA","GC:  39<br />Nor_cov: 0.405657<br />Sample: piMDA","GC:  40<br />Nor_cov: 0.415973<br />Sample: piMDA","GC:  41<br />Nor_cov: 0.425379<br />Sample: piMDA","GC:  42<br />Nor_cov: 0.437138<br />Sample: piMDA","GC:  43<br />Nor_cov: 0.452932<br />Sample: piMDA","GC:  44<br />Nor_cov: 0.468080<br />Sample: piMDA","GC:  45<br />Nor_cov: 0.486169<br />Sample: piMDA","GC:  46<br />Nor_cov: 0.505475<br />Sample: piMDA","GC:  47<br />Nor_cov: 0.527300<br />Sample: piMDA","GC:  48<br />Nor_cov: 0.548652<br />Sample: piMDA","GC:  49<br />Nor_cov: 0.575102<br />Sample: piMDA","GC:  50<br />Nor_cov: 0.602403<br />Sample: piMDA","GC:  51<br />Nor_cov: 0.630888<br />Sample: piMDA","GC:  52<br />Nor_cov: 0.656932<br />Sample: piMDA","GC:  53<br />Nor_cov: 0.683046<br />Sample: piMDA","GC:  54<br />Nor_cov: 0.717527<br />Sample: piMDA","GC:  55<br />Nor_cov: 0.757875<br />Sample: piMDA","GC:  56<br />Nor_cov: 0.811073<br />Sample: piMDA","GC:  57<br />Nor_cov: 0.879951<br />Sample: piMDA","GC:  58<br />Nor_cov: 0.966270<br />Sample: piMDA","GC:  59<br />Nor_cov: 1.061797<br />Sample: piMDA","GC:  60<br />Nor_cov: 1.159691<br />Sample: piMDA","GC:  61<br />Nor_cov: 1.253257<br />Sample: piMDA","GC:  62<br />Nor_cov: 1.334450<br />Sample: piMDA","GC:  63<br />Nor_cov: 1.391704<br />Sample: piMDA","GC:  64<br />Nor_cov: 1.428192<br />Sample: piMDA","GC:  65<br />Nor_cov: 1.453741<br />Sample: piMDA","GC:  66<br />Nor_cov: 1.467683<br />Sample: piMDA","GC:  67<br />Nor_cov: 1.472837<br />Sample: piMDA","GC:  68<br />Nor_cov: 1.477028<br />Sample: piMDA","GC:  69<br />Nor_cov: 1.480501<br />Sample: piMDA","GC:  70<br />Nor_cov: 1.477314<br />Sample: piMDA","GC:  71<br />Nor_cov: 1.472256<br />Sample: piMDA","GC:  72<br />Nor_cov: 1.468050<br />Sample: piMDA","GC:  73<br />Nor_cov: 1.468900<br />Sample: piMDA","GC:  74<br />Nor_cov: 1.463853<br />Sample: piMDA","GC:  75<br />Nor_cov: 1.456570<br />Sample: piMDA","GC:  76<br />Nor_cov: 1.450094<br />Sample: piMDA","GC:  77<br />Nor_cov: 1.437582<br />Sample: piMDA","GC:  78<br />Nor_cov: 1.424080<br />Sample: piMDA","GC:  79<br />Nor_cov: 1.412505<br />Sample: piMDA","GC:  80<br />Nor_cov: 1.387158<br />Sample: piMDA","GC:  81<br />Nor_cov: 1.375932<br />Sample: piMDA","GC:  82<br />Nor_cov: 1.361797<br />Sample: piMDA","GC:  83<br />Nor_cov: 1.363703<br />Sample: piMDA","GC:  84<br />Nor_cov: 1.351728<br />Sample: piMDA","GC:  85<br />Nor_cov: 1.352834<br />Sample: piMDA","GC:  86<br />Nor_cov: 1.347919<br />Sample: piMDA","GC:  87<br />Nor_cov: 1.281099<br />Sample: piMDA","GC:  88<br />Nor_cov: 1.323011<br />Sample: piMDA","GC:  89<br />Nor_cov: 1.098751<br />Sample: piMDA","GC:  90<br />Nor_cov: 0.788911<br />Sample: piMDA","GC:  91<br />Nor_cov: 1.521635<br />Sample: piMDA","GC:  92<br />Nor_cov: 0.436643<br />Sample: piMDA","GC:  93<br />Nor_cov: 0.254709<br />Sample: piMDA","GC:  94<br />Nor_cov: 0.000000<br />Sample: piMDA","GC:  95<br />Nor_cov: 0.000000<br />Sample: piMDA","GC:  96<br />Nor_cov: 0.000000<br />Sample: piMDA","GC:  97<br />Nor_cov: 0.000000<br />Sample: piMDA","GC:  98<br />Nor_cov: 0.000000<br />Sample: piMDA","GC:  99<br />Nor_cov: 0.000000<br />Sample: piMDA","GC: 100<br />Nor_cov: 0.000000<br />Sample: piMDA"],"type":"scatter","mode":"lines","line":{"width":5.66929133858268,"color":"rgba(0,154,205,1)","dash":"solid"},"hoveron":"points","name":"piMDA","legendgroup":"piMDA","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0,0,0,0,0,0,0,0,0,0,0.335421,0.31306,0.409959,0.402505,0.355072,0.431999,0.407042,0.431912,0.425948,0.539205,0.435968,0.394688,0.389205,0.381895,0.383746,0.375431,0.367307,0.371363,0.362105,0.358485,0.353494,0.350825,0.349834,0.353191,0.357318,0.363921,0.370888,0.375758,0.384051,0.391011,0.400167,0.40803,0.418412,0.432499,0.44677,0.463303,0.481949,0.504564,0.527837,0.552501,0.58046,0.607645,0.63056,0.656433,0.689599,0.730546,0.782693,0.8487,0.93241,1.027127,1.120589,1.216217,1.297908,1.354892,1.395782,1.427521,1.448207,1.461884,1.473356,1.489459,1.500349,1.507721,1.519211,1.532625,1.543404,1.552743,1.565729,1.573068,1.575979,1.580487,1.57055,1.566158,1.574472,1.596077,1.588901,1.601351,1.631539,1.583731,1.645128,1.414119,1.079181,1.768584,0.456173,0.436047,0,0,0,0,0,0,0],"text":["GC:   0<br />Nor_cov: 0.000000<br />Sample: piMDA2","GC:   1<br />Nor_cov: 0.000000<br />Sample: piMDA2","GC:   2<br />Nor_cov: 0.000000<br />Sample: piMDA2","GC:   3<br />Nor_cov: 0.000000<br />Sample: piMDA2","GC:   4<br />Nor_cov: 0.000000<br />Sample: piMDA2","GC:   5<br />Nor_cov: 0.000000<br />Sample: piMDA2","GC:   6<br />Nor_cov: 0.000000<br />Sample: piMDA2","GC:   7<br />Nor_cov: 0.000000<br />Sample: piMDA2","GC:   8<br />Nor_cov: 0.000000<br />Sample: piMDA2","GC:   9<br />Nor_cov: 0.000000<br />Sample: piMDA2","GC:  10<br />Nor_cov: 0.335421<br />Sample: piMDA2","GC:  11<br />Nor_cov: 0.313060<br />Sample: piMDA2","GC:  12<br />Nor_cov: 0.409959<br />Sample: piMDA2","GC:  13<br />Nor_cov: 0.402505<br />Sample: piMDA2","GC:  14<br />Nor_cov: 0.355072<br />Sample: piMDA2","GC:  15<br />Nor_cov: 0.431999<br />Sample: piMDA2","GC:  16<br />Nor_cov: 0.407042<br />Sample: piMDA2","GC:  17<br />Nor_cov: 0.431912<br />Sample: piMDA2","GC:  18<br />Nor_cov: 0.425948<br />Sample: piMDA2","GC:  19<br />Nor_cov: 0.539205<br />Sample: piMDA2","GC:  20<br />Nor_cov: 0.435968<br />Sample: piMDA2","GC:  21<br />Nor_cov: 0.394688<br />Sample: piMDA2","GC:  22<br />Nor_cov: 0.389205<br />Sample: piMDA2","GC:  23<br />Nor_cov: 0.381895<br />Sample: piMDA2","GC:  24<br />Nor_cov: 0.383746<br />Sample: piMDA2","GC:  25<br />Nor_cov: 0.375431<br />Sample: piMDA2","GC:  26<br />Nor_cov: 0.367307<br />Sample: piMDA2","GC:  27<br />Nor_cov: 0.371363<br />Sample: piMDA2","GC:  28<br />Nor_cov: 0.362105<br />Sample: piMDA2","GC:  29<br />Nor_cov: 0.358485<br />Sample: piMDA2","GC:  30<br />Nor_cov: 0.353494<br />Sample: piMDA2","GC:  31<br />Nor_cov: 0.350825<br />Sample: piMDA2","GC:  32<br />Nor_cov: 0.349834<br />Sample: piMDA2","GC:  33<br />Nor_cov: 0.353191<br />Sample: piMDA2","GC:  34<br />Nor_cov: 0.357318<br />Sample: piMDA2","GC:  35<br />Nor_cov: 0.363921<br />Sample: piMDA2","GC:  36<br />Nor_cov: 0.370888<br />Sample: piMDA2","GC:  37<br />Nor_cov: 0.375758<br />Sample: piMDA2","GC:  38<br />Nor_cov: 0.384051<br />Sample: piMDA2","GC:  39<br />Nor_cov: 0.391011<br />Sample: piMDA2","GC:  40<br />Nor_cov: 0.400167<br />Sample: piMDA2","GC:  41<br />Nor_cov: 0.408030<br />Sample: piMDA2","GC:  42<br />Nor_cov: 0.418412<br />Sample: piMDA2","GC:  43<br />Nor_cov: 0.432499<br />Sample: piMDA2","GC:  44<br />Nor_cov: 0.446770<br />Sample: piMDA2","GC:  45<br />Nor_cov: 0.463303<br />Sample: piMDA2","GC:  46<br />Nor_cov: 0.481949<br />Sample: piMDA2","GC:  47<br />Nor_cov: 0.504564<br />Sample: piMDA2","GC:  48<br />Nor_cov: 0.527837<br />Sample: piMDA2","GC:  49<br />Nor_cov: 0.552501<br />Sample: piMDA2","GC:  50<br />Nor_cov: 0.580460<br />Sample: piMDA2","GC:  51<br />Nor_cov: 0.607645<br />Sample: piMDA2","GC:  52<br />Nor_cov: 0.630560<br />Sample: piMDA2","GC:  53<br />Nor_cov: 0.656433<br />Sample: piMDA2","GC:  54<br />Nor_cov: 0.689599<br />Sample: piMDA2","GC:  55<br />Nor_cov: 0.730546<br />Sample: piMDA2","GC:  56<br />Nor_cov: 0.782693<br />Sample: piMDA2","GC:  57<br />Nor_cov: 0.848700<br />Sample: piMDA2","GC:  58<br />Nor_cov: 0.932410<br />Sample: piMDA2","GC:  59<br />Nor_cov: 1.027127<br />Sample: piMDA2","GC:  60<br />Nor_cov: 1.120589<br />Sample: piMDA2","GC:  61<br />Nor_cov: 1.216217<br />Sample: piMDA2","GC:  62<br />Nor_cov: 1.297908<br />Sample: piMDA2","GC:  63<br />Nor_cov: 1.354892<br />Sample: piMDA2","GC:  64<br />Nor_cov: 1.395782<br />Sample: piMDA2","GC:  65<br />Nor_cov: 1.427521<br />Sample: piMDA2","GC:  66<br />Nor_cov: 1.448207<br />Sample: piMDA2","GC:  67<br />Nor_cov: 1.461884<br />Sample: piMDA2","GC:  68<br />Nor_cov: 1.473356<br />Sample: piMDA2","GC:  69<br />Nor_cov: 1.489459<br />Sample: piMDA2","GC:  70<br />Nor_cov: 1.500349<br />Sample: piMDA2","GC:  71<br />Nor_cov: 1.507721<br />Sample: piMDA2","GC:  72<br />Nor_cov: 1.519211<br />Sample: piMDA2","GC:  73<br />Nor_cov: 1.532625<br />Sample: piMDA2","GC:  74<br />Nor_cov: 1.543404<br />Sample: piMDA2","GC:  75<br />Nor_cov: 1.552743<br />Sample: piMDA2","GC:  76<br />Nor_cov: 1.565729<br />Sample: piMDA2","GC:  77<br />Nor_cov: 1.573068<br />Sample: piMDA2","GC:  78<br />Nor_cov: 1.575979<br />Sample: piMDA2","GC:  79<br />Nor_cov: 1.580487<br />Sample: piMDA2","GC:  80<br />Nor_cov: 1.570550<br />Sample: piMDA2","GC:  81<br />Nor_cov: 1.566158<br />Sample: piMDA2","GC:  82<br />Nor_cov: 1.574472<br />Sample: piMDA2","GC:  83<br />Nor_cov: 1.596077<br />Sample: piMDA2","GC:  84<br />Nor_cov: 1.588901<br />Sample: piMDA2","GC:  85<br />Nor_cov: 1.601351<br />Sample: piMDA2","GC:  86<br />Nor_cov: 1.631539<br />Sample: piMDA2","GC:  87<br />Nor_cov: 1.583731<br />Sample: piMDA2","GC:  88<br />Nor_cov: 1.645128<br />Sample: piMDA2","GC:  89<br />Nor_cov: 1.414119<br />Sample: piMDA2","GC:  90<br />Nor_cov: 1.079181<br />Sample: piMDA2","GC:  91<br />Nor_cov: 1.768584<br />Sample: piMDA2","GC:  92<br />Nor_cov: 0.456173<br />Sample: piMDA2","GC:  93<br />Nor_cov: 0.436047<br />Sample: piMDA2","GC:  94<br />Nor_cov: 0.000000<br />Sample: piMDA2","GC:  95<br />Nor_cov: 0.000000<br />Sample: piMDA2","GC:  96<br />Nor_cov: 0.000000<br />Sample: piMDA2","GC:  97<br />Nor_cov: 0.000000<br />Sample: piMDA2","GC:  98<br />Nor_cov: 0.000000<br />Sample: piMDA2","GC:  99<br />Nor_cov: 0.000000<br />Sample: piMDA2","GC: 100<br />Nor_cov: 0.000000<br />Sample: piMDA2"],"type":"scatter","mode":"lines","line":{"width":5.66929133858268,"color":"rgba(0,154,205,1)","dash":"solid"},"hoveron":"points","name":"piMDA2","legendgroup":"piMDA2","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0,0,0,0,0,0,0,0,0,0,0.075576,0.201536,0.109165,0.183365,0.186268,0.221516,0.224537,0.230524,0.249484,0.266385,0.244038,0.235433,0.225728,0.225659,0.223626,0.225989,0.223356,0.226498,0.229356,0.228524,0.229599,0.232143,0.237505,0.24264,0.251025,0.258558,0.265471,0.274289,0.283322,0.291784,0.298693,0.306325,0.315156,0.324986,0.335261,0.346828,0.359283,0.372393,0.385114,0.402774,0.420384,0.439275,0.460393,0.482095,0.509941,0.548062,0.599518,0.672919,0.764834,0.868962,0.982676,1.101075,1.210972,1.296141,1.368028,1.430778,1.48157,1.529997,1.580276,1.634482,1.680828,1.725514,1.778607,1.835466,1.888023,1.943085,2.004494,2.055656,2.10157,2.139141,2.147582,2.157967,2.181915,2.207912,2.209391,2.219833,2.250351,2.145263,2.170983,1.802232,1.364749,2.679512,0.967373,0.642396,0,0,0,0,0,0,0],"text":["GC:   0<br />Nor_cov: 0.000000<br />Sample: piMDA+D","GC:   1<br />Nor_cov: 0.000000<br />Sample: piMDA+D","GC:   2<br />Nor_cov: 0.000000<br />Sample: piMDA+D","GC:   3<br />Nor_cov: 0.000000<br />Sample: piMDA+D","GC:   4<br />Nor_cov: 0.000000<br />Sample: piMDA+D","GC:   5<br />Nor_cov: 0.000000<br />Sample: piMDA+D","GC:   6<br />Nor_cov: 0.000000<br />Sample: piMDA+D","GC:   7<br />Nor_cov: 0.000000<br />Sample: piMDA+D","GC:   8<br />Nor_cov: 0.000000<br />Sample: piMDA+D","GC:   9<br />Nor_cov: 0.000000<br />Sample: piMDA+D","GC:  10<br />Nor_cov: 0.075576<br />Sample: piMDA+D","GC:  11<br />Nor_cov: 0.201536<br />Sample: piMDA+D","GC:  12<br />Nor_cov: 0.109165<br />Sample: piMDA+D","GC:  13<br />Nor_cov: 0.183365<br />Sample: piMDA+D","GC:  14<br />Nor_cov: 0.186268<br />Sample: piMDA+D","GC:  15<br />Nor_cov: 0.221516<br />Sample: piMDA+D","GC:  16<br />Nor_cov: 0.224537<br />Sample: piMDA+D","GC:  17<br />Nor_cov: 0.230524<br />Sample: piMDA+D","GC:  18<br />Nor_cov: 0.249484<br />Sample: piMDA+D","GC:  19<br />Nor_cov: 0.266385<br />Sample: piMDA+D","GC:  20<br />Nor_cov: 0.244038<br />Sample: piMDA+D","GC:  21<br />Nor_cov: 0.235433<br />Sample: piMDA+D","GC:  22<br />Nor_cov: 0.225728<br />Sample: piMDA+D","GC:  23<br />Nor_cov: 0.225659<br />Sample: piMDA+D","GC:  24<br />Nor_cov: 0.223626<br />Sample: piMDA+D","GC:  25<br />Nor_cov: 0.225989<br />Sample: piMDA+D","GC:  26<br />Nor_cov: 0.223356<br />Sample: piMDA+D","GC:  27<br />Nor_cov: 0.226498<br />Sample: piMDA+D","GC:  28<br />Nor_cov: 0.229356<br />Sample: piMDA+D","GC:  29<br />Nor_cov: 0.228524<br />Sample: piMDA+D","GC:  30<br />Nor_cov: 0.229599<br />Sample: piMDA+D","GC:  31<br />Nor_cov: 0.232143<br />Sample: piMDA+D","GC:  32<br />Nor_cov: 0.237505<br />Sample: piMDA+D","GC:  33<br />Nor_cov: 0.242640<br />Sample: piMDA+D","GC:  34<br />Nor_cov: 0.251025<br />Sample: piMDA+D","GC:  35<br />Nor_cov: 0.258558<br />Sample: piMDA+D","GC:  36<br />Nor_cov: 0.265471<br />Sample: piMDA+D","GC:  37<br />Nor_cov: 0.274289<br />Sample: piMDA+D","GC:  38<br />Nor_cov: 0.283322<br />Sample: piMDA+D","GC:  39<br />Nor_cov: 0.291784<br />Sample: piMDA+D","GC:  40<br />Nor_cov: 0.298693<br />Sample: piMDA+D","GC:  41<br />Nor_cov: 0.306325<br />Sample: piMDA+D","GC:  42<br />Nor_cov: 0.315156<br />Sample: piMDA+D","GC:  43<br />Nor_cov: 0.324986<br />Sample: piMDA+D","GC:  44<br />Nor_cov: 0.335261<br />Sample: piMDA+D","GC:  45<br />Nor_cov: 0.346828<br />Sample: piMDA+D","GC:  46<br />Nor_cov: 0.359283<br />Sample: piMDA+D","GC:  47<br />Nor_cov: 0.372393<br />Sample: piMDA+D","GC:  48<br />Nor_cov: 0.385114<br />Sample: piMDA+D","GC:  49<br />Nor_cov: 0.402774<br />Sample: piMDA+D","GC:  50<br />Nor_cov: 0.420384<br />Sample: piMDA+D","GC:  51<br />Nor_cov: 0.439275<br />Sample: piMDA+D","GC:  52<br />Nor_cov: 0.460393<br />Sample: piMDA+D","GC:  53<br />Nor_cov: 0.482095<br />Sample: piMDA+D","GC:  54<br />Nor_cov: 0.509941<br />Sample: piMDA+D","GC:  55<br />Nor_cov: 0.548062<br />Sample: piMDA+D","GC:  56<br />Nor_cov: 0.599518<br />Sample: piMDA+D","GC:  57<br />Nor_cov: 0.672919<br />Sample: piMDA+D","GC:  58<br />Nor_cov: 0.764834<br />Sample: piMDA+D","GC:  59<br />Nor_cov: 0.868962<br />Sample: piMDA+D","GC:  60<br />Nor_cov: 0.982676<br />Sample: piMDA+D","GC:  61<br />Nor_cov: 1.101075<br />Sample: piMDA+D","GC:  62<br />Nor_cov: 1.210972<br />Sample: piMDA+D","GC:  63<br />Nor_cov: 1.296141<br />Sample: piMDA+D","GC:  64<br />Nor_cov: 1.368028<br />Sample: piMDA+D","GC:  65<br />Nor_cov: 1.430778<br />Sample: piMDA+D","GC:  66<br />Nor_cov: 1.481570<br />Sample: piMDA+D","GC:  67<br />Nor_cov: 1.529997<br />Sample: piMDA+D","GC:  68<br />Nor_cov: 1.580276<br />Sample: piMDA+D","GC:  69<br />Nor_cov: 1.634482<br />Sample: piMDA+D","GC:  70<br />Nor_cov: 1.680828<br />Sample: piMDA+D","GC:  71<br />Nor_cov: 1.725514<br />Sample: piMDA+D","GC:  72<br />Nor_cov: 1.778607<br />Sample: piMDA+D","GC:  73<br />Nor_cov: 1.835466<br />Sample: piMDA+D","GC:  74<br />Nor_cov: 1.888023<br />Sample: piMDA+D","GC:  75<br />Nor_cov: 1.943085<br />Sample: piMDA+D","GC:  76<br />Nor_cov: 2.004494<br />Sample: piMDA+D","GC:  77<br />Nor_cov: 2.055656<br />Sample: piMDA+D","GC:  78<br />Nor_cov: 2.101570<br />Sample: piMDA+D","GC:  79<br />Nor_cov: 2.139141<br />Sample: piMDA+D","GC:  80<br />Nor_cov: 2.147582<br />Sample: piMDA+D","GC:  81<br />Nor_cov: 2.157967<br />Sample: piMDA+D","GC:  82<br />Nor_cov: 2.181915<br />Sample: piMDA+D","GC:  83<br />Nor_cov: 2.207912<br />Sample: piMDA+D","GC:  84<br />Nor_cov: 2.209391<br />Sample: piMDA+D","GC:  85<br />Nor_cov: 2.219833<br />Sample: piMDA+D","GC:  86<br />Nor_cov: 2.250351<br />Sample: piMDA+D","GC:  87<br />Nor_cov: 2.145263<br />Sample: piMDA+D","GC:  88<br />Nor_cov: 2.170983<br />Sample: piMDA+D","GC:  89<br />Nor_cov: 1.802232<br />Sample: piMDA+D","GC:  90<br />Nor_cov: 1.364749<br />Sample: piMDA+D","GC:  91<br />Nor_cov: 2.679512<br />Sample: piMDA+D","GC:  92<br />Nor_cov: 0.967373<br />Sample: piMDA+D","GC:  93<br />Nor_cov: 0.642396<br />Sample: piMDA+D","GC:  94<br />Nor_cov: 0.000000<br />Sample: piMDA+D","GC:  95<br />Nor_cov: 0.000000<br />Sample: piMDA+D","GC:  96<br />Nor_cov: 0.000000<br />Sample: piMDA+D","GC:  97<br />Nor_cov: 0.000000<br />Sample: piMDA+D","GC:  98<br />Nor_cov: 0.000000<br />Sample: piMDA+D","GC:  99<br />Nor_cov: 0.000000<br />Sample: piMDA+D","GC: 100<br />Nor_cov: 0.000000<br />Sample: piMDA+D"],"type":"scatter","mode":"lines","line":{"width":5.66929133858268,"color":"rgba(0,0,139,1)","dash":"solid"},"hoveron":"points","name":"piMDA+D","legendgroup":"piMDA+D","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100],"y":[0,0,0,0,0,0,0,0,0,0,0.240001,0.720003,0.513336,0.539019,0.49576,0.580347,0.567249,0.604934,0.614761,0.65471,0.610793,0.58797,0.561742,0.559596,0.561565,0.543715,0.534207,0.52897,0.520889,0.512767,0.506492,0.507964,0.508718,0.507692,0.513546,0.520284,0.525898,0.531125,0.539989,0.546206,0.55269,0.55872,0.568099,0.575422,0.584395,0.595793,0.607633,0.620117,0.63207,0.649201,0.665348,0.680806,0.69624,0.712291,0.73172,0.75757,0.793174,0.841885,0.900262,0.96548,1.031052,1.098058,1.161703,1.204553,1.240947,1.271746,1.296476,1.319472,1.342002,1.37292,1.396185,1.417963,1.447786,1.478716,1.506016,1.535069,1.566152,1.597261,1.624071,1.643336,1.645813,1.654362,1.684127,1.71456,1.716196,1.72388,1.742098,1.689337,1.732294,1.486387,1.146092,1.745462,0.408002,0.720003,0,0,0,0,0,0,0],"text":["GC:   0<br />Nor_cov: 0.000000<br />Sample: piMDA+D2","GC:   1<br />Nor_cov: 0.000000<br />Sample: piMDA+D2","GC:   2<br />Nor_cov: 0.000000<br />Sample: piMDA+D2","GC:   3<br />Nor_cov: 0.000000<br />Sample: piMDA+D2","GC:   4<br />Nor_cov: 0.000000<br />Sample: piMDA+D2","GC:   5<br />Nor_cov: 0.000000<br />Sample: piMDA+D2","GC:   6<br />Nor_cov: 0.000000<br />Sample: piMDA+D2","GC:   7<br />Nor_cov: 0.000000<br />Sample: piMDA+D2","GC:   8<br />Nor_cov: 0.000000<br />Sample: piMDA+D2","GC:   9<br />Nor_cov: 0.000000<br />Sample: piMDA+D2","GC:  10<br />Nor_cov: 0.240001<br />Sample: piMDA+D2","GC:  11<br />Nor_cov: 0.720003<br />Sample: piMDA+D2","GC:  12<br />Nor_cov: 0.513336<br />Sample: piMDA+D2","GC:  13<br />Nor_cov: 0.539019<br />Sample: piMDA+D2","GC:  14<br />Nor_cov: 0.495760<br />Sample: piMDA+D2","GC:  15<br />Nor_cov: 0.580347<br />Sample: piMDA+D2","GC:  16<br />Nor_cov: 0.567249<br />Sample: piMDA+D2","GC:  17<br />Nor_cov: 0.604934<br />Sample: piMDA+D2","GC:  18<br />Nor_cov: 0.614761<br />Sample: piMDA+D2","GC:  19<br />Nor_cov: 0.654710<br />Sample: piMDA+D2","GC:  20<br />Nor_cov: 0.610793<br />Sample: piMDA+D2","GC:  21<br />Nor_cov: 0.587970<br />Sample: piMDA+D2","GC:  22<br />Nor_cov: 0.561742<br />Sample: piMDA+D2","GC:  23<br />Nor_cov: 0.559596<br />Sample: piMDA+D2","GC:  24<br />Nor_cov: 0.561565<br />Sample: piMDA+D2","GC:  25<br />Nor_cov: 0.543715<br />Sample: piMDA+D2","GC:  26<br />Nor_cov: 0.534207<br />Sample: piMDA+D2","GC:  27<br />Nor_cov: 0.528970<br />Sample: piMDA+D2","GC:  28<br />Nor_cov: 0.520889<br />Sample: piMDA+D2","GC:  29<br />Nor_cov: 0.512767<br />Sample: piMDA+D2","GC:  30<br />Nor_cov: 0.506492<br />Sample: piMDA+D2","GC:  31<br />Nor_cov: 0.507964<br />Sample: piMDA+D2","GC:  32<br />Nor_cov: 0.508718<br />Sample: piMDA+D2","GC:  33<br />Nor_cov: 0.507692<br />Sample: piMDA+D2","GC:  34<br />Nor_cov: 0.513546<br />Sample: piMDA+D2","GC:  35<br />Nor_cov: 0.520284<br />Sample: piMDA+D2","GC:  36<br />Nor_cov: 0.525898<br />Sample: piMDA+D2","GC:  37<br />Nor_cov: 0.531125<br />Sample: piMDA+D2","GC:  38<br />Nor_cov: 0.539989<br />Sample: piMDA+D2","GC:  39<br />Nor_cov: 0.546206<br />Sample: piMDA+D2","GC:  40<br />Nor_cov: 0.552690<br />Sample: piMDA+D2","GC:  41<br />Nor_cov: 0.558720<br />Sample: piMDA+D2","GC:  42<br />Nor_cov: 0.568099<br />Sample: piMDA+D2","GC:  43<br />Nor_cov: 0.575422<br />Sample: piMDA+D2","GC:  44<br />Nor_cov: 0.584395<br />Sample: piMDA+D2","GC:  45<br />Nor_cov: 0.595793<br />Sample: piMDA+D2","GC:  46<br />Nor_cov: 0.607633<br />Sample: piMDA+D2","GC:  47<br />Nor_cov: 0.620117<br />Sample: piMDA+D2","GC:  48<br />Nor_cov: 0.632070<br />Sample: piMDA+D2","GC:  49<br />Nor_cov: 0.649201<br />Sample: piMDA+D2","GC:  50<br />Nor_cov: 0.665348<br />Sample: piMDA+D2","GC:  51<br />Nor_cov: 0.680806<br />Sample: piMDA+D2","GC:  52<br />Nor_cov: 0.696240<br />Sample: piMDA+D2","GC:  53<br />Nor_cov: 0.712291<br />Sample: piMDA+D2","GC:  54<br />Nor_cov: 0.731720<br />Sample: piMDA+D2","GC:  55<br />Nor_cov: 0.757570<br />Sample: piMDA+D2","GC:  56<br />Nor_cov: 0.793174<br />Sample: piMDA+D2","GC:  57<br />Nor_cov: 0.841885<br />Sample: piMDA+D2","GC:  58<br />Nor_cov: 0.900262<br />Sample: piMDA+D2","GC:  59<br />Nor_cov: 0.965480<br />Sample: piMDA+D2","GC:  60<br />Nor_cov: 1.031052<br />Sample: piMDA+D2","GC:  61<br />Nor_cov: 1.098058<br />Sample: piMDA+D2","GC:  62<br />Nor_cov: 1.161703<br />Sample: piMDA+D2","GC:  63<br />Nor_cov: 1.204553<br />Sample: piMDA+D2","GC:  64<br />Nor_cov: 1.240947<br />Sample: piMDA+D2","GC:  65<br />Nor_cov: 1.271746<br />Sample: piMDA+D2","GC:  66<br />Nor_cov: 1.296476<br />Sample: piMDA+D2","GC:  67<br />Nor_cov: 1.319472<br />Sample: piMDA+D2","GC:  68<br />Nor_cov: 1.342002<br />Sample: piMDA+D2","GC:  69<br />Nor_cov: 1.372920<br />Sample: piMDA+D2","GC:  70<br />Nor_cov: 1.396185<br />Sample: piMDA+D2","GC:  71<br />Nor_cov: 1.417963<br />Sample: piMDA+D2","GC:  72<br />Nor_cov: 1.447786<br />Sample: piMDA+D2","GC:  73<br />Nor_cov: 1.478716<br />Sample: piMDA+D2","GC:  74<br />Nor_cov: 1.506016<br />Sample: piMDA+D2","GC:  75<br />Nor_cov: 1.535069<br />Sample: piMDA+D2","GC:  76<br />Nor_cov: 1.566152<br />Sample: piMDA+D2","GC:  77<br />Nor_cov: 1.597261<br />Sample: piMDA+D2","GC:  78<br />Nor_cov: 1.624071<br />Sample: piMDA+D2","GC:  79<br />Nor_cov: 1.643336<br />Sample: piMDA+D2","GC:  80<br />Nor_cov: 1.645813<br />Sample: piMDA+D2","GC:  81<br />Nor_cov: 1.654362<br />Sample: piMDA+D2","GC:  82<br />Nor_cov: 1.684127<br />Sample: piMDA+D2","GC:  83<br />Nor_cov: 1.714560<br />Sample: piMDA+D2","GC:  84<br />Nor_cov: 1.716196<br />Sample: piMDA+D2","GC:  85<br />Nor_cov: 1.723880<br />Sample: piMDA+D2","GC:  86<br />Nor_cov: 1.742098<br />Sample: piMDA+D2","GC:  87<br />Nor_cov: 1.689337<br />Sample: piMDA+D2","GC:  88<br />Nor_cov: 1.732294<br />Sample: piMDA+D2","GC:  89<br />Nor_cov: 1.486387<br />Sample: piMDA+D2","GC:  90<br />Nor_cov: 1.146092<br />Sample: piMDA+D2","GC:  91<br />Nor_cov: 1.745462<br />Sample: piMDA+D2","GC:  92<br />Nor_cov: 0.408002<br />Sample: piMDA+D2","GC:  93<br />Nor_cov: 0.720003<br />Sample: piMDA+D2","GC:  94<br />Nor_cov: 0.000000<br />Sample: piMDA+D2","GC:  95<br />Nor_cov: 0.000000<br />Sample: piMDA+D2","GC:  96<br />Nor_cov: 0.000000<br />Sample: piMDA+D2","GC:  97<br />Nor_cov: 0.000000<br />Sample: piMDA+D2","GC:  98<br />Nor_cov: 0.000000<br />Sample: piMDA+D2","GC:  99<br />Nor_cov: 0.000000<br />Sample: piMDA+D2","GC: 100<br />Nor_cov: 0.000000<br />Sample: piMDA+D2"],"type":"scatter","mode":"lines","line":{"width":5.66929133858268,"color":"rgba(0,0,139,1)","dash":"solid"},"hoveron":"points","name":"piMDA+D2","legendgroup":"piMDA+D2","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[-5.495,105.495],"y":[1,1],"text":"yintercept: 1","type":"scatter","mode":"lines","line":{"width":1.88976377952756,"color":"rgba(0,0,0,1)","dash":"dash"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":29.0178497301785,"r":9.29846409298464,"b":51.1415525114155,"l":39.983395599834},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":18.5969281859693},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-5.495,105.495],"tickmode":"array","ticktext":["0","25","50","75","100"],"tickvals":[0,25,50,75,100],"categoryorder":"array","categoryarray":["0","25","50","75","100"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":4.64923204649232,"tickwidth":0.845314917544058,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":14.8775425487754},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.845314917544058,"zeroline":false,"anchor":"y","title":{"text":"GC","font":{"color":"rgba(0,0,0,1)","family":"","size":18.5969281859693}},"hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-0.3772349,7.9219329],"tickmode":"array","ticktext":["0","2","4","6"],"tickvals":[0,2,4,6],"categoryorder":"array","categoryarray":["0","2","4","6"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":4.64923204649232,"tickwidth":0.845314917544058,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":14.8775425487754},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.845314917544058,"zeroline":false,"anchor":"x","title":{"text":"Normalized coverage","font":{"color":"rgba(0,0,0,1)","family":"","size":18.5969281859693}},"hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(51,51,51,1)","width":0.845314917544058,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1}],"showlegend":true,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":2.40515390121689,"font":{"color":"rgba(0,0,0,1)","family":"","size":14.8775425487754},"title":{"text":"Sample","font":{"color":"rgba(0,0,0,1)","family":"","size":18.5969281859693}}},"hovermode":"closest","barmode":"relative"},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"1701d21cb86a3":{"x":{},"y":{},"type":"bar"},"1701d38f448aa":{"x":{},"y":{},"colour":{}},"1701d2dceae1c":{"yintercept":{}}},"cur_data":"1701d21cb86a3","visdat":{"1701d21cb86a3":["function (y) ","x"],"1701d38f448aa":["function (y) ","x"],"1701d2dceae1c":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>Notwithstanding the difference in the method, this results are overall in agreement with the Figure 2. The control samples show a mild decrease in the coverage at GC values over 50%, with values always near to 1 (right Y axis). However, the <strong>normalized coverage of RP-MDA and PrimPol-MDA samples span a wide range at different GC content</strong>, with a very high normalized coverage of low GC sequences (&gt;2.5, with peaks up to 4-5) sharply decreasing the coverage for GC content values over 50-60%, down to near negligible coverage. Strikingly, the piPolB and piPolB+D reads show a lower coverage, with some scattered peaks within the range of 30-50%. This may be due to the preference for that GC content in the piPolB initiation events, which would be in agreement with a higher rate of over-represented sequences.</p>
<p>However, the piMDA samples show less variability in the coverage, with a low coverage (around 0.3-0.5) at GC&lt;50 and a strong increase in the coverage at values over 50-60%, up to 1-2. The piMDA and piMDA+D protocols show a very similar profile, although the piMDA+D reaches higher coverage levels for extremely high GC content values( &gt;75%)</p>
<section id="statistic-comparisons-of-gc-bias" class="level5" data-number="5.3.1.1">
<h5 data-number="5.3.1.1" class="anchored" data-anchor-id="statistic-comparisons-of-gc-bias"><span class="header-section-number">5.3.1.1</span> Statistic comparisons of GC bias</h5>
<p>Normality test prompted us to non-parametrics group difference tests. Here we are removing outliers from each genome, using the same criteria: Picard variable WINDOWS that represent the number of reference metagenome sequences of a 100 bp window length in that GC value. We use a threshold of WINDOWS &gt;5. Then, we use the function <code>cor.mtest()</code> to obtain the R<sup>2</sup> and the p-value for the correlations within a confidence interval of 95%, thus avoiding the noise from outliers.</p>
<div class="cell">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb65"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb65-1"><a href="#cb65-1" aria-hidden="true" tabindex="-1"></a><span class="co"># gc_cov_melted2 &lt;- gc_cov_melted[gc_cov_melted$GC&gt;19 &amp; gc_cov_melted$GC&lt;81,]</span></span>
<span id="cb65-2"><a href="#cb65-2" aria-hidden="true" tabindex="-1"></a>gc_cov_melted2 <span class="ot">&lt;-</span> gc_cov_melted[gc_cov_melted<span class="sc">$</span>W <span class="sc">&gt;</span> <span class="dv">5</span>, ]</span>
<span id="cb65-3"><a href="#cb65-3" aria-hidden="true" tabindex="-1"></a><span class="co"># test for normality and homogeneous variance intra groups</span></span>
<span id="cb65-4"><a href="#cb65-4" aria-hidden="true" tabindex="-1"></a>res_aov <span class="ot">&lt;-</span> <span class="fu">aov</span>(Nor_cov <span class="sc">~</span> Sample, <span class="at">data =</span> gc_cov_melted2)</span>
<span id="cb65-5"><a href="#cb65-5" aria-hidden="true" tabindex="-1"></a><span class="fu">shapiro.test</span>(res_aov<span class="sc">$</span>residuals)  <span class="co">#NO normality </span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stdout">
<pre><code>
    Shapiro-Wilk normality test

data:  res_aov$residuals
W = 0.90583, p-value &lt; 2.2e-16</code></pre>
</div>
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb67"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb67-1"><a href="#cb67-1" aria-hidden="true" tabindex="-1"></a><span class="fu">leveneTest</span>(Nor_cov <span class="sc">~</span> Sample, <span class="at">data =</span> gc_cov_melted2)  <span class="co">#NO homogeneus variance</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stdout">
<pre><code>Levene's Test for Homogeneity of Variance (center = median)
       Df F value    Pr(&gt;F)    
group  10  34.747 &lt; 2.2e-16 ***
      869                      
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</code></pre>
</div>
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb69"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb69-1"><a href="#cb69-1" aria-hidden="true" tabindex="-1"></a><span class="co"># non parametric, pairwise (paired) tests</span></span>
<span id="cb69-2"><a href="#cb69-2" aria-hidden="true" tabindex="-1"></a>kk <span class="ot">&lt;-</span> <span class="fu">pairwise.wilcox.test</span>(<span class="at">x =</span> gc_cov_melted2<span class="sc">$</span>Nor_cov, <span class="at">g =</span> gc_cov_melted2<span class="sc">$</span>Sample, <span class="at">paired =</span> <span class="cn">TRUE</span>)</span>
<span id="cb69-3"><a href="#cb69-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb69-4"><a href="#cb69-4" aria-hidden="true" tabindex="-1"></a><span class="fu">as.data.frame</span>(<span class="fu">format</span>(kk<span class="sc">$</span>p.value, <span class="at">scientific =</span> <span class="cn">FALSE</span>, <span class="at">digits =</span> <span class="dv">1</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb69-5"><a href="#cb69-5" aria-hidden="true" tabindex="-1"></a>    <span class="fu">replace</span>(., . <span class="sc">&lt;</span> <span class="dv">0</span>, <span class="st">""</span>) <span class="sc">%&gt;%</span></span>
<span id="cb69-6"><a href="#cb69-6" aria-hidden="true" tabindex="-1"></a>    <span class="fu">mutate_all</span>(<span class="sc">~</span><span class="fu">cell_spec</span>(.x, <span class="at">color =</span> <span class="fu">ifelse</span>(.x <span class="sc">&lt;</span> <span class="fl">0.01</span>, <span class="st">"firebrick"</span>, <span class="fu">ifelse</span>(.x <span class="sc">&lt;</span> <span class="fl">0.05</span>, <span class="st">"steelblue"</span>,</span>
<span id="cb69-7"><a href="#cb69-7" aria-hidden="true" tabindex="-1"></a>        <span class="st">"black"</span>)))) <span class="sc">%&gt;%</span></span>
<span id="cb69-8"><a href="#cb69-8" aria-hidden="true" tabindex="-1"></a>    <span class="fu">kable</span>(<span class="at">escape =</span> F, <span class="at">align =</span> <span class="st">"c"</span>, <span class="at">caption =</span> <span class="st">"Table 6. P-value of pairwise comparison of GC bias in the reference metagenome"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb69-9"><a href="#cb69-9" aria-hidden="true" tabindex="-1"></a>    <span class="fu">kable_styling</span>(<span class="at">bootstrap_options =</span> <span class="st">"striped"</span>, <span class="at">full_width =</span> F, <span class="at">position =</span> <span class="st">"center"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb69-10"><a href="#cb69-10" aria-hidden="true" tabindex="-1"></a>    <span class="fu">column_spec</span>(<span class="dv">1</span>, <span class="at">bold =</span> T)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">

<table class="table table-striped" style="width: auto !important; margin-left: auto; margin-right: auto;">
<caption>Table 6. P-value of pairwise comparison of GC bias in the reference metagenome</caption>
 <thead>
  <tr>
   <th style="text-align:left;">   </th>
   <th style="text-align:center;"> NA </th>
   <th style="text-align:center;"> NA2 </th>
   <th style="text-align:center;"> RP-MDA </th>
   <th style="text-align:center;"> RP-MDA2 </th>
   <th style="text-align:center;"> PrimPol-MDA </th>
   <th style="text-align:center;"> piPolB </th>
   <th style="text-align:center;"> piPolB+D </th>
   <th style="text-align:center;"> piMDA </th>
   <th style="text-align:center;"> piMDA2 </th>
   <th style="text-align:center;"> piMDA+D </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;font-weight: bold;"> NA2 </td>
   <td style="text-align:center;"> <span style="     color: black !important;">1.000000</span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
  </tr>
  <tr>
   <td style="text-align:left;font-weight: bold;"> RP-MDA </td>
   <td style="text-align:center;"> <span style="     color: steelblue !important;">0.028756</span> </td>
   <td style="text-align:center;"> <span style="     color: steelblue !important;">0.030977</span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
  </tr>
  <tr>
   <td style="text-align:left;font-weight: bold;"> RP-MDA2 </td>
   <td style="text-align:center;"> <span style="     color: steelblue !important;">0.027509</span> </td>
   <td style="text-align:center;"> <span style="     color: steelblue !important;">0.030977</span> </td>
   <td style="text-align:center;"> <span style="     color: black !important;">1.000000</span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
  </tr>
  <tr>
   <td style="text-align:left;font-weight: bold;"> PrimPol-MDA </td>
   <td style="text-align:center;"> <span style="     color: steelblue !important;">0.027509</span> </td>
   <td style="text-align:center;"> <span style="     color: steelblue !important;">0.030977</span> </td>
   <td style="text-align:center;"> <span style="     color: black !important;">1.000000</span> </td>
   <td style="text-align:center;"> <span style="     color: black !important;">0.757072</span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
  </tr>
  <tr>
   <td style="text-align:left;font-weight: bold;"> piPolB </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;">0.000217</span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;">0.000514</span> </td>
   <td style="text-align:center;"> <span style="     color: black !important;">0.195928</span> </td>
   <td style="text-align:center;"> <span style="     color: black !important;">0.164302</span> </td>
   <td style="text-align:center;"> <span style="     color: black !important;">0.070335</span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
  </tr>
  <tr>
   <td style="text-align:left;font-weight: bold;"> piPolB+D </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;">0.001396</span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;">0.002213</span> </td>
   <td style="text-align:center;"> <span style="     color: steelblue !important;">0.039915</span> </td>
   <td style="text-align:center;"> <span style="     color: black !important;">0.050675</span> </td>
   <td style="text-align:center;"> <span style="     color: steelblue !important;">0.047076</span> </td>
   <td style="text-align:center;"> <span style="     color: black !important;">1.000000</span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
  </tr>
  <tr>
   <td style="text-align:left;font-weight: bold;"> piMDA </td>
   <td style="text-align:center;"> <span style="     color: steelblue !important;">0.013655</span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;">0.005165</span> </td>
   <td style="text-align:center;"> <span style="     color: steelblue !important;">0.024453</span> </td>
   <td style="text-align:center;"> <span style="     color: steelblue !important;">0.023303</span> </td>
   <td style="text-align:center;"> <span style="     color: steelblue !important;">0.021405</span> </td>
   <td style="text-align:center;"> <span style="     color: steelblue !important;">0.028571</span> </td>
   <td style="text-align:center;"> <span style="     color: black !important;">0.170378</span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
  </tr>
  <tr>
   <td style="text-align:left;font-weight: bold;"> piMDA2 </td>
   <td style="text-align:center;"> <span style="     color: steelblue !important;">0.030977</span> </td>
   <td style="text-align:center;"> <span style="     color: steelblue !important;">0.017934</span> </td>
   <td style="text-align:center;"> <span style="     color: steelblue !important;">0.027509</span> </td>
   <td style="text-align:center;"> <span style="     color: steelblue !important;">0.027509</span> </td>
   <td style="text-align:center;"> <span style="     color: steelblue !important;">0.027509</span> </td>
   <td style="text-align:center;"> <span style="     color: steelblue !important;">0.027509</span> </td>
   <td style="text-align:center;"> <span style="     color: black !important;">0.123799</span> </td>
   <td style="text-align:center;"> <span style="     color: black !important;">0.123327</span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
  </tr>
  <tr>
   <td style="text-align:left;font-weight: bold;"> piMDA+D </td>
   <td style="text-align:center;"> <span style="     color: black !important;">1.000000</span> </td>
   <td style="text-align:center;"> <span style="     color: black !important;">1.000000</span> </td>
   <td style="text-align:center;"> <span style="     color: steelblue !important;">0.047076</span> </td>
   <td style="text-align:center;"> <span style="     color: steelblue !important;">0.039915</span> </td>
   <td style="text-align:center;"> <span style="     color: black !important;">0.054922</span> </td>
   <td style="text-align:center;"> <span style="     color: black !important;">0.247848</span> </td>
   <td style="text-align:center;"> <span style="     color: black !important;">1.000000</span> </td>
   <td style="text-align:center;"> <span style="     color: black !important;">1.000000</span> </td>
   <td style="text-align:center;"> <span style="     color: black !important;">1.000000</span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;"></span> </td>
  </tr>
  <tr>
   <td style="text-align:left;font-weight: bold;"> piMDA+D2 </td>
   <td style="text-align:center;"> <span style="     color: black !important;">1.000000</span> </td>
   <td style="text-align:center;"> <span style="     color: black !important;">0.375833</span> </td>
   <td style="text-align:center;"> <span style="     color: steelblue !important;">0.030977</span> </td>
   <td style="text-align:center;"> <span style="     color: steelblue !important;">0.030977</span> </td>
   <td style="text-align:center;"> <span style="     color: steelblue !important;">0.030977</span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;">0.007189</span> </td>
   <td style="text-align:center;"> <span style="     color: steelblue !important;">0.032833</span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;">0.000002</span> </td>
   <td style="text-align:center;"> <span style="     color: firebrick !important;">0.000040</span> </td>
   <td style="text-align:center;"> <span style="     color: black !important;">1.000000</span> </td>
  </tr>
</tbody>
</table>

</div>
</div>
<p>Only piMDA+D samples are not significantly different than controls samples.</p>
</section>
<section id="correlations-gc-content-vs.-normalized-coverage" class="level5" data-number="5.3.1.2">
<h5 data-number="5.3.1.2" class="anchored" data-anchor-id="correlations-gc-content-vs.-normalized-coverage"><span class="header-section-number">5.3.1.2</span> Correlations: GC content vs.&nbsp;Normalized Coverage</h5>
<p>Now we are going to analyze the correlation between the GC-bias in each sample (Normalized coverage by Picard) and the actual GC-bias in the reference metagenome (WINDOWS variable in the Picard output). Again, we use a threshold of WINDOWS &gt;5. Then, we use the function <code>cor.mtest()</code> to obtain the R<sup>2</sup> and the p-value for the correlations within a confidence interval of 95%, thus avoiding the noise from outliers.</p>
<div class="cell">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb70"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb70-1"><a href="#cb70-1" aria-hidden="true" tabindex="-1"></a><span class="co"># multiple correlation</span></span>
<span id="cb70-2"><a href="#cb70-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb70-3"><a href="#cb70-3" aria-hidden="true" tabindex="-1"></a>cor_matrix2 <span class="ot">&lt;-</span> gc[[<span class="dv">1</span>]]<span class="sc">$</span>GC</span>
<span id="cb70-4"><a href="#cb70-4" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">length</span>(gc)) {</span>
<span id="cb70-5"><a href="#cb70-5" aria-hidden="true" tabindex="-1"></a>    gc[[i]] <span class="ot">&lt;-</span> gc[[i]][gc[[i]]<span class="sc">$</span>WINDOWS <span class="sc">&gt;</span> <span class="dv">5</span>, ]  <span class="co">#remove outliers</span></span>
<span id="cb70-6"><a href="#cb70-6" aria-hidden="true" tabindex="-1"></a>    cor_matrix2 <span class="ot">&lt;-</span> <span class="fu">cbindX</span>(<span class="fu">as.data.frame</span>(cor_matrix2), <span class="fu">as.data.frame</span>(gc[[i]]<span class="sc">$</span>NORMALIZED_COVERAGE))</span>
<span id="cb70-7"><a href="#cb70-7" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb70-8"><a href="#cb70-8" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(cor_matrix2) <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"GC"</span>, <span class="st">"NA"</span>, <span class="st">"NA2"</span>, <span class="st">"RP-MDA"</span>, <span class="st">"RP-MDA2"</span>, <span class="st">"PrimPol-MDA"</span>, <span class="st">"piPolB"</span>,</span>
<span id="cb70-9"><a href="#cb70-9" aria-hidden="true" tabindex="-1"></a>    <span class="st">"piPolB+D"</span>, <span class="st">"piMDA"</span>, <span class="st">"piMDA2"</span>, <span class="st">"piMDA+D"</span>, <span class="st">"piMDA+D2"</span>)</span>
<span id="cb70-10"><a href="#cb70-10" aria-hidden="true" tabindex="-1"></a>testRes <span class="ot">&lt;-</span> <span class="fu">cor.mtest</span>(cor_matrix2, <span class="at">conf.level =</span> <span class="fl">0.99</span>)</span>
<span id="cb70-11"><a href="#cb70-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb70-12"><a href="#cb70-12" aria-hidden="true" tabindex="-1"></a><span class="fu">corrplot</span>(<span class="fu">cor</span>(cor_matrix2, <span class="at">use =</span> <span class="st">"complete.obs"</span>), <span class="at">tl.col =</span> <span class="st">"black"</span>, <span class="at">p.mat =</span> testRes<span class="sc">$</span>p, <span class="at">number.cex =</span> <span class="dv">1</span>,</span>
<span id="cb70-13"><a href="#cb70-13" aria-hidden="true" tabindex="-1"></a>    <span class="at">addCoef.col =</span> <span class="st">"grey60"</span>, <span class="at">type =</span> <span class="st">"upper"</span>, <span class="at">col =</span> <span class="fu">viridis</span>(<span class="dv">100</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="PlotGCbias_cor-1.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption">Figure 13. Correlations between GC content and normalized coverage with all data</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>In agreement with Figure 12, the GC content bias shows an moderate negative correlation with the NA samples. However, there is a clear difference between the two experiments. Thus, the coverage from one of the samples (from the first batch sequenced) show a strong negative correlation with the GC content (first row, R<sup>2</sup>=-0.93), whereas in the other sample the bias was less clear (R<sup>2</sup>=-0.53). This difference could be an artifact from the libraries preparation, as has been already pointed out (<a href="https://pubmed.ncbi.nlm.nih.gov/22323520/" target="_blank">Benjamini et al.&nbsp;2022</a>).</p>
<p>Regarding the amplified samples, which is stronger with the RP and PrimPol-MDA reads (R<sup>2</sup>=-0.91 to -0.95), whereas piMDA samples overall show positive correlation (R<sup>2</sup>≃0.9). In line with this, there is a significant and positive correlation among NA, RP and PrimPol samples and among piMDA samples.</p>
</section>
</section>
<section id="gc-bias-detailed-per-reference-genome" class="level4" data-number="5.3.2">
<h4 data-number="5.3.2" class="anchored" data-anchor-id="gc-bias-detailed-per-reference-genome"><span class="header-section-number">5.3.2</span> GC bias detailed per reference genome</h4>
<div class="cell">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb71"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb71-1"><a href="#cb71-1" aria-hidden="true" tabindex="-1"></a><span class="co"># load all the files as a list of dataframes</span></span>
<span id="cb71-2"><a href="#cb71-2" aria-hidden="true" tabindex="-1"></a>gc_picard <span class="ot">=</span> <span class="fu">c</span>(<span class="st">"ctrl_gc_bias_coli"</span>, <span class="st">"ctrl_gc_bias_subtilis"</span>, <span class="st">"ctrl_gc_bias_PAE"</span>, <span class="st">"ctrl_gc_bias_kocuria"</span>,</span>
<span id="cb71-3"><a href="#cb71-3" aria-hidden="true" tabindex="-1"></a>    <span class="st">"ctrl2_gc_bias_coli"</span>, <span class="st">"ctrl2_gc_bias_subtilis"</span>, <span class="st">"ctrl2_gc_bias_PAE"</span>, <span class="st">"ctrl2_gc_bias_kocuria"</span>,</span>
<span id="cb71-4"><a href="#cb71-4" aria-hidden="true" tabindex="-1"></a>    <span class="st">"N4_gc_bias_coli"</span>, <span class="st">"N4_gc_bias_subtilis"</span>, <span class="st">"N4_gc_bias_PAE"</span>, <span class="st">"N4_gc_bias_kocuria"</span>, <span class="st">"D3_gc_bias_coli"</span>,</span>
<span id="cb71-5"><a href="#cb71-5" aria-hidden="true" tabindex="-1"></a>    <span class="st">"D3_gc_bias_subtilis"</span>, <span class="st">"D3_gc_bias_PAE"</span>, <span class="st">"D3_gc_bias_kocuria"</span>, <span class="st">"C3_gc_bias_coli"</span>, <span class="st">"C3_gc_bias_subtilis"</span>,</span>
<span id="cb71-6"><a href="#cb71-6" aria-hidden="true" tabindex="-1"></a>    <span class="st">"C3_gc_bias_PAE"</span>, <span class="st">"C3_gc_bias_kocuria"</span>, <span class="st">"N1_gc_bias_coli"</span>, <span class="st">"N1_gc_bias_subtilis"</span>, <span class="st">"N1_gc_bias_PAE"</span>,</span>
<span id="cb71-7"><a href="#cb71-7" aria-hidden="true" tabindex="-1"></a>    <span class="st">"N1_gc_bias_kocuria"</span>, <span class="st">"F2_gc_bias_coli"</span>, <span class="st">"F2_gc_bias_subtilis"</span>, <span class="st">"F2_gc_bias_PAE"</span>, <span class="st">"F2_gc_bias_kocuria"</span>,</span>
<span id="cb71-8"><a href="#cb71-8" aria-hidden="true" tabindex="-1"></a>    <span class="st">"N2_gc_bias_coli"</span>, <span class="st">"N2_gc_bias_subtilis"</span>, <span class="st">"N2_gc_bias_PAE"</span>, <span class="st">"N2_gc_bias_kocuria"</span>, <span class="st">"B2_gc_bias_coli"</span>,</span>
<span id="cb71-9"><a href="#cb71-9" aria-hidden="true" tabindex="-1"></a>    <span class="st">"B2_gc_bias_subtilis"</span>, <span class="st">"B2_gc_bias_PAE"</span>, <span class="st">"B2_gc_bias_kocuria"</span>, <span class="st">"N6_gc_bias_coli"</span>, <span class="st">"N6_gc_bias_subtilis"</span>,</span>
<span id="cb71-10"><a href="#cb71-10" aria-hidden="true" tabindex="-1"></a>    <span class="st">"N6_gc_bias_PAE"</span>, <span class="st">"N6_gc_bias_kocuria"</span>, <span class="st">"A2_gc_bias_coli"</span>, <span class="st">"A2_gc_bias_subtilis"</span>, <span class="st">"A2_gc_bias_PAE"</span>,</span>
<span id="cb71-11"><a href="#cb71-11" aria-hidden="true" tabindex="-1"></a>    <span class="st">"A2_gc_bias_kocuria"</span>)</span>
<span id="cb71-12"><a href="#cb71-12" aria-hidden="true" tabindex="-1"></a>gc <span class="ot">&lt;-</span> <span class="fu">lapply</span>(gc_picard, <span class="cf">function</span>(x) <span class="fu">read.csv2</span>(<span class="fu">paste</span>(<span class="st">"gc/"</span>, x, <span class="st">".txt"</span>, <span class="at">sep =</span> <span class="st">""</span>), <span class="at">skip =</span> <span class="dv">6</span>,</span>
<span id="cb71-13"><a href="#cb71-13" aria-hidden="true" tabindex="-1"></a>    <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>, <span class="at">colClasses =</span> <span class="fu">c</span>(<span class="st">"NULL"</span>, <span class="st">"NULL"</span>, <span class="st">"numeric"</span>, <span class="st">"numeric"</span>, <span class="st">"numeric"</span>,</span>
<span id="cb71-14"><a href="#cb71-14" aria-hidden="true" tabindex="-1"></a>        <span class="st">"numeric"</span>, <span class="st">"numeric"</span>, <span class="st">"numeric"</span>, <span class="st">"numeric"</span>, <span class="st">"numeric"</span>, <span class="st">"numeric"</span>)))</span>
<span id="cb71-15"><a href="#cb71-15" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb71-16"><a href="#cb71-16" aria-hidden="true" tabindex="-1"></a><span class="co"># plot all the samples using a loop</span></span>
<span id="cb71-17"><a href="#cb71-17" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb71-18"><a href="#cb71-18" aria-hidden="true" tabindex="-1"></a>cor_matrix <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="dv">44</span>, <span class="dv">3</span>)</span>
<span id="cb71-19"><a href="#cb71-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb71-20"><a href="#cb71-20" aria-hidden="true" tabindex="-1"></a>samples <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"NA"</span>, <span class="st">"NA2"</span>, <span class="st">"RP-MDA"</span>, <span class="st">"RP-MDA2"</span>, <span class="st">"PrimPol-MDA"</span>, <span class="st">"piPolB"</span>, <span class="st">"piPolB+D"</span>, <span class="st">"piMDA"</span>,</span>
<span id="cb71-21"><a href="#cb71-21" aria-hidden="true" tabindex="-1"></a>    <span class="st">"piMDA2"</span>, <span class="st">"piMDA+D"</span>, <span class="st">"piMDA+D2"</span>)</span>
<span id="cb71-22"><a href="#cb71-22" aria-hidden="true" tabindex="-1"></a>templates <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"E. coli"</span>, <span class="st">"B. subtilis 110NA"</span>, <span class="st">"P. aeruginosa PAER4"</span>, <span class="st">"K. rhizophila"</span>)</span>
<span id="cb71-23"><a href="#cb71-23" aria-hidden="true" tabindex="-1"></a>genomas <span class="ot">&lt;-</span> <span class="fu">merge</span>(templates, samples, <span class="at">all =</span> <span class="cn">TRUE</span>)</span>
<span id="cb71-24"><a href="#cb71-24" aria-hidden="true" tabindex="-1"></a><span class="co"># subset color palette for genomes and replicate for each sample (subtitles of each plot)</span></span>
<span id="cb71-25"><a href="#cb71-25" aria-hidden="true" tabindex="-1"></a>col_genomes <span class="ot">&lt;-</span> <span class="fu">rep</span>(colorines[<span class="dv">12</span><span class="sc">:</span><span class="dv">15</span>], <span class="at">times =</span> <span class="dv">11</span>)</span>
<span id="cb71-26"><a href="#cb71-26" aria-hidden="true" tabindex="-1"></a><span class="co"># create the plot list</span></span>
<span id="cb71-27"><a href="#cb71-27" aria-hidden="true" tabindex="-1"></a>plot_list2 <span class="ot">&lt;-</span> <span class="fu">list</span>()</span>
<span id="cb71-28"><a href="#cb71-28" aria-hidden="true" tabindex="-1"></a>plot_list2 <span class="ot">&lt;-</span> <span class="fu">lapply</span>(<span class="dv">1</span><span class="sc">:</span><span class="fu">length</span>(gc), <span class="at">FUN =</span> <span class="cf">function</span>(i) {</span>
<span id="cb71-29"><a href="#cb71-29" aria-hidden="true" tabindex="-1"></a>    plot_list2[[i]] <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(<span class="at">data =</span> gc[[i]], <span class="fu">aes</span>(<span class="at">x =</span> gc[[i]]<span class="sc">$</span>GC, <span class="at">y =</span> gc[[i]]<span class="sc">$</span>WINDOWS)) <span class="sc">+</span> <span class="fu">geom_bar</span>(<span class="at">stat =</span> <span class="st">"identity"</span>,</span>
<span id="cb71-30"><a href="#cb71-30" aria-hidden="true" tabindex="-1"></a>        <span class="at">fill =</span> <span class="st">"steelblue"</span>) <span class="sc">+</span> <span class="fu">geom_line</span>(<span class="fu">aes</span>(<span class="at">y =</span> gc[[i]]<span class="sc">$</span>MEAN_BASE_QUALITY <span class="sc">*</span> <span class="dv">15000</span>), <span class="at">color =</span> <span class="st">"grey"</span>) <span class="sc">+</span></span>
<span id="cb71-31"><a href="#cb71-31" aria-hidden="true" tabindex="-1"></a>        <span class="fu">geom_point</span>(<span class="fu">aes</span>(<span class="at">y =</span> gc[[i]]<span class="sc">$</span>NORMALIZED_COVERAGE <span class="sc">*</span> <span class="fl">2e+05</span>), <span class="at">color =</span> <span class="st">"firebrick"</span>) <span class="sc">+</span> <span class="fu">scale_y_continuous</span>(<span class="st">"Frequency at GC"</span>,</span>
<span id="cb71-32"><a href="#cb71-32" aria-hidden="true" tabindex="-1"></a>        <span class="at">sec.axis =</span> <span class="fu">sec_axis</span>(<span class="sc">~</span>.<span class="sc">/</span><span class="fl">2e+05</span>, <span class="at">name =</span> <span class="st">"Normalized coverage"</span>)) <span class="sc">+</span> <span class="fu">scale_color_manual</span>(<span class="at">name =</span> <span class="st">""</span>,</span>
<span id="cb71-33"><a href="#cb71-33" aria-hidden="true" tabindex="-1"></a>        <span class="at">values =</span> colors) <span class="sc">+</span> <span class="fu">xlab</span>(<span class="st">"GC content"</span>) <span class="sc">+</span> <span class="fu">ggtitle</span>(<span class="fu">paste</span>(genomas[i, <span class="dv">2</span>]), <span class="at">subtitle =</span> <span class="fu">paste</span>(genomas[i,</span>
<span id="cb71-34"><a href="#cb71-34" aria-hidden="true" tabindex="-1"></a>        <span class="dv">1</span>])) <span class="sc">+</span> <span class="fu">theme_bw</span>(<span class="at">base_size =</span> <span class="dv">14</span>) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">color =</span> colorines[<span class="fu">ceiling</span>(i<span class="sc">/</span><span class="dv">4</span>)],</span>
<span id="cb71-35"><a href="#cb71-35" aria-hidden="true" tabindex="-1"></a>        <span class="at">hjust =</span> <span class="fl">0.5</span>, <span class="at">size =</span> <span class="dv">15</span>, <span class="at">face =</span> <span class="st">"bold"</span>), <span class="at">plot.subtitle =</span> <span class="fu">element_text</span>(<span class="at">color =</span> col_genomes[i],</span>
<span id="cb71-36"><a href="#cb71-36" aria-hidden="true" tabindex="-1"></a>        <span class="at">hjust =</span> <span class="fl">0.5</span>, <span class="at">face =</span> <span class="st">"italic"</span>))</span>
<span id="cb71-37"><a href="#cb71-37" aria-hidden="true" tabindex="-1"></a>})</span>
<span id="cb71-38"><a href="#cb71-38" aria-hidden="true" tabindex="-1"></a><span class="fu">do.call</span>(<span class="st">"grid.arrange"</span>, <span class="fu">c</span>(plot_list2, <span class="at">ncol =</span> <span class="dv">2</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="PlotGCbias_per_genome-1.png" class="img-fluid figure-img" width="960"></p>
<p></p><figcaption class="figure-caption">Figure 14. GC distribution of sequence and reads for each reference genome. Sequence coverage (red) is normalized by the GC range frequency in 100 bp windows (blue). The average base quality is also shown (grey)</figcaption><p></p>
</figure>
</div>
</div>
</div>
<section id="correlations-gc-content-vs.-normalized-coverage-per-reference-genome" class="level5" data-number="5.3.2.1">
<h5 data-number="5.3.2.1" class="anchored" data-anchor-id="correlations-gc-content-vs.-normalized-coverage-per-reference-genome"><span class="header-section-number">5.3.2.1</span> Correlations: GC content vs.&nbsp;Normalized Coverage per reference genome</h5>
<p>Here we are removing outliers from each genome, using the same criteria: Picard variable WINDOWS that represent the number of reference sequences of a 100 bp window length in that GC value. We use a threshold of WINDOWS &gt;5. Then, we use the function <code>cor.mtest()</code> to obtain the R<sup>2</sup> and the p-value for the correlations within a confidence interval of 95%, thus avoiding the noise from outliers.</p>
<div class="cell">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb72"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb72-1"><a href="#cb72-1" aria-hidden="true" tabindex="-1"></a>cor_matrix <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="dv">44</span>, <span class="dv">3</span>)</span>
<span id="cb72-2"><a href="#cb72-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb72-3"><a href="#cb72-3" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">length</span>(gc)) {</span>
<span id="cb72-4"><a href="#cb72-4" aria-hidden="true" tabindex="-1"></a>    gc[[i]] <span class="ot">&lt;-</span> gc[[i]][gc[[i]]<span class="sc">$</span>WINDOWS <span class="sc">&gt;</span> <span class="dv">5</span>, ]  <span class="co">#remove &lt;5 values </span></span>
<span id="cb72-5"><a href="#cb72-5" aria-hidden="true" tabindex="-1"></a>    cor_matrix[i, <span class="dv">1</span>] <span class="ot">&lt;-</span> gc_picard[i]</span>
<span id="cb72-6"><a href="#cb72-6" aria-hidden="true" tabindex="-1"></a>    tmp <span class="ot">&lt;-</span> <span class="fu">cor.test</span>(gc[[i]]<span class="sc">$</span>GC, gc[[i]]<span class="sc">$</span>NORMALIZED_COVERAGE)</span>
<span id="cb72-7"><a href="#cb72-7" aria-hidden="true" tabindex="-1"></a>    cor_matrix[i, <span class="dv">2</span>] <span class="ot">&lt;-</span> tmp<span class="sc">$</span>estimate</span>
<span id="cb72-8"><a href="#cb72-8" aria-hidden="true" tabindex="-1"></a>    cor_matrix[i, <span class="dv">3</span>] <span class="ot">&lt;-</span> tmp<span class="sc">$</span>p.value</span>
<span id="cb72-9"><a href="#cb72-9" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb72-10"><a href="#cb72-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb72-11"><a href="#cb72-11" aria-hidden="true" tabindex="-1"></a>corr_frame <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="fu">matrix</span>(<span class="at">ncol =</span> <span class="dv">11</span>, <span class="at">nrow =</span> <span class="dv">4</span>))</span>
<span id="cb72-12"><a href="#cb72-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb72-13"><a href="#cb72-13" aria-hidden="true" tabindex="-1"></a><span class="co"># split the data to genome vs sample GC vs Normalized Coverage</span></span>
<span id="cb72-14"><a href="#cb72-14" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">ncol</span>(corr_frame)) {</span>
<span id="cb72-15"><a href="#cb72-15" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span> (i <span class="sc">==</span> <span class="dv">1</span>) {</span>
<span id="cb72-16"><a href="#cb72-16" aria-hidden="true" tabindex="-1"></a>        corr_frame[<span class="dv">1</span><span class="sc">:</span><span class="dv">4</span>, i] <span class="ot">&lt;-</span> cor_matrix[<span class="dv">1</span><span class="sc">:</span><span class="dv">4</span>, <span class="dv">2</span>]</span>
<span id="cb72-17"><a href="#cb72-17" aria-hidden="true" tabindex="-1"></a>    } <span class="cf">else</span> {</span>
<span id="cb72-18"><a href="#cb72-18" aria-hidden="true" tabindex="-1"></a>        corr_frame[<span class="dv">1</span><span class="sc">:</span><span class="dv">4</span>, i] <span class="ot">&lt;-</span> cor_matrix[((<span class="dv">4</span> <span class="sc">*</span> i) <span class="sc">-</span> <span class="dv">3</span>)<span class="sc">:</span>(<span class="dv">4</span> <span class="sc">*</span> i), <span class="dv">2</span>]</span>
<span id="cb72-19"><a href="#cb72-19" aria-hidden="true" tabindex="-1"></a>    }</span>
<span id="cb72-20"><a href="#cb72-20" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb72-21"><a href="#cb72-21" aria-hidden="true" tabindex="-1"></a>corr_frame <span class="ot">&lt;-</span> <span class="fu">sapply</span>(corr_frame, as.numeric)</span>
<span id="cb72-22"><a href="#cb72-22" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(corr_frame) <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"NA"</span>, <span class="st">"NA2"</span>, <span class="st">"RP-MDA"</span>, <span class="st">"RP-MDA2"</span>, <span class="st">"PrimPol-MDA"</span>, <span class="st">"piPolB"</span>, <span class="st">"piPolB+D"</span>,</span>
<span id="cb72-23"><a href="#cb72-23" aria-hidden="true" tabindex="-1"></a>    <span class="st">"piMDA"</span>, <span class="st">"piMDA2"</span>, <span class="st">"piMDA+D"</span>, <span class="st">"piMDA+D2"</span>)</span>
<span id="cb72-24"><a href="#cb72-24" aria-hidden="true" tabindex="-1"></a><span class="fu">row.names</span>(corr_frame) <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"E. coli"</span>, <span class="st">"B. subtilis 110NA"</span>, <span class="st">"P. aeruginosa PAER4"</span>, <span class="st">"K. rhizophila"</span>)</span>
<span id="cb72-25"><a href="#cb72-25" aria-hidden="true" tabindex="-1"></a><span class="co"># plot</span></span>
<span id="cb72-26"><a href="#cb72-26" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb72-27"><a href="#cb72-27" aria-hidden="true" tabindex="-1"></a><span class="co"># include the p-values</span></span>
<span id="cb72-28"><a href="#cb72-28" aria-hidden="true" tabindex="-1"></a>corr_frame2 <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="fu">matrix</span>(<span class="at">ncol =</span> <span class="dv">11</span>, <span class="at">nrow =</span> <span class="dv">4</span>))</span>
<span id="cb72-29"><a href="#cb72-29" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb72-30"><a href="#cb72-30" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">ncol</span>(corr_frame2)) {</span>
<span id="cb72-31"><a href="#cb72-31" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span> (i <span class="sc">==</span> <span class="dv">1</span>) {</span>
<span id="cb72-32"><a href="#cb72-32" aria-hidden="true" tabindex="-1"></a>        corr_frame2[<span class="dv">1</span><span class="sc">:</span><span class="dv">4</span>, i] <span class="ot">&lt;-</span> cor_matrix[<span class="dv">1</span><span class="sc">:</span><span class="dv">4</span>, <span class="dv">3</span>]</span>
<span id="cb72-33"><a href="#cb72-33" aria-hidden="true" tabindex="-1"></a>    } <span class="cf">else</span> {</span>
<span id="cb72-34"><a href="#cb72-34" aria-hidden="true" tabindex="-1"></a>        corr_frame2[<span class="dv">1</span><span class="sc">:</span><span class="dv">4</span>, i] <span class="ot">&lt;-</span> cor_matrix[((<span class="dv">4</span> <span class="sc">*</span> i) <span class="sc">-</span> <span class="dv">3</span>)<span class="sc">:</span>(<span class="dv">4</span> <span class="sc">*</span> i), <span class="dv">3</span>]</span>
<span id="cb72-35"><a href="#cb72-35" aria-hidden="true" tabindex="-1"></a>    }</span>
<span id="cb72-36"><a href="#cb72-36" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb72-37"><a href="#cb72-37" aria-hidden="true" tabindex="-1"></a>corr_frame2 <span class="ot">&lt;-</span> <span class="fu">sapply</span>(corr_frame2, as.numeric)</span>
<span id="cb72-38"><a href="#cb72-38" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(corr_frame2) <span class="ot">&lt;-</span> <span class="fu">colnames</span>(corr_frame)</span>
<span id="cb72-39"><a href="#cb72-39" aria-hidden="true" tabindex="-1"></a><span class="fu">row.names</span>(corr_frame2) <span class="ot">&lt;-</span> <span class="fu">row.names</span>(corr_frame)</span>
<span id="cb72-40"><a href="#cb72-40" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb72-41"><a href="#cb72-41" aria-hidden="true" tabindex="-1"></a><span class="fu">corrplot</span>(<span class="fu">as.matrix</span>(corr_frame), <span class="at">tl.col =</span> <span class="st">"black"</span>, <span class="at">p.mat =</span> corr_frame2, <span class="at">number.cex =</span> <span class="fl">1.6</span>, <span class="at">addCoef.col =</span> <span class="st">"grey60"</span>,</span>
<span id="cb72-42"><a href="#cb72-42" aria-hidden="true" tabindex="-1"></a>    <span class="at">col =</span> <span class="fu">viridis</span>(<span class="dv">100</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="PlotGCbias_cor_genome-1.png" class="img-fluid figure-img" width="1152"></p>
<p></p><figcaption class="figure-caption">Figure 15. Correlations among normalized coverage by GC content per reference genome</figcaption><p></p>
</figure>
</div>
</div>
</div>
</section>
</section>
</section>
</section>
<section id="assemblies" class="level2" data-number="6">
<h2 data-number="6" class="anchored" data-anchor-id="assemblies"><span class="header-section-number">6</span> Assemblies</h2>
<section id="assembly-basic-stats" class="level3" data-number="6.1">
<h3 data-number="6.1" class="anchored" data-anchor-id="assembly-basic-stats"><span class="header-section-number">6.1</span> Assembly basic stats</h3>
<p>Assemblies were obtained with <a href="https://github.com/ablab/spades" target="_blank">MetaSpades</a>. The piPolB sample could not be assembled due to a high proportion of low complexity reads, particularly in the R2 file (see <a href="https://www.seqme.eu/en/magazine/sequencing-low-diversity-libraries-on-illumina" target="_blank">https://www.seqme.eu/en/magazine/sequencing-low-diversity-libraries-on-illumina</a>). We tried to bypass this problem using different trimming alternatives, like <a href="https://github.com/Adrian-Cantu/PRINSEQ-plus-plus" target="_blank">PRINSEQ++</a>, <a href="https://github.com/OpenGene/fastp" target="_blank">FastP</a> or <a href="https://github.com/jdidion/atropos" target="_blank">Atropos</a>. Only after using FastP for trimming out reads with complexity &lt;70 (option -Y 70), we could assembly the piPolB sample (N1 raw reads), but the assembly was poor and non comparable with the other samples. Therefore, the piPolB sample was not included in the following analysis.</p>
<p>The assemblies were evaluated with <a href="https://github.com/ablab/quast" target="_blank">MetaQuast</a> v. 5.1.0rc1. A comparative analysis of all the samples was performed without the reads to save computational resources (check full report <a href="quast_results/results_2022_03_10_13_57_32/combined_reference/report.html" target="_blank">here</a>). A full analysis including the reads were performed independently for each sample but due to size limitations it will be available along with the raw reads. All data is available in the repo folder <em>quast_results</em> and individual tables from detailed reports were merged using the script <a href="scripts/merge_quast_reports.R" target="_blank"><em>merge_quast_reports.R</em></a>.</p>
<div class="cell" data-layout-align="left">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb73"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb73-1"><a href="#cb73-1" aria-hidden="true" tabindex="-1"></a>quast <span class="ot">&lt;-</span> <span class="fu">fread</span>(<span class="st">"quast_results/merged_quast.csv"</span>, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">na.strings =</span> <span class="st">""</span>, <span class="at">strip.white =</span> <span class="cn">TRUE</span>)</span>
<span id="cb73-2"><a href="#cb73-2" aria-hidden="true" tabindex="-1"></a>tquast <span class="ot">&lt;-</span> <span class="fu">t</span>(quast)</span>
<span id="cb73-3"><a href="#cb73-3" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(tquast) <span class="ot">&lt;-</span> tquast[<span class="dv">1</span>, ]</span>
<span id="cb73-4"><a href="#cb73-4" aria-hidden="true" tabindex="-1"></a>tquast <span class="ot">&lt;-</span> tquast[<span class="sc">-</span><span class="dv">1</span>, ]</span>
<span id="cb73-5"><a href="#cb73-5" aria-hidden="true" tabindex="-1"></a>muestras <span class="ot">&lt;-</span> <span class="fu">rownames</span>(tquast)</span>
<span id="cb73-6"><a href="#cb73-6" aria-hidden="true" tabindex="-1"></a>tquast <span class="ot">&lt;-</span> <span class="fu">cbind</span>(muestras, tquast)</span>
<span id="cb73-7"><a href="#cb73-7" aria-hidden="true" tabindex="-1"></a>tquast <span class="ot">&lt;-</span> <span class="fu">as.data.frame</span>(tquast)</span>
<span id="cb73-8"><a href="#cb73-8" aria-hidden="true" tabindex="-1"></a><span class="co"># order samples</span></span>
<span id="cb73-9"><a href="#cb73-9" aria-hidden="true" tabindex="-1"></a>tquast[<span class="dv">5</span>, <span class="dv">1</span>] <span class="ot">&lt;-</span> <span class="st">"piPolB+D"</span></span>
<span id="cb73-10"><a href="#cb73-10" aria-hidden="true" tabindex="-1"></a>tquast<span class="sc">$</span>muestras <span class="ot">&lt;-</span> <span class="fu">factor</span>(tquast<span class="sc">$</span>muestras, <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">"NA"</span>, <span class="st">"RP-MDA"</span>, <span class="st">"PrimPol-MDA"</span>, <span class="st">"piPolB+D"</span>,</span>
<span id="cb73-11"><a href="#cb73-11" aria-hidden="true" tabindex="-1"></a>    <span class="st">"piMDA"</span>, <span class="st">"piMDA+D"</span>))</span>
<span id="cb73-12"><a href="#cb73-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb73-13"><a href="#cb73-13" aria-hidden="true" tabindex="-1"></a><span class="co"># plots</span></span>
<span id="cb73-14"><a href="#cb73-14" aria-hidden="true" tabindex="-1"></a>contigs <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(tquast) <span class="sc">+</span> <span class="fu">geom_point</span>(<span class="fu">aes</span>(<span class="at">x =</span> muestras, <span class="at">y =</span> <span class="fu">as.numeric</span>(tquast[, <span class="dv">4</span>]), <span class="at">color =</span> muestras,</span>
<span id="cb73-15"><a href="#cb73-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">fill =</span> muestras, <span class="at">group =</span> Experiment), <span class="at">size =</span> <span class="dv">4</span>, <span class="at">alpha =</span> <span class="fl">0.7</span>, <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="at">width =</span> <span class="fl">0.5</span>)) <span class="sc">+</span></span>
<span id="cb73-16"><a href="#cb73-16" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_point</span>(<span class="fu">aes</span>(<span class="at">x =</span> muestras, <span class="at">y =</span> <span class="fu">as.numeric</span>(tquast[, <span class="dv">21</span>]) <span class="sc">*</span> <span class="dv">30</span>, <span class="at">fill =</span> muestras), <span class="at">shape =</span> <span class="dv">21</span>,</span>
<span id="cb73-17"><a href="#cb73-17" aria-hidden="true" tabindex="-1"></a>        <span class="at">color =</span> <span class="st">"black"</span>, <span class="at">size =</span> <span class="dv">2</span>, <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="at">width =</span> <span class="fl">0.5</span>)) <span class="sc">+</span> <span class="fu">scale_y_continuous</span>(<span class="at">name =</span> <span class="st">"Contigs &gt;1kb"</span>,</span>
<span id="cb73-18"><a href="#cb73-18" aria-hidden="true" tabindex="-1"></a>    <span class="at">breaks =</span> <span class="fu">seq</span>(<span class="dv">0</span>, <span class="dv">2000</span>, <span class="dv">250</span>), <span class="at">sec.axis =</span> <span class="fu">sec_axis</span>(<span class="sc">~</span>.<span class="sc">/</span><span class="dv">30</span>, <span class="at">name =</span> <span class="st">"L50"</span>, <span class="at">breaks =</span> <span class="fu">seq</span>(<span class="dv">0</span>, <span class="dv">50</span>,</span>
<span id="cb73-19"><a href="#cb73-19" aria-hidden="true" tabindex="-1"></a>        <span class="dv">5</span>))) <span class="sc">+</span> <span class="fu">theme_bw</span>() <span class="sc">+</span> <span class="fu">scale_color_manual</span>(<span class="at">values =</span> colorines) <span class="sc">+</span> <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> colorines) <span class="sc">+</span></span>
<span id="cb73-20"><a href="#cb73-20" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme</span>(<span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">15</span>)) <span class="sc">+</span> <span class="fu">ggtitle</span>(<span class="st">"Assembly contigs"</span>) <span class="sc">+</span> <span class="fu">xlab</span>(<span class="st">"Sample"</span>) <span class="sc">+</span></span>
<span id="cb73-21"><a href="#cb73-21" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_shape_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="dv">21</span>, <span class="dv">22</span>, <span class="dv">23</span>, <span class="dv">24</span>)) <span class="sc">+</span> <span class="fu">guides</span>(<span class="at">shape =</span> <span class="cn">FALSE</span>, <span class="at">fill =</span> <span class="cn">FALSE</span>, <span class="at">color =</span> <span class="cn">FALSE</span>,</span>
<span id="cb73-22"><a href="#cb73-22" aria-hidden="true" tabindex="-1"></a>    <span class="at">scale =</span> <span class="st">"none"</span>) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">45</span>, <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">16</span>)) <span class="sc">+</span></span>
<span id="cb73-23"><a href="#cb73-23" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme</span>(<span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">hjust =</span> <span class="fl">0.5</span>))</span>
<span id="cb73-24"><a href="#cb73-24" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb73-25"><a href="#cb73-25" aria-hidden="true" tabindex="-1"></a>mismatches <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(tquast) <span class="sc">+</span> <span class="fu">geom_point</span>(<span class="fu">aes</span>(<span class="at">x =</span> muestras, <span class="at">y =</span> <span class="fu">as.numeric</span>(tquast[, <span class="dv">47</span>]), <span class="at">color =</span> muestras,</span>
<span id="cb73-26"><a href="#cb73-26" aria-hidden="true" tabindex="-1"></a>    <span class="at">fill =</span> muestras, <span class="at">group =</span> Experiment), <span class="at">size =</span> <span class="dv">4</span>, <span class="at">alpha =</span> <span class="fl">0.7</span>, <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="at">width =</span> <span class="fl">0.5</span>)) <span class="sc">+</span></span>
<span id="cb73-27"><a href="#cb73-27" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_point</span>(<span class="fu">aes</span>(<span class="at">x =</span> muestras, <span class="at">y =</span> <span class="fu">as.numeric</span>(tquast[, <span class="dv">48</span>]) <span class="sc">*</span> <span class="dv">16</span>, <span class="at">fill =</span> muestras), <span class="at">shape =</span> <span class="dv">21</span>,</span>
<span id="cb73-28"><a href="#cb73-28" aria-hidden="true" tabindex="-1"></a>        <span class="at">color =</span> <span class="st">"black"</span>, <span class="at">size =</span> <span class="dv">2</span>, <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="at">width =</span> <span class="fl">0.5</span>)) <span class="sc">+</span> <span class="fu">scale_y_continuous</span>(<span class="at">name =</span> <span class="st">"Mismatches per 100 kbp"</span>,</span>
<span id="cb73-29"><a href="#cb73-29" aria-hidden="true" tabindex="-1"></a>    <span class="at">breaks =</span> <span class="fu">seq</span>(<span class="dv">0</span>, <span class="dv">225</span>, <span class="dv">25</span>), <span class="at">limit =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">230</span>), <span class="at">sec.axis =</span> <span class="fu">sec_axis</span>(<span class="sc">~</span>.<span class="sc">/</span><span class="dv">16</span>, <span class="at">name =</span> <span class="st">"InDels per 100 kbp"</span>,</span>
<span id="cb73-30"><a href="#cb73-30" aria-hidden="true" tabindex="-1"></a>        <span class="at">breaks =</span> <span class="fu">seq</span>(<span class="dv">0</span>, <span class="dv">20</span>, <span class="dv">5</span>))) <span class="sc">+</span> <span class="fu">theme_bw</span>() <span class="sc">+</span> <span class="fu">scale_color_manual</span>(<span class="at">values =</span> colorines) <span class="sc">+</span> <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> colorines) <span class="sc">+</span></span>
<span id="cb73-31"><a href="#cb73-31" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme</span>(<span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">15</span>)) <span class="sc">+</span> <span class="fu">ggtitle</span>(<span class="st">"Assembly mismatches"</span>) <span class="sc">+</span> <span class="fu">xlab</span>(<span class="st">"Sample"</span>) <span class="sc">+</span></span>
<span id="cb73-32"><a href="#cb73-32" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_shape_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="dv">21</span>, <span class="dv">22</span>, <span class="dv">23</span>, <span class="dv">24</span>)) <span class="sc">+</span> <span class="fu">guides</span>(<span class="at">shape =</span> <span class="cn">FALSE</span>, <span class="at">fill =</span> <span class="cn">FALSE</span>, <span class="at">color =</span> <span class="cn">FALSE</span>,</span>
<span id="cb73-33"><a href="#cb73-33" aria-hidden="true" tabindex="-1"></a>    <span class="at">scale =</span> <span class="st">"none"</span>) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">45</span>, <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">16</span>)) <span class="sc">+</span></span>
<span id="cb73-34"><a href="#cb73-34" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme</span>(<span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">hjust =</span> <span class="fl">0.5</span>))</span>
<span id="cb73-35"><a href="#cb73-35" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb73-36"><a href="#cb73-36" aria-hidden="true" tabindex="-1"></a><span class="co"># misassemblies report</span></span>
<span id="cb73-37"><a href="#cb73-37" aria-hidden="true" tabindex="-1"></a>misas <span class="ot">&lt;-</span> <span class="fu">read.table</span>(<span class="st">"quast_results/transposed_report_misassemblies.tsv"</span>, <span class="at">head =</span> <span class="cn">TRUE</span>, <span class="at">comment.char =</span> <span class="st">"@"</span>,</span>
<span id="cb73-38"><a href="#cb73-38" aria-hidden="true" tabindex="-1"></a>    <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>, <span class="at">na.strings =</span> <span class="st">""</span>)</span>
<span id="cb73-39"><a href="#cb73-39" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">nrow</span>(misas)) {</span>
<span id="cb73-40"><a href="#cb73-40" aria-hidden="true" tabindex="-1"></a>    misas<span class="sc">$</span>Assembly[i] <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">"_2"</span>, <span class="st">""</span>, misas<span class="sc">$</span>Assembly[i])</span>
<span id="cb73-41"><a href="#cb73-41" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb73-42"><a href="#cb73-42" aria-hidden="true" tabindex="-1"></a>misas<span class="sc">$</span>Assembly[<span class="dv">3</span><span class="sc">:</span><span class="dv">4</span>] <span class="ot">&lt;-</span> <span class="st">"RP-MDA"</span></span>
<span id="cb73-43"><a href="#cb73-43" aria-hidden="true" tabindex="-1"></a>misas<span class="sc">$</span>Assembly[<span class="dv">5</span>] <span class="ot">&lt;-</span> <span class="st">"PrimPol-MDA"</span></span>
<span id="cb73-44"><a href="#cb73-44" aria-hidden="true" tabindex="-1"></a>misas<span class="sc">$</span>Assembly <span class="ot">&lt;-</span> <span class="fu">factor</span>(misas<span class="sc">$</span>Assembly, <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">"NA"</span>, <span class="st">"RP-MDA"</span>, <span class="st">"PrimPol-MDA"</span>, <span class="st">"piPolB+D"</span>,</span>
<span id="cb73-45"><a href="#cb73-45" aria-hidden="true" tabindex="-1"></a>    <span class="st">"piMDA"</span>, <span class="st">"piMDA+D"</span>))</span>
<span id="cb73-46"><a href="#cb73-46" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb73-47"><a href="#cb73-47" aria-hidden="true" tabindex="-1"></a>misas_plot <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(misas) <span class="sc">+</span> <span class="fu">geom_point</span>(<span class="fu">aes</span>(<span class="at">x =</span> Assembly, <span class="at">y =</span> <span class="fu">as.numeric</span>(misas[, <span class="dv">2</span>]), <span class="at">color =</span> Assembly,</span>
<span id="cb73-48"><a href="#cb73-48" aria-hidden="true" tabindex="-1"></a>    <span class="at">fill =</span> Assembly), <span class="at">size =</span> <span class="dv">4</span>, <span class="at">alpha =</span> <span class="fl">0.7</span>, <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="at">width =</span> <span class="fl">0.5</span>)) <span class="sc">+</span> <span class="fu">geom_point</span>(<span class="fu">aes</span>(<span class="at">x =</span> Assembly,</span>
<span id="cb73-49"><a href="#cb73-49" aria-hidden="true" tabindex="-1"></a>    <span class="at">y =</span> <span class="fu">as.numeric</span>(misas[, <span class="dv">17</span>])<span class="sc">/</span><span class="dv">4</span>, <span class="at">fill =</span> Assembly), <span class="at">shape =</span> <span class="dv">21</span>, <span class="at">color =</span> <span class="st">"black"</span>, <span class="at">size =</span> <span class="dv">2</span>,</span>
<span id="cb73-50"><a href="#cb73-50" aria-hidden="true" tabindex="-1"></a>    <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="at">width =</span> <span class="fl">0.8</span>)) <span class="sc">+</span> <span class="fu">scale_y_continuous</span>(<span class="at">name =</span> <span class="st">"Contig misassemblies"</span>,</span>
<span id="cb73-51"><a href="#cb73-51" aria-hidden="true" tabindex="-1"></a>    <span class="at">breaks =</span> <span class="fu">seq</span>(<span class="dv">0</span>, <span class="dv">150</span>, <span class="dv">20</span>), <span class="at">limit =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">100</span>), <span class="at">sec.axis =</span> <span class="fu">sec_axis</span>(<span class="sc">~</span>. <span class="sc">*</span> <span class="dv">4</span>, <span class="at">name =</span> <span class="st">"Local misassemblies"</span>,</span>
<span id="cb73-52"><a href="#cb73-52" aria-hidden="true" tabindex="-1"></a>        <span class="at">breaks =</span> <span class="fu">seq</span>(<span class="dv">0</span>, <span class="dv">400</span>, <span class="dv">50</span>))) <span class="sc">+</span> <span class="fu">theme_bw</span>() <span class="sc">+</span> <span class="fu">scale_color_manual</span>(<span class="at">values =</span> colorines) <span class="sc">+</span></span>
<span id="cb73-53"><a href="#cb73-53" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> colorines) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">15</span>)) <span class="sc">+</span> <span class="fu">ggtitle</span>(<span class="st">"Misassemblies"</span>) <span class="sc">+</span></span>
<span id="cb73-54"><a href="#cb73-54" aria-hidden="true" tabindex="-1"></a>    <span class="fu">xlab</span>(<span class="st">"Sample"</span>) <span class="sc">+</span> <span class="fu">scale_shape_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="dv">21</span>, <span class="dv">22</span>, <span class="dv">23</span>, <span class="dv">24</span>)) <span class="sc">+</span> <span class="fu">guides</span>(<span class="at">shape =</span> <span class="cn">FALSE</span>,</span>
<span id="cb73-55"><a href="#cb73-55" aria-hidden="true" tabindex="-1"></a>    <span class="at">fill =</span> <span class="cn">FALSE</span>, <span class="at">color =</span> <span class="cn">FALSE</span>, <span class="at">scale =</span> <span class="st">"none"</span>) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">45</span>,</span>
<span id="cb73-56"><a href="#cb73-56" aria-hidden="true" tabindex="-1"></a>    <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">16</span>)) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">hjust =</span> <span class="fl">0.5</span>))</span>
<span id="cb73-57"><a href="#cb73-57" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb73-58"><a href="#cb73-58" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb73-59"><a href="#cb73-59" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb73-60"><a href="#cb73-60" aria-hidden="true" tabindex="-1"></a>grid_contigs <span class="ot">&lt;-</span> <span class="fu">grid.arrange</span>(contigs, mismatches, misas_plot, <span class="at">nrow =</span> <span class="dv">1</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-left">
<figure class="figure">
<p><img src="Plotasssemblies-1.png" class="img-fluid figure-img" width="768"></p>
<p></p><figcaption class="figure-caption">Figure 16. Assembly basic stats in samples amplified by all the different MDA methods tested</figcaption><p></p>
</figure>
</div>
</div>
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb74"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb74-1"><a href="#cb74-1" aria-hidden="true" tabindex="-1"></a>grid_contigs</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stdout">
<pre><code>TableGrob (1 x 3) "arrange": 3 grobs
  z     cells    name           grob
1 1 (1-1,1-1) arrange gtable[layout]
2 2 (1-1,2-2) arrange gtable[layout]
3 3 (1-1,3-3) arrange gtable[layout]</code></pre>
</div>
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb76"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb76-1"><a href="#cb76-1" aria-hidden="true" tabindex="-1"></a><span class="co"># ggsave('figures/contigs.svg',grid_contigs,width=12,height=5)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
</section>
<section id="stats-per-individual-reference-genome" class="level3" data-number="6.2">
<h3 data-number="6.2" class="anchored" data-anchor-id="stats-per-individual-reference-genome"><span class="header-section-number">6.2</span> Stats per individual reference genome</h3>
<p>Given the high background of mismatches in the <em>P aeruginosa</em> reference, we decided to normalize the number of mismatches to the first NA sample.</p>
<div class="cell" data-layout-align="left">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb77"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb77-1"><a href="#cb77-1" aria-hidden="true" tabindex="-1"></a>quast_ref <span class="ot">&lt;-</span> <span class="fu">read.table</span>(<span class="st">"quast_results/num_contigs.tsv"</span>, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">na.strings =</span> <span class="st">""</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>,</span>
<span id="cb77-2"><a href="#cb77-2" aria-hidden="true" tabindex="-1"></a>    <span class="at">strip.white =</span> <span class="cn">TRUE</span>)</span>
<span id="cb77-3"><a href="#cb77-3" aria-hidden="true" tabindex="-1"></a><span class="fu">names</span>(quast_ref) <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"Reference"</span>, <span class="st">"NA"</span>, <span class="st">"NA"</span>, <span class="st">"RP-MDA"</span>, <span class="st">"RP-MDA"</span>, <span class="st">"PrimPol-MDA"</span>, <span class="st">"piPolB+D"</span>,</span>
<span id="cb77-4"><a href="#cb77-4" aria-hidden="true" tabindex="-1"></a>    <span class="st">"piMDA"</span>, <span class="st">"piMDA"</span>, <span class="st">"piMDA+D"</span>, <span class="st">"piMDA+D"</span>)</span>
<span id="cb77-5"><a href="#cb77-5" aria-hidden="true" tabindex="-1"></a>quast_ref[, <span class="dv">1</span>] <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"E. coli"</span>, <span class="st">"B. subtilis"</span>, <span class="st">"P. aeruginosa"</span>, <span class="st">"K. rhizophila"</span>, <span class="st">"Not_aligned"</span>)</span>
<span id="cb77-6"><a href="#cb77-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb77-7"><a href="#cb77-7" aria-hidden="true" tabindex="-1"></a>quast_ref_melted <span class="ot">&lt;-</span> <span class="fu">cbind</span>(<span class="at">Category =</span> quast_ref[[<span class="dv">1</span>]], <span class="fu">stack</span>(quast_ref[<span class="sc">-</span><span class="dv">1</span>]))</span>
<span id="cb77-8"><a href="#cb77-8" aria-hidden="true" tabindex="-1"></a><span class="co"># fix names</span></span>
<span id="cb77-9"><a href="#cb77-9" aria-hidden="true" tabindex="-1"></a>quast_ref_melted <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="fu">lapply</span>(quast_ref_melted, <span class="cf">function</span>(x) {</span>
<span id="cb77-10"><a href="#cb77-10" aria-hidden="true" tabindex="-1"></a>    <span class="fu">gsub</span>(<span class="st">"A.1"</span>, <span class="st">"A"</span>, x)</span>
<span id="cb77-11"><a href="#cb77-11" aria-hidden="true" tabindex="-1"></a>}))</span>
<span id="cb77-12"><a href="#cb77-12" aria-hidden="true" tabindex="-1"></a>quast_ref_melted <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="fu">lapply</span>(quast_ref_melted, <span class="cf">function</span>(x) {</span>
<span id="cb77-13"><a href="#cb77-13" aria-hidden="true" tabindex="-1"></a>    <span class="fu">gsub</span>(<span class="st">"D.1"</span>, <span class="st">"D"</span>, x)</span>
<span id="cb77-14"><a href="#cb77-14" aria-hidden="true" tabindex="-1"></a>}))</span>
<span id="cb77-15"><a href="#cb77-15" aria-hidden="true" tabindex="-1"></a>color_ref <span class="ot">&lt;-</span> <span class="fu">c</span>(colorines[<span class="dv">12</span><span class="sc">:</span><span class="dv">15</span>], <span class="at">Not_aligned =</span> <span class="st">"grey"</span>)</span>
<span id="cb77-16"><a href="#cb77-16" aria-hidden="true" tabindex="-1"></a>quast_ref_melted<span class="sc">$</span>ind <span class="ot">&lt;-</span> <span class="fu">factor</span>(quast_ref_melted<span class="sc">$</span>ind, <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">"NA"</span>, <span class="st">"RP-MDA"</span>, <span class="st">"PrimPol-MDA"</span>,</span>
<span id="cb77-17"><a href="#cb77-17" aria-hidden="true" tabindex="-1"></a>    <span class="st">"piPolB+D"</span>, <span class="st">"piMDA"</span>, <span class="st">"piMDA+D"</span>))</span>
<span id="cb77-18"><a href="#cb77-18" aria-hidden="true" tabindex="-1"></a>quast_ref_melted<span class="sc">$</span>Category <span class="ot">&lt;-</span> <span class="fu">factor</span>(quast_ref_melted<span class="sc">$</span>Category, <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">"E. coli"</span>, <span class="st">"B. subtilis"</span>,</span>
<span id="cb77-19"><a href="#cb77-19" aria-hidden="true" tabindex="-1"></a>    <span class="st">"P. aeruginosa"</span>, <span class="st">"K. rhizophila"</span>, <span class="st">"Not_aligned"</span>))</span>
<span id="cb77-20"><a href="#cb77-20" aria-hidden="true" tabindex="-1"></a>plot <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(quast_ref_melted) <span class="sc">+</span> <span class="fu">geom_point</span>(<span class="fu">aes</span>(<span class="at">x =</span> ind, <span class="at">y =</span> <span class="fu">as.numeric</span>(values), <span class="at">group =</span> Category,</span>
<span id="cb77-21"><a href="#cb77-21" aria-hidden="true" tabindex="-1"></a>    <span class="at">color =</span> Category, <span class="at">fill =</span> Category, <span class="at">shape =</span> Category), <span class="at">size =</span> <span class="dv">4</span>, <span class="at">alpha =</span> <span class="fl">0.7</span>, <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="at">width =</span> <span class="fl">0.5</span>)) <span class="sc">+</span></span>
<span id="cb77-22"><a href="#cb77-22" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme_bw</span>() <span class="sc">+</span> <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> color_ref) <span class="sc">+</span> <span class="fu">scale_color_manual</span>(<span class="at">values =</span> color_ref) <span class="sc">+</span></span>
<span id="cb77-23"><a href="#cb77-23" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_shape_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="dv">21</span>, <span class="dv">22</span>, <span class="dv">23</span>, <span class="dv">24</span>, <span class="dv">20</span>)) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">15</span>)) <span class="sc">+</span></span>
<span id="cb77-24"><a href="#cb77-24" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggtitle</span>(<span class="st">"Number of contigs per reference"</span>) <span class="sc">+</span> <span class="fu">xlab</span>(<span class="st">"Sample"</span>) <span class="sc">+</span> <span class="fu">labs</span>(<span class="at">color =</span> <span class="st">"Reference genome"</span>) <span class="sc">+</span></span>
<span id="cb77-25"><a href="#cb77-25" aria-hidden="true" tabindex="-1"></a>    <span class="fu">guides</span>(<span class="at">shape =</span> <span class="cn">FALSE</span>, <span class="at">color =</span> <span class="cn">FALSE</span>, <span class="at">fill =</span> <span class="cn">FALSE</span>, <span class="at">scale =</span> <span class="st">"none"</span>) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">45</span>,</span>
<span id="cb77-26"><a href="#cb77-26" aria-hidden="true" tabindex="-1"></a>    <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">16</span>)) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">legend.text =</span> <span class="fu">element_text</span>(<span class="at">face =</span> <span class="st">"italic"</span>)) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">hjust =</span> <span class="fl">0.5</span>)) <span class="sc">+</span></span>
<span id="cb77-27"><a href="#cb77-27" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_y_continuous</span>(<span class="at">name =</span> <span class="st">"Contigs"</span>)</span>
<span id="cb77-28"><a href="#cb77-28" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb77-29"><a href="#cb77-29" aria-hidden="true" tabindex="-1"></a>quast_ref_mism <span class="ot">&lt;-</span> <span class="fu">read.table</span>(<span class="st">"quast_results/num_mismatches_per_100_kbp.tsv"</span>, <span class="at">header =</span> <span class="cn">TRUE</span>,</span>
<span id="cb77-30"><a href="#cb77-30" aria-hidden="true" tabindex="-1"></a>    <span class="at">na.strings =</span> <span class="st">""</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>, <span class="at">strip.white =</span> <span class="cn">TRUE</span>)</span>
<span id="cb77-31"><a href="#cb77-31" aria-hidden="true" tabindex="-1"></a><span class="fu">names</span>(quast_ref_mism) <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"Reference"</span>, <span class="st">"NA"</span>, <span class="st">"NA"</span>, <span class="st">"RP-MDA"</span>, <span class="st">"RP-MDA"</span>, <span class="st">"PrimPol-MDA"</span>, <span class="st">"piPolB+D"</span>,</span>
<span id="cb77-32"><a href="#cb77-32" aria-hidden="true" tabindex="-1"></a>    <span class="st">"piMDA"</span>, <span class="st">"piMDA"</span>, <span class="st">"piMDA+D"</span>, <span class="st">"piMDA+D"</span>)</span>
<span id="cb77-33"><a href="#cb77-33" aria-hidden="true" tabindex="-1"></a>quast_ref_mism[, <span class="dv">1</span>] <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"E. coli"</span>, <span class="st">"B. subtilis"</span>, <span class="st">"P. aeruginosa"</span>, <span class="st">"K. rhizophila"</span>)</span>
<span id="cb77-34"><a href="#cb77-34" aria-hidden="true" tabindex="-1"></a><span class="co"># normalize data</span></span>
<span id="cb77-35"><a href="#cb77-35" aria-hidden="true" tabindex="-1"></a>quast_ref_mism[, <span class="dv">2</span><span class="sc">:</span><span class="dv">11</span>] <span class="ot">&lt;-</span> quast_ref_mism[, <span class="dv">2</span><span class="sc">:</span><span class="dv">11</span>]<span class="sc">/</span>quast_ref_mism[, <span class="dv">2</span>]</span>
<span id="cb77-36"><a href="#cb77-36" aria-hidden="true" tabindex="-1"></a><span class="co"># melt</span></span>
<span id="cb77-37"><a href="#cb77-37" aria-hidden="true" tabindex="-1"></a>quast_ref_mism_melted <span class="ot">&lt;-</span> <span class="fu">cbind</span>(<span class="at">Category =</span> quast_ref_mism[[<span class="dv">1</span>]], <span class="fu">stack</span>(quast_ref_mism[<span class="sc">-</span><span class="dv">1</span>]))</span>
<span id="cb77-38"><a href="#cb77-38" aria-hidden="true" tabindex="-1"></a><span class="co"># fix names</span></span>
<span id="cb77-39"><a href="#cb77-39" aria-hidden="true" tabindex="-1"></a>quast_ref_mism_melted <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="fu">lapply</span>(quast_ref_mism_melted, <span class="cf">function</span>(x) {</span>
<span id="cb77-40"><a href="#cb77-40" aria-hidden="true" tabindex="-1"></a>    <span class="fu">gsub</span>(<span class="st">"A.1"</span>, <span class="st">"A"</span>, x)</span>
<span id="cb77-41"><a href="#cb77-41" aria-hidden="true" tabindex="-1"></a>}))</span>
<span id="cb77-42"><a href="#cb77-42" aria-hidden="true" tabindex="-1"></a>quast_ref_mism_melted <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="fu">lapply</span>(quast_ref_mism_melted, <span class="cf">function</span>(x) {</span>
<span id="cb77-43"><a href="#cb77-43" aria-hidden="true" tabindex="-1"></a>    <span class="fu">gsub</span>(<span class="st">"D.1"</span>, <span class="st">"D"</span>, x)</span>
<span id="cb77-44"><a href="#cb77-44" aria-hidden="true" tabindex="-1"></a>}))</span>
<span id="cb77-45"><a href="#cb77-45" aria-hidden="true" tabindex="-1"></a>quast_ref_mism_melted<span class="sc">$</span>ind <span class="ot">&lt;-</span> <span class="fu">factor</span>(quast_ref_mism_melted<span class="sc">$</span>ind, <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">"NA"</span>, <span class="st">"RP-MDA"</span>, <span class="st">"PrimPol-MDA"</span>,</span>
<span id="cb77-46"><a href="#cb77-46" aria-hidden="true" tabindex="-1"></a>    <span class="st">"piPolB+D"</span>, <span class="st">"piMDA"</span>, <span class="st">"piMDA+D"</span>))</span>
<span id="cb77-47"><a href="#cb77-47" aria-hidden="true" tabindex="-1"></a>quast_ref_mism_melted<span class="sc">$</span>Category <span class="ot">&lt;-</span> <span class="fu">factor</span>(quast_ref_mism_melted<span class="sc">$</span>Category, <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">"E. coli"</span>,</span>
<span id="cb77-48"><a href="#cb77-48" aria-hidden="true" tabindex="-1"></a>    <span class="st">"B. subtilis"</span>, <span class="st">"P. aeruginosa"</span>, <span class="st">"K. rhizophila"</span>, <span class="st">"Not_aligned"</span>))</span>
<span id="cb77-49"><a href="#cb77-49" aria-hidden="true" tabindex="-1"></a>plot2 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(quast_ref_mism_melted) <span class="sc">+</span> <span class="fu">geom_point</span>(<span class="fu">aes</span>(<span class="at">x =</span> ind, <span class="at">y =</span> <span class="fu">as.numeric</span>(values), <span class="at">group =</span> Category,</span>
<span id="cb77-50"><a href="#cb77-50" aria-hidden="true" tabindex="-1"></a>    <span class="at">color =</span> Category, <span class="at">fill =</span> Category, <span class="at">shape =</span> Category), <span class="at">size =</span> <span class="dv">4</span>, <span class="at">alpha =</span> <span class="fl">0.7</span>, <span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="at">width =</span> <span class="fl">0.5</span>)) <span class="sc">+</span></span>
<span id="cb77-51"><a href="#cb77-51" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme_bw</span>() <span class="sc">+</span> <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> colorines[<span class="dv">12</span><span class="sc">:</span><span class="dv">15</span>]) <span class="sc">+</span> <span class="fu">scale_color_manual</span>(<span class="at">values =</span> colorines[<span class="dv">12</span><span class="sc">:</span><span class="dv">15</span>]) <span class="sc">+</span></span>
<span id="cb77-52"><a href="#cb77-52" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_shape_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="dv">21</span>, <span class="dv">22</span>, <span class="dv">23</span>, <span class="dv">24</span>)) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">14</span>)) <span class="sc">+</span></span>
<span id="cb77-53"><a href="#cb77-53" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggtitle</span>(<span class="st">"Normalized mismatches per sample"</span>) <span class="sc">+</span> <span class="fu">xlab</span>(<span class="st">"Sample"</span>) <span class="sc">+</span> <span class="fu">labs</span>(<span class="at">color =</span> <span class="st">"Reference genome"</span>) <span class="sc">+</span></span>
<span id="cb77-54"><a href="#cb77-54" aria-hidden="true" tabindex="-1"></a>    <span class="fu">guides</span>(<span class="at">shape =</span> <span class="cn">FALSE</span>, <span class="at">fill =</span> <span class="cn">FALSE</span>, <span class="at">color =</span> <span class="cn">FALSE</span>, <span class="at">scale =</span> <span class="st">"none"</span>) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">45</span>,</span>
<span id="cb77-55"><a href="#cb77-55" aria-hidden="true" tabindex="-1"></a>    <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">16</span>)) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">legend.text =</span> <span class="fu">element_text</span>(<span class="at">face =</span> <span class="st">"italic"</span>)) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">hjust =</span> <span class="fl">0.5</span>)) <span class="sc">+</span></span>
<span id="cb77-56"><a href="#cb77-56" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_y_continuous</span>(<span class="at">name =</span> <span class="st">"Normalized mismatches per 100 kbp"</span>)</span>
<span id="cb77-57"><a href="#cb77-57" aria-hidden="true" tabindex="-1"></a>grid_contigs_refs <span class="ot">&lt;-</span> <span class="fu">grid.arrange</span>(plot, plot2, <span class="at">nrow =</span> <span class="dv">1</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-left">
<figure class="figure">
<p><img src="Plotassemblies_per_ref-1.png" class="img-fluid figure-img" width="768"></p>
<p></p><figcaption class="figure-caption">Figure 17. Assembly basic stats per reference genome</figcaption><p></p>
</figure>
</div>
</div>
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb78"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb78-1"><a href="#cb78-1" aria-hidden="true" tabindex="-1"></a>grid_contigs_refs</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stdout">
<pre><code>TableGrob (1 x 2) "arrange": 2 grobs
  z     cells    name           grob
1 1 (1-1,1-1) arrange gtable[layout]
2 2 (1-1,2-2) arrange gtable[layout]</code></pre>
</div>
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb80"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb80-1"><a href="#cb80-1" aria-hidden="true" tabindex="-1"></a><span class="co"># ggsave('figures/quast_refs.svg',grid_contigs_refs,width=8,height=5)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
</section>
<section id="number-of-contigs-by-gc-content" class="level3" data-number="6.3">
<h3 data-number="6.3" class="anchored" data-anchor-id="number-of-contigs-by-gc-content"><span class="header-section-number">6.3</span> Number of contigs by GC-content</h3>
<div class="cell" data-layout-align="left">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb81"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb81-1"><a href="#cb81-1" aria-hidden="true" tabindex="-1"></a>gc_contigs <span class="ot">&lt;-</span> <span class="fu">read.table</span>(<span class="st">"quast_results/gc_contigs_distributions.csv"</span>, <span class="at">head =</span> <span class="cn">FALSE</span>, <span class="at">sep =</span> <span class="st">","</span>,</span>
<span id="cb81-2"><a href="#cb81-2" aria-hidden="true" tabindex="-1"></a>    <span class="at">na.strings =</span> <span class="st">""</span>)</span>
<span id="cb81-3"><a href="#cb81-3" aria-hidden="true" tabindex="-1"></a>gc_contigs <span class="ot">&lt;-</span> <span class="fu">t</span>(gc_contigs)</span>
<span id="cb81-4"><a href="#cb81-4" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(gc_contigs) <span class="ot">&lt;-</span> gc_contigs[<span class="dv">1</span>, ]</span>
<span id="cb81-5"><a href="#cb81-5" aria-hidden="true" tabindex="-1"></a>gc_contigs <span class="ot">&lt;-</span> gc_contigs[<span class="sc">-</span><span class="dv">1</span>, ]</span>
<span id="cb81-6"><a href="#cb81-6" aria-hidden="true" tabindex="-1"></a>gc_contigs <span class="ot">&lt;-</span> <span class="fu">as.data.frame</span>(gc_contigs, <span class="at">row.names =</span> <span class="cn">FALSE</span>)</span>
<span id="cb81-7"><a href="#cb81-7" aria-hidden="true" tabindex="-1"></a>gc_melt <span class="ot">&lt;-</span> <span class="fu">melt</span>(gc_contigs, <span class="at">id.vars =</span> <span class="st">"GC"</span>, <span class="at">na.rm =</span> F)</span>
<span id="cb81-8"><a href="#cb81-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb81-9"><a href="#cb81-9" aria-hidden="true" tabindex="-1"></a>gc_melt<span class="sc">$</span>value <span class="ot">&lt;-</span> <span class="fu">as.numeric</span>(gc_melt<span class="sc">$</span>value)</span>
<span id="cb81-10"><a href="#cb81-10" aria-hidden="true" tabindex="-1"></a>gc_melt<span class="sc">$</span>GC <span class="ot">&lt;-</span> <span class="fu">as.numeric</span>(gc_melt<span class="sc">$</span>GC)</span>
<span id="cb81-11"><a href="#cb81-11" aria-hidden="true" tabindex="-1"></a><span class="co"># full version</span></span>
<span id="cb81-12"><a href="#cb81-12" aria-hidden="true" tabindex="-1"></a>gc_melt<span class="sc">$</span>variable <span class="ot">&lt;-</span> <span class="fu">factor</span>(gc_melt<span class="sc">$</span>variable, <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">"NA"</span>, <span class="st">"NA2"</span>, <span class="st">"RP-MDA"</span>, <span class="st">"RP-MDA2"</span>, <span class="st">"PrimPol-MDA"</span>,</span>
<span id="cb81-13"><a href="#cb81-13" aria-hidden="true" tabindex="-1"></a>    <span class="st">"piPolB+D"</span>, <span class="st">"piMDA"</span>, <span class="st">"piMDA2"</span>, <span class="st">"piMDA+D"</span>, <span class="st">"piMDA+D2"</span>))</span>
<span id="cb81-14"><a href="#cb81-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb81-15"><a href="#cb81-15" aria-hidden="true" tabindex="-1"></a>contigs_gc2 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(gc_melt, <span class="fu">aes</span>(<span class="at">y =</span> value, <span class="at">x =</span> GC, <span class="at">fill =</span> variable, <span class="at">label =</span> variable)) <span class="sc">+</span></span>
<span id="cb81-16"><a href="#cb81-16" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_y_continuous</span>(<span class="at">limits =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">1500</span>)) <span class="sc">+</span> <span class="fu">geom_bar</span>(<span class="at">stat =</span> <span class="st">"identity"</span>, <span class="at">position =</span> <span class="st">"dodge"</span>,</span>
<span id="cb81-17"><a href="#cb81-17" aria-hidden="true" tabindex="-1"></a>    <span class="at">alpha =</span> <span class="fl">0.8</span>) <span class="sc">+</span> <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> colorines[<span class="dv">1</span><span class="sc">:</span><span class="dv">11</span>]) <span class="sc">+</span> <span class="fu">scale_color_manual</span>(<span class="at">values =</span> colorines[<span class="dv">1</span><span class="sc">:</span><span class="dv">11</span>]) <span class="sc">+</span></span>
<span id="cb81-18"><a href="#cb81-18" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ylab</span>(<span class="st">"Number of contigs"</span>) <span class="sc">+</span> <span class="fu">xlab</span>(<span class="st">"GC content"</span>) <span class="sc">+</span> <span class="fu">theme_bw</span>() <span class="sc">+</span> <span class="fu">scale_y_break</span>(<span class="fu">c</span>(<span class="dv">75</span>, <span class="dv">200</span>),</span>
<span id="cb81-19"><a href="#cb81-19" aria-hidden="true" tabindex="-1"></a>    <span class="at">scales =</span> <span class="fl">0.5</span>, <span class="at">ticklabels =</span> <span class="fu">c</span>(<span class="dv">200</span>, <span class="dv">600</span>, <span class="dv">1000</span>, <span class="dv">1400</span>), <span class="at">space =</span> <span class="dv">0</span>) <span class="sc">+</span> <span class="fu">scale_x_continuous</span>(<span class="at">limit =</span> <span class="fu">c</span>(<span class="dv">30</span>,</span>
<span id="cb81-20"><a href="#cb81-20" aria-hidden="true" tabindex="-1"></a>    <span class="dv">80</span>), <span class="at">expand =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="fl">0.1</span>)) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">18</span>)) <span class="sc">+</span> <span class="fu">labs</span>(<span class="at">fill =</span> <span class="st">"Sample"</span>)</span>
<span id="cb81-21"><a href="#cb81-21" aria-hidden="true" tabindex="-1"></a>contigs_gc2</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-left">
<figure class="figure">
<p><img src="Plotcontigs_by_GC-1.png" class="img-fluid figure-img" width="1056"></p>
<p></p><figcaption class="figure-caption">Figure 18. Number of contigs by GC content in all samples</figcaption><p></p>
</figure>
</div>
</div>
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb82"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb82-1"><a href="#cb82-1" aria-hidden="true" tabindex="-1"></a><span class="co"># ggsave('figures/GC_contigs2.svg', contigs_gc2, width=10, height=6)</span></span>
<span id="cb82-2"><a href="#cb82-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb82-3"><a href="#cb82-3" aria-hidden="true" tabindex="-1"></a><span class="co"># stats test for normality and homogeneous variance intra groups</span></span>
<span id="cb82-4"><a href="#cb82-4" aria-hidden="true" tabindex="-1"></a>res_aov <span class="ot">&lt;-</span> <span class="fu">aov</span>(value <span class="sc">~</span> variable, <span class="at">data =</span> gc_melt)</span>
<span id="cb82-5"><a href="#cb82-5" aria-hidden="true" tabindex="-1"></a><span class="fu">shapiro.test</span>(res_aov<span class="sc">$</span>residuals)  <span class="co">#NO normality </span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stdout">
<pre><code>
    Shapiro-Wilk normality test

data:  res_aov$residuals
W = 0.32817, p-value &lt; 2.2e-16</code></pre>
</div>
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb84"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb84-1"><a href="#cb84-1" aria-hidden="true" tabindex="-1"></a><span class="fu">leveneTest</span>(value <span class="sc">~</span> variable, <span class="at">data =</span> gc_melt)  <span class="co">#NO homogeneus variance</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stdout">
<pre><code>Levene's Test for Homogeneity of Variance (center = median)
       Df F value Pr(&gt;F)  
group   9  2.1723 0.0254 *
      200                 
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</code></pre>
</div>
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb86"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb86-1"><a href="#cb86-1" aria-hidden="true" tabindex="-1"></a><span class="co"># non parametric, pairwise (paired) tests</span></span>
<span id="cb86-2"><a href="#cb86-2" aria-hidden="true" tabindex="-1"></a><span class="fu">pairwise.wilcox.test</span>(<span class="at">x =</span> gc_melt<span class="sc">$</span>value, <span class="at">g =</span> gc_melt<span class="sc">$</span>variable, <span class="at">paired =</span> <span class="cn">TRUE</span>, <span class="at">p.adjust.method =</span> <span class="st">"BH"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stdout">
<pre><code>
    Pairwise comparisons using Wilcoxon signed rank test with continuity correction 

data:  gc_melt$value and gc_melt$variable 

            NA   NA2  RP-MDA RP-MDA2 PrimPol-MDA piPolB+D piMDA piMDA2 piMDA+D
NA2         0.34 -    -      -       -           -        -     -      -      
RP-MDA      1.00 0.17 -      -       -           -        -     -      -      
RP-MDA2     0.12 0.12 0.12   -       -           -        -     -      -      
PrimPol-MDA 0.13 0.14 0.14   0.34    -           -        -     -      -      
piPolB+D    0.34 0.34 0.30   0.12    0.17        -        -     -      -      
piMDA       0.34 0.14 0.34   0.12    0.20        0.30     -     -      -      
piMDA2      0.17 0.13 0.27   0.14    0.25        0.20     0.24  -      -      
piMDA+D     0.45 0.65 0.20   0.12    0.12        0.34     0.23  0.17   -      
piMDA+D2    1.00 0.14 0.83   0.12    0.14        0.34     0.23  0.17   0.47   

P value adjustment method: BH </code></pre>
</div>
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb88"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb88-1"><a href="#cb88-1" aria-hidden="true" tabindex="-1"></a><span class="co"># no significant differences</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<p>The coverage by the GC content can be also represented by Circos plots. The outer circle represents reference sequences with GC (%) heatmap, from 25% (white) to 79% (black). Color bars help to distinguish between different references. Assembly tracks are combined with mismatches visualization: green contigs indicate perfect assembly, whereas red contigs contains misassemblies, higher black columns indicate larger mismatch rate. Finally, the inner circle represents read coverage histogram.</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<figure class="figure">
<img src="images/circos_all.png" title="Circos plots of all samples by the four reference genomes" class="img-fluid figure-img">
</figure>
<p></p><figcaption class="figure-caption">External Figure 3. Circos plots of all samples by the four reference genomes.</figcaption><p></p>
</figure>
</div>
<p>The analysis of assemblies shows that for all the samples <em>P. aeruginosa</em> genome seems different to the reference genome. On the other hand, <em>B. subtilis 110NA</em> is a 168 derivative with a deletion spanning Spo0A gene (see <a href="https://gallica.bnf.fr/ark:/12148/bpt6k6436891f/f9.item" target="_blank">Michel, J.F. y Cami, B., Ann. Inst. Pasteur 116(N1), 3-18 (1969)</a>). On the contrary the assemblies of <em>E. coli</em> and <em>K. rhizophila</em> fit with the reference genomes in most samples.</p>
</section>
<section id="assembly-annotation" class="level3" data-number="6.4">
<h3 data-number="6.4" class="anchored" data-anchor-id="assembly-annotation"><span class="header-section-number">6.4</span> Assembly annotation</h3>
<p><a href="https://github.com/oschwengers/bakta">Bakta</a> v.1.5.1 with Bakta Database v.4.0. (<a href="https://pubmed.ncbi.nlm.nih.gov/34739369/">Schwengers et al, 2021</a>). Results in <em>bakta_results</em> folder in the repo.</p>
<div class="cell">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb89"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb89-1"><a href="#cb89-1" aria-hidden="true" tabindex="-1"></a>samples <span class="ot">=</span> <span class="fu">c</span>(<span class="st">"ctrl"</span>, <span class="st">"ctrl2"</span>, <span class="st">"N4"</span>, <span class="st">"D3"</span>, <span class="st">"C3"</span>, <span class="st">"F2"</span>, <span class="st">"N2"</span>, <span class="st">"B2"</span>, <span class="st">"N6"</span>, <span class="st">"A2"</span>)</span>
<span id="cb89-2"><a href="#cb89-2" aria-hidden="true" tabindex="-1"></a>bakta <span class="ot">&lt;-</span> <span class="fu">lapply</span>(samples, <span class="cf">function</span>(x) <span class="fu">read.csv</span>(<span class="at">file =</span> <span class="fu">paste0</span>(<span class="st">"bakta_stats/"</span>, x, <span class="st">".txt"</span>), <span class="at">sep =</span> <span class="st">":"</span>,</span>
<span id="cb89-3"><a href="#cb89-3" aria-hidden="true" tabindex="-1"></a>    <span class="at">header =</span> F, <span class="at">skip =</span> <span class="dv">1</span>, <span class="at">col.names =</span> <span class="fu">c</span>(<span class="st">"V1"</span>, <span class="fu">paste</span>(x))))</span>
<span id="cb89-4"><a href="#cb89-4" aria-hidden="true" tabindex="-1"></a>bakta_merged <span class="ot">&lt;-</span> <span class="fu">Reduce</span>(<span class="cf">function</span>(x, y) <span class="fu">merge</span>(x, y, <span class="at">by =</span> <span class="st">"V1"</span>, <span class="at">all =</span> <span class="cn">TRUE</span>), bakta)</span>
<span id="cb89-5"><a href="#cb89-5" aria-hidden="true" tabindex="-1"></a>bakta_final <span class="ot">&lt;-</span> bakta_merged[<span class="fu">c</span>(<span class="dv">3</span>, <span class="dv">4</span>, <span class="dv">15</span>, <span class="dv">16</span>, <span class="dv">20</span>, <span class="dv">21</span>, <span class="dv">24</span>, <span class="dv">25</span>, <span class="dv">26</span>), ]</span>
<span id="cb89-6"><a href="#cb89-6" aria-hidden="true" tabindex="-1"></a>bakta_final <span class="ot">=</span> <span class="fu">data.frame</span>(bakta_final, <span class="at">row.names =</span> <span class="cn">NULL</span>)</span>
<span id="cb89-7"><a href="#cb89-7" aria-hidden="true" tabindex="-1"></a><span class="fu">names</span>(bakta_final) <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"Parameter"</span>, <span class="st">"NA"</span>, <span class="st">"NA2"</span>, <span class="st">"RP-MDA"</span>, <span class="st">"RP-MDA2"</span>, <span class="st">"PrimPol-MDA"</span>, <span class="st">"piPolB+D"</span>,</span>
<span id="cb89-8"><a href="#cb89-8" aria-hidden="true" tabindex="-1"></a>    <span class="st">"piMDA"</span>, <span class="st">"piMDA2"</span>, <span class="st">"piMDA+D"</span>, <span class="st">"piMDA+D2"</span>)</span>
<span id="cb89-9"><a href="#cb89-9" aria-hidden="true" tabindex="-1"></a><span class="fu">kbl</span>(bakta_final, <span class="at">align =</span> <span class="st">"c"</span>, <span class="at">caption =</span> <span class="st">"Table 7. Bakta annotation stats"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb89-10"><a href="#cb89-10" aria-hidden="true" tabindex="-1"></a>    <span class="fu">kable_styling</span>(<span class="at">bootstrap_options =</span> <span class="st">"striped"</span>, <span class="at">full_width =</span> F, <span class="at">position =</span> <span class="st">"center"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb89-11"><a href="#cb89-11" aria-hidden="true" tabindex="-1"></a>    <span class="fu">column_spec</span>(<span class="dv">1</span>, <span class="at">italic =</span> T)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">

<table class="table table-striped" style="width: auto !important; margin-left: auto; margin-right: auto;">
<caption>Table 7. Bakta annotation stats</caption>
 <thead>
  <tr>
   <th style="text-align:center;"> Parameter </th>
   <th style="text-align:center;"> NA </th>
   <th style="text-align:center;"> NA2 </th>
   <th style="text-align:center;"> RP-MDA </th>
   <th style="text-align:center;"> RP-MDA2 </th>
   <th style="text-align:center;"> PrimPol-MDA </th>
   <th style="text-align:center;"> piPolB+D </th>
   <th style="text-align:center;"> piMDA </th>
   <th style="text-align:center;"> piMDA2 </th>
   <th style="text-align:center;"> piMDA+D </th>
   <th style="text-align:center;"> piMDA+D2 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:center;font-style: italic;"> CDSs </td>
   <td style="text-align:center;"> 16612 </td>
   <td style="text-align:center;"> 16609 </td>
   <td style="text-align:center;"> 14502 </td>
   <td style="text-align:center;"> 16713 </td>
   <td style="text-align:center;"> 16609 </td>
   <td style="text-align:center;"> 17326 </td>
   <td style="text-align:center;"> 16671 </td>
   <td style="text-align:center;"> 16697 </td>
   <td style="text-align:center;"> 16627 </td>
   <td style="text-align:center;"> 16627 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> coding density </td>
   <td style="text-align:center;"> 89.3 </td>
   <td style="text-align:center;"> 89.3 </td>
   <td style="text-align:center;"> 79.9 </td>
   <td style="text-align:center;"> 87.3 </td>
   <td style="text-align:center;"> 89.3 </td>
   <td style="text-align:center;"> 74.2 </td>
   <td style="text-align:center;"> 89.2 </td>
   <td style="text-align:center;"> 89.1 </td>
   <td style="text-align:center;"> 89.2 </td>
   <td style="text-align:center;"> 89.3 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> ncRNA regions </td>
   <td style="text-align:center;"> 161 </td>
   <td style="text-align:center;"> 161 </td>
   <td style="text-align:center;"> 149 </td>
   <td style="text-align:center;"> 159 </td>
   <td style="text-align:center;"> 160 </td>
   <td style="text-align:center;"> 160 </td>
   <td style="text-align:center;"> 160 </td>
   <td style="text-align:center;"> 162 </td>
   <td style="text-align:center;"> 160 </td>
   <td style="text-align:center;"> 160 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> ncRNAs </td>
   <td style="text-align:center;"> 292 </td>
   <td style="text-align:center;"> 286 </td>
   <td style="text-align:center;"> 287 </td>
   <td style="text-align:center;"> 287 </td>
   <td style="text-align:center;"> 291 </td>
   <td style="text-align:center;"> 285 </td>
   <td style="text-align:center;"> 293 </td>
   <td style="text-align:center;"> 289 </td>
   <td style="text-align:center;"> 291 </td>
   <td style="text-align:center;"> 289 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> pseudogenes </td>
   <td style="text-align:center;"> 45 </td>
   <td style="text-align:center;"> 44 </td>
   <td style="text-align:center;"> 125 </td>
   <td style="text-align:center;"> 76 </td>
   <td style="text-align:center;"> 43 </td>
   <td style="text-align:center;"> 120 </td>
   <td style="text-align:center;"> 44 </td>
   <td style="text-align:center;"> 46 </td>
   <td style="text-align:center;"> 45 </td>
   <td style="text-align:center;"> 45 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> rRNAs </td>
   <td style="text-align:center;"> 12 </td>
   <td style="text-align:center;"> 14 </td>
   <td style="text-align:center;"> 12 </td>
   <td style="text-align:center;"> 12 </td>
   <td style="text-align:center;"> 13 </td>
   <td style="text-align:center;"> 12 </td>
   <td style="text-align:center;"> 12 </td>
   <td style="text-align:center;"> 12 </td>
   <td style="text-align:center;"> 12 </td>
   <td style="text-align:center;"> 13 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> sORFs </td>
   <td style="text-align:center;"> 90 </td>
   <td style="text-align:center;"> 91 </td>
   <td style="text-align:center;"> 84 </td>
   <td style="text-align:center;"> 89 </td>
   <td style="text-align:center;"> 92 </td>
   <td style="text-align:center;"> 84 </td>
   <td style="text-align:center;"> 89 </td>
   <td style="text-align:center;"> 89 </td>
   <td style="text-align:center;"> 91 </td>
   <td style="text-align:center;"> 92 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> tmRNAs </td>
   <td style="text-align:center;"> 5 </td>
   <td style="text-align:center;"> 5 </td>
   <td style="text-align:center;"> 3 </td>
   <td style="text-align:center;"> 5 </td>
   <td style="text-align:center;"> 5 </td>
   <td style="text-align:center;"> 5 </td>
   <td style="text-align:center;"> 5 </td>
   <td style="text-align:center;"> 5 </td>
   <td style="text-align:center;"> 5 </td>
   <td style="text-align:center;"> 5 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> tRNAs </td>
   <td style="text-align:center;"> 254 </td>
   <td style="text-align:center;"> 248 </td>
   <td style="text-align:center;"> 210 </td>
   <td style="text-align:center;"> 244 </td>
   <td style="text-align:center;"> 246 </td>
   <td style="text-align:center;"> 237 </td>
   <td style="text-align:center;"> 248 </td>
   <td style="text-align:center;"> 251 </td>
   <td style="text-align:center;"> 242 </td>
   <td style="text-align:center;"> 239 </td>
  </tr>
</tbody>
</table>

</div>
</div>
<p>The annotation of all the metagenomes was very similar, except for those from RP-MDA and piPolB+D amplified DNA, which contain a high number of pseudogenes, most likely as consequence of poor assembly</p>
</section>
<section id="variant-calling" class="level3" data-number="6.5">
<h3 data-number="6.5" class="anchored" data-anchor-id="variant-calling"><span class="header-section-number">6.5</span> Variant calling</h3>
<p>We performed short nucleotide variant (SNV) detection with <a href="https://github.com/tseemann/snippy">Snippy</a> 4.6.0, which has been benchmarked as the top variant caller for microbial genomes,showing both strong and uniform performance across different species (<a href="https://pubmed.ncbi.nlm.nih.gov/32025702/">Bush et al.&nbsp;2020</a>).</p>
<div class="cell">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb90"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb90-1"><a href="#cb90-1" aria-hidden="true" tabindex="-1"></a>samples<span class="ot">=</span><span class="fu">c</span>(<span class="st">"Ctrl"</span>,<span class="st">"Ctrl2"</span>,<span class="st">"N4"</span>,<span class="st">"D3"</span>,<span class="st">"C3"</span>,<span class="st">"N1"</span>,<span class="st">"F2"</span>,<span class="st">"N2"</span>,<span class="st">"B2"</span>,<span class="st">"N6"</span>,<span class="st">"A2"</span>)</span>
<span id="cb90-2"><a href="#cb90-2" aria-hidden="true" tabindex="-1"></a>snps <span class="ot">&lt;-</span> <span class="fu">lapply</span>(samples, <span class="cf">function</span>(x) <span class="fu">read.csv</span>(<span class="at">file=</span><span class="fu">paste0</span>(<span class="st">"snippy_results/"</span>,x,<span class="st">"_snps.csv"</span>),<span class="at">sep=</span><span class="st">","</span>, <span class="at">header=</span>T))</span>
<span id="cb90-3"><a href="#cb90-3" aria-hidden="true" tabindex="-1"></a>shared_samples_NA <span class="ot">&lt;-</span> <span class="fu">list</span>()</span>
<span id="cb90-4"><a href="#cb90-4" aria-hidden="true" tabindex="-1"></a>samples_NA <span class="ot">&lt;-</span> <span class="fu">list</span>()</span>
<span id="cb90-5"><a href="#cb90-5" aria-hidden="true" tabindex="-1"></a>NA_samples <span class="ot">&lt;-</span> <span class="fu">list</span>()</span>
<span id="cb90-6"><a href="#cb90-6" aria-hidden="true" tabindex="-1"></a>snps_table <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="fu">matrix</span>(<span class="cn">NA</span>,    <span class="co"># Create empty data frame</span></span>
<span id="cb90-7"><a href="#cb90-7" aria-hidden="true" tabindex="-1"></a>                          <span class="at">nrow =</span> <span class="dv">11</span>,</span>
<span id="cb90-8"><a href="#cb90-8" aria-hidden="true" tabindex="-1"></a>                          <span class="at">ncol =</span> <span class="dv">9</span>))</span>
<span id="cb90-9"><a href="#cb90-9" aria-hidden="true" tabindex="-1"></a>snps_table[,<span class="dv">1</span>] <span class="ot">&lt;-</span>  <span class="fu">c</span>(<span class="st">"NA"</span>,<span class="st">"NA2"</span>,<span class="st">"RP-MDA"</span>,<span class="st">"RP-MDA2"</span>,<span class="st">"PrimPol-MDA"</span>,<span class="st">"piPolB"</span>,<span class="st">"piPolB+D"</span>,<span class="st">"piMDA"</span>,<span class="st">"piMDA2"</span>,<span class="st">"piMDA+D"</span>,<span class="st">"piMDA+D2"</span>)</span>
<span id="cb90-10"><a href="#cb90-10" aria-hidden="true" tabindex="-1"></a><span class="st">`</span><span class="at">%notin%</span><span class="st">`</span> <span class="ot">&lt;-</span> <span class="fu">Negate</span>(<span class="st">`</span><span class="at">%in%</span><span class="st">`</span>)</span>
<span id="cb90-11"><a href="#cb90-11" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">length</span>(samples)){</span>
<span id="cb90-12"><a href="#cb90-12" aria-hidden="true" tabindex="-1"></a>  shared_samples_NA[[i]] <span class="ot">&lt;-</span> snps[[i]][snps[[i]]<span class="sc">$</span>POS <span class="sc">%in%</span> snps[[<span class="dv">1</span>]]<span class="sc">$</span>POS,]</span>
<span id="cb90-13"><a href="#cb90-13" aria-hidden="true" tabindex="-1"></a>  samples_NA[[i]] <span class="ot">&lt;-</span> snps[[i]][snps[[i]]<span class="sc">$</span>POS <span class="sc">%notin%</span> snps[[<span class="dv">1</span>]]<span class="sc">$</span>POS,]</span>
<span id="cb90-14"><a href="#cb90-14" aria-hidden="true" tabindex="-1"></a>  NA_samples[[i]] <span class="ot">&lt;-</span> snps[[<span class="dv">1</span>]][snps[[<span class="dv">1</span>]]<span class="sc">$</span>POS <span class="sc">%notin%</span> snps[[i]]<span class="sc">$</span>POS,]</span>
<span id="cb90-15"><a href="#cb90-15" aria-hidden="true" tabindex="-1"></a>  snps_table[i,<span class="dv">2</span>] <span class="ot">&lt;-</span> <span class="fu">nrow</span>(snps[[i]])</span>
<span id="cb90-16"><a href="#cb90-16" aria-hidden="true" tabindex="-1"></a>  snps_table[i,<span class="dv">3</span>] <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="fu">nrow</span>(shared_samples_NA[[i]]))</span>
<span id="cb90-17"><a href="#cb90-17" aria-hidden="true" tabindex="-1"></a>  snps_table[i,<span class="dv">4</span>] <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="fu">nrow</span>(samples_NA[[i]]))</span>
<span id="cb90-18"><a href="#cb90-18" aria-hidden="true" tabindex="-1"></a>  snps_table[i,<span class="dv">5</span>] <span class="ot">&lt;-</span> <span class="fu">table</span>(samples_NA[[i]]<span class="sc">$</span>TYPE<span class="sc">==</span><span class="st">"snp"</span>)[<span class="st">"TRUE"</span>]</span>
<span id="cb90-19"><a href="#cb90-19" aria-hidden="true" tabindex="-1"></a>  snps_table[i,<span class="dv">6</span>] <span class="ot">&lt;-</span> <span class="fu">table</span>(samples_NA[[i]]<span class="sc">$</span>TYPE<span class="sc">==</span><span class="st">"complex"</span>)[<span class="st">"TRUE"</span>]</span>
<span id="cb90-20"><a href="#cb90-20" aria-hidden="true" tabindex="-1"></a>  snps_table[i,<span class="dv">7</span>] <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="fu">nrow</span>(NA_samples[[i]]))</span>
<span id="cb90-21"><a href="#cb90-21" aria-hidden="true" tabindex="-1"></a>  snps_table[i,<span class="dv">8</span>] <span class="ot">&lt;-</span> <span class="fu">table</span>(NA_samples[[i]]<span class="sc">$</span>TYPE<span class="sc">==</span><span class="st">"snp"</span>)[<span class="st">"TRUE"</span>]</span>
<span id="cb90-22"><a href="#cb90-22" aria-hidden="true" tabindex="-1"></a>  snps_table[i,<span class="dv">9</span>] <span class="ot">&lt;-</span> <span class="fu">table</span>(NA_samples[[i]]<span class="sc">$</span>TYPE<span class="sc">==</span><span class="st">"complex"</span>)[<span class="st">"TRUE"</span>]</span>
<span id="cb90-23"><a href="#cb90-23" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb90-24"><a href="#cb90-24" aria-hidden="true" tabindex="-1"></a><span class="fu">names</span>(snps_table) <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"Sample"</span>,<span class="st">"Variants"</span>,<span class="st">"Variants in NA"</span>,<span class="st">"New Variants"</span>,<span class="st">"New SNPS"</span>,<span class="st">"New Complex"</span>,<span class="st">"Missing Variants"</span>,<span class="st">"Missing SNPS"</span>,<span class="st">"Missing Complex"</span>)</span>
<span id="cb90-25"><a href="#cb90-25" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb90-26"><a href="#cb90-26" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb90-27"><a href="#cb90-27" aria-hidden="true" tabindex="-1"></a><span class="fu">kbl</span>(snps_table, <span class="at">align=</span><span class="st">"c"</span>,<span class="at">caption=</span><span class="st">"Table 8. Variant detection"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb90-28"><a href="#cb90-28" aria-hidden="true" tabindex="-1"></a>    <span class="fu">kable_styling</span>(<span class="at">bootstrap_options =</span> <span class="st">"striped"</span>, <span class="at">full_width =</span> F,</span>
<span id="cb90-29"><a href="#cb90-29" aria-hidden="true" tabindex="-1"></a>        <span class="at">position =</span> <span class="st">"center"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb90-30"><a href="#cb90-30" aria-hidden="true" tabindex="-1"></a>     <span class="fu">column_spec</span>(<span class="dv">1</span>, <span class="at">italic =</span> T) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">

<table class="table table-striped" style="width: auto !important; margin-left: auto; margin-right: auto;">
<caption>Table 8. Variant detection</caption>
 <thead>
  <tr>
   <th style="text-align:center;"> Sample </th>
   <th style="text-align:center;"> Variants </th>
   <th style="text-align:center;"> Variants in NA </th>
   <th style="text-align:center;"> New Variants </th>
   <th style="text-align:center;"> New SNPS </th>
   <th style="text-align:center;"> New Complex </th>
   <th style="text-align:center;"> Missing Variants </th>
   <th style="text-align:center;"> Missing SNPS </th>
   <th style="text-align:center;"> Missing Complex </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:center;font-style: italic;"> NA </td>
   <td style="text-align:center;"> 61 </td>
   <td style="text-align:center;"> 61 </td>
   <td style="text-align:center;"> 0 </td>
   <td style="text-align:center;"> NA </td>
   <td style="text-align:center;"> NA </td>
   <td style="text-align:center;"> 0 </td>
   <td style="text-align:center;"> NA </td>
   <td style="text-align:center;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> NA2 </td>
   <td style="text-align:center;"> 61 </td>
   <td style="text-align:center;"> 61 </td>
   <td style="text-align:center;"> 0 </td>
   <td style="text-align:center;"> NA </td>
   <td style="text-align:center;"> NA </td>
   <td style="text-align:center;"> 0 </td>
   <td style="text-align:center;"> NA </td>
   <td style="text-align:center;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> RP-MDA </td>
   <td style="text-align:center;"> 26 </td>
   <td style="text-align:center;"> 19 </td>
   <td style="text-align:center;"> 7 </td>
   <td style="text-align:center;"> 6 </td>
   <td style="text-align:center;"> 1 </td>
   <td style="text-align:center;"> 42 </td>
   <td style="text-align:center;"> 32 </td>
   <td style="text-align:center;"> 10 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> RP-MDA2 </td>
   <td style="text-align:center;"> 36 </td>
   <td style="text-align:center;"> 31 </td>
   <td style="text-align:center;"> 5 </td>
   <td style="text-align:center;"> 4 </td>
   <td style="text-align:center;"> 1 </td>
   <td style="text-align:center;"> 30 </td>
   <td style="text-align:center;"> 21 </td>
   <td style="text-align:center;"> 9 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> PrimPol-MDA </td>
   <td style="text-align:center;"> 46 </td>
   <td style="text-align:center;"> 40 </td>
   <td style="text-align:center;"> 6 </td>
   <td style="text-align:center;"> 5 </td>
   <td style="text-align:center;"> 1 </td>
   <td style="text-align:center;"> 21 </td>
   <td style="text-align:center;"> 14 </td>
   <td style="text-align:center;"> 7 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> piPolB </td>
   <td style="text-align:center;"> 1 </td>
   <td style="text-align:center;"> 0 </td>
   <td style="text-align:center;"> 1 </td>
   <td style="text-align:center;"> NA </td>
   <td style="text-align:center;"> 1 </td>
   <td style="text-align:center;"> 61 </td>
   <td style="text-align:center;"> 45 </td>
   <td style="text-align:center;"> 16 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> piPolB+D </td>
   <td style="text-align:center;"> 12 </td>
   <td style="text-align:center;"> 11 </td>
   <td style="text-align:center;"> 1 </td>
   <td style="text-align:center;"> 1 </td>
   <td style="text-align:center;"> NA </td>
   <td style="text-align:center;"> 50 </td>
   <td style="text-align:center;"> 36 </td>
   <td style="text-align:center;"> 14 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> piMDA </td>
   <td style="text-align:center;"> 47 </td>
   <td style="text-align:center;"> 41 </td>
   <td style="text-align:center;"> 6 </td>
   <td style="text-align:center;"> 5 </td>
   <td style="text-align:center;"> 1 </td>
   <td style="text-align:center;"> 20 </td>
   <td style="text-align:center;"> 16 </td>
   <td style="text-align:center;"> 4 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> piMDA2 </td>
   <td style="text-align:center;"> 60 </td>
   <td style="text-align:center;"> 49 </td>
   <td style="text-align:center;"> 11 </td>
   <td style="text-align:center;"> 7 </td>
   <td style="text-align:center;"> 3 </td>
   <td style="text-align:center;"> 12 </td>
   <td style="text-align:center;"> 10 </td>
   <td style="text-align:center;"> 2 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> piMDA+D </td>
   <td style="text-align:center;"> 40 </td>
   <td style="text-align:center;"> 36 </td>
   <td style="text-align:center;"> 4 </td>
   <td style="text-align:center;"> 4 </td>
   <td style="text-align:center;"> NA </td>
   <td style="text-align:center;"> 25 </td>
   <td style="text-align:center;"> 19 </td>
   <td style="text-align:center;"> 6 </td>
  </tr>
  <tr>
   <td style="text-align:center;font-style: italic;"> piMDA+D2 </td>
   <td style="text-align:center;"> 61 </td>
   <td style="text-align:center;"> 61 </td>
   <td style="text-align:center;"> 0 </td>
   <td style="text-align:center;"> NA </td>
   <td style="text-align:center;"> NA </td>
   <td style="text-align:center;"> 0 </td>
   <td style="text-align:center;"> NA </td>
   <td style="text-align:center;"> NA </td>
  </tr>
</tbody>
</table>

</div>
</div>
<p>As shown in Table 8, variant calling capacity varies greatly samples. For example, RP-MDA and piPolB(+D) amplification result in poor variant detection of both single nucleotide polymorphisms (snps) and complex variants. It should be noted that with the exception of samples NA and NA2, which called the same SNVs, there are differences between the duplicates and the second batch of samples from RP-MDA. piMDA and especially piMDA+D allow better variant detection, with piMDA+D being identical to the control samples. Nevertheless, the PrimPol-MDA and piMDA(+D) methods show similar and good performance in variant detection, with up to 60-80% of variants overlapping with the non-amplified samples.</p>
</section>
<section id="assembly-k-mer-content" class="level3" data-number="6.6">
<h3 data-number="6.6" class="anchored" data-anchor-id="assembly-k-mer-content"><span class="header-section-number">6.6</span> Assembly k-mer content</h3>
<p>We used KAT v.2.4.1 to compare the Illumina reads with their corresponding Spades assembly.</p>
<div class="cell" data-layout-align="left">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb91"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb91-1"><a href="#cb91-1" aria-hidden="true" tabindex="-1"></a>reads <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"Ctrl"</span>, <span class="st">"Ctrl2"</span>, <span class="st">"N4"</span>, <span class="st">"D3"</span>, <span class="st">"C3"</span>, <span class="st">"F2"</span>, <span class="st">"N2"</span>, <span class="st">"B2"</span>, <span class="st">"N6"</span>, <span class="st">"A2"</span>)</span>
<span id="cb91-2"><a href="#cb91-2" aria-hidden="true" tabindex="-1"></a>samples <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"NA"</span>, <span class="st">"NA2"</span>, <span class="st">"RP-MDA"</span>, <span class="st">"RP-MDA2"</span>, <span class="st">"PrimPol-MDA"</span>, <span class="st">"piPolB+D"</span>, <span class="st">"piMDA"</span>, <span class="st">"piMDA2"</span>,</span>
<span id="cb91-3"><a href="#cb91-3" aria-hidden="true" tabindex="-1"></a>    <span class="st">"piMDA+D"</span>, <span class="st">"piMDA+D2"</span>)</span>
<span id="cb91-4"><a href="#cb91-4" aria-hidden="true" tabindex="-1"></a>kat <span class="ot">&lt;-</span> <span class="fu">list</span>()</span>
<span id="cb91-5"><a href="#cb91-5" aria-hidden="true" tabindex="-1"></a>p_kat <span class="ot">&lt;-</span> <span class="fu">list</span>()</span>
<span id="cb91-6"><a href="#cb91-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb91-7"><a href="#cb91-7" aria-hidden="true" tabindex="-1"></a>kat <span class="ot">&lt;-</span> <span class="fu">lapply</span>(reads, <span class="cf">function</span>(x) <span class="fu">read.csv</span>(<span class="fu">paste0</span>(<span class="st">"kat/sample_vs_assembly/"</span>, x, <span class="st">"_m16_pe_v_asm_test-main.mx"</span>),</span>
<span id="cb91-8"><a href="#cb91-8" aria-hidden="true" tabindex="-1"></a>    <span class="at">sep =</span> <span class="st">" "</span>, <span class="at">header =</span> <span class="cn">FALSE</span>, <span class="at">skip =</span> <span class="dv">12</span>))</span>
<span id="cb91-9"><a href="#cb91-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb91-10"><a href="#cb91-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb91-11"><a href="#cb91-11" aria-hidden="true" tabindex="-1"></a>p_kat <span class="ot">&lt;-</span> <span class="fu">lapply</span>(<span class="dv">1</span><span class="sc">:</span><span class="fu">length</span>(reads), <span class="at">FUN =</span> <span class="cf">function</span>(i) {</span>
<span id="cb91-12"><a href="#cb91-12" aria-hidden="true" tabindex="-1"></a>    p_kat[[i]] <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(<span class="at">data =</span> kat[[i]]) <span class="sc">+</span> <span class="fu">geom_point</span>(<span class="fu">aes_</span>(<span class="at">x =</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">length</span>(kat[[i]][, <span class="dv">5</span>]), <span class="at">y =</span> (kat[[i]][,</span>
<span id="cb91-13"><a href="#cb91-13" aria-hidden="true" tabindex="-1"></a>        <span class="dv">5</span>])), <span class="at">col =</span> <span class="st">"cyan"</span>, <span class="at">alpha =</span> <span class="fl">0.5</span>) <span class="sc">+</span> <span class="fu">geom_point</span>(<span class="fu">aes_</span>(<span class="at">x =</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">length</span>(kat[[i]][, <span class="dv">4</span>]), <span class="at">y =</span> (kat[[i]][,</span>
<span id="cb91-14"><a href="#cb91-14" aria-hidden="true" tabindex="-1"></a>        <span class="dv">4</span>])), <span class="at">col =</span> <span class="st">"green"</span>, <span class="at">alpha =</span> <span class="fl">0.5</span>) <span class="sc">+</span> <span class="fu">geom_point</span>(<span class="fu">aes_</span>(<span class="at">x =</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">length</span>(kat[[i]][, <span class="dv">3</span>]), <span class="at">y =</span> (kat[[i]][,</span>
<span id="cb91-15"><a href="#cb91-15" aria-hidden="true" tabindex="-1"></a>        <span class="dv">3</span>])), <span class="at">col =</span> <span class="st">"orange"</span>, <span class="at">alpha =</span> <span class="fl">0.5</span>) <span class="sc">+</span> <span class="fu">geom_point</span>(<span class="fu">aes_</span>(<span class="at">x =</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">length</span>(kat[[i]][, <span class="dv">2</span>]), <span class="at">y =</span> (kat[[i]][,</span>
<span id="cb91-16"><a href="#cb91-16" aria-hidden="true" tabindex="-1"></a>        <span class="dv">2</span>])), <span class="at">col =</span> <span class="st">"red"</span>, <span class="at">alpha =</span> <span class="fl">0.5</span>) <span class="sc">+</span> <span class="fu">geom_point</span>(<span class="fu">aes_</span>(<span class="at">x =</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">length</span>(kat[[i]][, <span class="dv">1</span>]), <span class="at">y =</span> (kat[[i]][,</span>
<span id="cb91-17"><a href="#cb91-17" aria-hidden="true" tabindex="-1"></a>        <span class="dv">1</span>])), <span class="at">alpha =</span> <span class="fl">0.5</span>) <span class="sc">+</span> <span class="fu">ylim</span>(<span class="dv">0</span>, <span class="fl">2e+05</span>) <span class="sc">+</span> <span class="fu">theme_bw</span>() <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">13</span>)) <span class="sc">+</span></span>
<span id="cb91-18"><a href="#cb91-18" aria-hidden="true" tabindex="-1"></a>        <span class="fu">xlab</span>(<span class="st">"k-mer frequency"</span>) <span class="sc">+</span> <span class="fu">ylab</span>(<span class="st">"Number of distinct k-mers"</span>) <span class="sc">+</span> <span class="fu">ggtitle</span>(samples[i])</span>
<span id="cb91-19"><a href="#cb91-19" aria-hidden="true" tabindex="-1"></a>})</span>
<span id="cb91-20"><a href="#cb91-20" aria-hidden="true" tabindex="-1"></a><span class="fu">do.call</span>(<span class="st">"grid.arrange"</span>, <span class="fu">c</span>(p_kat, <span class="at">ncol =</span> <span class="dv">5</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-left">
<figure class="figure">
<p><img src="Plotkat_assembly-1.png" class="img-fluid figure-img" width="1056"></p>
<p></p><figcaption class="figure-caption">Figure 19. KAT k-mer plots comparing k-mer content of Illumina PCR-free reads with the corresponding assembly. The black dots of the graph represents the distribution of k-mers present in the reads but not in the assembly, and the red area represents the distribution of k-mers present in the reads and once in the assembly</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>The black line in piPolB+D has a larger extension along the x-axis, indicating a higher number of k-mers excluded from the assembly. The red area in all graphs represents a normal distribution of k-mers found in the reads and occurring once in the assembly. The presence of orange dots in some samples, namely RP-MDA, RP-MDA2, PrimPol-MDA and piPolB+D, reflects the presence of repetitive content throughout the assembly, with k-mers in the reads occurring &gt;2 times in the assembly, in agreement with a higher level of biased overamplification (Figure 4).</p>
</section>
<section id="assembly-binning" class="level3" data-number="6.7">
<h3 data-number="6.7" class="anchored" data-anchor-id="assembly-binning"><span class="header-section-number">6.7</span> Assembly binning</h3>
<p>With <a href="https://github.com/Vini2/MetaCoAG" target="_blank">MetacoAG</a> 1.0. MetacoAG uses the information of assembly graphs as well as composition (<em>k</em>-mers), gene markers and coverage information. Obtained bins were then assigned to the reference genomes by the mash distances, selecting those with coverage &gt;5% and p-value &lt; 10<sup>-10</sup>. Results are available in the folder <em>binning</em> in the repo.</p>
<div class="cell" data-layout-align="left">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb92"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb92-1"><a href="#cb92-1" aria-hidden="true" tabindex="-1"></a><span class="co"># read data</span></span>
<span id="cb92-2"><a href="#cb92-2" aria-hidden="true" tabindex="-1"></a>dist <span class="ot">&lt;-</span> <span class="fu">read.csv</span>(<span class="at">file =</span> <span class="st">"binning/all_distances.csv"</span>, <span class="at">sep =</span> <span class="st">";"</span>, <span class="at">header =</span> T)</span>
<span id="cb92-3"><a href="#cb92-3" aria-hidden="true" tabindex="-1"></a><span class="co"># rename references and samples</span></span>
<span id="cb92-4"><a href="#cb92-4" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>reference <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">"../refs/NC_000913.3.fasta"</span>, <span class="st">"E. coli"</span>, dist<span class="sc">$</span>reference)</span>
<span id="cb92-5"><a href="#cb92-5" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>reference <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">"../refs/NC_000964.3.fasta"</span>, <span class="st">"B. subtilis"</span>, dist<span class="sc">$</span>reference)</span>
<span id="cb92-6"><a href="#cb92-6" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>reference <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">"../refs/CP013113.1_PAER4.fasta"</span>, <span class="st">"P. aeruginosa"</span>, dist<span class="sc">$</span>reference)</span>
<span id="cb92-7"><a href="#cb92-7" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>reference <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">"../refs/Kocuria_rhizophila_ATCC_9341.fasta"</span>, <span class="st">"K. rhizophila"</span>, dist<span class="sc">$</span>reference)</span>
<span id="cb92-8"><a href="#cb92-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb92-9"><a href="#cb92-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb92-10"><a href="#cb92-10" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>sample <span class="ot">&lt;-</span> <span class="fu">substr</span>(dist<span class="sc">$</span>bin, <span class="dv">1</span>, <span class="dv">5</span>)</span>
<span id="cb92-11"><a href="#cb92-11" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>sample <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">"Ctrl/"</span>, <span class="st">"NA"</span>, dist<span class="sc">$</span>sample)</span>
<span id="cb92-12"><a href="#cb92-12" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>sample <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">"Ctrl2"</span>, <span class="st">"NA2"</span>, dist<span class="sc">$</span>sample)</span>
<span id="cb92-13"><a href="#cb92-13" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>sample <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">"A2/bi"</span>, <span class="st">"piMDA+D2"</span>, dist<span class="sc">$</span>sample)</span>
<span id="cb92-14"><a href="#cb92-14" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>sample <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">"B2/bi"</span>, <span class="st">"piMDA2"</span>, dist<span class="sc">$</span>sample)</span>
<span id="cb92-15"><a href="#cb92-15" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>sample <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">"F2/bi"</span>, <span class="st">"piPolB+D"</span>, dist<span class="sc">$</span>sample)</span>
<span id="cb92-16"><a href="#cb92-16" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>sample <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">"C3/bi"</span>, <span class="st">"PrimPol-MDA"</span>, dist<span class="sc">$</span>sample)</span>
<span id="cb92-17"><a href="#cb92-17" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>sample <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">"D3/bi"</span>, <span class="st">"RP-MDA2"</span>, dist<span class="sc">$</span>sample)</span>
<span id="cb92-18"><a href="#cb92-18" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>sample <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">"N2/bi"</span>, <span class="st">"piMDA"</span>, dist<span class="sc">$</span>sample)</span>
<span id="cb92-19"><a href="#cb92-19" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>sample <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">"N4/bi"</span>, <span class="st">"RP-MDA"</span>, dist<span class="sc">$</span>sample)</span>
<span id="cb92-20"><a href="#cb92-20" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>sample <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">"N6/bi"</span>, <span class="st">"piMDA+D"</span>, dist<span class="sc">$</span>sample)</span>
<span id="cb92-21"><a href="#cb92-21" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb92-22"><a href="#cb92-22" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>bin <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">".fasta"</span>, <span class="st">""</span>, dist<span class="sc">$</span>bin)</span>
<span id="cb92-23"><a href="#cb92-23" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>bin <span class="ot">&lt;-</span> <span class="fu">substr</span>(dist<span class="sc">$</span>bin, <span class="fu">nchar</span>(dist<span class="sc">$</span>bin) <span class="sc">-</span> <span class="dv">5</span>, <span class="fu">nchar</span>(dist<span class="sc">$</span>bin))</span>
<span id="cb92-24"><a href="#cb92-24" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>bin <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">"/"</span>, <span class="st">""</span>, dist<span class="sc">$</span>bin)</span>
<span id="cb92-25"><a href="#cb92-25" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>length <span class="ot">&lt;-</span> <span class="fu">substr</span>(dist<span class="sc">$</span>length, <span class="dv">1</span>, <span class="fu">nchar</span>(dist<span class="sc">$</span>length) <span class="sc">-</span> <span class="dv">5</span>)</span>
<span id="cb92-26"><a href="#cb92-26" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>length <span class="ot">&lt;-</span> <span class="fu">as.integer</span>(dist<span class="sc">$</span>length)<span class="sc">/</span><span class="dv">10</span></span>
<span id="cb92-27"><a href="#cb92-27" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb92-28"><a href="#cb92-28" aria-hidden="true" tabindex="-1"></a><span class="co"># select the bins</span></span>
<span id="cb92-29"><a href="#cb92-29" aria-hidden="true" tabindex="-1"></a>selection <span class="ot">&lt;-</span> dist[dist<span class="sc">$</span>length <span class="sc">&gt;</span> <span class="dv">5</span>, ]</span>
<span id="cb92-30"><a href="#cb92-30" aria-hidden="true" tabindex="-1"></a>selection<span class="sc">$</span>sample <span class="ot">&lt;-</span> <span class="fu">factor</span>(selection<span class="sc">$</span>sample, <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">"NA"</span>, <span class="st">"NA2"</span>, <span class="st">"RP-MDA"</span>, <span class="st">"RP-MDA2"</span>, <span class="st">"PrimPol-MDA"</span>,</span>
<span id="cb92-31"><a href="#cb92-31" aria-hidden="true" tabindex="-1"></a>    <span class="st">"piPolB+D"</span>, <span class="st">"piMDA"</span>, <span class="st">"piMDA2"</span>, <span class="st">"piMDA+D"</span>, <span class="st">"piMDA+D2"</span>))</span>
<span id="cb92-32"><a href="#cb92-32" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb92-33"><a href="#cb92-33" aria-hidden="true" tabindex="-1"></a>g <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(selection, <span class="fu">aes</span>(<span class="at">fill =</span> reference, <span class="at">y =</span> length, <span class="at">x =</span> sample, <span class="at">label =</span> <span class="fu">paste</span>(length,</span>
<span id="cb92-34"><a href="#cb92-34" aria-hidden="true" tabindex="-1"></a>    <span class="st">"%"</span>))) <span class="sc">+</span> <span class="fu">geom_bar</span>(<span class="at">position =</span> <span class="st">"stack"</span>, <span class="at">stat =</span> <span class="st">"identity"</span>, <span class="at">colour =</span> <span class="st">"black"</span>) <span class="sc">+</span> <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> colorines[<span class="dv">12</span><span class="sc">:</span><span class="dv">15</span>]) <span class="sc">+</span></span>
<span id="cb92-35"><a href="#cb92-35" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ylab</span>(<span class="st">"Cumulative bin length (%)"</span>) <span class="sc">+</span> <span class="fu">theme_bw</span>()</span>
<span id="cb92-36"><a href="#cb92-36" aria-hidden="true" tabindex="-1"></a>g <span class="sc">+</span> <span class="fu">geom_text</span>(<span class="at">size =</span> <span class="dv">3</span>, <span class="at">position =</span> <span class="fu">position_stack</span>(<span class="at">vjust =</span> <span class="fl">0.5</span>)) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">45</span>,</span>
<span id="cb92-37"><a href="#cb92-37" aria-hidden="true" tabindex="-1"></a>    <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">14</span>), <span class="at">axis.text.y =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">14</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-left">
<figure class="figure">
<p><img src="Plotbinning-1.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption">Figure 20. Metagenome Binning</figcaption><p></p>
</figure>
</div>
</div>
</div>
<p>Try the same but with different MetaCoAG parameters: <code>–mg_threshold 0.4 –bin_mg_threshold 0.2</code></p>
<div class="cell" data-layout-align="left">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb93"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb93-1"><a href="#cb93-1" aria-hidden="true" tabindex="-1"></a>dist <span class="ot">&lt;-</span> <span class="fu">read.csv</span>(<span class="at">file =</span> <span class="st">"binning/all_distances_final.csv"</span>, <span class="at">sep =</span> <span class="st">";"</span>, <span class="at">header =</span> T)</span>
<span id="cb93-2"><a href="#cb93-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb93-3"><a href="#cb93-3" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>reference <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">"../refs/NC_000913.3.fasta"</span>, <span class="st">"E. coli"</span>, dist<span class="sc">$</span>reference)</span>
<span id="cb93-4"><a href="#cb93-4" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>reference <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">"../refs/NC_000964.3.fasta"</span>, <span class="st">"B. subtilis"</span>, dist<span class="sc">$</span>reference)</span>
<span id="cb93-5"><a href="#cb93-5" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>reference <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">"../refs/CP013113.1_PAER4.fasta"</span>, <span class="st">"P. aeruginosa"</span>, dist<span class="sc">$</span>reference)</span>
<span id="cb93-6"><a href="#cb93-6" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>reference <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">"../refs/Kocuria_rhizophila_ATCC_9341.fasta"</span>, <span class="st">"K. rhizophila"</span>, dist<span class="sc">$</span>reference)</span>
<span id="cb93-7"><a href="#cb93-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb93-8"><a href="#cb93-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb93-9"><a href="#cb93-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb93-10"><a href="#cb93-10" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>sample <span class="ot">&lt;-</span> <span class="fu">substr</span>(dist<span class="sc">$</span>bin, <span class="dv">1</span>, <span class="dv">5</span>)</span>
<span id="cb93-11"><a href="#cb93-11" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>sample <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">"Ctrlb"</span>, <span class="st">"NA"</span>, dist<span class="sc">$</span>sample)</span>
<span id="cb93-12"><a href="#cb93-12" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>sample <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">"Ctrl2"</span>, <span class="st">"NA2"</span>, dist<span class="sc">$</span>sample)</span>
<span id="cb93-13"><a href="#cb93-13" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>sample <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">"A2b/b"</span>, <span class="st">"piMDA+D2"</span>, dist<span class="sc">$</span>sample)</span>
<span id="cb93-14"><a href="#cb93-14" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>sample <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">"B2b/b"</span>, <span class="st">"piMDA2"</span>, dist<span class="sc">$</span>sample)</span>
<span id="cb93-15"><a href="#cb93-15" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>sample <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">"F2b/b"</span>, <span class="st">"piPolB+D"</span>, dist<span class="sc">$</span>sample)</span>
<span id="cb93-16"><a href="#cb93-16" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>sample <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">"C3b/b"</span>, <span class="st">"PrimPol-MDA"</span>, dist<span class="sc">$</span>sample)</span>
<span id="cb93-17"><a href="#cb93-17" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>sample <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">"D3b/b"</span>, <span class="st">"RP-MDA2"</span>, dist<span class="sc">$</span>sample)</span>
<span id="cb93-18"><a href="#cb93-18" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>sample <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">"N2b/b"</span>, <span class="st">"piMDA"</span>, dist<span class="sc">$</span>sample)</span>
<span id="cb93-19"><a href="#cb93-19" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>sample <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">"N4b/b"</span>, <span class="st">"RP-MDA"</span>, dist<span class="sc">$</span>sample)</span>
<span id="cb93-20"><a href="#cb93-20" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>sample <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">"N6b/b"</span>, <span class="st">"piMDA+D"</span>, dist<span class="sc">$</span>sample)</span>
<span id="cb93-21"><a href="#cb93-21" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb93-22"><a href="#cb93-22" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>bin <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">".fasta"</span>, <span class="st">""</span>, dist<span class="sc">$</span>bin)</span>
<span id="cb93-23"><a href="#cb93-23" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>bin <span class="ot">&lt;-</span> <span class="fu">substr</span>(dist<span class="sc">$</span>bin, <span class="fu">nchar</span>(dist<span class="sc">$</span>bin) <span class="sc">-</span> <span class="dv">5</span>, <span class="fu">nchar</span>(dist<span class="sc">$</span>bin))</span>
<span id="cb93-24"><a href="#cb93-24" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>bin <span class="ot">&lt;-</span> <span class="fu">gsub</span>(<span class="st">"/"</span>, <span class="st">""</span>, dist<span class="sc">$</span>bin)</span>
<span id="cb93-25"><a href="#cb93-25" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>length <span class="ot">&lt;-</span> <span class="fu">substr</span>(dist<span class="sc">$</span>length, <span class="dv">1</span>, <span class="fu">nchar</span>(dist<span class="sc">$</span>length) <span class="sc">-</span> <span class="dv">5</span>)</span>
<span id="cb93-26"><a href="#cb93-26" aria-hidden="true" tabindex="-1"></a>dist<span class="sc">$</span>length <span class="ot">&lt;-</span> <span class="fu">as.integer</span>(dist<span class="sc">$</span>length)<span class="sc">/</span><span class="dv">10</span></span>
<span id="cb93-27"><a href="#cb93-27" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb93-28"><a href="#cb93-28" aria-hidden="true" tabindex="-1"></a><span class="co"># select the bins</span></span>
<span id="cb93-29"><a href="#cb93-29" aria-hidden="true" tabindex="-1"></a>dist <span class="ot">&lt;-</span> dist[<span class="sc">-</span><span class="dv">184</span>, ]</span>
<span id="cb93-30"><a href="#cb93-30" aria-hidden="true" tabindex="-1"></a>selection <span class="ot">&lt;-</span> dist[dist<span class="sc">$</span>length <span class="sc">&gt;</span> <span class="dv">5</span>, ]</span>
<span id="cb93-31"><a href="#cb93-31" aria-hidden="true" tabindex="-1"></a>selection<span class="sc">$</span>sample <span class="ot">&lt;-</span> <span class="fu">factor</span>(selection<span class="sc">$</span>sample, <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">"NA"</span>, <span class="st">"NA2"</span>, <span class="st">"RP-MDA"</span>, <span class="st">"RP-MDA2"</span>, <span class="st">"PrimPol-MDA"</span>,</span>
<span id="cb93-32"><a href="#cb93-32" aria-hidden="true" tabindex="-1"></a>    <span class="st">"piPolB+D"</span>, <span class="st">"piMDA"</span>, <span class="st">"piMDA2"</span>, <span class="st">"piMDA+D"</span>, <span class="st">"piMDA+D2"</span>))</span>
<span id="cb93-33"><a href="#cb93-33" aria-hidden="true" tabindex="-1"></a><span class="co"># plot</span></span>
<span id="cb93-34"><a href="#cb93-34" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb93-35"><a href="#cb93-35" aria-hidden="true" tabindex="-1"></a>g <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(selection, <span class="fu">aes</span>(<span class="at">fill =</span> reference, <span class="at">y =</span> length, <span class="at">x =</span> sample, <span class="at">label =</span> <span class="fu">paste</span>(length,</span>
<span id="cb93-36"><a href="#cb93-36" aria-hidden="true" tabindex="-1"></a>    <span class="st">"%"</span>))) <span class="sc">+</span> <span class="fu">geom_bar</span>(<span class="at">position =</span> <span class="st">"stack"</span>, <span class="at">stat =</span> <span class="st">"identity"</span>, <span class="at">colour =</span> <span class="st">"black"</span>, <span class="at">alpha =</span> <span class="fl">0.8</span>) <span class="sc">+</span></span>
<span id="cb93-37"><a href="#cb93-37" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggtitle</span>(<span class="st">"Assembly binning by reference"</span>) <span class="sc">+</span> <span class="fu">labs</span>(<span class="at">fill =</span> <span class="st">"Reference genome"</span>) <span class="sc">+</span> <span class="fu">ylab</span>(<span class="st">"Cumulative bin length (%)"</span>) <span class="sc">+</span></span>
<span id="cb93-38"><a href="#cb93-38" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> colorines[<span class="dv">12</span><span class="sc">:</span><span class="dv">15</span>]) <span class="sc">+</span> <span class="fu">geom_text</span>(<span class="at">size =</span> <span class="dv">3</span>, <span class="at">position =</span> <span class="fu">position_stack</span>(<span class="at">vjust =</span> <span class="fl">0.5</span>)) <span class="sc">+</span></span>
<span id="cb93-39"><a href="#cb93-39" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme_bw</span>() <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">axis.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">15</span>), <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">45</span>,</span>
<span id="cb93-40"><a href="#cb93-40" aria-hidden="true" tabindex="-1"></a>    <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">14</span>), <span class="at">axis.text.y =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">14</span>))</span>
<span id="cb93-41"><a href="#cb93-41" aria-hidden="true" tabindex="-1"></a>g <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">legend.text =</span> <span class="fu">element_text</span>(<span class="at">face =</span> <span class="st">"italic"</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="quarto-figure quarto-figure-left">
<figure class="figure">
<p><img src="Plotbinning2-1.png" class="img-fluid figure-img" width="672"></p>
<p></p><figcaption class="figure-caption">Figure 21. Assembly bins extracted and color-coded by the reference genome</figcaption><p></p>
</figure>
</div>
</div>
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb94"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb94-1"><a href="#cb94-1" aria-hidden="true" tabindex="-1"></a><span class="co"># ggsave('figures/binning.svg', g, width=8, height=5)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<p>Despite the low coverage, the piPolB+D assembly surprisingly good in the binning analysis and extraction.</p>
<p>Interestingly, despite containing more contigs in the assemblies (Figures 16 and 18), piMDA outperforms the piMDA+D protocol in the binning of <em>B. subtilis</em> sequences, suggesting that the alkaline denaturation step may somewhat reduce the overall quality of the MAGs from (pi)MDA samples.</p>
</section>
</section>
<section id="index-of-figures" class="level2" data-number="7">
<h2 data-number="7" class="anchored" data-anchor-id="index-of-figures"><span class="header-section-number">7</span> Index of Figures</h2>
<div class="cell">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb95"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb95-1"><a href="#cb95-1" aria-hidden="true" tabindex="-1"></a><span class="fu">names</span>(figuritas) <span class="ot">&lt;-</span> <span class="st">"List of Plots captions"</span></span>
<span id="cb95-2"><a href="#cb95-2" aria-hidden="true" tabindex="-1"></a><span class="fu">kbl</span>(figuritas, <span class="at">align =</span> <span class="st">"l"</span>, <span class="at">caption =</span> <span class="st">""</span>) <span class="sc">%&gt;%</span></span>
<span id="cb95-3"><a href="#cb95-3" aria-hidden="true" tabindex="-1"></a>    <span class="fu">kable_styling</span>(<span class="at">bootstrap_options =</span> <span class="st">"striped"</span>, <span class="at">full_width =</span> F, <span class="at">font_size =</span> <span class="dv">12</span>, <span class="at">position =</span> <span class="st">"left"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">

<table class="table table-striped" style="font-size: 12px; width: auto !important; ">
<caption style="font-size: initial !important;"></caption>
 <thead>
  <tr>
   <th style="text-align:left;"> List of Plots captions </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> Figure 1. Optimization of piMDA buffer components. </td>
  </tr>
  <tr>
   <td style="text-align:left;"> Figure 2. DNA amplification yield comparison. All reactions were carried out in 50 µL and incubated at 30ºC for 3 h, except those of RP-MDA protocol that were incubated for 16 h. Shown are results from 4 independent experiments. </td>
  </tr>
  <tr>
   <td style="text-align:left;"> Figure 3. DNA amplification time-course. All the amplification assays contained 10 ng of input DNA sample. Shown is the total DNA product in 50 µL reactions. Amplification yield from 2-4 experiments are shown. </td>
  </tr>
  <tr>
   <td style="text-align:left;"> Figure 4. Reads duplication level before (top) and after (bottom) trimming </td>
  </tr>
  <tr>
   <td style="text-align:left;"> Figure 5. Paired reads after trimming </td>
  </tr>
  <tr>
   <td style="text-align:left;"> Figure 6. Reads' GC content plot. GC content is indicated in the plot leyend for the post-trimming reads </td>
  </tr>
  <tr>
   <td style="text-align:left;"> Figure 7. Metagenomic samples reads profiling of amplified DNA by different MDA protocols. Note that both plots show the same data, with different representation </td>
  </tr>
  <tr>
   <td style="text-align:left;"> Figure 8. Clustering of samples by the metagenomic profiling results </td>
  </tr>
  <tr>
   <td style="text-align:left;"> Figure 9. Reference genomes coverage depth by diverse MDA methods </td>
  </tr>
  <tr>
   <td style="text-align:left;"> Figure 10. Reference genomes coverage depth (left axis) and variation coefficient (small points, right axis) by MDA methods </td>
  </tr>
  <tr>
   <td style="text-align:left;"> Figure 11. Reads Mismatch rate </td>
  </tr>
  <tr>
   <td style="text-align:left;"> Figure 12. Comparative Normalized coverage by GC distribution of sequence and reads. Sequence coverage (red) is normalized by the GC range frequency in 100 bp windows (blue). The average GC content of each sample mapped reads is also indicated in the leyend </td>
  </tr>
  <tr>
   <td style="text-align:left;"> Figure 13. Correlations between GC content and normalized coverage with all data </td>
  </tr>
  <tr>
   <td style="text-align:left;"> Figure 14. GC distribution of sequence and reads for each reference genome. Sequence coverage (red) is normalized by the GC range frequency in 100 bp windows (blue). The average base quality is also shown (grey) </td>
  </tr>
  <tr>
   <td style="text-align:left;"> Figure 15. Correlations among normalized coverage by GC content per reference genome </td>
  </tr>
  <tr>
   <td style="text-align:left;"> Figure 16. Assembly basic stats in samples amplified by all the different MDA methods tested </td>
  </tr>
  <tr>
   <td style="text-align:left;"> Figure 17. Assembly basic stats per reference genome </td>
  </tr>
  <tr>
   <td style="text-align:left;"> Figure 18. Number of contigs by GC content in all samples </td>
  </tr>
  <tr>
   <td style="text-align:left;"> Figure 19. KAT k-mer plots comparing k-mer content of Illumina PCR-free reads with the corresponding assembly. The black dots of the graph represents the distribution of k-mers present in the reads but not in the assembly, and the red area represents the distribution of k-mers present in the reads and once in the assembly </td>
  </tr>
  <tr>
   <td style="text-align:left;"> Figure 20. Metagenome Binning </td>
  </tr>
  <tr>
   <td style="text-align:left;"> Figure 21. Assembly bins extracted and color-coded by the reference genome </td>
  </tr>
</tbody>
</table>

</div>
</div>
</section>
<section id="acknowledments-and-funding" class="level2" data-number="8">
<h2 data-number="8" class="anchored" data-anchor-id="acknowledments-and-funding"><span class="header-section-number">8</span> Acknowledments and Funding</h2>
<p>We are indebted to Professor Margarita Salas for her longstanding support and contributions to the discovery of piPolB and to the early development of this project. We also thank Dr.&nbsp;Miguel de Vega for valuable suggestions and critical reading of the manuscript. This work was supported by a grants PGC2018-093723-A-100 and PID2021-123403NB-I00 funded by MCIN/AEI/10.13039/501100011033/FEDER A way to make Europe to M.R.R. C. Egas’ laboratory was funded by the European Union´s Horizon 2020 Research and Innovation Program under Grant Agreement No.&nbsp;685474, the METAFLUIDICS project. CDO was holder of a Fellowship from the Spanish Ministry of University FPU16/02665.</p>
</section>
<section id="session-info" class="level2" data-number="9">
<h2 data-number="9" class="anchored" data-anchor-id="session-info"><span class="header-section-number">9</span> Session Info</h2>
<div class="cell">
<details>
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb96"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb96-1"><a href="#cb96-1" aria-hidden="true" tabindex="-1"></a><span class="fu">sessionInfo</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stdout">
<pre><code>R version 4.2.2 (2022-10-31)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Big Sur ... 10.16

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.2/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.2/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] dplyr_1.1.0        geomtextpath_0.1.1 patchwork_1.1.2    reshape2_1.4.4    
 [5] heatmaply_1.4.2    viridis_0.6.2      viridisLite_0.4.1  plotly_4.10.1     
 [9] corrplot_0.92      kableExtra_1.3.4   gridExtra_2.3      pzfx_0.3.0        
[13] ggrepel_0.9.2      gdata_2.18.0.1     stringr_1.5.0      car_3.1-1         
[17] carData_3.0-5      ggh4x_0.2.3        ggbreak_0.1.1      data.table_1.14.6 
[21] ggplot2_3.4.0      formatR_1.14       knitr_1.42        

loaded via a namespace (and not attached):
 [1] httr_1.4.4         tidyr_1.3.0        splines_4.2.2      jsonlite_1.8.4    
 [5] foreach_1.5.2      gtools_3.9.4       assertthat_0.2.1   highr_0.10        
 [9] yulab.utils_0.0.6  yaml_2.3.7         lattice_0.20-45    pillar_1.8.1      
[13] glue_1.6.2         digest_0.6.31      RColorBrewer_1.1-3 rvest_1.0.3       
[17] colorspace_2.1-0   ggfun_0.0.9        Matrix_1.5-3       plyr_1.8.8        
[21] htmltools_0.5.4    pkgconfig_2.0.3    purrr_1.0.1        scales_1.2.1      
[25] webshot_0.5.4      ggplotify_0.1.0    svglite_2.1.1      tibble_3.1.8      
[29] mgcv_1.8-41        farver_2.1.1       generics_0.1.3     ellipsis_0.3.2    
[33] withr_2.5.0        lazyeval_0.2.2     cli_3.6.0          magrittr_2.0.3    
[37] evaluate_0.20      fansi_1.0.4        nlme_3.1-161       xml2_1.3.3        
[41] textshaping_0.3.6  tools_4.2.2        registry_0.5-1     lifecycle_1.0.3   
[45] aplot_0.1.9        munsell_0.5.0      compiler_4.2.2     ca_0.71.1         
[49] gridGraphics_0.5-1 systemfonts_1.0.4  rlang_1.0.6        grid_4.2.2        
[53] iterators_1.0.14   rstudioapi_0.14    htmlwidgets_1.6.1  crosstalk_1.2.0   
[57] labeling_0.4.2     rmarkdown_2.20     gtable_0.3.1       codetools_0.2-18  
[61] abind_1.4-5        TSP_1.2-2          R6_2.5.1           seriation_1.4.1   
[65] fastmap_1.1.0      utf8_1.2.2         dendextend_1.16.0  stringi_1.7.12    
[69] Rcpp_1.0.10        vctrs_0.5.2        tidyselect_1.2.0   xfun_0.36         </code></pre>
</div>
</div>
</section>


<div id="quarto-appendix" class="default"><section id="footnotes" class="footnotes footnotes-end-of-document" role="doc-endnotes"><h2 class="anchored quarto-appendix-heading">Footnotes</h2>

<ol>
<li id="fn1"><p>JM83 assembly was published in July 2022, but it shares an ANI of 99.98 with MG1655, so we decided not to change the reference genome.<a href="#fnref1" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
</ol>
</section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>