\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{10}{|r}|}
\hline
Assembly & NA & NA\_2 & Phi29+RP & Phi29+RP\_2 & PrimPol+Phi29 & piPolB+D & piMDA & piMDA\_2 & piMDA+D & piMDA+D\_2 \\ \hline
\# contigs ($\geq$ 1000 bp) & 51 & 44 & 980 & 138 & 50 & 92 & 62 & 54 & 50 & 46 \\ \hline
\# contigs ($\geq$ 5000 bp) & 44 & 37 & 168 & 97 & 40 & 78 & 52 & 47 & 42 & 39 \\ \hline
\# contigs ($\geq$ 10000 bp) & 42 & 35 & 80 & 87 & 39 & 72 & 50 & 44 & 40 & 36 \\ \hline
\# contigs ($\geq$ 25000 bp) & 38 & 32 & 20 & 69 & 36 & 60 & 44 & 39 & 36 & 33 \\ \hline
\# contigs ($\geq$ 50000 bp) & 29 & 25 & 5 & 38 & 26 & 35 & 30 & 30 & 27 & 25 \\ \hline
Total length ($\geq$ 1000 bp) & 6558761 & 6558420 & 4028207 & 6445070 & 6468312 & 6465764 & {\bf 6579073} & 6568959 & 6562876 & 6552093 \\ \hline
Total length ($\geq$ 5000 bp) & 6541004 & 6542541 & 2366557 & 6348498 & 6439688 & 6433494 & {\bf 6553655} & 6553131 & 6542652 & 6536222 \\ \hline
Total length ($\geq$ 10000 bp) & 6529610 & 6531146 & 1760483 & 6278756 & 6433498 & 6392007 & {\bf 6542260} & 6535961 & 6531257 & 6518326 \\ \hline
Total length ($\geq$ 25000 bp) & 6449756 & 6468850 & 878109 & 5956942 & 6368385 & 6190327 & 6436260 & 6432868 & 6451403 & {\bf 6470656} \\ \hline
Total length ($\geq$ 50000 bp) & 6104448 & {\bf 6193012} & 332346 & 4796604 & 5943874 & 5274503 & 5914261 & 6067363 & 6137277 & 6166316 \\ \hline
\# contigs & 52 & 45 & 1750 & 151 & 53 & 97 & 65 & 57 & 51 & 49 \\ \hline
Largest contig & 480222 & {\bf 637715} & 85618 & 418864 & 480222 & 412635 & 495927 & 459318 & 447001 & 453250 \\ \hline
Total length & 6559585 & 6559244 & 4571010 & 6454610 & 6470701 & 6469217 & {\bf 6580888} & 6571466 & 6563700 & 6554700 \\ \hline
Reference length & 6264404 & 6264404 & 6264404 & 6264404 & 6264404 & 6264404 & 6264404 & 6264404 & 6264404 & 6264404 \\ \hline
GC (\%) & 66.29 & 66.29 & 65.40 & 66.32 & 66.33 & 66.37 & 66.28 & 66.29 & 66.28 & 66.28 \\ \hline
Reference GC (\%) & 66.56 & 66.56 & 66.56 & 66.56 & 66.56 & 66.56 & 66.56 & 66.56 & 66.56 & 66.56 \\ \hline
N50 & 271299 & {\bf 320157} & 5425 & 103965 & 250423 & 164779 & 268827 & 212077 & 320156 & 320081 \\ \hline
NG50 & 271299 & 320157 & 2501 & 107002 & 250423 & 170905 & 295760 & 212077 & {\bf 341015} & 320081 \\ \hline
N90 & 81823 & 103656 & 904 & 29447 & 68627 & 37040 & 47924 & 94138 & 68627 & {\bf 103684} \\ \hline
NG90 & 91181 & {\bf 173536} & - & 33213 & 99345 & 40717 & 65668 & 103750 & 103656 & 145236 \\ \hline
L50 & 9 & {\bf 8} & 153 & 18 & 9 & 13 & 10 & 10 & 9 & 9 \\ \hline
LG50 & 9 & {\bf 8} & 389 & 17 & 9 & 12 & 9 & 10 & {\bf 8} & 9 \\ \hline
L90 & 26 & {\bf 21} & 1071 & 64 & 25 & 48 & 31 & 28 & 24 & 22 \\ \hline
LG90 & 23 & {\bf 19} & - & 59 & 23 & 44 & 26 & 25 & 21 & 20 \\ \hline
\# misassemblies & 49 & 49 & 56 & 46 & 49 & {\bf 42} & 58 & 52 & 48 & 49 \\ \hline
\# misassembled contigs & 21 & {\bf 20} & 53 & 26 & 21 & 27 & 24 & 25 & {\bf 20} & 21 \\ \hline
Misassembled contigs length & 4779725 & 5087640 & {\bf 576399} & 3104594 & 4679577 & 3836857 & 4688132 & 4802028 & 5025451 & 5082348 \\ \hline
\# local misassemblies & 43 & 42 & 244 & 45 & 42 & {\bf 37} & 46 & 56 & 40 & 41 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & {\bf 0} & {\bf 0} & 6 & 1 & {\bf 0} & 1 & {\bf 0} & {\bf 0} & {\bf 0} & {\bf 0} \\ \hline
\# unaligned contigs & 0 + 29 part & {\bf 0 + 27 part} & 1 + 51 part & 0 + 33 part & {\bf 0 + 27 part} & 0 + 37 part & 0 + 31 part & 0 + 30 part & 0 + 29 part & {\bf 0 + 27 part} \\ \hline
Unaligned length & 546235 & 546756 & {\bf 427364} & 467211 & 467631 & 467349 & 552365 & 543578 & 552824 & 547719 \\ \hline
Genome fraction (\%) & {\bf 95.847} & 95.836 & 66.002 & 95.432 & 95.731 & 95.630 & 95.826 & 95.837 & 95.811 & 95.753 \\ \hline
Duplication ratio & 1.001 & 1.001 & {\bf 0.999} & 1.001 & 1.001 & 1.001 & 1.004 & 1.004 & 1.001 & 1.001 \\ \hline
\# N's per 100 kbp & 0.00 & 0.00 & 0.00 & 0.00 & 0.00 & 0.00 & 0.00 & 0.00 & 0.00 & 0.00 \\ \hline
\# mismatches per 100 kbp & 507.33 & 507.77 & 517.00 & {\bf 502.35} & 507.53 & 505.25 & 508.10 & 507.77 & 508.43 & 507.25 \\ \hline
\# indels per 100 kbp & 15.17 & 15.06 & 32.80 & {\bf 14.71} & 15.11 & 15.20 & 14.94 & 14.92 & 15.05 & 14.89 \\ \hline
Largest alignment & 400768 & 400768 & 44438 & 247330 & 400265 & 319885 & 341687 & {\bf 401504} & 400804 & 400768 \\ \hline
Total aligned length & 6010661 & 6010413 & 4130952 & 5983876 & 6000773 & 5998992 & {\bf 6025430} & 6024611 & 6008336 & 6004547 \\ \hline
NA50 & 116871 & {\bf 120701} & 3470 & 63369 & 116793 & 87493 & 101721 & 108849 & 116871 & {\bf 120701} \\ \hline
NGA50 & 120701 & {\bf 137726} & 1548 & 63948 & 118868 & 88056 & 105008 & 116792 & 120701 & {\bf 137726} \\ \hline
NA90 & 14460 & 15400 & 503 & 6914 & {\bf 16241} & 14107 & 9545 & 14460 & 14515 & 14515 \\ \hline
NGA90 & 27440 & {\bf 28274} & - & 14615 & 27533 & 18849 & 24696 & 24966 & 26372 & 27795 \\ \hline
LA50 & 17 & {\bf 16} & 263 & 31 & {\bf 16} & 22 & 20 & 19 & 17 & {\bf 16} \\ \hline
LGA50 & 16 & {\bf 15} & 631 & 29 & {\bf 15} & 21 & 19 & 17 & 16 & {\bf 15} \\ \hline
LA90 & 72 & {\bf 66} & 1761 & 118 & {\bf 66} & 92 & 85 & 79 & 70 & 68 \\ \hline
LGA90 & 59 & {\bf 55} & - & 102 & 57 & 81 & 68 & 65 & 58 & 56 \\ \hline
K-mer-based compl. (\%) & 65.83 & 65.84 & 42.21 & 65.56 & 65.86 & 65.78 & 65.70 & 65.71 & 65.83 & {\bf 65.87} \\ \hline
K-mer-based cor. length (\%) & 87.74 & 98.92 & 64.35 & 98.01 & 95.51 & 97.01 & {\bf 98.99} & 98.90 & 94.79 & 98.80 \\ \hline
K-mer-based mis. length (\%) & 11.18 & {\bf 0.00} & {\bf 0.00} & {\bf 0.00} & 3.62 & {\bf 0.00} & {\bf 0.00} & {\bf 0.00} & 4.10 & {\bf 0.00} \\ \hline
\# k-mer-based misjoins & 2 & {\bf 0} & {\bf 0} & {\bf 0} & 2 & {\bf 0} & {\bf 0} & {\bf 0} & 1 & {\bf 0} \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
