\documentclass[12pt,a4paper]{article}
\begin{document}
\begin{table}[ht]
\begin{center}
\caption{All statistics are based on contigs of size $\geq$ 500 bp, unless otherwise noted (e.g., "\# contigs ($\geq$ 0 bp)" and "Total length ($\geq$ 0 bp)" include all contigs).}
\begin{tabular}{|l*{10}{|r}|}
\hline
Assembly & NA & NA\_2 & Phi29+RP & Phi29+RP\_2 & PrimPol+Phi29 & piPolB+D & piMDA & piMDA\_2 & piMDA+D & piMDA+D\_2 \\ \hline
\# contigs ($\geq$ 1000 bp) & 16 & 8 & 149 & 465 & 12 & 18 & 29 & 16 & 13 & 9 \\ \hline
\# contigs ($\geq$ 5000 bp) & 15 & 7 & 19 & 110 & 9 & 17 & 24 & 14 & 12 & 6 \\ \hline
\# contigs ($\geq$ 10000 bp) & 13 & 7 & 7 & 48 & 9 & 16 & 21 & 13 & 10 & 6 \\ \hline
\# contigs ($\geq$ 25000 bp) & 13 & 7 & 1 & 15 & 9 & 15 & 19 & 12 & 10 & 6 \\ \hline
\# contigs ($\geq$ 50000 bp) & 11 & 7 & 0 & 2 & 7 & 15 & 16 & 11 & 10 & 5 \\ \hline
Total length ($\geq$ 1000 bp) & 2684147 & 2688755 & 433790 & 2272855 & 2683274 & 2684182 & 2691800 & {\bf 2711637} & 2684100 & 2683297 \\ \hline
Total length ($\geq$ 5000 bp) & 2681293 & 2685901 & 213276 & 1524421 & 2675542 & 2681328 & 2683316 & {\bf 2707497} & 2681246 & 2675547 \\ \hline
Total length ($\geq$ 10000 bp) & 2669522 & 2685901 & 124412 & 1102994 & 2675542 & 2675768 & 2661944 & {\bf 2701937} & 2669477 & 2675547 \\ \hline
Total length ($\geq$ 25000 bp) & 2669522 & {\bf 2685901} & 30963 & 585480 & 2675542 & 2665475 & 2628554 & 2677778 & 2669477 & 2675547 \\ \hline
Total length ($\geq$ 50000 bp) & 2604546 & {\bf 2685901} & 0 & 152349 & 2598727 & 2665475 & 2529178 & 2638016 & 2669477 & 2631807 \\ \hline
\# contigs & 16 & 8 & 401 & 661 & 13 & 18 & 30 & 17 & 13 & 10 \\ \hline
Largest contig & 583354 & 840041 & 30963 & 85713 & 840047 & 369657 & 305864 & 593849 & 705563 & {\bf 958637} \\ \hline
Total length & 2684147 & 2688755 & 604110 & 2418532 & 2684018 & 2684182 & 2692778 & {\bf 2712557} & 2684100 & 2684013 \\ \hline
Reference length & 2696980 & 2696980 & 2696980 & 2696980 & 2696980 & 2696980 & 2696980 & 2696980 & 2696980 & 2696980 \\ \hline
GC (\%) & 71.24 & 71.20 & 68.41 & 71.01 & 71.23 & 71.23 & 71.24 & 71.23 & 71.23 & 71.23 \\ \hline
Reference GC (\%) & 71.16 & 71.16 & 71.16 & 71.16 & 71.16 & 71.16 & 71.16 & 71.16 & 71.16 & 71.16 \\ \hline
N50 & 422898 & {\bf 583329} & 2058 & 8324 & 543595 & 239654 & 198003 & 376610 & 304667 & 583319 \\ \hline
NG50 & 422898 & {\bf 583329} & - & 6605 & 543595 & 239654 & 198003 & 376610 & 304667 & 583319 \\ \hline
N90 & 75931 & {\bf 273772} & 620 & 1291 & 164698 & 78031 & 58552 & 100307 & 112842 & {\bf 273772} \\ \hline
NG90 & 67282 & {\bf 273772} & - & - & 164698 & 78031 & 58552 & 100307 & 112842 & {\bf 273772} \\ \hline
L50 & 3 & {\bf 2} & 49 & 60 & {\bf 2} & 5 & 6 & 3 & 3 & {\bf 2} \\ \hline
LG50 & 3 & {\bf 2} & - & 79 & {\bf 2} & 5 & 6 & 3 & 3 & {\bf 2} \\ \hline
L90 & 8 & {\bf 5} & 293 & 381 & 6 & 12 & 15 & 9 & 7 & {\bf 5} \\ \hline
LG90 & 9 & {\bf 5} & - & - & 6 & 12 & 15 & 9 & 7 & {\bf 5} \\ \hline
\# misassemblies & {\bf 0} & {\bf 0} & 11 & 5 & {\bf 0} & {\bf 0} & 3 & 6 & {\bf 0} & {\bf 0} \\ \hline
\# misassembled contigs & {\bf 0} & {\bf 0} & 11 & 5 & {\bf 0} & {\bf 0} & 3 & 3 & {\bf 0} & {\bf 0} \\ \hline
Misassembled contigs length & {\bf 0} & {\bf 0} & 34603 & 6627 & {\bf 0} & {\bf 0} & 450753 & 1091078 & {\bf 0} & {\bf 0} \\ \hline
\# local misassemblies & {\bf 3} & {\bf 3} & 71 & 125 & 4 & 4 & 9 & 22 & {\bf 3} & 5 \\ \hline
\# scaffold gap ext. mis. & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 \\ \hline
\# scaffold gap loc. mis. & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 \\ \hline
\# unaligned mis. contigs & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 \\ \hline
\# unaligned contigs & {\bf 0 + 0 part} & {\bf 0 + 0 part} & 0 + 4 part & {\bf 0 + 0 part} & {\bf 0 + 0 part} & {\bf 0 + 0 part} & {\bf 0 + 0 part} & {\bf 0 + 0 part} & {\bf 0 + 0 part} & {\bf 0 + 0 part} \\ \hline
Unaligned length & {\bf 0} & {\bf 0} & 10288 & {\bf 0} & {\bf 0} & {\bf 0} & {\bf 0} & {\bf 0} & {\bf 0} & {\bf 0} \\ \hline
Genome fraction (\%) & 99.403 & {\bf 99.585} & 22.013 & 89.821 & 99.479 & 99.401 & 99.402 & 99.479 & 99.407 & 99.479 \\ \hline
Duplication ratio & 1.000 & 1.000 & {\bf 0.998} & {\bf 0.998} & 1.000 & 1.000 & 1.003 & 1.011 & 1.000 & 1.000 \\ \hline
\# N's per 100 kbp & 0.00 & 0.00 & 0.00 & 0.00 & 0.00 & 0.00 & 0.00 & 0.00 & 0.00 & 0.00 \\ \hline
\# mismatches per 100 kbp & 1.34 & 2.12 & 26.83 & 14.03 & 1.83 & 1.49 & 1.08 & {\bf 1.03} & 1.16 & 1.04 \\ \hline
\# indels per 100 kbp & 2.54 & 2.98 & 28.69 & 14.40 & 2.50 & 2.83 & 2.53 & {\bf 2.40} & 2.50 & 2.42 \\ \hline
Largest alignment & 583354 & 840041 & 30963 & 85713 & 840047 & 369156 & 305339 & 461821 & 705563 & {\bf 958637} \\ \hline
Total aligned length & 2681248 & 2686026 & 592627 & 2416764 & 2683254 & 2681374 & 2689590 & {\bf 2711535} & 2681263 & 2683280 \\ \hline
NA50 & 420742 & {\bf 583329} & 1878 & 8324 & 543583 & 239654 & 174021 & 181444 & 304136 & 583319 \\ \hline
NGA50 & 420742 & {\bf 583329} & - & 6605 & 543583 & 239654 & 174021 & 181444 & 304136 & 583319 \\ \hline
NA90 & 67267 & {\bf 273772} & 585 & 1276 & 164470 & 78018 & 58501 & 68707 & 112829 & {\bf 273772} \\ \hline
NGA90 & 67267 & {\bf 273772} & - & - & 164470 & 78018 & 58501 & 68707 & 112829 & {\bf 273772} \\ \hline
LA50 & 3 & {\bf 2} & 52 & 60 & {\bf 2} & 5 & 6 & 5 & 3 & {\bf 2} \\ \hline
LGA50 & 3 & {\bf 2} & - & 79 & {\bf 2} & 5 & 6 & 5 & 3 & {\bf 2} \\ \hline
LA90 & 9 & {\bf 5} & 313 & 382 & 6 & 12 & 16 & 14 & 7 & {\bf 5} \\ \hline
LGA90 & 9 & {\bf 5} & - & - & 6 & 12 & 16 & 14 & 7 & {\bf 5} \\ \hline
K-mer-based compl. (\%) & 99.64 & {\bf 99.69} & 19.19 & 85.32 & 99.64 & 99.61 & 99.36 & 98.73 & 0.00 & {\bf 99.69} \\ \hline
K-mer-based cor. length (\%) & 77.95 & 78.20 & 50.68 & 83.77 & 79.43 & {\bf 99.69} & 88.41 & 99.61 & {\bf 99.69} & 77.95 \\ \hline
K-mer-based mis. length (\%) & 21.73 & 21.70 & {\bf 0.00} & {\bf 0.00} & 20.25 & {\bf 0.00} & 11.14 & {\bf 0.00} & {\bf 0.00} & 21.73 \\ \hline
\# k-mer-based misjoins & 1 & 1 & {\bf 0} & {\bf 0} & 1 & {\bf 0} & 1 & {\bf 0} & {\bf 0} & 1 \\ \hline
\end{tabular}
\end{center}
\end{table}
\end{document}
